<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">J Vis Exp</span> <span class="iso-abbrev" title="iso-abbrev">J Vis Exp</span> <span class="publisher-id" title="publisher-id">JoVE</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Journal of Visualized Experiments : JoVE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1940-087X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">MyJove Corporation</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5931759">5931759</a> </span> <span class="publisher-id" title="publisher-id">57223</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3791/57223">10.3791/57223</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Neuroscience</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Combining Optogenetics with Artificial microRNAs to Characterize the Effects of Gene Knockdown on Presynaptic Function within Intact Neuronal Circuits </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thalhammer</span> <span class="given-names" tagx="given-names" title="given-names">Agnes</span> </span> <a href="#ID1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jaudon</span> <span class="given-names" tagx="given-names" title="given-names">Fanny</span> </span> <a href="#ID1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cingolani</span> <span class="given-names" tagx="given-names" title="given-names">Lorenzo A.</span> </span> <a href="#ID1"> <sup>1</sup> </a> </span> </div> <span class="citation_author_institution" id="ID1">[], <sup>1</sup> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""> <p>Correspondence to: Lorenzo A. Cingolani at <span class="email" tagx="email" title="email">lorenzo.cingolani@iit.it</span> </p> </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-3-3</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-3-3</span> </span> <span class="issue" tagx="issue" title="issue">133</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">57223</span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by-nc-nd/3.0/">http://creativecommons.org/licenses/by-nc-nd/3.0/</a> </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The purpose of this protocol is to characterize the effect of gene knockdown on presynaptic function within intact neuronal circuits. We describe a workflow on how to combine artificial microRNA (miR)-mediated RNA interference with optogenetics to achieve selective stimulation of manipulated presynaptic boutons in acute brain slices. The experimental approach involves the use of a single viral construct and a single neuron-specific promoter to drive the expression of both an optogenetic probe and artificial miR(s) against presynaptic gene(s). When stereotactically injected in the brain region of interest, the expressed construct makes it possible to stimulate with light exclusively the neurons with reduced expression of the gene(s) under investigation. This strategy does not require the development and maintenance of genetically modified mouse lines and can in principle be applied to other organisms and to any neuronal gene of choice. We have recently applied it to investigate how the knockdown of alternative splice isoforms of presynaptic P/Q-type voltage-gated calcium channels (VGCCs) regulates short-term synaptic plasticity at CA3 to CA1 excitatory synapses in acute hippocampal slices. A similar approach could also be used to manipulate and probe the neuronal circuitry <i>in vivo</i>. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
Neuroscience</div> <div class="kwd" title="kwd">
Issue 133</div> <div class="kwd" title="kwd">
Optogenetics</div> <div class="kwd" title="kwd">
ultrafast channelrhodopsin</div> <div class="kwd" title="kwd">
ChETA</div> <div class="kwd" title="kwd">
microRNA</div> <div class="kwd" title="kwd">
RNA interference</div> <div class="kwd" title="kwd">
knockdown</div> <div class="kwd" title="kwd">
recombinant adeno-associated virus</div> <div class="kwd" title="kwd">
rAAV1/2</div> <div class="kwd" title="kwd">
presynaptic</div> <div class="kwd" title="kwd">
synaptic transmission</div> <div class="kwd" title="kwd">
synaptic plasticity</div> <div class="kwd" title="kwd">
RNA extraction</div> </div> </div> </div> <div class="body" title="body"> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/" /> </div>  </div> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>This protocol describes a new approach to characterize the effect of gene knockdown on presynaptic function within intact neuronal circuits. Investigating presynaptic function in intact neuronal circuits is challenging because many presynaptic boutons are too small and far away from the soma to allow combined molecular and electrophysiological interventions. Although RNA interference offers a powerful and flexible means to knockdown synaptic proteins <a href="#B0">1</a>, this approach has been used sparingly to investigate presynaptic function because it is difficult to detect the effects of knockdown using traditional electrical stimulations, which do not distinguish between manipulated and naïve presynaptic boutons <a href="#B1">2</a>. Here, we describe how to combine artificial microRNA (miR)-mediated RNA interference with recently developed optogenetic technology to achieve selective stimulation of manipulated presynaptic boutons in acute brain slices. </p> <p>While conditional knockout mice in combination with electrophysiology could also be used to investigate the function of presynaptic proteins <a href="#B2">3</a> <a href="#B3">4</a>, our strategy does not require the development and maintenance of genetically modified mouse lines and can also be easily employed to knock down specific isoforms of a gene. Relative to more commonly used short hairpin RNAs (shRNAs), artificial miRs, which we employ here, offer key advantages for knockdown in neurons. Unlike shRNAs, they can be expressed under the control of a polymerase II promoter <a href="#B0">1</a>. Thus, a single promoter can be used to drive the expression of both miR and an optogenetic probe, along with a fluorescent reporter. In this way, the size of the construct can be kept within the packaging limits of recombinant adeno-associated viruses (rAAV, <a href="#F1">Figure 1A</a>). Also, the use of a single construct and a single promoter reduces experimental variability because it allows for expression of the miR, the optogenetic probe and the fluorescent reporter in a fixed ratio. </p> <p>We have recently applied this technology to examine the role of alternatively spliced isoforms of presynaptic calcium channels in the hippocampus <a href="#B4">5</a>. Such a strategy is generally applicable to studying the physiological relevance of other presynaptically expressed proteins in any brain circuit of interest. </p> </div> <div class="protocol" title="sec"> <div class="title" tagx="title" title="title">
Protocol</div> <p>All experiments were carried out in accordance with the guidelines established by the European Communities Council (Directive 2010/63/EU of March 4, 2014), and were approved by the Italian Ministry of Health. </p> <div class="1.designofmicrornasforrnainterferenceandevaluationoftheirefficiencyinheterologousexpressionsystems" title="sec"> <div class="title" tagx="title" title="title">
1. Design of microRNAs for RNA Interference and Evaluation of their Efficiency in Heterologous Expression Systems </div> <p>NOTE: This protocol requires knowledge of the following well-established methods: molecular cloning, DNA sequencing, maintenance of cell lines, calcium phosphate transfection, quantitative real time PCR (qRT-PCR), preparation of cell lysates from cell lines and Western blotting. </p> <ol> <li> <p>Determine whether the gene of interest is subject to alternative splicing. For a general knockdown of the gene, select exclusively constitutive exons; for a knockdown of a specific alternatively spliced isoform, select the relevant alternatively spliced exon. </p> </li> <li> <p>Use dedicated software ( <i>e.g.</i> https://rnaidesigner.thermofisher.com/rnaiexpress/) to design artificial miRs for RNA interference against the sequence of interest. NOTE: It is important to use a Basic Local Alignment Search Tool (BLAST) to check for possible off-targets within the same species. </p> </li> <li> <p>Select the top three ranked miRs for the target gene.</p> </li> <li> <p>Order the top and bottom strands of the selected miRs from a suitable company. For negative control, use a scrambled version of one the selected miRs or a miR predicted not to target any known gene of the species used in the experiment ( <i>e.g</i>. for vertebrate genes, the miR present in pcDNA6.2-GW/EmGFP-miR-neg). </p> </li> <li> <p>Anneal the respective top and bottom strands of the selected miRs and clone them into a plasmid designed for expression of miRs ( <i>e.g.</i> pcDNA6.2-GW/EmGFP-miR) using standard cloning strategies. NOTE: The aim is to create a miR expression cassette consisting of a 5' miR flanking region, the selected miR sequence and a 3' miR flanking region that can be expressed from the 3' UTR of a reporter gene under the control of a RNA polymerase type II promoter. </p> </li> <li> <p>Use DNA sequencing to validate the correct insertion of the selected miRs.</p> </li> <li> <p>Grow HEK293 cells in a humidified cell culture incubator (37 °C, 5% CO <sub>2</sub>) using DMEM medium supplemented with 10% fetal bovine serum, 1 mM NaPyruvate, 10 mM HEPES (pH 7.39) and 1x penicillin/streptomycin (Pen/Strep). </p> </li> <li> <p>When HEK293 cells are ~70% confluent, co-transfect them with each of the miR expression vectors and a vector expressing the targeted gene in a 1:1 DNA ratio using the calcium phosphate method <a href="#B5">6</a>. Include the following controls: (i) untransfected cells, (ii) cells transfected with the vector expressing the targeted gene together with either carrier DNA ( <i>e.g.</i> pBluescript II SK(+)) or (iii) miR Control. NOTE: (i) The expressed targeted gene must be from the same species towards which the miRs were designed, unless the sequences targeted by the miRs are absolutely conserved at the nucleotide level between the two species. (ii) Cell lines other than HEK293 can be used. (iii) Alternative methods of transfection, such as electroporation or liposome-based methods, can be used. </p> </li> <li> <p> <b>48 h after transfection, lyse the cells, run protein gel, perform Western blot and analyze protein content with an antibody recognizing the target protein. Aim for a knockdown efficiency of ≥50%. </b> NOTE: Knockdown efficiency can alternatively, or in addition, be evaluated (i) by ELISA analysis, (ii) by measuring the activity of the target protein if applicable ( <i>e.g.</i> current densities for ion channels) or (iii) by qRT-PCR at the mRNA level if suitable antibodies are not available (protocol step 3). </p><ol> <li> <p>Place the culture dishes on ice and wash the cells once with ice-cold phosphate-buffered saline (PBS). </p> </li> <li> <p>Aspirate the PBS, then add ice-cold RIPA buffer (50 mM Tris-HCl pH 7.4, 150 mM NaCl, 2 mM EDTA, 1% NP40, 0.1% sodium dodecyl sulfate (SDS), containing protease and phosphatase inhibitors; 1 mL for a dish of Ø 100 mm, 0.5 mL for a dish of Ø 60 mm). </p> </li> <li> <p>Scrape adherent cells off the dish using a cell scraper and gently transfer the cell suspension into a pre-cooled microcentrifuge tube. </p> </li> <li> <p>Centrifuge the tube at 15,000 x g for 15 min at 4 °C, transfer the supernatant in a new pre-cooled microcentrifuge tube and discard the pellet. </p> </li> <li> <p>Determine the protein concentration using the BCA Protein Assay kit or other suitable method ( <i>e.g.</i> Bradford assay). NOTE: Samples can either be frozen at -20 °C or -80 °C for later use, or processed immediately. </p> </li> <li> <p>Transfer an appropriate volume of lysates to microcentrifuge tubes so that all samples have the same protein concentration and add adequate ice-cold lysis buffer to make up all the lysates to the same volume. NOTE: 30 to 50 µg of total protein should be sufficient for most of proteins, but the appropriate quantity to be loaded should be determined according to the abundance of the protein of interest. </p> </li> <li> <p>Add an appropriate amount of 2x Laemmli buffer (4% SDS, 10% 2-mercaptoethanol, 20% glycerol, 0.004% bromophenol blue, 0.125 M Tris-HCl, pH 6.8) and boil the samples at 95 °C for 5 min. </p> </li> <li> <p>Load the samples on an acrylamide gel along with a molecular weight marker. Run the gel at 100 V for 1-2 h. NOTE: The gel percentage depends on the size of the protein of interest. </p> </li> <li> <p>Assemble the transfer sandwich and transfer the proteins from the gel onto a membrane at 100 V for 2 h. The membrane can be either nitrocellulose or PVDF. Activate PVDF with methanol for 1 min and rinse it with transfer buffer before preparing the stack. </p> </li> <li> <p>Block the membrane for 1 h at room temperature using blocking buffer (5% milk in PBST (PBS + 0.1% Tween-20)), then incubate the membrane overnight at 4 °C with primary antibody diluted in blocking buffer. </p> </li> <li> <p>Wash the membrane for 10 min in PBST three times and incubate it with HRP-conjugated secondary antibody in blocking buffer for 1 h at room temperature. </p> </li> <li> <p>Wash the membrane for 10 min in PBS four times, then apply the chemiluminescent substrate to the membrane and capture the chemiluminescent signals using a CCD camera-based imager. </p> </li> <li> <p>Use image analysis software to quantify the knockdown efficiency.</p> </li> </ol> <p /> </li> <li> <p>Select the two most efficient miRs targeting non-overlapping sequences for further functional assays. NOTE: Off-targets effects can never be entirely ruled out for any miR. However, if two independent miRs have a similar effect, it is extremely unlikely that this is because of a same off-target knockdown. </p> </li> <li> <p>If the knockdown efficiency of the selected miRs is not satisfactory (&amp;lt;50%), screen for other miRs. Alternatively, express the most efficient miR(s) in tandem, <i>i.e.</i> express them in multiple copies from the same expression cassette, which may result in increased knockdown efficiency <a href="#B6">7</a>. </p> </li> </ol> </div> <div class="2.constructionofrecombinantadeno-associatedvectorsforcombinedexpressionofoptogeneticprobesandmicrornas" title="sec"> <div class="title" tagx="title" title="title">
2. Construction of Recombinant Adeno-associated Vectors for Combined Expression of Optogenetic Probes and microRNAs </div> <p>NOTE: This protocol requires knowledge of the following well-established methods: molecular cloning, DNA sequencing and rAAV production. </p> <ol> <li> <p>Clone each of the most efficient miR expression cassettes into the 3'UTR of constructs designed for production of recombinant rAAVs and expression of excitatory optogenetic probes, such as in pAAV-Syn-ChETA-TdT-miR-X ( <a href="#F1">Figure 1A</a>), where the neuron-specific synapsin promoter drives expression of the ultrafast channelrhodopsin ChETA, the red fluorescent reporter TdTomato, and miR(s) inserted between the NheI and EcoRI sites <a href="#B4">5</a>. NOTE: (i) Do not surpass the packaging limit of rAAVs, which is approximately 4.7 Kb in length from ITR (inverted terminal repeat) to ITR ( <a href="#F1">Figure 1A</a>, orange). (ii) The miR expression cassette needs to be inserted between the stop codon and WPRE (Woodchuck Hepatitis Virus Posttranscriptional Regulatory Element; <a href="#F1">Figure 1A</a>). (iii) Inclusion of a fluorescent protein, such as TdTomato, is extremely useful as it allows for readily monitoring the localization and intensity of infection (protocol step 4). </p> </li> <li> <p>Use DNA sequencing to validate the correct insertion of the miR cassettes.</p> </li> <li> <p>Produce and titer rAAV1/2 according to the previously published protocol <a href="#B7">8</a>, one rAAV1/2 for each selected miR. One typical experiment for the knockdown of one gene would include two independent miRs against the same gene and one miR control. Aim for a virus titer of ≥10 <sup>9</sup> viral genomes (vg)/µL. CAUTION: rAAVs need to be handled in a Biosafety Level 1 (BSL-1) facility. Please check with Institutional Biosafety Committee for detailed information. NOTE: If the laboratory does not have a viral facility or if viral titers are not satisfactory, rAAV production can be outsourced. For example, visit https://www.med.upenn.edu/gtp/vectorcore/ or https://vcf.charite.de/en/metas/. </p> </li> </ol> </div> <div class="3.extractionofrnafromprimaryneuronalculturesforevaluationofmirknockdownefficiencyofendogenousgenesbyqrt-pcr" title="sec"> <div class="title" tagx="title" title="title">
3. Extraction of RNA from Primary Neuronal Cultures for Evaluation of miR Knockdown Efficiency of Endogenous Genes by qRT-PCR </div> <p>NOTE: (i) This protocol requires knowledge of the following well-established methods: preparation and maintenance of primary neuronal cultures and qRT-PCR. (ii) Repeat the quantification of knockdown efficiency (steps 3.1–3.14) at least 3 times (biological replicates). (iii) Estimation of knockdown efficiency at the mRNA level by qRT-PCR is suitable when an analysis of the protein content is precluded, such as when knocking down alternatively spliced isoforms for which specific antibodies are not available <a href="#B4">5</a>. </p> <ol> <li> <p> <b>Prepare primary neuronal cultures from the brain region of interest. Follow the protocol for cortical cultures given in previous publication <a href="#B8">9</a>, with the following modifications: </b> </p><ol> <li> <p>Plate neurons in 6 well plates at a density of 500,000 neurons per well. Use 2.5 mL/well of attachment medium and 3.3 mL/well of maintenance medium. </p> </li> <li> <p>If astrocyte overgrowth is observed, add 0.5 mL/well of maintenance medium supplemented with 7.5 µM of cytosine β-D-arabinofuranoside (for a final concentration of 1 µM) at 3–4 days <i>in vitro</i> (DIV). </p> </li> </ol> <p /> </li> <li> <p> <b>Infect three wells per miR (technical replicates) at 5</b>– <b>6 DIV. Use the lowest infectious dose which will infect ≥99% of the neurons.</b> </p><ol> <li> <p>Remove 2 mL of medium from each well and collect it in a 50 mL falcon tube.</p> </li> <li> <p>Add virus directly to the neurons, mix gently and place the plates back in the incubator at 37 °C. </p> </li> <li> <p>Store the collected medium in the incubator. Loose the cap of the falcon tube to allow gas equilibration. </p> </li> <li> <p>After 24 h, add the removed medium back in each well (1.9 mL/well).</p> </li> </ol> <p /> </li> <li> <p> <b>At 17</b>– <b>18 DIV, lyse the neurons for RNA extraction.</b> CAUTION: Lysis Reagent and chloroform are highly toxic. Work under a chemical hood and wear protective gear; dispose of waste according to your national and institutional guidelines. NOTE: For steps 3.3–3.13, work in RNAse-free conditions. Wear gloves and use RNAse-free glassware and disposable plasticware. For general precautions on how to handle RNA, consult for example Appendix A of the RNeasy Micro Handbook available online. </p><ol> <li> <p>Incubate glass Pasteur pipettes overnight in an oven at 180 °C.</p> </li> <li> <p>Tilt the plates and aspirate the medium completely from each well using a glass Pasteur pipette connected to a vacuum pump. </p> </li> <li> <p>Immediately add 700 µL of Lysis Reagent to each well.</p> </li> <li> <p>Spread the solution evenly by rocking and shaking the plate carefully and briefly by hand. </p> </li> <li> <p>Pipette the solution up and down 4–5 times or until a homogeneous suspension is obtained.</p> </li> <li> <p>Transfer the solution from each well into a separate 1.5 mL Eppendorf tube. NOTE: The cell lysates can be stored at -80 °C or processed immediately. </p> </li> </ol> <p /> </li> <li> <p>If required, defrost cell the lysates at RT, and proceed immediately to step 3.5.</p> </li> <li> <p>Add 140 µL of chloroform to each sample of cell lysates. NOTE: Chloroform is volatile and difficult to pipette, close the Eppendorf tubes as soon as possible. </p> </li> <li> <p>Shake the Eppendorf tubes vigorously for 15 s or until the samples are fully emulsified.</p> </li> <li> <p>Keep the sample at RT for 1–2 min or until the liquid phases start to separate.</p> </li> <li> <p>Centrifuge the samples at 12,000 x g for 15 min at 4 °C.</p> </li> <li> <p>Transfer the upper aqueous phase (~320 µL) containing RNA to a new Eppendorf tube, and discard the remaining liquid. NOTE: Do not touch the lower pink organic phase or the white ring between the two phases with the tip of the pipette. </p> </li> <li> <p>Add 1.5 volumes of 100% ethanol (480 µL) to the aqueous phase, and pipette the solution slowly up and down 3 times to mix. </p> </li> <li> <p>Purify RNA using a commercially available kit designed for purification of RNA from small samples. </p> </li> <li> <p>Quantify RNA concentration and sample purity with a spectrophotometer. NOTE: (i) Expect yields of ≥3.5 µg/well; the 260/280 and 260/230 ratios for purity over proteins and organic compounds should both be ≥1.9. (ii) The RNA samples can be stored at -80 °C or processed immediately. </p> </li> <li> <p>Retrotranscribe 250, 500 or 1000 ng of RNA with a commercially available kit.</p> </li> <li> <p>Quantify the knockdown efficiency for the endogenous gene of interest by qRT-PCR. For a detailed protocol, see reference <a href="#B9">10</a>. Aim for a knockdown efficiency of ≥60% ( <a href="#F2">Figure 2</a>). Normalize the data to ≥2 housekeeping genes ( <i>e.g</i>. <i>GAPDH, ACTB, TUBB3, PPIA</i>) using the multiple internal control gene method <a href="#B10">11</a>. NOTE: If working in rat, use the following PCR primers for the housekeeping genes: GAPDH-fwd: 5' GGTGCTGAGTATGTCGTGGA 3' and GAPDH-rev: 5' GATGATGACCCTTTTGGC 3'; ACTB-fwd: 5' CATCACTATCGGCAATGAGC 3' and ACTB-rev: 5' TCATGGATGCCACAGGATT 3'; TUBB3-fwd: 5' GCCTTTGGACACCTATTCAG 3' and TUBB3-rev: 5' TCACATTCTTTCCTCACGAC 3'; PPIA-fwd: 5' CACTGGGGAGAAAGGATTTG 3' and PPIA-rev 5' CCATTATGGCGTGTGAAGTC 3'. </p> </li> </ol> </div> <div class="4.assessingtheroleofpresynapticproteinsinintactneuronalcircuitsbytargetedstimulationofknocked-downneuronswithoptogenetics" title="sec"> <div class="title" tagx="title" title="title">
4. Assessing the Role of Presynaptic Proteins in Intact Neuronal Circuits by Targeted Stimulation of Knocked-down Neurons with Optogenetics </div> <p>NOTE: The following protocol requires previous experience with electrophysiological recordings in acute brain slices and access to an electrophysiological setup. </p> <ol> <li> <p> <b>Stereotactically inject rAAV1/2 into the brain following a detailed protocol in previous publication <a href="#B11">12</a>. Determine the stereotactic coordinates for the brain region of interest using stereotactic atlases for mouse <a href="#B12">13</a> or rat brain <a href="#B13">14</a>. </b> </p><ol> <li> <p>Use a micropipette puller to pull injection micropipettes with long shanks of Ø 7–9 µm; clip the injection micropipettes on the shanks with scissors. Use a thin marker and graph paper to place calibrating marks on the injection micropipettes every 2 mm. </p> </li> <li> <p>Anesthetize the animal with isoflurane and fix it in the stereotactic apparatus.</p> </li> <li> <p>Keep the animal warm throughout the operation with a heating pad set at 37 °C.</p> </li> <li> <p>Protect the eyes with ocular lubricant.</p> </li> <li> <p>Shave the fur on the head with an electric razor.</p> </li> <li> <p>Spread povidone iodine on the shaved head using a cotton bud.</p> </li> <li> <p>Under a dissecting microscope, make a midline incision.</p> </li> <li> <p>Clean the skull surface with a cotton bud, so to make the bregma and lambda visible.</p> </li> <li> <p>Place the injection micropipette into its holder. Attach the holder to the stereotactic arm. </p> </li> <li> <p>Determine the x and y coordinates of the site of injection relative to the bregma and/or the lambda. </p> </li> <li> <p>Use a drill to thin the skull over the target area. NOTE: Use gentle circular movements and avoid drilling through the skull as this will damage the surface of the brain. </p> </li> <li> <p>If there is bleeding, absorb excessive blood with towel paper. NOTE: Excessive blood will make it difficult to determine correctly the z coordinate. </p> </li> <li> <p>Load ≤2 µL of virus into the injection micropipette by capillary action.</p> </li> <li> <p>Bring the injection micropipette to the x and y coordinates of the site of injection.</p> </li> <li> <p>Calculate the z coordinate from the dura and lower the pipette slowly into the brain.</p> </li> <li> <p>When the z coordinate is reached, wait 3 min to allow for tissue adjustment.</p> </li> <li> <p>Apply low positive pressure using a 1 mL syringe connected via a flexible tube to the back of the injection micropipette. Visually monitor the speed of ejection of the virus from the injection micropipette through the dissection microscope and using the calibrating marks as reference points. NOTE: Virus needs to be injected at slow rate (&amp;lt;100 nL/min) to avoid tissue damage. </p> </li> <li> <p>When injection is finished, wait 5 min, withdraw the injection micropipette by 0.2 mm, wait for another 5 min, to avoid backflow of the virus. </p> </li> <li> <p>Withdraw the injection micropipette slowly and completely from the brain and dispose of it into a container filled with bleach. </p> </li> <li> <p>Wet the skull with physiological solution and suture the skin with 3-4 stitches.</p> </li> <li> <p>Apply gentamicin ointment to the wound.</p> </li> <li> <p>Single-house the animal in a clean cage with food pellets and under a heat lamp till it fully recovers. </p> </li> </ol> <p /> </li> <li> <p>≥15 days post-injection, decapitate animals under deep isoflurane anesthesia.</p> </li> <li> <p>Prepare acute brain slices of the brain region of interest with a Vibratome using gassed (95% O <sub>2</sub>, 5% CO <sub>2</sub>) ice-cold aCSF solution, containing (in mM): 123 NaCl, 1.25 KCl, 1.25 KH <sub>2</sub>PO <sub>4</sub>, 1.5 MgCl <sub>2</sub>, 1 CaCl <sub>2</sub>, 25 NaHCO <sub>3</sub>, 2 NaPyruvate and 18 glucose (osmolarity adjusted to 300 mOsm). For example, if the aim is to analyze synaptic transmission from CA3 to CA1 pyramidal neurons, prepare sagittal slices of the hippocampal formation (350 µm thick). NOTE: From this step onwards work under low-light conditions to avoid activation of the optogenetic probe by environmental light. </p> </li> <li> <p>Let the slices recover for 30 min at 37 °C in the same aCSF in a chamber designed for holding brain slices. Keep the slices in the same brain slice chamber at room temperature till recording. NOTE: Using these conditions, slices can be maintained healthy for up to 6–8 h. </p> </li> <li> <p>Transfer one slice to a submerged recording chamber and superfuse it with 2 mL/min of the same aCSF used for recovery supplemented with 1.5 mM CaCl <sub>2</sub> (total Ca <sup>2+</sup>: 2.5 mM) and without NaPyruvate. NOTE: Slices are prepared and maintained in low concentration of Ca <sup>2+</sup> (1 mM) to minimize toxicity. Recordings are performed in 2.5 mM Ca <sup>2+</sup> to favor vesicle release. </p> </li> <li> <p>Check briefly the signal of the expressed fluorescent reporter ( <i>e.g</i>. TdTomato; <a href="#F3">Figure 3B</a>) to ascertain the localization and intensity of infection. </p> </li> <li> <p>Fill a patch electrode with an intracellular solution containing (in mM): 110 K-gluconate, 22 KCl, 5 NaCl, 0.5 EGTA, 3 MgCl <sub>2</sub>, 4 Mg-ATP, 0.5 Na <sub>3</sub>-GTP, 20 K <sub>2</sub>-creatine phosphate, 10 HEPES-KOH (pH 7.28, 290 mΩ). NOTE: Use patch electrode with a pipette resistance of 5–6 mΩ. </p> </li> <li> <p>Under infrared illumination, reach tight-seal whole-cell configuration from a neuron receiving synaptic inputs from the infected neurons. For example, if CA3 pyramidal neurons were infected, patch pyramidal neurons in the proximal to medial tract of the CA1 region ( <a href="#F3">Figure 3C</a>). NOTE: Series resistance can be left uncompensated, but it should be constant and low (≤20 MΩ). </p> </li> <li> <p>Use pharmacology to isolate the synaptic currents under investigation. For example, if the aim is to investigate excitatory synaptic transmission, block inhibitory synaptic transmission with 10 µM bicuculline. </p> </li> <li> <p> <b>Evoke synaptic currents, for example, excitatory postsynaptic currents (EPSCs), using a 473 nm blue laser coupled to an optical fiber (Ø ≤250 µm) positioned on the somata of the infected neurons ( <i>e.g</i>. CA3 pyramidal neurons). </b> </p><ol> <li> <p>Adjust stimulation length to a minimum, to reduce the possibility of evoking more than one action potential per light pulse. If using ChETA, set it to 2 ms <a href="#B14">15</a>. </p> </li> <li> <p>Adjust stimulation strength of the laser to yield small but clearly detectable synaptic currents (≤50 pA peak amplitude for EPSCs recorded at a holding potential of -70 mV between CA3 and CA1 pyramidal neurons; <a href="#F3">Figure 3D</a>). If using ChETA and an optical fiber of 250 µm in diameter, stimulation strengths of 1–3 mW at fiber exit should be suitable. NOTE: if laser strength cannot be regulated, use neutral density filters. </p> </li> <li> <p>Shine the 473 nm laser light on the somata of the infected neurons, but not on their axons. For example, shine the light on CA3 somata, and away from the Schaffer collaterals, to avoid direct depolarization of the axons. </p> </li> <li> <p>Apply tetrodotoxin (TTX; 0.5 µM), a blocker of sodium channels, to the sample to confirm the channels are action potential-driven. </p> </li> <li> <p>In different recordings, apply a selective blocker to the synaptic current under investigation to confirm the stimulation is selective. NOTE: For example, apply NBQX (10 µM) to AMPA-type glutamate receptor-mediated EPSCs. </p> </li> <li> <p>Compare optically and electrically evoked synaptic currents in the same acute brain slice in response to repetitive stimulation (≥2 stimuli at ≤20 Hz). A similar degree of synaptic facilitation or depression should be observed between electrical and optical stimulation when using a control miR, thus suggesting that similar cellular mechanisms are activated by the two types of stimulations. NOTE: The above steps are necessary to ensure that optical stimulation does not induce a direct depolarization of presynaptic boutons, thus bypassing some mechanisms of synaptic transmission. </p> </li> </ol> <p /> </li> <li> <p>Compare synaptic transmission in response to single and repetitive stimulations (≥2 stimuli at ≤20 Hz) between a control miR and miRs targeting the presynaptic proteins under investigation. </p> </li> </ol> </div> </div> <div class="representativeresults" title="sec"> <div class="title" tagx="title" title="title">
Representative Results</div> <p>The procedures described above provide a robust method to assess how synaptic transmission is affected by the knockdown of synaptic proteins in presynaptic neurons. Representative results on how the knockdown of alternative splice isoforms of presynaptic Ca <sub>v</sub>2.1 (P/Q-type) VGCCs regulates short-term synaptic plasticity at CA3 to CA1 excitatory synapses are given below as an example. </p> <p>Ca <sub>v</sub>2.1 (P/Q-type) channels are the predominant presynaptic VGCCs at most fast synapses in the central nervous system. Alternative splicing of the mutually exclusive exons 37a and 37b of the pore-forming α <sub>1</sub> subunit of Ca <sub>v</sub>2.1 (α <sub>1A</sub>) produces two major variants, Ca <sub>v</sub>2.1[EFa] and Ca <sub>v</sub>2.1[EFb] <a href="#B15">16</a> <a href="#B16">17</a> <a href="#B17">18</a>. To determine whether Ca <sub>v</sub>2.1[EFa] and Ca <sub>v</sub>2.1[EFb] differentially regulate synaptic transmission and plasticity in rat hippocampal pyramidal neurons, we first developed isoform-specific miRs to knockdown selectively Ca <sub>v</sub>2.1[EFa] or Ca <sub>v</sub>2.1[EFb] <a href="#B4">5</a>. Despite the short size (97 bp) and high similarity (61.86% identity at the nucleotide level) between exons 37a and 37b, we could design three miR sequences against rat Ca <sub>v</sub>2.1[EFa] (miR EFa1: TCCTTATAGTGAATGCGGCCG; miR EFa2: ATGTCCTTATAGTGAATGCGG; miR EFa3: TTGCAAGCAACCCTATGAGGA) and two against rat Ca <sub>v</sub>2.1[EFb] (miR EFb1: ATACATGTCCGGGTAAGGCAT; miR EFb2: ATCTGATACATGTCCGGGTAA) with predicted high knockdown efficiency. As negative control (miR Control), we used the pcDNA6.2-GW/EmGFP-miR-neg plasmid containing a sequence that does not target any known vertebrate gene. Based on a first screen in HEK 293 cells against heterologous channels, we selected miR EFa1, miR EFa3 and miR EFb2 and cloned their expression cassettes into the 3'UTR of the vector pAAV-Syn-ChETA-TdT-miR-X, which is designed for production of rAAVs and where the synapsin promoter drives expression of the ultrafast channelrhodopsin ChETA, the red fluorescent protein TdTomato and the inserted miR ( <a href="#F1">Figure 1</a>). We also duplicated the expression cassette of miR EFb2 to increase the knockdown efficiency of this miR. </p> <p>Next, we prepared rAAV1/2 for the above four constructs, and quantified their knockdown efficiency and selectivity in primary rat neuronal cultures using isoform-specific qRT-PCR. miR EFa1 and miR EFa3 reduced mRNA of native Ca <sub>v</sub>2.1[EFa] by ~70% but not that of Ca <sub>v</sub>2.1[EFb], whilst miR EFb reduced mRNA of native Ca <sub>v</sub>2.1[EFb] by ~60% but not that of Ca <sub>v</sub>2.1[EFa] ( <a href="#F2">Figure 2</a>). </p> <p>We then stereotactically injected each of the four rAAV1/2 into the CA3 area of the hippocampus of P18 rats ( <a href="#F3">Figure 3A</a> <b>-C</b>), with coordinates of (A-P/M-L/D-V from Bregma) −2.6/± 2.9/−2.9. Fifteen to twenty-four days post-injection, we prepared acute hippocampal slices from rAAV1/2-injected rats and used TdTomato fluorescence to confirm the expression and localization of rAAVs ( <a href="#F3">Figure 3B</a>). To investigate whether presynaptic knockdown of either Ca <sub>v</sub>2.1[EFa] or Ca <sub>v</sub>2.1[EFb] affected short-term synaptic plasticity at CA3 to CA1 synapses, we stimulated selectively infected CA3 neurons with brief 473 nm laser light pulses (2 ms-long; <a href="#F3">Figure 3C</a>), and recorded the resulting EPSCs by patching pyramidal neurons in the proximal to medial tract of the CA1 region ( <a href="#F3">Figure 3C</a>). We found that the knockdown of Ca <sub>v</sub>2.1 splice isoforms affected responses to paired-pulse stimulation in opposite directions: knockdown of Ca <sub>v</sub>2.1[EFa] (miR EFa1 or miR EFa3) boosted paired-pulse facilitation (PPF) whereas knockdown of Ca <sub>v</sub>2.1[EFb] (miR EFb2) abolished it ( <a href="#F3">Figure 3D</a> <b>, E</b>). </p> <p> </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/" /> </div>  </div> <b>Figure 1: Scheme of the constructs for combined expression of the ultrafast optogenetic probe ChETA and isoform-specific miRs against Ca <sub>v</sub>2.1[EFa] and Ca <sub>v</sub>2.1[EFb]. (A) </b>Map of the rAAV construct pAAV-Syn-ChETA-TdT-miR-X, containing a synapsin promoter (Syn), the ultrafast channelrhodopsin ChETA fused to TdTomato and, in the proximal 3'UTR, a Ca <sub>v</sub>2.1 splice isoform-specific miR. Restriction enzymes shown are single cutters. <b>(B) </b>Top, scheme of the expression cassette. The synapsin promoter drives expression of both ChETA-TdTomato and an isoform-specific miR. Bottom, reverse complement of the 21-nucleotide target sequences, which form part of the miR cassette. For miR EFb2 two identical miR cassettes were expressed in tandem one after the other. <a href="https://cloudflare.jove.com/files/ftp_upload/57223/57223fig1large.jpg">Please click here to view a larger version of this figure.</a> <p /> <p> </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/" /> </div>  </div> <b>Figure 2. Evaluation of the knockdown efficiency and selectivity of isoform-specific miRs for Ca <sub>v</sub>2.1[EFa] and Ca <sub>v</sub>2.1[EFb]. </b> Isoform-specific qRT-PCR analysis on RNA isolated from 17-18 DIV primary cultures infected at 6 DIV with rAAVs expressing miRs targeting either Ca <sub>v</sub>2.1[EFa] (miR EFa1 and miR EFa3) or Ca <sub>v</sub>2.1[EFb] (miR EFb2). Data are normalized to the negative control (miR Control). miR EFa1 and miR EFa3 significantly and selectively reduce mRNA of Ca <sub>v</sub>2.1[EFa] (n = 8 and 7 cultures, respectively), while miR EFb significantly and selectively reduces mRNA of Ca <sub>v</sub>2.1[EFb] (n = 4 cultures; ***p&amp;lt;0.001; one-way analysis of variance test followed by the Tukey-Kramer post-test). Data are presented as mean ± SEM. This figure has been adapted from Thalhammer, A. <i> et al.</i> <a href="#B4">5</a>. <a href="https://cloudflare.jove.com/files/ftp_upload/57223/57223fig2large.jpg">Please click here to view a larger version of this figure.</a> <p /> <p> </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/" /> </div>  </div> <b>Figure 3. Assessing the role of Ca <sub>v</sub>2.1[EFa] and Ca <sub>v</sub>2.1[EFb] in the native hippocampus by targeted stimulation of knocked-down neurons with optogenetics. </b> <b>(A)</b> Scheme of the expression cassette of the rAAV constructs used for <i>in vivo</i> infection. <b>(B)</b> Hippocampal section showing that TdTomato fluorescence is limited to the CA3 region and its projections. <b>(C)</b> Experimental configuration: laser beam was directed onto CA3 somata, and patch clamp recordings were performed from CA1 pyramidal neurons. <b>(D)</b> 2 ms-long blue (473 nm) laser light pulses shone at 20 Hz evoke EPSCs whose PPF is increased by miRs targeting Ca <sub>v</sub>2.1[EFa] and abolished by miR for Ca <sub>v</sub>2.1[EFb]. <b>(E)</b> Summary of paired-pulse ratio for experiments as in (D), showing an increase in PPF for miR EFa1 and miR EFa3 and a decrease for miR EFb2, relative to miR Control (n = 9 - 11 recordings; *p=0.02; **p=0.01; ***p&amp;lt;0.0004; analysis of covariance). Data are presented as mean ± SEM. This figure has been adapted from Thalhammer, A. <i> et al.</i> <a href="#B4">5</a>. <a href="https://cloudflare.jove.com/files/ftp_upload/57223/57223fig3large.jpg">Please click here to view a larger version of this figure.</a> <p /> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Co-expression of an optogenetic probe and a miR against a presynaptic gene of interest offers a powerful approach to characterize the effect of gene knockdown on presynaptic function within intact neuronal circuits. For this experimental approach, it is important to identify and characterize miRs that are highly efficient and selective in knocking down the gene of interest in native systems. If possible, two or more independent miRs against the same mRNA of interest should be used to control for eventual off-target effects. Rescue experiments, in which a miR-resistant gene is reintroduced in the system, can be used as a control for specificity. </p> <p>Using the same construct to express an optogenetic probe and a miR allows for stimulating optically only the presynaptic neurons that have been manipulated. This is not possible with electrical stimulations because they do not distinguish between infected and non-infected neurons, thus producing mixed and diluted results. Because optical stimulations can induce multiple action potentials <a href="#B14">15</a>, it is important to choose only the fastest optogenetic probes, among an expanding palette <a href="#B14">15</a> <a href="#B18">19</a> <a href="#B19">20</a> <a href="#B20">21</a>. In addition, it is essential to ensure that optical stimulation does not induce a direct depolarization of presynaptic boutons, to avoid bypassing some of the steps of synaptic transmission one wishes to investigate <a href="#B21">22</a>. </p> <p>The experimental approach we describe here, makes it possible to evaluate in parallel the physiological relevance of multiple presynaptic genes of interest within a limited time period (4–6 months). However, it is important to keep in mind that knockdowns rarely reach 100%. Moreover, rAAVs need to be expressed for at least two weeks to allow for maximal knockdown and full expression of the optogenetic probe, which may represent a time constraint when investigating early developmental processes. Though more time consuming, conditional knockout mice limited to presynaptic neurons, generally result in complete removal of the gene of interest and offer therefore a valid complementary approach. </p> <p>A specific advantage of the miR technology is that it enables the expression of multiple miRs from the same promoter. This property has mainly been used to increase knockdown efficiency by inserting multiple copies of the same miR or different miRs against the same target gene. It can however be used also to knockdown multiple genes by expressing miRs against different target genes <a href="#B6">7</a> <a href="#B22">23</a>. This property might be used to silence multiple presynaptic proteins to dissect presynaptic signaling pathways. </p> <p>Here, we have combined optogenetics with artificial miRs to characterize the effects of gene knockdown on presynaptic function in acute hippocampal slices. A similar approach could also be used to manipulate and probe neuronal circuitry <i>in vivo</i>. In addition, combining artificial miRs with chemogenetic approaches would enable one to interrogate neuronal circuitry on longer time scales. </p> </div> <div class="disclosures" title="sec"> <div class="title" tagx="title" title="title">
Disclosures</div> <p>The authors have nothing to disclose.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank F. Benfenati (Istituto Italiano di Tecnologia, IIT) for support and Carmela Vitale for helping with the demonstration. This work was funded by IIT and the Compagnia San Paolo (grant no. 9734 to LAC). </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="B0" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lam</span> <span class="given-names" tagx="given-names" title="given-names">JK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chow</span> <span class="given-names" tagx="given-names" title="given-names">MY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leung</span> <span class="given-names" tagx="given-names" title="given-names">SW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">siRNA Versus miRNA as Therapeutics for Gene Silencing</span> <span class="source" tagx="source" title="source">Mol Ther-Nucl Acids</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">e252</span> </span> </li> <li tag="ref"> <a name="B1" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kohl</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Hemisphere-specific optogenetic stimulation reveals left-right asymmetry of hippocampal plasticity </span> <span class="source" tagx="source" title="source">Nat Neurosci</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">1413</span> <span class="lpage" tagx="lpage" title="lpage">1415</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21946328">21946328</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maejima</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Postnatal loss of P/Q-type channels confined to rhombic-lip-derived neurons alters synaptic transmission at the parallel fiber to purkinje cell synapse and replicates genomic Cacna1a mutation phenotype of ataxia and seizures in mice </span> <span class="source" tagx="source" title="source">J Neurosci</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">5162</span> <span class="lpage" tagx="lpage" title="lpage">5174</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23516282">23516282</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mark</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Delayed postnatal loss of P/Q-type calcium channels recapitulates the absence epilepsy, dyskinesia, and ataxia phenotypes of genomic Cacna1a mutations </span> <span class="source" tagx="source" title="source">J Neurosci</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">4311</span> <span class="lpage" tagx="lpage" title="lpage">4326</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21411672">21411672</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thalhammer</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Alternative Splicing of P/Q-Type Ca2+ Channels Shapes Presynaptic Plasticity</span> <span class="source" tagx="source" title="source">Cell Rep</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">333</span> <span class="lpage" tagx="lpage" title="lpage">343</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28700936">28700936</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sambrook</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Russell</span> <span class="given-names" tagx="given-names" title="given-names">DW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Calcium-phosphate-mediated Transfection of Eukaryotic Cells with Plasmid DNAs</span> <span class="source" tagx="source" title="source">CSH Protoc</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">2006</span> <span class="issue" tagx="issue" title="issue">1</span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chung</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Polycistronic RNA polymerase II expression vectors for RNA interference based on BIC/miR-155</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">e53</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16614444">16614444</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McClure</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cole</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wulff</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klugmann</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murray</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Production and titering of recombinant adeno-associated viral vectors</span> <span class="source" tagx="source" title="source">J Vis Exp</span> <span class="year" tagx="year" title="year">2011</span> <span class="issue" tagx="issue" title="issue">57</span> <span class="fpage" tagx="fpage" title="fpage">e3348</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22143312">22143312</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cingolani</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Activity-dependent regulation of synaptic AMPA receptor composition and abundance by beta3 integrins </span> <span class="source" tagx="source" title="source">Neuron</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">749</span> <span class="lpage" tagx="lpage" title="lpage">762</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18549786">18549786</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Detection of alternative splicing during epithelial-mesenchymal transition</span> <span class="source" tagx="source" title="source">J Vis Exp</span> <span class="year" tagx="year" title="year">2014</span> <span class="issue" tagx="issue" title="issue">92</span> <span class="fpage" tagx="fpage" title="fpage">e51845</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25350517">25350517</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vandesompele</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes </span> <span class="source" tagx="source" title="source">Genome Biol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">7</span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cetin</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Komai</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eliava</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seeburg</span> <span class="given-names" tagx="given-names" title="given-names">PH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Osten</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Stereotaxic gene delivery in the rodent brain</span> <span class="source" tagx="source" title="source">Nat Protoc</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">3166</span> <span class="lpage" tagx="lpage" title="lpage">3173</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17406580">17406580</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paxinos</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Franklin</span> <span class="given-names" tagx="given-names" title="given-names">KB</span> </span> </span> <span class="source" tagx="source" title="source">The Mouse Brain in Stereotaxic Coordinates</span> <span class="year" tagx="year" title="year">2012</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press</span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paxinos</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Watson</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="source" tagx="source" title="source">The Rat Brain in Stereotaxic Coordinates</span> <span class="year" tagx="year" title="year">2013</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press</span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gunaydin</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Ultrafast optogenetic control</span> <span class="source" tagx="source" title="source">Nat Neurosci</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">387</span> <span class="lpage" tagx="lpage" title="lpage">392</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20081849">20081849</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bourinet</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Splicing of alpha 1A subunit gene generates phenotypic variants of P- and Q-type calcium channels </span> <span class="source" tagx="source" title="source">Nat Neurosci</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">407</span> <span class="lpage" tagx="lpage" title="lpage">415</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10321243">10321243</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chaudhuri</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Alternative splicing as a molecular switch for Ca2+/calmodulin-dependent facilitation of P/Q-type Ca2+ channels </span> <span class="source" tagx="source" title="source">J Neurosci</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="issue" tagx="issue" title="issue">28</span> <span class="fpage" tagx="fpage" title="fpage">6334</span> <span class="lpage" tagx="lpage" title="lpage">6342</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15254089">15254089</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soong</span> <span class="given-names" tagx="given-names" title="given-names">TW</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Systematic identification of splice variants in human P/Q-type channel alpha1(2.1) subunits: implications for current density and Ca2+-dependent inactivation </span> <span class="source" tagx="source" title="source">J Neurosci</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="issue" tagx="issue" title="issue">23</span> <span class="fpage" tagx="fpage" title="fpage">10142</span> <span class="lpage" tagx="lpage" title="lpage">10152</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12451115">12451115</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berndt</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High-efficiency channelrhodopsins for fast neuronal stimulation at low light levels</span> <span class="source" tagx="source" title="source">P Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">108</span> <span class="issue" tagx="issue" title="issue">18</span> <span class="fpage" tagx="fpage" title="fpage">7595</span> <span class="lpage" tagx="lpage" title="lpage">7600</span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klapoetke</span> <span class="given-names" tagx="given-names" title="given-names">NC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Independent optical excitation of distinct neural populations</span> <span class="source" tagx="source" title="source">Nat Methods</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">338</span> <span class="lpage" tagx="lpage" title="lpage">346</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24509633">24509633</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">JY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">MZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steinbach</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsien</span> <span class="given-names" tagx="given-names" title="given-names">RY</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of engineered channelrhodopsin variants with improved properties and kinetics </span> <span class="source" tagx="source" title="source">Biophys J</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">96</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">1803</span> <span class="lpage" tagx="lpage" title="lpage">1814</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19254539">19254539</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">YP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oertner</span> <span class="given-names" tagx="given-names" title="given-names">TG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optical induction of synaptic plasticity using a light-sensitive channel</span> <span class="source" tagx="source" title="source">Nat Methods</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">139</span> <span class="lpage" tagx="lpage" title="lpage">141</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17195846">17195846</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fowler</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerritsen</span> <span class="given-names" tagx="given-names" title="given-names">AT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Washbourne</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Improved knockdown from artificial microRNAs in an enhanced miR-155 backbone: a designer's guide to potent multi-target RNAi </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">e48</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26582923">26582923</a> </span> </span> </li> </ul> </div> </div>  </body></html>