<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">eNeuro</span> <span class="iso-abbrev" title="iso-abbrev">eNeuro</span> <span class="hwp" title="hwp">eneuro</span> <span class="pmc" title="pmc">eneuro</span> <span class="publisher-id" title="publisher-id">eNeuro</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">eNeuro</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2373-2822</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Society for Neuroscience</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6565376">6565376</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1523/ENEURO.0066-19.2019">10.1523/ENEURO.0066-19.2019</a> </span> <span class="publisher-id" title="publisher-id">eN-NWR-0066-19</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">1</span> <span class="subject" title="subject">1.1</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">New Research</span>: <span class="subject" title="subject">Cognition and Behavior</span> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
The NeuroD6 Subtype of VTA Neurons Contributes to Psychostimulant Sensitization and Behavioral Reinforcement </div> <div class="alt-title" title="alt-title">
NeuroD6 VTA Neurons in Reward-Related Behavior</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-8819-7957</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bimpisidis</span> <span class="given-names" tagx="given-names" title="given-names">Zisis</span> </span> <a href="#aff1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">König</span> <span class="given-names" tagx="given-names" title="given-names">Niclas</span> </span> <a href="#aff1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stagkourakis</span> <span class="given-names" tagx="given-names" title="given-names">Stefanos</span> </span> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zell</span> <span class="given-names" tagx="given-names" title="given-names">Vivien</span> </span> <a href="#aff3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-5442-2303</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vlcek</span> <span class="given-names" tagx="given-names" title="given-names">Bianca</span> </span> <a href="#aff1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dumas</span> <span class="given-names" tagx="given-names" title="given-names">Sylvie</span> </span> <a href="#aff5"> <sup>5</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-5876-9822</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giros</span> <span class="given-names" tagx="given-names" title="given-names">Bruno</span> </span> <a href="#aff6"> <sup>6</sup> </a> <a href="#aff7"> <sup>7</sup> </a> <a href="#aff8"> <sup>8</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-7050-8809</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Broberger</span> <span class="given-names" tagx="given-names" title="given-names">Christian</span> </span> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-6176-8513</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hnasko</span> <span class="given-names" tagx="given-names" title="given-names">Thomas S.</span> </span> <a href="#aff3"> <sup>3</sup> </a> <a href="#aff4"> <sup>4</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-8713-070X</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wallén-Mackenzie</span> <span class="given-names" tagx="given-names" title="given-names">Åsa</span> </span> <a href="#aff1"> <sup>1</sup> </a> </span> <span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Uppsala University</span> <span class="country" tagx="country" title="country">Sweden</span> </span> <span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">Karolinska Institutet</span> <span class="country" tagx="country" title="country">Sweden</span> </span> <span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">University of California, San Diego</span> </span> <span class="citation_author_institution" id="aff4">[4], <span class="institution" title="institution">Research Service VA San Diego Healthcare System</span> </span> <span class="citation_author_institution" id="aff5">[5], <span class="institution" title="institution">Oramacell</span> <span class="country" tagx="country" title="country">France</span> </span> <span class="citation_author_institution" id="aff6">[6], <span class="institution" title="institution">Institut National de la Santé et de la Recherche Médicale, INSERM UMRS 1130; Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8246; Sorbonne University Université Pierre-et-Marie-Curie </span> <span class="country" tagx="country" title="country">France</span> </span> <span class="citation_author_institution" id="aff7">[7], <span class="country" tagx="country" title="country">Canada</span> </span> <span class="citation_author_institution" id="aff8">[8], <span class="country" tagx="country" title="country">Canada</span> </span> </div> <div class="author-notes" title="author-notes"> <div class="fn-type-other" title="other"> <p>S.D. is the owner of Oramacell. All other authors declare no competing financial interests.</p> </div> <div class="fn-type-con" title="con"> <p>Author contributions: Å.W.M conceived the study and was in charge of overall direction and planning; Z.B., S.S., V.Z., S.D., C.B., T.S.H., and Å.W.-M. designed research; Z.B., N.K., S.S., V.Z., B.V., S.D., and Å.W.-M. performed research; Z.B., N.K., S.S., V.Z., B.V., S.D., C.B., T.S.H., and Å.W.-M. analyzed data; Z.B. and Å.W.-M. wrote the paper; B.G. contributed transgenic tool. </p> </div> <div class="fn-type-supported-by" title="supported-by"> <p>Work in Å.W.-M. lab was supported by Uppsala University, Vetenskapsrådet (Medicine &amp;amp; Health), Hjärnfonden, Parkinsonfonden, and the Research Foundations of Bertil Hållsten, OE &amp;amp; Edla Johansson, Zoologisk Forskning, and Åhlén. Work in C.B. lab was supported by Vetenskapsrådet (Medicine &amp;amp; Health), the European Research Council, Novo Nordisk Fonden, and the Strategic Research Programme for Diabetes Research at Karolinska Institutet. Work in T.S.H. lab was supported by the National Institutes of Health Grant DA036612 and the Veterans Affairs Grant BX003759. </p> </div> <div class="corresp" title="corresp">
Correspondence should be addressed to Åsa Wallén-Mackenzie at <span class="email" tagx="email" title="email">asa.mackenzie@ebc.uu.se</span> </div> </div> <span class="pub-date-epreprint" title="pub-date-epreprint">epreprint: <span>2019-5-5</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-6-6</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span> </span> <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">ENEURO.0066-19.2019</span> <span class="history" title="history"> <span class="received" title="received">received: 2019-2-22</span> <span class="rev-recd" title="rev-recd">rev-recd: 2019-5-7</span> <span class="accepted" title="accepted">accepted: 2019-5-9</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Bimpisidis et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>, which permits unrestricted use, distribution and reproduction in any medium provided that the original work is properly attributed. </span> </span> </div> <a class="self-uri" href="ENEURO.0066-19.2019.pdf" title="self-uri">ENEURO.0066-19.2019.pdf</a> <a class="self-uri" href="enu00319294100g1.jpg" title="self-uri">enu00319294100g1.jpg</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Reward-related behavior is complex and its dysfunction correlated with neuropsychiatric illness. Dopamine (DA) neurons of the ventral tegmental area (VTA) have long been associated with different aspects of reward function, but it remains to be disentangled how distinct VTA DA neurons contribute to the full range of behaviors ascribed to the VTA. Here, a recently identified subtype of VTA neurons molecularly defined by NeuroD6 (NEX1M) was addressed. Among all VTA DA neurons, less than 15% were identified as positive for NeuroD6. In addition to dopaminergic markers, sparse NeuroD6 neurons expressed the vesicular glutamate transporter 2 ( <i>Vglut2</i>) gene. To achieve manipulation of NeuroD6 VTA neurons, NeuroD6(NEX)-Cre-driven mouse genetics and optogenetics were implemented. First, expression of vesicular monoamine transporter 2 (VMAT2) was ablated to disrupt dopaminergic function in NeuroD6 VTA neurons. Comparing <i>Vmat2 <sup>lox/lox;NEX-Cre</sup> </i> conditional knock-out (cKO) mice with littermate controls, it was evident that baseline locomotion, preference for sugar and ethanol, and place preference upon amphetamine-induced and cocaine-induced conditioning were similar between genotypes. However, locomotion upon repeated psychostimulant administration was significantly elevated above control levels in cKO mice. Second, optogenetic activation of NEX-Cre VTA neurons was shown to induce DA release and glutamatergic postsynaptic currents within the nucleus accumbens. Third, optogenetic stimulation of NEX-Cre VTA neurons <i>in vivo</i> induced significant place preference behavior, while stimulation of VTA neurons defined by Calretinin failed to cause a similar response. The results show that NeuroD6 VTA neurons exert distinct regulation over specific aspects of reward-related behavior, findings that contribute to the current understanding of VTA neurocircuitry. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
accumbens</div> <div class="kwd" title="kwd">
dopamine</div> <div class="kwd" title="kwd">
mouse genetics</div> <div class="kwd" title="kwd">
optogenetics</div> <div class="kwd" title="kwd">
reward</div> <div class="kwd" title="kwd">
ventral tegmental area</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
hjärnfonden</div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
vetenskapsrådet</div> </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="equation-count" title="equation-count">equation-count: </span> <span class="ref-count" title="ref-count">ref-count: </span> <span class="page-count" title="page-count">page-count: </span> <span class="word-count" title="word-count">word-count: </span> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="cover-date" value="May/June 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Significance Statement</div> <p>Reward-related behavior is complex and its dysfunction is implicated in many neuropsychiatric disorders, including drug addiction. Midbrain dopamine (mDA) neurons of the ventral tegmental area (VTA) are crucial for reward behavior, but due to recently uncovered heterogeneity, it remains to be fully resolved how they regulate reward responsiveness and how their dysfunction might contribute to disease. Here we show that the recently described NeuroD6 (NEX) subtype of VTA DA neurons is involved in psychostimulant sensitization and that optogenetic stimulation of NEX-Cre VTA neurons induces DA release, glutamatergic postsynaptic currents, and real-time place preference behavior. NeuroD6 VTA neurons thus exert distinct regulation over specific aspects of reward-related behavior, findings that contribute to the current understanding of VTA neurocircuitry. </p> </div> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The midbrain dopamine (mDA) system mediates a diverse spectrum of behaviors and their dysfunction is correlated with a range of severe behavioral disorders including substance use disorder, schizophrenia, ADHD and Parkinson’s disease (PD). Consequently, therapies based on modulating the activity of the mDA system are commonly prescribed, however, due to their unselective nature, current treatments often fail to alleviate symptoms and instead cause adverse effects ( <a href="#B52">Weintraub, 2008</a>; <a href="#B10">Divac et al., 2014</a>). One reason for the lack of successful treatment is incomplete understanding of the underlying neurobiology. Indeed, it is increasingly understood that the mDA system is highly heterogeneous (for review, see <a href="#B40">Pupe and Wallén-Mackenzie, 2015</a>; <a href="#B34">Morales and Margolis, 2017</a>). Beyond the classical separation into the ventral tegmental area (VTA) and substantia nigra pars compacta (SNc), with VTA projections to cortical and limbic target areas and SNc projections to the dorsal striatum subserving cognitive/affective and motor functions, respectively ( <a href="#B3">Björklund and Dunnett, 2007</a>), a higher level of complexity is now being unfolded: afferent and efferent projections, electrophysiological patterns, capacity for glutamate or GABA co-release, and responsiveness to appetitive or aversive stimuli are some of the properties that distinguish mDA neurons from each other ( <a href="#B29">Lammel et al., 2011</a>; <a href="#B2">Beier et al., 2015</a>; <a href="#B33">Menegas et al., 2015</a>; <a href="#B12">Faget et al., 2016</a>). </p> <p>Likely coupled to this functional diversity is a complex diversity in molecular identity. Microarray-based analyses have identified gene expression patterns enriched in VTA over SNc DA neurons ( <a href="#B7">Chung et al., 2005</a>; <a href="#B16">Greene et al., 2005</a>; <a href="#B50">Viereckel et al., 2016</a>) while single cell profiling has begun to identify combinatorial gene expression patterns that molecularly define subtypes of mDA neurons ( <a href="#B38">Poulin et al., 2014</a>; <a href="#B28">La Manno et al., 2016</a>; <a href="#B18">Hook et al., 2018</a>). Based on this new knowledge, intersectional genetic approaches were recently described in which the distinct projection pathways of several newly defined subtypes of mDA neurons were identified ( <a href="#B39">Poulin et al., 2018</a>). By forwarding the current knowledge toward molecularly defined, and thus targetable, subtypes of mDA neurons with distinct projection patterns, these recent advances enhance the possibility of improving selectivity in treatment of dopaminergic disorders. However, a key issue that remains to be resolved is how each molecularly defined subtype of DA neuron contributes to the complex range of behaviors ascribed to the mDA system. </p> <p>The gene encoding the transcription factor NeuroD6 (also known as NEX1M) has recently gained attention due to its selective expression within subsets of VTA DA neurons while being excluded from the SNc ( <a href="#B50">Viereckel et al., 2016</a>; <a href="#B25">Khan et al., 2017</a>; <a href="#B27">Kramer et al., 2018</a>). VTA DA neurons are of particular interest for several reasons. First, the importance of VTA DA neurons in several aspects of behavioral reinforcement and conditioning has been established through classical studies (for review, see <a href="#B9">Di Chiara and Bassareo, 2007</a>; <a href="#B20">Ikemoto, 2007</a>), and more recently, by the use of optogenetics ( <a href="#B49">Tsai et al., 2009</a>; <a href="#B26">Kim et al., 2012</a>; <a href="#B22">Ilango et al., 2014</a>; <a href="#B37">Pascoli et al., 2015</a>). However, detailed knowledge of the exact nature of those particular DA neurons that contribute to each of these complex behaviors remains elusive. Second, medial DA neurons mediate the most potent responsiveness to addictive drugs via their projection to the nucleus accumbens shell (NAcSh; <a href="#B21">Ikemoto and Bonci, 2014</a>). The possibility to ascribe specific aspects of drug responses to a distinct subtype of VTA DA neurons would therefore enhance the understanding of addictive behavior. Third, certain VTA neurons show resistance to degeneration in PD ( <a href="#B5">Brichta and Greengard, 2014</a>); however, depending on their role in behavioral regulation, surviving VTA neurons might contribute to non-motor symptoms including behavioral addictions ( <a href="#B6">Cenci et al., 2015</a>). </p> <p>While <i>NeuroD6</i>-expressing DA neurons were recently identified as neuroprotected in experimental PD ( <a href="#B27">Kramer et al., 2018</a>), the potential role of this newly described subtype of VTA neurons in behavioral regulation has remained unexplored. Here, we implemented NeuroD6-Cre mice (also known as NEX-Cre) to create opportunities for targeting and manipulation of the NeuroD6 subtype VTA neurons. We show that gene targeting of vesicular monoamine transporter 2 (VMAT2) within this particular DA neuron subtype elevated the locomotor response to psychostimulants while activation of NeuroD6-Cre neurons by optogenetic stimulation in the medial VTA induced DA release and glutamatergic postsynaptic responses in the NAcSh. <i>In vivo</i> optogenetic activation of the NeuroD6-Cre VTA subpopulation in a real-time place preference (RT-PP) failed to trigger a conditioned response (CR) but induced place preference upon direct stimulation. These results advance the current understanding of the VTA circuitry by identifying discrete aspects of reward-related behavior correlated with the NeuroD6 subtype VTA neurons. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="mice" title="sec"> <div class="title" tagx="title" title="title">
Mice</div> <p>Mice were provided with food and water ad libitum and housed according to Swedish legislation (Animal Welfare Act SFS 1998:56) and European Union legislation (Convention ETS 123 and Directive 2010/63/EU). Mice of either sex were used. Experiments were conducted with permission from the local Animal Ethical Committees. DAT-Cre ( <a href="#B11">Ekstrand et al., 2007</a>), Vglut2-Cre ( <a href="#B4">Borgius et al., 2010</a>), Calb2-Cre (The Jackson Laboratory, RRID: <a href="https://scicrunch.org/resolver/MGI_4365741">MGI_4365741</a>), and NeuroD6-Cre/NEX-Cre ( <a href="#B15">Goebbels et al., 2006</a>) transgenic mice were bred with C57BL/6N Tac wild-type mice (Taconic) for optogenetics-based experiments. NEX-Cre mice were also bred with <i>Vmat2 <sup>lox/lox</sup> </i> mice, in which exon 2 of the <i>Vmat2</i> gene is flanked by LoxP sites ( <a href="#B35">Narboux-Nême et al., 2011</a>) to generate conditional knock-out (cKO; <i>Vmat2 <sup>lox/lox;NEX-Cre-tg</sup> </i>) mice in which <i>Vmat2</i> exon 2 is ablated on NEX-Cre-mediated recombination of LoxP sites. Littermate mice negative for the NEX-Cre-transgene served as control mice ( <i>Vmat2 <sup>lox/lox;NEX-Cre-wt</sup> </i>: Ctrl; illustrated in <a href="#F2">Fig. 2 <i>A</i> </a>). Mice were genotyped by PCR using the following primer sequences: Cre (applies to DAT-Cre, NEX-Cre, and Calb2-Cre): 5’-ACG AGT GAT GAG GTT CGC AAG A-3’; 5’-ACC GAC GAT GAA GCA TGT TTA G-3’; Vglut2-Cre: 5’-TTG CAT CGC ATT GTC TGA GTA G-3’; 5’-TTC CCA CAC AAG ATA CAG ACT CC-3’; Vmat2Lox: 5’-GAC TCA GGG CAG CAC AAA TCT CC-3’; 5’-GAA ACA TGA AGG ACA ACT GGG ACC C-3’. </p> </div> <div class="insituhybridization(ish)" title="sec"> <div class="title" tagx="title" title="title"> <i>In situ</i> hybridization (ISH) </div> <p>For ISH using radioactive oligoprobes, the following probes sequences were used:</p> <p>NeuroD6: NM_009717.2; bases 99–132, 933–966, and 1256–1288</p> <p>Th: NM_009377.1; bases 774–807, 272–305, and 1621–1655</p> <p>Vmat2exon1: NM_172523.3; bases 18–51 and 83–116</p> <p>Vmat2exon2: NM_172523.3; bases 201–237 and 240–276</p> <p>Oligoprobes were 3’ end-labeled with [alpha- <sup>35</sup>S]dATP using terminal deoxynucleotidyl transferase at a specific activity of 5 × 10 <sup>8</sup> d.p.m./µg. Sections were fixed in 3.7% formaldehyde in PBS for 1 h, washed in PBS, rinsed in water, dehydrated in 70% ethanol and air dried. Hybridization was conducted at 42°C for 16 h in hybridization medium (Oramacell) containing the labeled antisense oligonucleotides (3.10 <sup>5</sup> cpm/100 µl). Sections were washed to a final stringency of 0.5 SSC at 53°C, dehydrated in ethanol, air-dried and exposed to Fujifilm BioImaging Analyzer BAS-5000 for 15 d. </p> <p>For double and triple ISH using riboprobes [fluorescent ISH (FISH) or combined FISH/brightfield ISH (FISH/ISH)], the following probes sequences were used: </p> <p>Calb2: NM_007586.1; bases 80–793</p> <p>Dat (Slc6a3): NM_012694.2; bases 1015–1938</p> <p>NeuroD6: NM_009717.2; bases 635–1419</p> <p>Th: NM_009377.1; bases 456–1453</p> <p>Vglut2 (Slc17a6): NM_080853.3; bases 2315–3244</p> <p>Viaat (Slc32a1): NM_009508.2; bases 649–1488</p> <p>Vmat 2 probe 1: Vmat2: NM_0130331.1 (rat); bases 701–1439 (corresponds to exon 6–15 of mouse sequence NM_172523.3) </p> <p>Vmat2 probe 2: NM_172523.3; bases142–274, i.e., the whole exon 2.</p> <p>Detection of Th, Dat, Vglut2, Viaat, Calb2, NeuroD6 mRNA, and Vmat2 probe 1 and probe 2 mRNA in brain tissue using ISH was performed following a previously published protocol ( <a href="#B50">Viereckel et al., 2016</a>). Briefly, mice were sacrificed and brains dissected. Coronal cryosections were prepared, air-dried, fixed in 4% paraformaldehyde and acetylated in 0.25% acetic anhydride/100 mM triethanolamine (pH 8) followed by hybridization for 18 h at 65°C in 100 μl of formamide-buffer containing 1 μg/ml digoxigenin (DIG)-labeled probe for colorimetric detection or 1 μg/ml DIG- or 1 μg/ml fluorescein-labeled probes for fluorescent detection. Sections were washed at 65°C with SSC buffers of decreasing strength, and blocked with 20% FBS and 1% blocking solution. For colorimetric detection, DIG epitopes were detected with alkaline phosphatase-coupled anti-DIG fab fragments at 1:500 and signal developed with NBT/BCIP. For fluorescent detection, sections were incubated with HRP-conjugated anti-fluorescein antibody at 1:1000 concentration (Roche catalog #11426346910, RRID: <a href="https://scicrunch.org/resolver/AB_840257">AB_840257</a>). Signals were revealed with the TSA kit (PerkinElmer catalog #NEL749A001KT) using biotin tyramide at 1:75 concentration followed by incubation with neutravidin Oregon Green conjugate at 1:750 (Invitrogen catalog #A-6374, RRID: <a href="https://scicrunch.org/resolver/AB_2315961">AB_2315961</a>). HRP-activity was stopped by incubation of sections in 0.1 M glycine and 3% H <sub>2</sub>O <sub>2</sub>. DIG epitopes were detected with HRP-conjugated anti-DIG antibody at 1:1000 (Roche catalog #11207733910, RRID: <a href="https://scicrunch.org/resolver/AB_514500">AB_514500</a>) and revealed with TSA kit (PerkinElmer catalog #NEL744A001KT) using Cy3 tyramide at 1:200. For triple FISH, TH mRNA was detected with dinitrophenyl (DNP)-labeled probe; NeuroD6 mRNA with DIG-labeled probe and Vglut2 mRNA with fluorescein-labeled probe. The protocol was the same as described above until revelation: DIG epitopes were detected with HRP anti-DIG fab fragments at 1:3000 and revealed using Cy3 tyramide at 1:50 followed by glycine and H <sub>2</sub>O <sub>2</sub> treatment. Fluorescein epitopes were detected with HRP anti-fluorescein fab fragments at 1:5000 and revealed using Cy2 tyramide at 1:250 by glycine and H <sub>2</sub>O <sub>2</sub> treatment. DNP epitopes were detected with HRP anti-DNP fab fragments at 1:1000 and revealed using Cy5 tyramide at 1:50, followed by incubation with DAPI. Fluorophore tyramides were synthetized as previously described ( <a href="#B19">Hopman et al., 1998</a>). All slides were scanned and analyzed on NanoZoomer 2.0-HT Ndp2.view (Hamamatsu). Stereotaxic reference atlases ( <a href="#B13">Franklin and Paxinos, 2008</a>; <a href="#B14">Fu et al., 2012</a>) were used to outline anatomic borders. </p> </div> <div class="validationofnex-cre-mediatedrecombinationoffloxedvmat2exon2" title="sec"> <div class="title" tagx="title" title="title">
Validation of NEX-Cre-mediated recombination of floxed <i>Vmat2</i> exon 2 </div> <p>Upon genotyping, PCR-validated <i>Vmat2 <sup>lox/lox;NEX-Cre-tg</sup> </i>(cKO) and <i>Vmat2 <sup>lox/lox;NEX-Cre-wt</sup> </i> (Ctrl) mice were sacrificed and brains analyzed by ISH to verify NEX-Cre-driven recombination of the floxed exon 2 of the <i>Vmat2</i> gene in cKO mice. Littermate Ctrl mice were used to validate wild-type Vmat2 mRNA. A Vmat2 mRNA two-probe approach was implemented to visualize cells positive for wild-type Vmat2 mRNA and cells positive for a truncated Vmat2 mRNA generated on NEX-Cre-driven recombination of the floxed <i>Vmat2</i> exon 2. Probe 1 (green) was designed for detection of Vmat2 mRNA derived from exon 6–15 and probe 2 (blue) for detection of mRNA from exon 2. In control mice, both probe 1 and probe 2 can bind their target mRNA (wild-type Vmat2 mRNA). Combination of probe 1 and probe 2 gives rise to combined blue and green labeling in wild-type DA neurons. In cKO mice, <i>Vmat2</i> exon 2 will be deleted specifically in cells expressing the NEX-Cre transgene, leading to production of Vmat2 mRNA missing exon 2 but maintaining exons 6–15. In Vmat2-expressing cells that do not express the NEX-Cre transgene in cKO mice, wild-type Vmat2 mRNA will be produced. <i>Vmat2</i>-targeted cells can thus be identified based on lack of blue color (probe 2) and presence of green color only (probe 1). Thus, using the Vmat2 mRNA two-probe-approach, the color shift from complete overlap of blue and green color in Ctrl mice to the presence of green-only cells in cKO mice is used to verify Cre-LoxP-mediated cKO of the <i>Vmat2</i> gene. </p> </div> <div class="immunohistochemistry" title="sec"> <div class="title" tagx="title" title="title">
Immunohistochemistry</div> <p>Detection of TH and eYFP proteins took place according to standard immunohistochemical protocols using primary antibodies [mouse anti-TH (1:1000, Millipore catalog #MAB318, RRID: <a href="https://scicrunch.org/resolver/AB_2201528">AB_2201528</a>), chicken anti-GFP (1:1000, Abcam catalog #ab13970, RRID: <a href="https://scicrunch.org/resolver/AB_300798">AB_300798</a>)]. After overnight incubation, primary antibodies were removed and sections were incubated in specific fluorophore-conjugated secondary antibodies (donkey anti-mouse Cy3, Millipore catalog #AP192C, RRID: <a href="https://scicrunch.org/resolver/AB_11214096">AB_11214096</a>, donkey anti-chicken A488, Jackson ImmunoResearch catalog #703-545-155, RRID: <a href="https://scicrunch.org/resolver/AB_2340375">AB_2340375</a>, both 1:500). Upon rinses, slides were coverslipped using Fluoromount Aqueous mounting medium (Sigma-Aldrich catalog #F4680). For bright-field detection of TH, the peroxidase-based method (ABC kit; Vector Laboratories catalog #PK-4001, RRID: <a href="https://scicrunch.org/resolver/AB_2336810">AB_2336810</a>) with DAB chromogen was used. Quantifications were done manually on three mice per group. A stereotaxic atlas ( <a href="#B13">Franklin and Paxinos, 2008</a>) was used to outline anatomic borders. </p> </div> <div class="behavioralanalysis" title="sec"> <div class="title" tagx="title" title="title">
Behavioral analysis</div> <p> <i>Vmat2 <sup>lox/lox;NEX-Cre-tg</sup> </i> cKO and <i>Vmat2 <sup>lox/lox;NEX-Cre-wt</sup> </i> Ctrl mice were analyzed in the following behavioral tests. </p> <div class="baselinelocomotion" title="sec"> <div class="title" tagx="title" title="title">
Baseline locomotion</div> <p>Spontaneous locomotion and habituation in a novel environment were monitored for 30 min upon placing the mice in Makrolon polycarbonate boxes containing 1.5-cm bedding and a transparent Plexiglas lid. Locomotor behavior of the mice was recorded by the EthovisionXT software (Noldus, RRID: <a href="https://scicrunch.org/resolver/SCR_000441">SCR_000441</a>). </p> </div> <div class="sucrosepreferencetest" title="sec"> <div class="title" tagx="title" title="title">
Sucrose preference test</div> <p>Preference to sucrose was assessed in the home cage of the mice. The mice were housed individually in cages containing two drinking bottles. After 48 h of habituation to the experimental set up, they were presented to one bottle of tap water and one of sucrose solution (1%, 3%, and 10%) that were replaced and weighted every 24 h. Each concentration was tested twice and the position of the bottles was alternated to avoid side bias. </p> </div> <div class="ethanolpreferencetest" title="sec"> <div class="title" tagx="title" title="title">
Ethanol preference test</div> <p>Individually housed mice had access to one bottle of tap water and one of alcohol solution (3%, 6%, and 10%) that were replaced and weighted every 24 h. Each concentration of ethanol was tested four times. </p> </div> <div class="cocaine-inducedlocomotion" title="sec"> <div class="title" tagx="title" title="title">
Cocaine-induced locomotion</div> <p>Mice were placed in Makrolon polycarbonate boxes containing 1.5-cm bedding and a transparent Plexiglas lid and their locomotor behavior was recorded 30 min before and 60 min after injection of saline or cocaine (5, 10, and 20 mg/kg, i.p.) on four consecutive days. Locomotor behavior of the mice was recorded by the EthovisionXT software (Noldus, RRID: <a href="https://scicrunch.org/resolver/SCR_000441">SCR_000441</a>). </p> </div> <div class="amphetaminesensitization" title="sec"> <div class="title" tagx="title" title="title">
Amphetamine sensitization</div> <p>Upon habituation, mice received a saline injection (day 1) followed by 4 d of amphetamine injections (days 2–5, 3 mg/kg, i.p.) followed by a last injection on day 17. Locomotion was recorded 30 min before and 1.5 h after injection. Locomotor behavior of the mice was recorded by the EthovisionXT software (Noldus, RRID: <a href="https://scicrunch.org/resolver/SCR_000441">SCR_000441</a>). </p> </div> <div class="conditioned-placedpreference(cpp)" title="sec"> <div class="title" tagx="title" title="title">
Conditioned-placed preference (CPP)</div> <p>An apparatus (Panlab, Harvard Apparatus) consisting of two-main compartments [20 cm (W) × 18 cm (L) × 25 cm (H)] with distinct wall and floor texture patterns and one connecting, transparent compartment (20 × 7 × 25 cm) was used. The CPP procedure was conducted throughout 6 d. Firstly, during the pre-test, the mice were placed in the apparatus and left to freely explore. This session was used to assess initial preferences and to calculate the preference score (see below). During the next four consecutive conditioning days, the mice were constrained in one of the two main compartments and received drug injections (cocaine, 20 mg/kg or amphetamine, 3 mg/kg; i.p.) in the least preferred compartment or saline injections in the opposite one. The conditioning sessions were repeated twice a day [morning (A.M.), afternoon (P.M.)] and the treatment was alternated between days. Thus, the mice received in total four injections of saline and four injections of the drug, counterbalanced between sessions and genotypes. On the test day, the mice were placed again in the apparatus and were let to freely explore. The preference score was calculated by subtracting the time in seconds the animal spent in the drug-paired compartment during pre-test from the time spent in the same compartment during the test (ΔSec). All sessions lasted 30 min, and the locomotor behavior of the mice was recorded by the EthovisionXT software (Noldus, RRID: <a href="https://scicrunch.org/resolver/SCR_000441">SCR_000441</a>). </p> </div> </div> <div class="stereotaxicinjections" title="sec"> <div class="title" tagx="title" title="title">
Stereotaxic injections</div> <p>Optogenetic viruses were purchased from University of North Carolina, Vector Core Facilities. DAT-Cre, Vglut2-Cre, Calb2-Cre and NEX-Cre mice (more than eight weeks; &amp;gt;20 g) were deeply anesthetized with isofluorane and received infusions of 300 nl of <i>AAV5-EF1a-DIO-ChR2(H134)-eYFP</i> or <i>AAV5-EF1a-DIO-eYFP-WPREpA</i> in the right VTA (AP: –3.45 mm, L: –0.2 mm, V: –4.4 mm according to <a href="#B13">Franklin and Paxinos, 2008</a>) at 100 nl min <sup>−1</sup> flow rate. For behavioral analysis, an optic fiber was implanted and stabilized above the right VTA (AP: –3.4 mm, ML: –0.3 mm, DV: –4.0 mm) using anchor screws and dental cement. A subset of NEX-Cre mice was injected bilaterally with <i>AAV5-EF1a-DIO-ChR2(H134)-eYFP</i> before fiber implantation. After postmortem histological validation, mice with limited transfection in the VTA and/or misplaced optic fiber were excluded from statistical analysis. </p> </div> <div class="imaging,cellcounting,andanalysisofprojectiontargetareas" title="sec"> <div class="title" tagx="title" title="title">
Imaging, cell counting, and analysis of projection target areas</div> <div class="quantificationoffish" title="sec"> <div class="title" tagx="title" title="title">
Quantification of FISH</div> <p>Manual counting of cells expressing mRNAs of interest was performed in two to three mice per probe pair with Th mRNA as reference for outline of the VTA and Th, Dat, Viaat, or Vglut2 mRNA as reference for distinct cell soma. A signal for a particular mRNA was considered as specific for a particular cell when five contiguous fluorescent puncta were present within the outline of the cell soma. </p> </div> <div class="quantificationofimmunohistochemistry" title="sec"> <div class="title" tagx="title" title="title">
Quantification of immunohistochemistry</div> <p>Sections of Calb2-Cre and NEX-Cre mice injected with <i>AAV5-EF1a-DIO-ChR2(H134)-eYFP</i> containing the VTA (–3.28 to –3.80 mm from bregma according to <a href="#B13">Franklin and Paxinos, 2008</a>) were immunostained for eYFP and TH as described above. Z-stacks in four different positions within the VTA, (VTA1–VTA4, of which VTA1 and VTA3 represented medial VTA and VTA2 and VTA4 lateral VTA on two different bregma levels), were acquired using a Zeiss Confocal microscope (LSM 700, 20× magnification). Co-labeling of YFP and TH was identified for each fluorescent channel and counted manually using the ImageJ software (RRID: <a href="https://scicrunch.org/resolver/SCR_003070">SCR_003070</a>). A minimum of three mice of each genotype was processed and analyzed. </p> </div> <div class="analysisofprojectionareas" title="sec"> <div class="title" tagx="title" title="title">
Analysis of projection areas</div> <p>Fluorescent microscopy (Zeiss Confocal microscope) was used to detect eYFP-positive fibers in sections derived from the whole brain of NEX-Cre, Calb2-Cre, DAT-Cre, and NEX-Cre mice injected into the VTA with <i>AAV5-EF1a-DIO-ChR2(H134)-eYFP</i>. A minimum of two mice of each genotype was analyzed by two persons blind to the genotype of the mice. </p> </div> </div> <div class="fast-scancyclicvoltammetry(fscv)inslices" title="sec"> <div class="title" tagx="title" title="title">
Fast-scan cyclic voltammetry (FSCV) in slices</div> <p>For DA recordings in terminal areas upon photostimulation, DAT-Cre, Calb2-Cre, and NEX-Cre mice were injected with <i>AAV5-EF1a-DIO-ChR2(H134)-eYFP</i> or <i>AAV5-EF1a-DIO-eYFP-WPREpA</i> as described above. </p> <div class="carbonfibermicroelectrodes" title="sec"> <div class="title" tagx="title" title="title">
Carbon fiber microelectrodes</div> <p>Carbon fiber working electrodes were fabricated by aspirating 7-μm diameter carbon fibers (Cytec Engineered Materials) into borosilicate glass capillaries (1.2 mm O.D., 0.69 mm I.D., Sutter Instrument Co). Capillaries were adjusted (Sutter Instrument, P-97) and sealed with epoxy (EpoTek 301, Epoxy Technology). Electrodes were tested on bath applications of known concentrations of DA. Only electrodes showing good reaction kinetics (current vs time plots, and current vs voltage plots) were used. </p> </div> <div class="fscv" title="sec"> <div class="title" tagx="title" title="title">
FSCV</div> <p>A Dagan Chem-Clamp potentiostat (Dagan Corporation) and two data acquisition boards (PCI-6221, National Instruments) run by the TH 1.0 CV program (ESA) were used to collect all electrochemical data. Cyclic voltammograms were obtained by applying a triangular wave form potential (−0.4 to +1.3 V vs Ag/AgCl) repeated every 100 ms at a scan rate of 200 V/s (low pass Bessel filter at 3 kHz). Each cyclic voltammogram was a background-subtracted average of 10 successive cyclic voltammograms taken at the maximum oxidation peak current. All electrodes were allowed to cycle for at least 15 min before recording to stabilize the background current. The recorded current response was converted to DA concentration via <i>in vitro</i> electrode calibration with standard DA solution after each experiment. For optically evoked DA release, photostimulation during FSCV recordings was generated through a 3.4-W 447-nm LED mounted on the microscope oculars and delivered through the objective lens. Photostimulation was controlled via a DigiData 1440A, enabling control over duration and intensity. Illumination intensity typically scaled to 3 mW/mm <sup>2</sup>. Acquired data were analyzed and plotted using MATLAB (RRID: <a href="https://scicrunch.org/resolver/SCR_001622">SCR_001622</a>) routines and statistical analysis was performed using Prism 6.0 (GraphPad Software, RRID: <a href="https://scicrunch.org/resolver/SCR_002798">SCR_002798</a>) </p> </div> </div> <div class="patch-clampelectrophysiologyinslices" title="sec"> <div class="title" tagx="title" title="title">
Patch-clamp electrophysiology in slices</div> <p>For recordings of EPSCs and IPSCs upon optogenetic stimulation, Calb2-Cre and NEX-Cre mice (more than eight weeks, &amp;gt;20 g) were injected with <i>AAV5-EF1a-DIO-ChR2(H134R)-eYFP</i> as described above. Mice were deeply anaesthetized with pentobarbital (200 mg kg <sup>−1</sup>, i.p.; Virbac) and perfused intracardially with 10-ml ice-cold sucrose-artificial CSF (ACSF) containing: 75 mM sucrose, 87 mM NaCl, 2.5 mM KCl, 7 mM MgCl <sub>2</sub>, 0.5 mM CaCl <sub>2</sub>, 1.25 mM NaH <sub>2</sub>PO <sub>4</sub>, and 25 mM NaHCO <sub>3</sub> and continuously bubbled with carbogen (95% O <sub>2</sub>–5% CO <sub>2</sub>). A total of 200-µm coronal brain slices were cut in sucrose-ACSF. Slices were transferred to a perfusion chamber containing ACSF at 31°C: 126 mM NaCl, 2.5 mM KCl, 1.2 mM MgCl <sub>2</sub>, 2.4 mM CaCl <sub>2</sub>, 1.4 mM NaH <sub>2</sub>PO <sub>4</sub>, 25 mM NaHCO <sub>3</sub>, and 11 mM glucose, continuously bubbled with carbogen. After at least 45-min recovery, slices were transferred to a recording chamber continuously perfused with ACSF (2–3 ml min <sup>−1</sup>) maintained at 29°C–31°C. Patch pipettes (3.5–5.5 MΩ) were pulled from borosilicate glass and filled with internal recording solution containing: 120 mM CsCH <sub>3</sub>SO <sub>3</sub>, 20 mM HEPES, 0.4 mM EGTA, 2.8 mM NaCl, 5 mM TEA, 2.5 mM Mg-ATP, and 0.25 mM Na-GTP, at pH 7.25 and 285 ± 5 mOsm. VTA neurons and terminals were visualized by epifluorescence and visually guided patch recordings were achieved using infrared differential interference contrast (IR-DIC) illumination (Axiocam MRm, Zeiss). ChR2 was activated by flashing blue light (5-ms pulse width) through the light path of the microscope using a light-emitting diode (UHP-LED460, Prizmatix) under computer control. EPSCs and IPSCs were recorded in whole-cell voltage clamp (–60 and 0 mV holding potential, respectively, Multiclamp 700B amplifier, Molecular Devices), filtered at 2 kHz, digitized at 10 kHz (Axon Digidata 1550, Molecular Devices), and collected online using pClamp 10 software (Molecular Device). Series resistance and capacitance were electronically compensated before recordings. Estimated liquid-junction potential was 12 mV and left uncorrected. Series resistance and/or leak current were monitored during recordings and cells that showed &amp;gt;25% change during recordings were considered unstable and discarded. Single-pulse (5-ms) photostimuli were applied every 55 s, and 10 photo-evoked currents were averaged per neuron per condition. DMSO stock solution of 6,7-dinitroquinoxaline-2,3-dione (DNQX; 10 mM, Sigma) was diluted 1000-fold in ACSF and bath applied. Current sizes were calculated by using peak amplitude from baseline. Decay time constants (τ) were calculated by fitting an exponential function to each averaged current trace using the following formula: f(t) = e <sup>–t/τ</sup> + C. </p> </div> <div class="placepreferenceuponoptogeneticstimulation" title="sec"> <div class="title" tagx="title" title="title">
Place preference upon optogenetic stimulation</div> <p>The three-compartment apparatus (Panlab, Harvard Apparatus) used in the CPP experiments (above) was also implemented in the optogenetics-driven place preference experiments to address RT-PP upon photostimulation and CR, the association to compartment previously paired with photostimulation. Similar to protocols previously described by others ( <a href="#B44">Root et al., 2014</a>; <a href="#B41">Qi et al., 2016</a>), the entry of the mouse into one of the two main compartments was paired with intracranial VTA photostimulation (10-ms pulse width, 20 Hz, 10 mW) while the interconnecting compartment was not coupled to light stimulation (neutral) at all. The EthovisionXT tracking software (Noldus, RRID: <a href="https://scicrunch.org/resolver/SCR_000441">SCR_000441</a>) was used to monitor behavior and trigger laser stimulation. Behavior was assessed over the course of eight experimental days subdivided into two recording phases with a minimum 3-d rest period in between ("phase 1," days 3–5, and "reversal phase," days 6–8). On day 1 ("habituation"), the mouse was connected to the optic fiber cord and allowed to acclimatize. On day 2 ("pre-test"), the mouse was placed in the three-compartment apparatus for 15 min to freely explore, while attached to the optic fiber cord but without receiving any photostimulation; the preference for each compartment was evaluated. During 30 min-long recordings on days 3 and 4 ("RT-PP"), entry into the assigned light-paired compartment (non-preferred in pre-test) resulted in blue laser photostimulation delivered as continuous train of pulses (10-ms pulse width, 20 Hz, 10 mW). On day 5 ("CR"), the time spent in each compartment was measured for 15 min with no delivery of photostimulation. In the reversal phase, the protocol was repeated but with stimulation in the opposite compartment compared to phase 1. "High-power" experiments followed the same structure except that the mice received a stimulation of higher power (5-ms pulse width, 20 Hz, 20 mW). </p> <p>For the Neutral Compartment Preference (NCP) test, a modified version of the test described above was used with the following changes: Entry into either one of the two main compartments was coupled to light stimulation, while only entry the interconnecting compartment had no consequence. The experiment took place on three consecutive days: During the first 2 d (Stim1 and Stim2), the mice received stimulation upon entry in any of the main compartments during 30 min long sessions while the third day was stimulation-free (15 min long session) and used to study the presence of any CRs induced by the experience with the stimulation. </p> </div> <div class="experimentaldesignandstatisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Experimental design and statistical analysis</div> <p>Regular and repeated measures (RM) two-way ANOVA and unpaired <i>t</i> tests were used to compare mean scores of Ctrl and cKO mice in behavioral tests. To analyze cocaine-induced locomotion during CPP, a mixed-effects model was used. <i>Post hoc</i> comparisons were performed by Sidak’s multiple comparison test. Unpaired <i>t</i> test was used to compare mean DA release between ChR2- and eYFP (control)-injected DAT-Cre, Calb2-Cre and NEX-Cre mice for each region where the measurements were performed. Paired <i>t</i> tests were used to compare pre-DNQX and post-DNQX EPSP recordings. Two-way RM ANOVA with day and chamber were used as factors throughout the optogenetic experiments followed by Tukey’s <i>post hoc</i> test. When the days of stimulation were averaged, one-way ANOVA was used to unravel the effect of compartment (paired, unpaired, neutral) on time spent and Tukey’s multiple comparison test for <i>post hoc</i> analysis. Data are presented as mean ± SEM unless stated otherwise. Data analysis was performed with Prism8 (RRID: <a href="https://scicrunch.org/resolver/SCR_002798">SCR_002798</a>). Detailed statistical information is shown in <a href="#T1">Table 1</a>. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1.</span>  <p>Statistical analysis of results obtained in behavioral and electrophysiological experiments</p>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Figure</th> <th>Data structure</th> <th>Type of test</th> <th>Sample Size</th> <th>Statistical data</th> </tr> </thead> <tbody> <tr> <td> <a href="#F3">Figure 3 <i>A</i> </a> Weight analysis of ctrl and cKO mice </td> <td>Normally distributed</td> <td>Two-way ANOVA followed by Sidak’s multiple comparison test</td> <td>Ctrl <i>N</i> = 14(M = 8, F = 6)cKO <i>N</i> = 23(M = 15, F = 8) </td> <td>Interaction: <i>p</i> = 0.996, <i>F</i> <sub>(4,158)</sub> = 0.0447Week: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(4,158)</sub> = 79.8Genotype: <i>p</i> = 0.032, <i>F</i> <sub>(1,158)</sub> = 4.67Multiple comparisonsCtrl vs cKOw4 <i>p</i> = 0.908; 95% CI: –3.55 to 1.75w5 <i>p</i> = 0.966; 95% CI: –2.70 to 1.57w6 <i>p</i> = 0.876; 95% CI: –2.91 to 1.35w7 <i>p</i> = 0.720; 95% CI: –3.19 to 1.15w8 <i>p</i> = 0.783; 95%CI: –3.11 to 1.23 </td> </tr> <tr> <td> <a href="#F3">Figure 3 <i>B</i> </a> Baseline locomotion of ctrl and cKO mice for 30 min in 5-min bins </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Sidak’s multiple comparison test</td> <td>Ctrl <i>N</i> = 17(M = 8, F = 9)cKO <i>N</i> = 17(M = 13, F = 4) </td> <td>Interaction: <i>p</i> = 0.256, <i>F</i> <sub>(5,160)</sub> = 1.33Time: <i>p &amp;lt;</i> 0.001, <i>F</i> <sub>(5,160)</sub> = 69.5Genotype: <i>p</i> = 0.535, <i>F</i> <sub>(1,32)</sub> = 0.00912Multiple comparisonsCtrl vs cKO5 <i>p &amp;gt;</i> 0.999; 95% CI: –287 to 21110 <i>p &amp;gt;</i> 0.999; 95% CI: –217 to 28215 <i>p</i> = 0.952; 95% CI: –170 to 32920 <i>p &amp;gt;</i> 0.999; 95% CI: –236 to 26325 <i>p</i> = 0.886; 95% CI: –346 to 15330 <i>p</i> = 0.993; 95% CI: –195 to 304 </td> </tr> <tr> <td> <a href="#F3">Figure 3 <i>C</i> </a> Sucrose preference of ctrl and cKO mice for 1%, 3%, and 10% sucrose solutions </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Sidak’s multiple comparison test</td> <td>Ctrl <i>N</i> = 14(M = 8, F = 6)cKO <i>N</i> = 21(M = 13, F = 8) </td> <td>Interaction: <i>p</i> = 0.475, <i>F</i> <sub>(2,66)</sub> = 0.752Concentration: <i>p &amp;lt;</i> 0.001, <i>F</i> <sub>(2,66)</sub> = 151Genotype: <i>p</i> = 0.297, <i>F</i> <sub>(1,33)</sub> = 1.12Multiple comparisonsCtrl vs cKO1% <i>p &amp;gt;</i> 0.999; 95% CI: –5.21 to 5.693% p = 0.294; 95% CI: –1.83 to 9.0810% <i>p</i> = 0.991; 95% CI: –4.85 to 6.05 </td> </tr> <tr> <td> <a href="#F3">Figure 3 <i>D</i> </a> Ethanol preference of ctrl and cKO mice for 3%, 6%, and 10% ethanol concentrations </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Sidak’s multiple comparison test</td> <td>Ctrl <i>N</i> = 14(M = 7, F = 7)cKO <i>N</i> = 14(M = 6, F = 8) </td> <td>Interaction: <i>p</i> = 0.129, <i>F</i> <sub>(2,52)</sub> = 2.13Concentration: <i>p &amp;lt;</i> 0.001, <i>F</i> <sub>(2,52)</sub> = 14.2Genotype: <i>p</i> = 0.334, <i>F</i> <sub>(1,26)</sub> = 0.969Multiple comparisonsCtrl vs cKO3% <i>p</i> = 0.983; 95% CI: –9.31 to 7.116% p = 0.453; 95% CI: –3.68 to 12.710% <i>p</i> = 0.396; 95% CI: –3.38 to 13.0Ctrl3% vs 6% <i>p</i> &amp;lt; 0.001; 95% CI: –14.7 to –3.453% vs 10% <i>p</i> &amp;lt; 0.001; 95% CI: –16.9 to –5.586% vs 10% <i>p</i> = 0.733; 95% CI: –7.78 to 3.52cKO3% vs 6% <i>p</i> = 0.354; 95% CI: –9.11 to 2.183% vs 10% <i>p</i> = 0.072; 95% CI: –10.9 to 0.3546% vs 10% <i>p</i> = 0.814; 95% CI: –7.47 to 3.82 </td> </tr> <tr> <td> <a href="#F3">Figure 3 <i>E</i> </a> Injection-induced locomotion for ctrl and cKO mice after saline and 5, 10, and 20 mg/kg injections of cocaine </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Sidak’s multiple comparison test</td> <td>Ctrl <i>N</i> = 14(M = 8, F = 6)cKO <i>N</i> = 21(M = 13, F = 8) </td> <td>Interaction: <i>p</i> = 0.396, <i>F</i> <sub>(3,99)</sub> = 1Session: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(3,99)</sub> = 108Genotype: <i>p</i> = 0.208, <i>F</i> <sub>(1,33)</sub> = 1.65Multiple comparisonsCtrl vs cKOSaline <i>p</i> = 0.966; 95% CI: –3437 to 54365 mg/kg. <i>p</i> = 0.962; 95% CI: –3410 to 546410 mg/kg. <i>p</i> = 0.887; 95% CI: –3015 to 585820 mg/kg. <i>p</i> = 0.152; 95% CI: –802 to 8071 </td> </tr> <tr> <td> <a href="#F3">Figure 3 <i>F</i> </a> Amphetamine- induced (3 mg/kg) locomotion under a sensitization protocol for ctrl and cKO mice </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Sidak’s multiple comparison test</td> <td>Ctrl <i>N</i> = 17(M = 8, F = 9)cKO <i>N</i> = 17(M = 13, F = 4) </td> <td>Interaction: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(5,160)</sub> = 4.79Session: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(5,160)</sub> = 40.9Genotype: <i>p</i> = 0.005, <i>F</i> <sub>(1,32)</sub> = 9.09Multiple comparisonsCtrl vs cKODay1 <i>p</i> &amp;gt; 0.999; 95% CI: –13,977 to 12,091Day2 <i>p</i> = 0.266; 95% CI: –3371 to 22,696Day3 <i>p</i> = 0.063; 95% CI: –407 to 25,661Day4 <i>p</i> = 0.011; 95% CI: –2481 to 28,549Day5 <i>p</i> &amp;lt; 0.001; 95% CI: –6873 to 32,941Day17 <i>p</i> = 0.029; 95% CI: –928 to 26,996 </td> </tr> <tr> <td> <a href="#F3">Figure 3 <i>H</i> </a>Cocaine (20 mg/kg, i.p) CPP for ctrl and cKO mice </td> <td>Normally distributed</td> <td>Unpaired <i>t</i> test </td> <td>Ctrl <i>N</i> = 12 (M = 6, F = 6)cKO <i>N</i> = 15 (M = 6, F = 9) </td> <td> <i>t</i> testctrl vs cKO <i>p</i> = 0.860; 95% CI: –162.0 to 136.1 </td> </tr> <tr> <td> <a href="#F3">Figure 3 <i>H</i> </a>, bottom panelAmphetamine (3 mg/kg, i.p.) CPP for ctrl and cKO mice </td> <td>Assumed normality</td> <td>Unpaired <i>t</i> test </td> <td>Ctrl <i>N</i> = 13 (M = 6, F = 7)cKO <i>N</i> = 16 (M = 9, F = 7) </td> <td> <i>t</i> testctrl vs cKO <i>p</i> = 0.744; 95% CI: –365.5 to 264.3 </td> </tr> <tr> <td /> </tr> <tr> <td> <a href="#F3">Figure 3 <i>I</i> </a>Cocaine-induced locomotion during the CPP for ctrl and cKO mice </td> <td>Assumed normality</td> <td>Two-way RM ANOVA followed by Sidak’s multiple comparison test</td> <td>Ctrl <i>N</i> = 12 (M = 6, F = 6)cKO <i>N</i> = 15 (M = 6, F = 9) </td> <td>Interaction: <i>p</i> = 0.652, <i>F</i> <sub>(3,75)</sub> = 0.5Session: <i>p</i> = 0.006, <i>F</i> <sub>(3,75)</sub> = 4.4Genotype: <i>p</i> = 0.031, <i>F</i> <sub>(1,25)</sub> = 5.2Multiple comparisonsCtrl vs cKOInjection 1: <i>p</i> = 0.373; 95% CI: –6850 to 1526Injection 2: <i>p</i> = 0.067; 95% CI: –8185 to 191Injection 3: <i>p</i> = 0.115; 95% CI: –7818 to 558Injection 4: <i>p</i> = 0.475; 95% CI: –6591 to 1785 </td> </tr> <tr> <td> <a href="#F3">Figure 3 <i>K</i> </a>, bottom panelAmphetamine-induced locomotion during the CPP for ctrl and cKO mice </td> <td>Normally distributed</td> <td>Mixed-effects model (REML) followed by Sidak’s multiple comparison test</td> <td>Ctrl <i>N</i> = 15 (M = 7, F = 8)cKO <i>N</i> = 17 (M = 10, F = 7) </td> <td>Interaction: <i>p</i> = 0.567, <i>F</i> <sub>(3,85)</sub> = 0.680Session: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(3,85)</sub> = 24.0Genotype: <i>p</i> = 0.803, <i>F</i> <sub>(1,30)</sub> = 0.0631Multiple comparisonsCtrl vs cKOInjection 1: <i>p</i> = 0.941; 95% CI: –2522 to 1473Injection 2: <i>p</i> = 0.931; 95% CI: –1431 to 2517Injection 3: <i>p</i> = 0.995; 95% CI: –1783 to 2331Injection 4: <i>p</i> = 0.989; 95% CI: –1671 to 2327 </td> </tr> <tr> <td> <a href="#F6">Figure 6 <i>D</i> </a>, leftOptically evoked DA release in NAcSh of DAT-, NEX-, and Calb2-Cre mice injected with ChR2 or eYFP </td> <td>Normally distributed</td> <td>Unpaired <i>t</i> test </td> <td>10 observations for each group and virusDAT-Cre/Chr2 <i>N</i> = 2(M = 0, F = 2)DAT-Cre/eYFP <i>N</i> = 2(M = 1, F = 1)NEX-Cre/Chr2 <i>N</i> = 3(M = 2, F = 1)NEX-Cre/eYFP <i>N</i> = 2(M = 0, F = 2)Calb2-Cre/Chr2 <i>N</i> = 3(M = 1, F = 2)Calb2-Cre/eYFP <i>N</i> = 2(M = 0, F = 2) </td> <td> <i>t</i> testDAT-Cre/ChR2 vs DAT-Cre/eYFP <i>p</i> &amp;lt; 0.0001; 95% CI: –1.272 to –0.6540NEX-Cre/ChR2 vs NEX-Cre/eYFP <i>p</i> &amp;lt; 0.0001; 95% CI: –0.6289 to –0.2909Calb2-Cre/ChR2 vs Calb2-Cre/eYFP <i>p</i> = 0.0148; 95% CI: –0.01602 to –0.001988 </td> </tr> <tr> <td> <a href="#F6">Figure 6 <i>D</i> </a>, rightOptically evoked DA release in OT of DAT-, NEX-, and Calb2-Cre mice injected with ChR2 or eYFP </td> <td>Normally distributed</td> <td>Unpaired <i>t</i> test </td> <td>As above</td> <td> <i>t</i> testDAT-Cre/ChR2 vs DAT-Cre/eYFP <i>p</i> &amp;lt; 0.0001; 95% CI: –0.2354 to –0.1810NEX-Cre/ChR2 vs NEX-Cre/eYFP <i>p</i> = 0.0049; 95% CI: –0.01295 to –0.002704Calb2-Cre/ChR2 vs Calb2-Cre/eYFP <i>p</i> = 0.0002; 95% CI: –0.02022 to –0.007554 </td> </tr> <tr> <td> <a href="#F7">Figure 7 <i>D</i> </a>, left panelOptically evoked EPSCs in NAcSh of NEX-Cre/ChR2 mice before (control) and after DNQX bath application </td> <td>Assumed normality</td> <td>Paired <i>t</i> test </td> <td>6 cells from 3 NEX-Cre/ChR2 mice(M = 3, F = 0)</td> <td> <i>p</i> = 0.0481; 95% CI: –61.86 to –0.3739 </td> </tr> <tr> <td> <a href="#F7">Figure 7 <i>D</i> </a>, right panelOptically evoked EPSCs in OT of Calb2-Cre/ChR2 mice before (control) and after DNQX bath application </td> <td>Assumed normality</td> <td>Paired <i>t</i> test </td> <td>5 cells from 3 Calb2-Cre/ChR2 mice(M = 2, F = 1)</td> <td> <i>p</i> = 0.0456; 95% CI: –89.88 to –1.444 </td> </tr> <tr> <td> <a href="#F8">Figure 8 <i>B</i> </a>, leftBehavioral analysis of DAT-Cre/ChR2 mice throughout the opto-behavioral experiments </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 10(M = 2, F = 8) </td> <td>Interaction: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(12,108)</sub> = 33Day: <i>p</i> = 0.435, <i>F</i> <sub>(6,54)</sub> = 1Compartment: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,18)</sub> = 51.8Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> = 0.513; 95% CI: –38.4 to –6.29Day3 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 21.7 to 66.4Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 35.7 to 80.4Day4 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 37.6 to 82.3Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 47.3 to 92.0Day5 (CR)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 11.5 to 56.1Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 26.7 to 71.4Day6 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –65.7 to –21.0Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 34.4 to 79.1Day7 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –85.0 to –40.4Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 43.1 to 87.7Day8 (CR)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –67.2 to –22.5Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 33.6 to 78.3 </td> </tr> <tr> <td /> </tr> <tr> <td /> <td /> <td /> <td /> <td>Reversal parameters:Day3 paired vs Day6 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 22.8 to 67.4Day3 paired vs Day7 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 32.5 to 77.2Day4 paired vs Day6 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 32.3 to 76.9Day4 paired vs Day7 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 42.0 to 86.7Day5 paired vs Day8 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 17.5 to 62.2 </td> </tr> <tr> <td> <a href="#F8">Figure 8 <i>B</i> </a>, rightTime spent in paired, unpaired, and neutral compartments during the four RT-PP days for DAT-Cre/ChR2 mice </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 10(M = 2, F = 8) </td> <td>Compartment <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 166Multiple comparisonsPaired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 41.1 to 63.9Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 51.7 to 74.5Unpaired vs neutral <i>p =</i> 0.066; 95% CI: –0.808 to 22.0 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>A</i> </a>, leftBehavioral analysis of DAT-Cre-negative mice injected with AAV-ChR2 throughout the opto-behavioral experiments </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 3(M = 0, F = 3) </td> <td>Interaction: <i>p</i> = 0.562, <i>F</i> <sub>(12,24)</sub> = 0.898Day: <i>p</i> = 0.569, <i>F</i> <sub>(6,12)</sub> = 0.830Compartment: <i>p</i> = 0.102, <i>F</i> <sub>(2,4)</sub> = 4.26Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> = 0.010; 95% CI: –49.6 to –4.05Day3 (RT-PP)Paired vs unpaired <i>p</i> = 0.074; 95% CI: 44.5 to 1.05Paired vs neutral <i>p</i> = 0.292; 95% CI: –5.3 to 40.3Day4 (RT-PP)Paired vs unpaired <i>p</i> = 0.236; 95% CI: –41.0 to 4.57Paired vs neutral <i>p</i> = 0.055; 95% CI: –0.241 to 45.3Day5 (CR)Paired vs unpaired <i>p</i> = 0.074; 95% CI: –44.5 to 1.08Paired vs neutral <i>p</i> = 0.204; 95% CI: –4.08 to 41. 5Day6 (RT-PP)Paired vs unpaired <i>p</i> = 0.998; 95% CI: –30.3 to 15.3Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 12.0 to 57.6Day7 (RT-PP)Paired vs unpaired <i>p</i> = 0.863; 95% CI: –34.5 to 11.0Paired vs neutral <i>p</i> = 0.001; 95% CI: 9.18 to 54.8Day8 (CR)Paired vs unpaired <i>p</i> = 0.012; 95% CI: –49.2 to –3.6Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 22.5 to 68.1Reversal parameters:Day3 paired vs Day6 unpaired <i>p</i> = 0.995; 95% CI: –30.8 to 14.8Day3 paired vs Day7 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –27.0 to 18.5Day4 paired vs Day6 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –28.0 to 17.6Day4 paired vs Day7 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –24.2 to 21.4Day5 paired vs Day8 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –21.3 to 24.3 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>A</i> </a>, rightTime spent in paired, unpaired, and neutral compartments during the four RT-PP days for DAT-Cre-negative/ChR2 mice </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 3(M = 0, F = 3) </td> <td>Compartment <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 48.7Multiple comparisonsPaired vs unpaired <i>p =</i> 0.358; 95% CI: –15.8 to 5.46Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 16.1 to 37.3Unpaired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 21.2 to 42.5 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>B</i> </a>, leftBehavioral analysis of DAT-Cre/eYFP throughout the opto-behavioral experiments </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 3(M = 0, F = 3) </td> <td>Interaction: <i>p</i> = 0.677, <i>F</i> <sub>(12,24)</sub> = 0.767Day: <i>p</i> = 0.935, <i>F</i> <sub>(6,12)</sub> = 0.281Compartment: <i>p</i> = 0.004, <i>F</i> <sub>(2,4)</sub> = 27.9Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –62.8 to –22.5Day3 (RT-PP)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –64.9 to –24.6Paired vs neutral <i>p</i> = 0.198; 95% CI: –3.52 to 36.7Day4 (RT-PP)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –63.5 to –23.2Paired vs neutral <i>p</i> = 0.222; 95% CI: –3.85 to 36.4Day5 (CR)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –65.9 to –25.6Paired vs neutral <i>p</i> = 0.251; 95% CI: –4.21 to 36.1Day6 (RT-PP)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –70.7 to –30.5Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 45.8 to 86.1Day7 (RT-PP)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –55.5 to –15.2Paired vs neutral <i>p</i> &amp;lt; 0.001; 95% CI: 31.3 to 71.6Day8 (CR)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –66.1 to –25.8Paired vs neutral <i>p</i> &amp;lt; 0.001; 95% CI: –42.7 to 82.9 </td> </tr> <tr> <td /> </tr> <tr> <td /> <td /> <td /> <td /> <td>Reversal parameters:Day3 paired vs Day6 unpaired <i>p</i> &amp;gt; 0.999; 95% CI: –17.8 to 22.5Day3 paired vs Day7 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –23.1 to 17.2Day4 paired vs Day6 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –17.4 to 22.9Day4 paired vs Day7 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –22.7 to 17.5Day5 paired vs Day8 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –20.4 to 19.9 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>B</i> </a>, rightTime spent in paired, unpaired, and neutral compartments during the four RT-PP days for DAT-Cre/eYFP mice </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 3(M = 0, F = 3) </td> <td>Compartment <i>p</i> = 0.127, <i>F</i> <sub>(2,6)</sub> = 2.97Multiple comparisonsPaired vs unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –55.6 to 54.5Paired vs neutral <i>p =</i> 0.171; 95% CI: –17.5 to 92.6Unpaired vs neutral <i>p =</i> 0.165; 95% CI: –16.9 to 93.1 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>C</i> </a>, leftBehavioral analysis of DAT-Cre controls (pooled) throughout the opto-behavioral experiments </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 6(M = 0, F = 6) </td> <td>Interaction: <i>p</i> = 0.494, <i>F</i> <sub>(12,60)</sub> = 0.963Day: <i>p</i> = 0.929, <i>F</i> <sub>(6,30)</sub> = 0.306Compartment: <i>p</i> &amp;lt; 0.001 <i>F</i> <sub>(2,10)</sub> = 18.6Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –48.6 to –20.8Day3 (RT-PP)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –47.1 to –19.4Paired vs neutral <i>p</i> = 0.004; 95% CI: 3.15 to 30.9Day4 (RT-PP)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –44.7 to –16.9Paired vs neutral <i>p</i> &amp;lt; 0.001; 95% CI: 5.52 to 33.3Day5 (CR)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –47.6 to –19.8Paired vs neutral <i>p</i> = 0.003; 95% CI: 3.42 to 31.2Day6 (RT-PP)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –42.9 to –15.1Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 36.5 to 64.3Day7 (RT-PP)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –37.4 to –9.64Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 27.8 to 55.6Day8 (CR)Paired vs unpaired <i>p</i> &amp;lt; 0.001; 95% CI: –50.1 to –22.3Paired vs neutral <i>p</i> &amp;lt; 0.001; 95% CI: 40.2 to 67.9Reversal parameters:Day3 paired vs Day6 unpaired <i>p</i> &amp;gt; 0.999; 95% CI: –16.7 to 11.1Day3 paired vs Day7 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –17.5 to 10.3Day4 paired vs Day6 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –15.1 to 12.7Day4 paired vs Day7 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –15.9 to 11.9Day5 paired vs Day8 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –13.3 to 14.5 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>C</i> </a>, rightTime spent in paired, unpaired, and neutral compartments during the four RT-PP days for DAT-Cre control mice (pooled) </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 6(M = 0, F = 6) </td> <td>Compartment <i>p</i> = 0.015, <i>F</i> <sub>(2,6)</sub> = 9.27Multiple comparisonsPaired vs unpaired <i>p =</i> 0.946; 95% CI: –30.6 to 24.8Paired vs neutral <i>p =</i> 0.028; 95% CI: 4.44 to 59.8Unpaired vs neutral <i>p =</i> 0.019; 95% CI: 7.30 to 62.7 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>E</i> </a>, leftBehavioral analysis of DAT-Cre/ChR2 mice tested on high power, throughout the opto-behavioral experiments </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 10(M = 2, F = 8) </td> <td>Interaction: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(12,36)</sub> = 22.6Day: <i>p</i> = 0.455, <i>F</i> <sub>(6,18)</sub> = 1Compartment: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 105Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> &amp;gt; 0.999; 95% CI: –45.7 to 26.7Day3 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 26.2 to 98.6Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 38.5 to 111Day4 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 42.7 to 115Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 47.1 to 119Day5 (CR)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 17.4 to 89.8Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 16.5 to 88.8Day6 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –98.0 to –25.6Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 36.5 to 109Day7 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –109 to –37.0Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 42.3 to 115Day8 (CR)Paired vs unpaired <i>p =</i> 0.407; 95% CI: –62.5 to 9.72Paired vs neutral <i>p =</i> 0.030; 95% CI: 1.98 to 74.3 </td> </tr> <tr> <td /> </tr> <tr> <td /> <td /> <td /> <td /> <td>Reversal parameters:Day3 paired vs Day6 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 26.5 to 98.9Day3 paired vs Day7 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 32.1 to 104Day4 paired vs Day6 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 34.8 to 107Day4 paired vs Day7 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 40.5 to 113Day5 paired vs Day8 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 4.15 to 76.5 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>E</i> </a>, rightTime spent in paired, unpaired, and neutral compartments during the four RT-PP days for DAT-Cre/ChR2 mice under high power stimulation </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 4(M = 0, F = 4) </td> <td>Compartment <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 404Multiple comparisonsPaired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 59.9 to 78.2Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 68.1 to 86.5Unpaired vs neutral <i>p =</i> 0.074; 95% CI: –0.934 to 17.4 </td> </tr> <tr> <td> <a href="#F8">Figure 8 <i>C</i> </a>, leftBehavioral analysis of Vglut2-Cre/ChR2 mice throughout the opto-behavioral experiments </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 7(M = 2, F = 5) </td> <td>Interaction: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(12,72)</sub> = 16.1Day: <i>p</i> = 0.181, <i>F</i> <sub>(6,36)</sub> = 1.58Compartment: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,12)</sub> = 40.9Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> &amp;gt; 0.999; 95% CI: –28.1 to 26.6Day3 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –71.9 to –17.2Paired vs neutral <i>p</i> &amp;gt; 0.999; 95% CI: –18.0 to 36.7Day4 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –68.1 to –13.4Paired vs neutral <i>p</i> &amp;gt; 0.997; 95% CI: –16.9 to 37.8Day5 (CR)Paired vs unpaired <i>p</i> = 0.998; 95% CI: –37.8 to 16.9Paired vs neutral <i>p</i> = 0.019; 95% CI: 2.42 to 57.1Day6 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 32.9 to 87.6Paired vs neutral <i>p</i> &amp;gt; 0.999; 95% CI: –28.8 to 25.9Day7 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 27.0 to 81.7Paired vs neutral <i>p</i> &amp;gt; 0.999; 95% CI: –28.9 to 25.8Day8 (CR)Paired vs unpaired <i>p</i> = 0.783; 95% CI: –10.8 to 43.9Paired vs neutral <i>p</i> = 0.055; 95% CI: –0.268 to 54.4Reversal parameters:Day3 paired vs Day6 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –78.7 to –24.1Day3 paired vs Day7 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –74.8 to –20.1Day4 paired vs Day6 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –77.1 to –22.4Day4 paired vs Day7 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –73.2 to –18.5Day5 paired vs Day8 unpaired <i>p</i> = 0.952; 95% CI: –41.0 to 13.7 </td> </tr> <tr> <td> <a href="#F8">Figure 8 <i>C</i> </a> (right) time spent in paired, unpaired and neutral compartments during the 4 RT-PP days for Vglut2-Cre/ChR2 mice </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 7(M = 2, F = 5) </td> <td>Compartment <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 162Multiple comparisonsPaired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –60.2 to –39.7Paired vs neutral <i>p =</i> 0.469; 95% CI: –6.08 to 14.5Unpaired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 43.9 to 64.4 </td> </tr> <tr> <td> <a href="#F8">Figure 8 <i>D</i> </a>, leftBehavioral analysis of Calb2-Cre/ChR2 mice throughout the opto-behavioral experiments </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 7(M = 0, F = 7) </td> <td>Interaction: <i>p</i> = 0.163, <i>F</i> <sub>(12,72)</sub> = 1.45Day: <i>p</i> = 0.567, <i>F</i> <sub>(6,36)</sub> = 0.813Compartment: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,12)</sub> = 27Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> = 0.096; 95% CI: –33.3 to 1.13Day3 (RT-PP)Paired vs unpaired <i>p =</i> 0.343; 95% CI: –30.6 to 3.82Paired vs neutral <i>p</i> = 0.010; 95% CI: 2.52 to 37.0Day4 (RT-PP)Paired vs unpaired <i>p</i> &amp;gt; 0.999; 95% CI: –21.1 to 13.4Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 8.22 to 42.7Day5 (CR)Paired vs unpaired <i>p</i> &amp;gt; 0.999; 95% CI: 13.4 to 47.9Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: –15.7 to 18.8Day6 (RT-PP)Paired vs unpaired <i>p</i> &amp;gt; 0.999; 95% CI: –18.2 to 16.2Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 13.0 to 47.5Day7 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –17.6 to 16.9Paired vs neutral <i>p</i> &amp;gt; 0.999; 95% CI: 10.8 to 45.2Day8 (CR)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –13.1 to 21.3Paired vs neutral <i>p</i> &amp;gt; 0.999; 95% CI: 7.45 to 41.9 </td> </tr> <tr> <td /> </tr> <tr> <td /> <td /> <td /> <td /> <td>Reversal parameters:Day3 paired vs Day6 unpaired <i>p =</i> 0.991; 95% CI: –24.6 to 9.91Day3 paired vs Day7 unpaired <i>p =</i> 0.995; 95% CI: –24.2 to 10.2Day4 paired vs Day6 unpaired <i>p</i> &amp;gt; 0.999; 95% CI: –19.5 to 15.0Day4 paired vs Day7 unpaired <i>p</i> &amp;gt; 0.999; 95% CI: –19.1 to 15.3Day5 paired vs Day8 unpaired <i>p</i> &amp;gt; 0.999; 95% CI: –17.2 to 17.3 </td> </tr> <tr> <td> <a href="#F8">Figure 8 <i>D</i> </a>, rightTime spent in paired, unpaired, and neutral compartments during the four RT-PP days for Calb2-Cre/ChR2 mice </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 7(M = 0, F = 7) </td> <td>Compartment <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 90.1Multiple comparisonsPaired vs unpaired <i>p =</i> 0.297; 95% CI: –11.4 to 3.42Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 18.5 to 33.3Unpaired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 22.4 to 37.3 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>F</i> </a>Behavioral analysis of Calb2-Cre/ChR2 mice tested on high power, throughout the opto-behavioral experiments </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 7(M = 0, F = 7) </td> <td>Interaction: <i>p</i> = 0.927, <i>F</i> <sub>(12,72)</sub> = 0.469Day: <i>p</i> = 0.661, <i>F</i> <sub>(6,36)</sub> = 0.688Compartment: <i>p</i> = 0.001, <i>F</i> <sub>(2,12)</sub> = 12.5Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> = 0.104; 95% CI: –33.5 to 1.28Day3 (RT-PP)Paired vs unpaired <i>p =</i> 0.995; 95% CI: –24.4 to 10.4Paired vs neutral <i>p =</i> 0.019; 95% CI: 1.56 to 36.3Day4 (RT-PP)Paired vs unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –22.6 to 12.2Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 47.1 to 119Day5 (CR)Paired vs unpaired <i>p =</i> 0.742; 95% CI: –28.2 to 6.54Paired vs neutral <i>p =</i> 0.015; 95% CI: 1.87 to 36.6Day6 (RT-PP)Paired vs unpaired <i>p =</i> 0.937; 95% CI: –26.3 to 8.46Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 11.1 to 45.9Day7 (RT-PP)Paired vs unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –22.3 to 12.5Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 7.32 to 42.1Day8 (CR)Paired vs unpaired <i>p =</i> 0.976; 95% CI: –25.5 to 9.30Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 7.52 to 42.3Reversal parameters:Day3 paired vs Day6 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –17.0 to 17.8Day3 paired vs Day7 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –18.4 to 16.4Day4 paired vs Day6 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –15.1 to 19.6Day4 paired vs Day7 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –16.5 to 18.2Day5 paired vs Day8 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –17.5 to 17.3 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>F</i> </a>, rightTime spent in paired, unpaired, and neutral compartments during the four RT-PP days for Calb2-Cre/ChR2 mice under high power stimulation </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 7(M = 0, F = 7) </td> <td>Compartment <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 47.3Multiple comparisonsPaired vs unpaired <i>p =</i> 0.988; 95% CI: –8.15 to 8.97Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 15.1 to 32.3Unpaired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 14.7 to 31.8 </td> </tr> <tr> <td> <a href="#F8">Figure 8 <i>E</i> </a>, leftBehavioral analysis of NEX-Cre/ChR2 mice throughout the opto-behavioral experiments </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 5(M = 1, F = 4) </td> <td>Interaction: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(12,48)</sub> = 4.63Day: <i>p</i> = 0.307, <i>F</i> <sub>(6,24)</sub> = 1.27Compartment: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,8)</sub> = 76.8Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> &amp;gt; 0.999; 95% CI: –18.7 to 24.9Day3 (RT-PP)Paired vs unpaired <i>p =</i> 0.414; 95% CI: –5.70 to 37.9Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 17.1 to 60.7Day4 (RT-PP)Paired vs unpaired <i>p</i> &amp;gt; 0.999; 95% CI: –17.5 to 26.1Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 16.6 to 60.3Day5 (CR)Paired vs unpaired <i>p</i> &amp;gt; 0.999; 95% CI: 12.0 to 55.6Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: –5.03 to 38.6Day6 (RT-PP)Paired vs unpaired <i>p</i> = 0.020; 95% CI: –45.5 to –1.92Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 15.1 to 58.7Day7 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –51.8 to –8.16Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 20.5 to 64.1Day8 (CR)Paired vs unpaired <i>p =</i> 0.937; 95% CI: –32.7 to 10.9Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 16.3 to 59.9 </td> </tr> <tr> <td /> </tr> <tr> <td /> <td /> <td /> <td /> <td>Reversal parameters:Day3 paired vs Day6 unpaired <i>p =</i> 0.049; 95% CI: 0.0239 to 43.6Day3 paired vs Day7 unpaired <i>p =</i> 0.016; 95% CI: 2.38 to 46.0Day4 paired vs Day6 unpaired <i>p =</i> 0.252; 95% CI: –4.06 to 39.5Day4 paired vs Day7 unpaired <i>p =</i> 0.105; 95% CI: –1.71 to 41.9Day5 paired vs Day8 unpaired <i>p =</i> 0.998; 95% CI: –14.0 to 29.6 </td> </tr> <tr> <td> <a href="#F8">Figure 8 <i>E</i> </a>, rightTime spent in paired, unpaired, and neutral compartments during the four RT-PP days for NEX-Cre/ChR2 mice </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 5(M = 1, F = 4) </td> <td>Compartment <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 39.7Multiple comparisonsPaired vs unpaired <i>p =</i> 0.013; 95% CI: 5.03 to 32.0Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 25.7 to 52.6Unpaired vs neutral <i>p =</i> 0.008; 95% CI: 7.16 to 34.1 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>G</i> </a>, leftBehavioral analysis of NEX-Cre/ChR2 mice tested on high power, throughout the opto-behavioral experiments </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 4(M = 1, F = 3) </td> <td>Interaction: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(12,36)</sub> = 8.58Day: <i>p</i> = 0.252, <i>F</i> <sub>(6,18)</sub> = 1.44Compartment: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 48.3Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> = 0.369; 95% CI: –25.0 to 3.62Day3 (RT-PP)Paired vs unpaired <i>p =</i> 0.358; 95% CI: –3.54 to 25.1Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 16.0 to 44.6Day4 (RT-PP)Paired vs unpaired <i>p =</i> 0.003; 95% CI: 3.97 to 32.6Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 24.9 to 53.5Day5 (CR)Paired vs unpaired <i>p =</i> 0.084; 95% CI: –0.819 to 27.8Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 19.7 to 48.3Day6 (RT-PP)Paired vs unpaired <i>p =</i> 0.087; 95% CI: –27.8 to 0.877Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 17.7 to 46.4Day7 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –34.7 to –6.03Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 21.8 to 50.5Day8 (CR)Paired vs unpaired <i>p =</i> 0.798; 95% CI: –22.5 to 6.11Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 13.7 to 42.3Reversal parameters:Day3 paired vs Day6 unpaired <i>p =</i> 0.203; 95% CI: –2.36 to 26.3Day3 paired vs Day7 unpaired <i>p =</i> 0.028; 95% CI: 0.881 to 29.5Day4 paired vs Day6 unpaired <i>p =</i> 0.005; 95% CI: 3.12 to 31.8Day4 paired vs Day7 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 6.36 to 35.0Day5 paired vs Day8 unpaired <i>p =</i> 0.202; 95% CI: –2.35 to 26.3 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>G</i> </a>, rightTime spent in paired, unpaired, and neutral compartments during the four RT-PP days for NEX-Cre/ChR2 mice under high-power stimulation </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 4(M = 1, F = 3) </td> <td>Compartment <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 178Multiple comparisonsPaired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 10.1 to 21.3Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 28.8 to 40.0Unpaired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 13.1 to 24.3 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>H</i> </a>, leftBehavioral analysis of bilaterally injected NEX-Cre/ChR2 mice throughout the opto-behavioral experiments </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 4(M = 0, F = 4) </td> <td>Interaction: <i>p</i> = 0.040, <i>F</i> <sub>(12,36)</sub> = 2.13Day: <i>p</i> = 0.384, <i>F</i> <sub>(6,18)</sub> = 1.13Compartment: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 43.3Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> = 0.999; 95% CI: –50.7 to 25.1Day3 (RT-PP)Paired vs unpaired <i>p =</i> 0.998; 95% CI: –24.6 to 51.2Paired vs neutral <i>p =</i> 0.017; 95% CI: 4.27 to 80.1Day4 (RT-PP)Paired vs unpaired <i>p =</i> 0.768; 95% CI: –15.7 to 60.2Paired vs neutral <i>p =</i> 0.003; 95% CI: 10.3 to 86.1Day5 (CR)Paired vs unpaired <i>p =</i> 0.974; 95% CI: –21.2 to 54.7Paired vs neutral <i>p =</i> 0.015; 95% CI: 4.63 to 80.5Day6 (RT-PP)Paired vs unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –49.6 to 26.3Paired vs neutral <i>p =</i> 0.029; 95% CI: 2.13 to 78.0Day7 (RT-PP)Paired vs unpaired <i>p =</i> 0.999; 95% CI: –52.5 to 23.3Paired vs neutral <i>p =</i> 0.019; 95% CI: 3.69 to 79.5Day8 (CR)Paired vs unpaired <i>p =</i> 0.185; 95% CI: –70.1 to 5.78Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 14.2 to 90.0 </td> </tr> <tr> <td /> </tr> <tr> <td /> <td /> <td /> <td /> <td>Reversal parameters:Day3 paired vs Day6 unpaired <i>p =</i> 0.999; 95% CI: –25.0 to 50.8Day3 paired vs Day7 unpaired <i>p =</i> 0.995; 95% CI: –23.6 to 52.3Day4 paired vs Day6 unpaired <i>p =</i> 0.952; 95% CI: –20.0 to 55.8Day4 paired vs Day7 unpaired <i>p =</i> 0.999; 95% CI: –18.6 to 57.3Day5 paired vs Day8 unpaired <i>p =</i> 0.668; 95% CI: –14.1 to 61.8 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>H</i> </a>, rightTime spent in paired, unpaired, and neutral compartments during the four RT-PP days for bilaterally injected NEX-Cre/ChR2 mice </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 4(M = 0, F = 4) </td> <td>Compartment <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 331Multiple comparisonsPaired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 10.1 to 20.8Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 37.7 to 48.4Unpaired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 22.2 to 32.9 </td> </tr> <tr> <td> <a href="#F8">Figure 8 <i>F</i> </a>, left Behavioral analysis of bilaterally injected NEX-Cre/ChR2 mice throughout the opto-behavioral experiments, tested on high power </td> <td>Normally distributed</td> <td>Two-way RM ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 4(M = 0, F = 4) </td> <td>Interaction: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(12,36)</sub> = 9.03Day: <i>p</i> = 0.310, <i>F</i> <sub>(6,18)</sub> = 1.29Compartment: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 36.5Multiple comparisons (of interest)Day2 (pre-test)Paired vs unpaired <i>p</i> = 0.982; 95% CI: –42.9 to 17.4Day3 (RT-PP)Paired vs unpaired <i>p =</i> 0.349; 95% CI: –7.34 to 53.0Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 20.0 to 80.3Day4 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 14.1 to 74.4Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 30.8 to 91.1Day5 (CR)Paired vs unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –29.7 to 30.6Paired vs neutral <i>p =</i> 0.002; 95% CI: 9.24 to 69.5Day6 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –76.1 to –15.8Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 28.5 to 88.8Day7 (RT-PP)Paired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: –74.4 to –14.1Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 27.0 to 87.3Day8 (CR)Paired vs unpaired <i>p =</i> 0.989; 95% CI: –42.4 to 17.9Paired vs neutral <i>p =</i> 0.006; 95% CI: 6.19 to 66.5Reversal parameters:Day3 paired vs Day6 unpaired <i>p =</i> 0.009; 95% CI: 5.24 to 65.5Day3 paired vs Day7 unpaired <i>p =</i> 0.011; 95% CI: 4.65 to 64.9Day4 paired vs Day6 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 16.0 to 76.3Day4 paired vs Day7 unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 15.4 to 75.7Day5 paired vs Day8 unpaired <i>p &amp;gt;</i> 0.999; 95% CI: –20.8 to 39.5 </td> </tr> <tr> <td> <a href="#F8">Figure 8 <i>F</i> </a>, rightTime spent in paired, unpaired, and neutral compartments during the four RT-PP days for NEX-Cre/ChR2 mice bilaterally injected and under high-power stimulation </td> <td>Assumed normality</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 4(M = 0, F = 4) </td> <td>Compartment <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,6)</sub> = 106Multiple comparisonsPaired vs unpaired <i>p &amp;lt;</i> 0.001; 95% CI: 27.1 to 51.6Paired vs neutral <i>p &amp;lt;</i> 0.001; 95% CI: 44.5 to 69.0Unpaired vs neutral <i>p =</i> 0.011; 95% CI: 5.19 to 29.7 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>Iii</i> </a>Behavioral analysis of Vglut2-Cre throughout the NCP experiments </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 5(M = 0, F = 5) </td> <td>Interaction: <i>p</i> = 0.002, <i>F</i> <sub>(4,16)</sub> = 6.90Day: <i>p</i> = 0.410, <i>F</i> <sub>(2,8)</sub> = 1Compartment: <i>p</i> &amp;lt; 0.001, <i>F</i> <sub>(2,8)</sub> = 70.9Multiple comparisons (of interest)Stimulation 1Neutral vs Paired1 <i>p</i> = 0.009; 95% CI: –93.5 to –10.6Neutral vs Paired2 <i>p</i> = 0.004; 95% CI: –98.6 to –15.7Paired 1 vs Paired2 <i>p</i> &amp;gt; 0.999; 95% CI: –36.4 to 46.5Stimulation 2Neutral vs Paired1 <i>p</i> &amp;lt; 0.001; 95% CI: –113 to –29.9Neutral vs Paired2 <i>p</i> &amp;lt; 0.001; 95% CI: –112 to –29Paired 1 vs Paired2 <i>p</i> &amp;gt; 0.999; 95% CI: –42.3 to 40.6CRNeutral vs Paired1 <i>p</i> = 0.998; 95% CI: –49.8 to 33.1Neutral vs Paired2 <i>p</i> &amp;gt; 0.999; 95% CI: –35.5 to 47.4Paired 1 vs Paired2 <i>p</i> = 0.938; 95% CI: –55.7 to 27.2 </td> </tr> <tr> <td>Extended Data <a href="#fig8-1">Figure 8-1 <i>Iiii</i> </a>Time spent in paired1, paired2, and neutral compartments during the two NCP days for Vglut2-Cre/ChR2 mice </td> <td>Normally distributed</td> <td>RM one-way ANOVA followed by Tukey’s multiple comparison test</td> <td> <i>N</i> = 5(M = 0, F = 5) </td> <td>Compartment <i>p</i> = 0.018, <i>F</i> <sub>(2,2)</sub> = 54.2Multiple comparisonsPaired1 vs Paired2 <i>p =</i> 0.951; 95% CI: –38.9 to 43.2Paired1 vs Neutral <i>p =</i> 0.023; 95% CI: –103 to –20.7Paired2 vs Neutral <i>p =</i> 0.021; 95% CI: –105 to –22.8 </td> </tr> </tbody> </table> </div> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="neurod6mrnaisfoundinamodestpopulationofthemedialvtawhereitco-localizesextensivelywithdopaminergicmarkersandwithaglutamatergicmarkertominordegree" title="sec"> <div class="title" tagx="title" title="title">
NeuroD6 mRNA is found in a modest population of the medial VTA where it co-localizes extensively with dopaminergic markers and with a glutamatergic marker to minor degree </div> <p>To address the distribution pattern and neurotransmitter identity of <i>NeuroD6</i>-expressing neurons, double-labeling FISH was first performed in which NeuroD6 mRNA ( <a href="#F1">Fig. 1 <i>A</i>, <i>C</i> </a>) was compared to tyrosine hydroxylase (Th) mRNA encoding the rate-limiting enzyme (TH) of DA synthesis ( <a href="#F1">Fig. 1 <i>B</i>, <i>C</i> </a>). Using the distribution pattern of Th mRNA as reference, DA neurons of the SNc and VTA were identified, including the paranigral (PN), parainterfascicular (PIF), parabrachial pigmented nucleus (PBP), interfascicular nucleus (IF), and rostral linear nucleus (RLi) subareas of the VTA ( <a href="#F1">Fig. 1 <i>A–C</i> </a>). NeuroD6 mRNA was excluded from the SNc, but was detected in scattered VTA neurons. Most NeuroD6 neurons were found within the PN, PIF, and PBP subareas of the VTA, followed by fewer NeuroD6 neurons in the IF and RLi ( <a href="#F1">Fig. 1 <i>A</i>, <i>C</i> </a>). Co-detection analysis showed that all neurons detected as positive for NeuroD6 mRNA within the PN, PIF, PBP, IF, and RLi were positive for Th mRNA ( <a href="#F1">Fig. 1 <i>C</i> </a>). Quantification verified that 100% of NeuroD6 mRNA-positive cells in the PN/PIF, PBP, IF, and RLi were positive for Th mRNA, while 12% of all Th-expressing neurons within these VTA subareas contained NeuroD6 mRNA ( <a href="#F1">Fig. 1 <i>D</i> </a>). To further address the dopaminergic identity of NeuroD6 neurons, co-detection of NeuroD6 mRNA with Dat mRNA, encoding the DA transporter (DAT), was performed. Similar to the overlap between NeuroD6 and Th, all neurons detected as positive for NeuroD6 mRNA in the VTA were positive for Dat mRNA ( <a href="#F1">Fig. 1 <i>E</i> </a>). To further address the neurotransmitter identity of the NeuroD6-mRNA-positive VTA neurons, co-detection analyses of NeuroD6 mRNA with vesicular glutamate transporter 2 (Vglut2) and vesicular inhibitory amino acid transporter (Viaat) mRNAs were performed for identification of glutamatergic and GABAergic properties, respectively. NeuroD6 mRNA showed some co-localization with Vglut2 mRNA ( <a href="#F1">Fig. 1 <i>F</i> </a>), while no or very few NeuroD6-positive cells in the VTA were detected as positive for Viaat mRNA ( <a href="#F1">Fig. 1 <i>G</i> </a>). To address the overlap of NeuroD6 mRNA with Vglut2 and Th mRNA in detail, triple-labeling ISH of NeuroD6, Th and Vglut2 mRNAs was performed ( <a href="#F1">Fig. 1 <i>H–P</i> </a>). This experiment confirmed that all NeuroD6 VTA neurons within the PN, PIF, PBP, IF, and RLi were detected as positive for Th ( <a href="#F1">Fig. 1 <i>H</i>, <i>K</i>, <i>N</i> </a>) and that some NeuroD6 neurons co-localized with Vglut2 ( <a href="#F1">Fig. 1 <i>I</i>, <i>L</i>, <i>N</i> </a>). Further, the experiment identified that these NeuroD6/Vglut2 double positive cells in the VTA were positive for Th mRNA ( <a href="#F1">Fig. 1 <i>J</i>, <i>M</i>, <i>N</i> </a>). Quantification verified that 100% of NeuroD6 VTA neurons were positive for Th (NeuroD6+/Th+), and showed that 12% of these NeuroD6+/Th+ VTA neurons were also positive for Vglut2 mRNA. 12% thus displayed a NeuroD6+/Th+/Vglut2+ triple-positive molecular phenotype, while the remaining 88% of NeuroD6/Th neurons were negative for Vglut2 (NeuroD6+/Th+/Vglut2-; <a href="#F1">Fig. 1 <i>O</i> </a>). NeuroD6+/Th+/Vglut2+ and NeuroD6+/Th+/Vglut2- VTA neurons were distributed throughout the VTA with highest density in PN, PIF, and PBP subareas ( <a href="#F1">Fig. 1 <i>M</i>, <i>P</i> </a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1.</span> </a> </div>   <p> NeuroD6 mRNA is found in a modest population of the VTA, co-localizes with dopaminergic markers and partially with a glutamatergic marker. <b> <i>A–G</i> </b>, Double FISH in the ventral midbrain of adult wild-type mice detecting the following mRNAs. <b> <i>A</i> </b>, <b> <i>A’</i> </b>, NeuroD6 (red). <b> <i>B</i> </b>, <b> <i>B’</i> </b>, Th (green). <b> <i>C</i> </b>, <b> <i>C’</i> </b>, NeuroD6 (red) and Th (green). Th/NeuroD6 mRNA overlap shown in yellow. Low magnification to the left; close-ups to the right. Schematic outline shows borders for SNc and subregions of VTA: PN, PIF, PBP, IF, RLi. <b> <i>D</i> </b>, Quantification of percentage of NeuroD6-positive cells among all Th VTA cells; all NeuroD6 cells are positive for Th mRNA. <b> <i>E</i> </b>, NeuroD6 (red) and Dat (green), inset with high magnification of Dat/NeuroD6 mRNA overlap (yellow). <b> <i>F</i> </b>, NeuroD6 (red) and Vglut2 (green). <b> <i>G</i> </b>, NeuroD6 (red) and Viaat (green), inset with high magnification of Viaat-negative/NeuroD6-positive (red). <b> <i>H–P</i> </b>, Triple-labeling FISH in the ventral midbrain of adult wild-type mice detecting: ( <b> <i>H</i> </b>) Th (blue); ( <b> <i>I</i> </b>) NeuroD6 (red); ( <b> <i>J</i> </b>) Vglut2 (green) mRNAs and their co-localization: ( <b> <i>K</i> </b>) NeuroD6/Th; ( <b> <i>L</i> </b>) NeuroD6/Vglut2; ( <b> <i>M</i> </b>) Th/NeuroD6/Vglut2. Cellular closeups: ( <b> <i>N</i> </b>) NeuroD6/Th (top), NeuroD6/Vglut2 (middle), Th/NeuroD6/Vglut2 (bottom). Arrows point to NeuroD6 mRNA-positive cells. <b> <i>O</i> </b>, Quantification of percentage of NeuroD6+/Th+/Vglut2+ and NeuroD6+/Th+/Vglut2− neurons of the VTA. <b> <i>P</i> </b>, Schematic illustration of distribution pattern of NeuroD6+/Th+/Vglut2+ and NeuroD6+/Th+/Vglut2− neurons within the VTA (same as shown with experimental data in <b> <i>M</i> </b>). NeuroD6+/Th+/Vglut2- cells in magenta; NeuroD6+/Th+/Vglut2+ cells in cyan. VTA, ventral tegmental area; SNc, substantia nigra pars compacta; PBP, parabrachial pigmented nucleus; PN, paranigral nucleus; PIF, parainterfascicular nucleus; RLi, rostral linear nucleus; IF, interfascicular nucleus. FISH, fluorescent <i>in situ</i>; Dat, Dopamine transporter; Th, Tyrosine hydroxylase; Vglut2, Vesicular glutamate transporter 2; Viaat, Vesicular inhibitory amino acid transporter. </p>   </div> </div> <div class="conditionalablationofthevmat2geneinneurod6-crevtaneurons,amodelforspatiallyrestricteddadeficiency" title="sec"> <div class="title" tagx="title" title="title">
Conditional ablation of the <i>Vmat2</i> gene in NeuroD6-Cre VTA neurons, a model for spatially restricted DA deficiency </div> <p>To analyze the consequences of lost ability for vesicular packaging of DA in NeuroD6 VTA DA neurons, the <i>Slc18a2/Vmat2</i> gene encoding VMAT2 was targeted using a NeuroD6-Cre (NEX-Cre) transgenic mouse line. By breeding NEX-Cre mice with <i>Vmat2 <sup>lox/lox</sup> </i> mice, <i>Vmat2 <sup>lox/lox;NEX-Cre-tg</sup> </i> (cKO), and littermate control (Ctrl) mice were produced ( <a href="#F2">Fig. 2 <i>A</i> </a>). Upon PCR-based analysis of genotype, brains from Ctrl and cKO mice were analyzed by ISH to verify loss of full-length Vmat2 mRNA in cKO mice. Due to the scarcity of NeuroD6-positive neurons in the VTA, a Vmat2 mRNA two-probe approach was used to allow detection of gene-targeted neurons. Vmat2 probe 1 was designed to detect all cells positive for Vmat2 mRNA, while Vmat2 probe 2 was designed to bind mRNA derived from exon 2, the exon targeted for recombination by Cre recombinase ( <a href="#F2">Fig. 2 <i>B</i> </a>). In the ventral midbrain of control mice, probe 1 (green) and probe 2 (blue) were detected throughout the VTA and SNc areas with complete overlap ( <a href="#F2">Fig. 2 <i>C</i> </a>, left panel). In the corresponding area of cKO mice, the majority of cells were positive for both probe 1 and probe 2 with complete overlap ( <a href="#F2">Fig. 2 <i>C</i> </a>, right panel). However, throughout the PN, PIF, PBP, and IF VTA subareas, sparse cells showing green color only (probe 1) were detected, thus visualizing <i>Vmat2</i>-gene targeted cells among the mass of VTA DA neurons positive for both Vmat2 probes 1 and 2 ( <a href="#F2">Fig. 2 <i>C</i> </a>, right panel). Having confirmed NEX-Cre-mediated recombination of the floxed <i>Vmat2</i> gene within scattered neurons of the VTA, other brain areas in which monoaminergic neurons reside were addressed by oligo ISH. Apart from the modest VTA population positive for NeuroD6 mRNA, NeuroD6 mRNA was not detected within any other monoaminergic cell group, identified by Th and Vmat2 mRNA (Extended Data <a href="#fig2-1">Fig. 2-1</a>). However, as previously reported ( <a href="#B15">Goebbels et al., 2006</a>), NeuroD6 was abundant in several non-dopaminergic brain structures, primarily the cerebral cortex and hippocampus (Extended Data <a href="#fig2-1">Fig. 2-1</a>). In accordance with the lack of NeuroD6 in all monoaminergic cell groups apart from the VTA, Vmat2 probe 1 and probe 2 showed complete overlap in these areas, including locus coeruleus, ventromedial hypothalamus, and nucleus raphe obscurus, while none displayed labeling from probe 1 only ( <a href="#F2">Fig. 2 <i>D</i> </a>). These experiments showed that in cKO mice, Vmat2 mRNA was selectively ablated within the VTA. To address whether the targeted deletion of <i>Vmat2</i> in NeuroD6 neurons of the VTA affected the morphology of the midbrain DA system, distribution patterns of Th mRNA and TH protein were addressed, none of which revealed any gross anatomic difference in the dopaminergic system between Ctrl and cKO mice ( <a href="#F2">Fig. 2 <i>E</i> </a>; Extended Data <a href="#fig2-1">Fig. 2-1</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2.</span> </a> </div>   <p>Conditional ablation of the Vmat2 gene in NEX-Cre neurons, a model for spatially restricted DA deficiency. <b> <i>A</i> </b>, Breeding strategy for generation of mice gene-targeted for Vmat2 in VTA NEX-Cre neurons. NEX-Cre transgenic mice were mated to <i>Vmat2 <sup>lox/lox</sup> </i> mice to generate NEX-Cre-positive mice homozygous for <i>Vmat2 <sup>lox/lox</sup> </i> ( <i>Vmat2 <sup>lox/lox;NEX-Cre-tg</sup> </i>: cKO mice) and littermate control mice homozygous for <i>Vmat2 <sup>lox/lox</sup> </i> and negative for the NEX-Cre transgene ( <i>Vmat2 <sup>lox/lox;NEX-Cre-wt</sup> </i>: Ctrl mice). <b> <i>B</i> </b>, Two-probe approach for detection of Vmat2 mRNA by ISH. Probe 1 detects exons 6–15 and probe 2 detects exon 2 of the Vmat2 gene. Exon 2 is floxed in <i>Vmat2 <sup>lox/lox</sup> </i> mice leading to failure of probe 2-binding to Vmat2 mRNA in cKO neurons. <b> <i>C</i> </b>, Implementation of Vmat2 mRNA two-probe approach in <i>Vmat2 <sup>lox/lox;NEX-Cre-wt</sup> </i> (Ctrl, left panel) and <i>Vmat2 <sup>lox/lox;NEX-Cre-tg</sup> </i> (cKO, right panel) brains. Wild-type neurons are positive for both Vmat2 probes, while cKO neurons are only positive for probe 1 due to targeted deletion of exon 2 (detected by probe 2). Probe 1 detected in green and probe 2 detected in blue results in green-blue double-labeling in wild-type cells and green-only labeling in cKO cells. Green arrows point to green-only cells, i.e., VMAT2 cKO cells. <b> <i>D</i> </b>, Vmat2 mRNA two-probe ISH in additional monoaminergic areas. <b> <i>E</i> </b>, TH immunohistochemistry in Ctrl and cKO midbrain and striatum. LC, locus coeruleus; ROB, raphe nucleus obscurus; VMH, ventromedial hypothalamus; VTA, ventral tegmental area; SNc, substantia nigra pars compacta <i>;</i> DStr, dorsal striatum; NAc, nucleus accumbens; OT, olfactory tubercle. TH, Tyrosine hydroxylase; Vmat2/VMAT2, Vesicular monoamine transporter 2; Ctrl, control; cKO, conditional knockout. </p>   </div> <div class="local-data" title="local-data"> <a href="https://dx.doi.org/10.1523/ENEURO.0066-19.2019.f2-1">10.1523/ENEURO.0066-19.2019.f2-1</a> <span class="label" tagx="label" title="label">Extended Data Figure 2-1</span>  <p>ISH for detection of Th, NeuroD6, and Vmat2 mRNA. Analysis of Th, NeuroD6, and Vmat2 (two probes, covering mRNA derived from exon 1 and exon 2, respectively) mRNAs using radioactively labeled oligo-probes on sections throughout the whole brains of <i>Vmat2 <sup>lox/lox/NEX-Cre-wt</sup> </i> (Ctrl) and <i>Vmat2 <sup>lox/lox/NEX-Cre-tg</sup> </i> (cKO) mice. Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">Figure 2-1, EPS file</span>. </p>  </div> </div> <div class="heightenedlocomotorresponsetopsychostimulantsupongene-targetingofvmat2innex-crevtaneurons" title="sec"> <div class="title" tagx="title" title="title">
Heightened locomotor response to psychostimulants upon gene-targeting of <i>Vmat2</i> in NEX-Cre VTA neurons </div> <p>To address whether it is possible to dissociate an explicit behavioral role of DA neurotransmission exerted by NeuroD6 VTA DA neurons from the range of behaviors ascribed to the mDA system, <i>Vmat2 <sup>lox/lox;NEX-Cre-tg</sup> </i> cKO mice were tested in a battery o <i>f</i> tests relevant to the mDA system and compared to <i>Vmat2 <sup>lox/lox;NEX-Cre-wt</sup> </i> Ctrl mice. To assess body weight, mice were weighed every week from weaning to adulthood. cKO mice were similar to their Ctrl littermates weight-wise (effect of age: <i>F</i> <sub>(4158)</sub> = 79.8, <i>p</i> &amp;lt; 0.001; genotype: <i>F</i> <sub>(1158)</sub> = 4.67 <i>p</i> = 0.032; no age × genotype interaction, no <i>post hoc</i> differences between genotypes; <a href="#F3">Fig. 3 <i>A</i> </a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3.</span> </a> </div>   <p>Altered responsiveness to psychostimulants upon ablation of Vmat2 gene expression in NeuroD6 VTA neurons. Color coding: <i>Vmat2 <sup>lox/lox;NEX-Cre-wt</sup> </i> (Ctrl) in white; <i>Vmat2 <sup>lox/lox;NEX-Cre-tg</sup> </i>(cKO) in green. <b> <i>A</i> </b>, Weight curve for Ctrl ( <i>N</i> = 14) and cKO ( <i>N</i> = 23) mice. Data presented as mean weight in grams for each week ± SEM ( <b>+</b> <i>p</i> &amp;lt; 0.05 effect of genotype; ### <i>p</i> &amp;lt; 0.001, effect of age). <b> <i>B</i> </b>, Baseline locomotion in novel environment. Ctrl ( <i>N</i> = 17) and cKO ( <i>N</i> = 17). Data expressed as mean distance moved in 5-min bins ± SEM (### <i>p</i> &amp;lt; 0.001, effect of time). <b> <i>C</i> </b>, Sucrose preference expressed as percentage of preference for sucrose over tap water ± SEM. Ctrl ( <i>N</i> = 14) and cKO ( <i>N</i> = 21; ### <i>p</i> &amp;lt; 0.001 effect of sucrose concentration). <b> <i>D</i> </b>, Ethanol preference expressed as percentage of preference for ethanol solution over tap water ± SEM. Ctrl ( <i>N</i> = 14) and cKO ( <i>N</i> = 14; ### <i>p</i> &amp;lt; 0.001 effect of ethanol concentration, §§§ <i>p</i> &amp;lt; 0.001 3% vs 6% and 10% in ctrl mice). <b> <i>E</i> </b>, Cocaine-induced locomotion. Top, Administration schedule. Bottom, Average distance moved 1 h after injection of saline and 5, 10, 20 mg/kg of cocaine; Ctrl ( <i>N</i> = 14) and cKO ( <i>N</i> = 21) mice. Data expressed as total distance moved during the 1-h recording period ± SEM (### <i>p</i> &amp;lt; 0.001 effect of session). <b> <i>F</i> </b>, Amphetamine-induced locomotion. Top, Administration schedule. Bottom, Average distance moved 1.5 h after injection; Ctrl ( <i>N</i> = 17) and cKO mice ( <i>N</i> = 17). Data presented as mean of total distance moved in cm ± SEM for each session; <b>++</b> <i>p</i> &amp;lt; 0.01 effect of genotype, ### <i>p</i> &amp;lt; 0.001 effect of session, * <i>p</i> &amp;lt; 0.05 and *** <i>p</i> &amp;lt; 0.001 cKO versus Ctrl. <b> <i>G</i> </b>, CPP. Illustration of setup and administration schedule. <b> <i>H</i> </b>, <b> <i>J</i> </b>, Preference score displayed as Δsec, the difference between time spent in drug-paired compared during pretest and test ± SEM, positive value indicates preference (cocaine: Ctrl <i>N</i> = 12, cKO <i>N</i> = 15; amphetamine: Ctrl <i>N</i> = 13, cKO <i>N</i> = 16). <b> <i>I</i> </b>, <b> <i>K</i> </b>, Cocaine-induced and amphetamine-induced locomotion during conditioning in the CPP setup displayed as distance moved in 30 min ± SEM (cocaine: Ctrl <i>N</i> = 12, cKO <i>N</i> = 15; amphetamine: Ctrl <i>N</i> = 15, cKO <i>N</i> = 17, <b>+</b> <i>p</i> = 0.031 effect of genotype, ## <i>p</i> = 0.006, ### <i>p</i> &amp;lt; 0.001 effect of session). Ctrl, Control; cKO, conditional knockout; CPP, Conditioned place preference. </p>   </div> <div class="baselinelocomotion" title="sec"> <div class="title" tagx="title" title="title">
Baseline locomotion</div> <p>The habituation response to a novel environment, a gross measure of stress and exploratory behavior, was addressed. Both Ctrl and cKO mice showed the same rate of reaching a stable plateau in baseline locomotion (effect of time: <i>F</i> <sub>(5160)</sub> = 69.5, <i>p</i> &amp;lt; 0.001; effect of genotype: <i>F</i> <sub>(1,32)</sub> = 0.00912, <i>p</i> = 0.535; <a href="#F3">Fig. 3 <i>B</i> </a>). </p> </div> <div class="sucroseandethanolpreference" title="sec"> <div class="title" tagx="title" title="title">
Sucrose and ethanol preference</div> <p>A sucrose bottle preference test was next performed. Both Ctrl and cKO mice preferred the ascending concentrations of sucrose solutions over water (effect of concentration: <i>F</i> <sub>(2,66)</sub> = 151, <i>p</i> &amp;lt; 0.001), but no differences between the genotypes were observed (effect of genotype: <i>F</i> <sub>(1,33)</sub> = 1.12, <i>p</i> = 0.297; <a href="#F3">Fig. 3 <i>C</i> </a>). The rewarding effect of alcohol was subsequently measured by using increasing concentrations of ethanol (3%, 6%, 10%) presented in a bottle preference test. Again, both Ctrl and cKO mice preferred the presented reward over water (effect of concentration: <i>F</i> <sub>(2,52)</sub> = 14.2, <i>p</i> &amp;lt; 0.001), but there was no difference between the genotypes (effect of genotype: <i>F</i> <sub>(1,26)</sub> = 0.969, <i>p</i> = 0.334). However, <i>post hoc</i> analysis showed that Ctrl mice significantly preferred the 6% and 10% concentrations over the 3% solution (§§§ <i>p</i> &amp;lt; 0.001 3% vs 6% and 10% ethanol in ctrl mice), while a trend toward significant differences in cKO mice was observed only between the 3% and 10% ethanol solutions (3% vs 10%: <i>p</i> &amp;lt; 0.072; <a href="#F3">Fig. 3 <i>D</i> </a>). </p> </div> <div class="cocaine-inducedandamphetamine-inducedlocomotion" title="sec"> <div class="title" tagx="title" title="title">
Cocaine-induced and amphetamine-induced locomotion</div> <p>To address locomotor responses on psychostimulant-injections, cocaine and amphetamine administration protocols were applied and locomotion was measured. Following administration of acute ascending doses of cocaine (5, 10, and 20 mg/kg), both Ctrl and cKO mice displayed increased locomotion in a dose-dependent manner; however, no significant differences were observed between genotypes (effect of session: <i>F</i> <sub>(3,99)</sub> = 108, <i>p</i> &amp;lt; 0.001; genotype, <i>F</i> <sub>(1,33)</sub> = 1.65, <i>p</i> = 0.208; session × genotype interaction: <i>F</i> <sub>(3,99)</sub> = 1, <i>p</i> = 0.396; <a href="#F3">Fig. 3 <i>E</i> </a>). Next, an amphetamine sensitization protocol was applied. All mice responded to amphetamine with hyperlocomotion, but the effect was significantly higher in cKO mice than control mice in days 4, 5, and 17 of the experiment (effect of day: <i>F</i> <sub>(5160)</sub> = 40.9, <i>p</i> &amp;lt; 0.001; genotype, <i>F</i> <sub>(1,32)</sub> = 9.09, <i>p</i> = 0.005; day × genotype interaction: <i>F</i> <sub>(5160)</sub> = 4.79; <i>p</i> &amp;lt; 0.001; ctrl vs cKO day 4 <i>p</i> = 0.011, day 5 <i>p</i> &amp;lt; 0.001, day 17 <i>p</i> = 0.029; <a href="#F3">Fig. 3 <i>F</i> </a>). </p> </div> <div class="cpp" title="sec"> <div class="title" tagx="title" title="title">
CPP</div> <p>To study the reinforcing effects of psychostimulants, a CPP procedure was applied ( <a href="#F3">Fig. 3 <i>G</i> </a>). Both Ctrl and cKO mice showed preference for the cocaine-paired or amphetamine-paired compartment over the saline-paired compartment with no significant difference between genotypes (ctrl vs cKO cocaine: <i>p</i> = 0.860, amphetamine <i>p</i> = 0.744; <a href="#F3">Fig. 3 <i>H</i>, <i>J</i> </a>). In addition to preference, locomotion was monitored during the conditioning sessions. cKO mice displayed increased locomotor responses after repeated administration of cocaine compared to Ctrl mice (effect of session; <i>F</i> <sub>(3,75)</sub> = 4.4, <i>p</i> = 0.006; effect of genotype <i>F</i> <sub>(1,25)</sub> = 5.2, <i>p</i> = 0.031, no differences in <i>post hoc</i> analysis; <a href="#F3">Fig. 3 <i>I</i> </a>). In contrast, in the CPP paradigm, repeated administration of amphetamine did not induce elevated locomotion in cKO over Ctrl mice (effect of session; <i>F</i> <sub>(3,85)</sub> = 24.0, <i>p</i> &amp;lt; 0.001; effect of genotype <i>F</i> <sub>(1,30)</sub> = 0.0631, <i>p</i> = 0.803; <a href="#F3">Fig. 3 <i>K</i> </a>). </p> </div> </div> <div class="neurod6mrnaco-localizespartlywithcalb2mrna,butcalb2mrnaisabundantthroughoutvtaandsnc" title="sec"> <div class="title" tagx="title" title="title">
NeuroD6 mRNA co-localizes partly with Calb2 mRNA, but Calb2 mRNA is abundant throughout VTA and SNc </div> <p>To further characterize the molecular identity of NeuroD6 VTA neurons, FISH was next used to address the putative overlap between NeuroD6 and Calb2 mRNAs. Distribution patterns of NeuroD6 and Calb2 mRNAs within midbrain DA neurons were recently described without addressing their putative overlap ( <a href="#B50">Viereckel et al., 2016</a>). In contrast to the selective localization of NeuroD6 mRNA within the VTA and its exclusion from the SNc, Calb2 mRNA was abundant in both VTA and SNc ( <a href="#F4">Fig. 4 <i>A</i> </a>). The restricted number of NeuroD6 neurons in the VTA showed partial overlap with Calb2 mRNA: 54% of all NeuroD6 VTA neurons were positive for Calb2 mRNA while 20% of Calb2 neurons expressed NeuroD6 mRNA ( <a href="#F4">Fig. 4 <i>A</i> </a>). Further quantification within the VTA showed that Calb2 mRNA was detected in 51% of all Th-neurons, with a similar match of Calb2/Dat co-localization at 50% ( <a href="#F4">Fig. 4 <i>B</i>, <i>C</i> </a>). Some Calb2 neurons in the VTA were positive for Vglut2 mRNA (7%; <a href="#F4">Fig. 4 <i>D</i> </a>), while 20% of all Calb2 neurons in the VTA were positive for Viaat mRNA ( <a href="#F4">Fig. 4 <i>E</i> </a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>NeuroD6 mRNA co-localizes partly with Calb2 mRNA, but Calb2 mRNA is abundant throughout the VTA and SNc. Double-labeling FISH in the ventral midbrain of adult wild-type mice detecting the following mRNAs. <b> <i>A</i> </b>, NeuroD6 (red) and Calb2 (green), inset with high magnification of overlap (yellow), pie charts illustrating quantification of overlap between NeuroD6 and Calb2. <b> <i>B</i> </b>, Calb (red) and Th (green), inset with high magnification of overlap (yellow), pie charts illustrating quantification of overlap between Th and Calb2. <b> <i>C</i> </b>, Calb2 (red) and Dat (green), inset with high magnification of Dat/Calb2 mRNA overlap (yellow), pie chart illustrating quantification of overlap between Th and Calb2. <b> <i>D</i> </b>, Calb2 (red) and Vglut2 (green), inset with high magnification of Vglut2/Calb2 mRNA overlap (yellow) in blue square and Vglut2-negative/Calb2-positive (red) in white square, pie chart illustrating quantification of overlap between Th and Calb2. <b> <i>E</i> </b>, Calb2 (red) and Viaat (green), inset with high magnification of Viaat negative or positive (red, yellow, green) in white square. VTA, ventral tegmental area; SNc, substantia nigra pars compacta; PBP, parabrachial pigmented nucleus; PN, paranigral nucleus; PIF, parainterfascicular nucleus; RLi, rostral linear nucleus; IF, interfascicular nucleus. Calb2, Calbindin 2 (Calretinin); Dat, Dopamine transporter; Th, Tyrosine hydroxylase; Vglut2, Vesicular glutamate transporter 2; Viaat, Vesicular inhibitory amino acid transporter; FISH, fluorescent <i>in situ</i> hybridization. </p>   </div> </div> <div class="spatiallyrestrictedstriatalinnervationbyneurod6-creandcalb2-crevtaneurons" title="sec"> <div class="title" tagx="title" title="title">
Spatially restricted striatal innervation by NeuroD6-Cre and Calb2-Cre VTA neurons</div> <p>Next, to allow analysis of projections, signaling properties and behavioral regulation of NEX-Cre and Calb2-Cre VTA neurons, optogenetics was implemented. Upon infusion of viral particles carrying a double-floxed <i>DIO-ChR2-eYFP</i> genetic construct encoding both Channelrhodopsin (ChR2) and the enhanced yellow fluorescent protein (eYFP) into the VTA, mice were analyzed in different parameters. DAT-Cre and Vglut2-Cre transgenic mice were used as controls based on their representation of VTA and SNc dopaminergic and glutamatergic neurons, respectively ( <a href="#B46">Stuber et al., 2010</a>; <a href="#B17">Hnasko et al., 2012</a>; <a href="#B37">Pascoli et al., 2015</a>; <a href="#B41">Qi et al., 2016</a>; <a href="#B53">Yoo et al., 2016</a>). First, Cre-driven expression of the <i>DIO-ChR2-eYFP</i> construct in DAT-Cre, Vglut2-Cre, Calb2-Cre and NEX-Cre mice was analyzed histologically by comparing YFP with TH immunolabeling ( <a href="#F5">Fig. 5 <i>A</i> </a>). In DAT-Cre, Vglut2-Cre, Calb2-Cre, and NEX-Cre mice, YFP fluorescent labeling was identified in the VTA, verifying the activity of each Cre-driver to recombine the floxed optogenetic construct ( <a href="#F5">Fig. 5 <i>B–F</i> </a>). YFP co-localized extensively with TH in the VTA. YFP was strongest and most abundant in the VTA of DAT-Cre mice, while Vglut2-Cre, Calb2-Cre, and NEX-Cre mice all showed lower amount of cells positive for YFP ( <a href="#F5">Fig. 5 <i>B–F</i> </a>). Next, to reveal target areas, sections throughout the entire brain of all four Cre-driver mouse lines were analyzed and compared. Some target areas were the same for all four Cre-drivers, including the NAcSh and ventral pallidum, while others differed, such as the distribution within the medial and lateral habenula ( <a href="#T2">Table 2</a>). Overall, the density of YFP-positive fibers was substantially lower in NEX-Cre and Calb2-Cre mice than in DAT-Cre and Vglut2-Cre mice. Following analysis of sections throughout the brain, the VTA and striatum were analyzed in more detail. DAT-Cre mice showed strong cellular YFP labeling within all VTA subareas (sparse in RLi) and within the SNc, primarily on the injected side ( <a href="#F5">Fig. 5 <i>C–C’</i> <i>’</i> </a>). YFP-positive fibers were distributed across the striatal complex including primarily the dorsomedial striatum, NAcSh, NAc core and the olfactory tubercle (OT; <a href="#F5">Fig. 5 <i>C–C’</i> <i>’</i> </a>). Vglut2-Cre mice showed YFP-labeled cell bodies primarily in the medial VTA with fibers innervating the NAc and OT ( <a href="#F5">Fig. 5 <i>D–D’</i> <i>’</i> </a>). Next, Calb2-Cre and NEX-Cre mice were addressed. Calb2-Cre mice showed similar distribution of YFP-labeling as DAT-Cre within VTA, but the density was sparser than in DAT-Cre mice ( <a href="#F5">Fig. 5 <i>E–E’’</i> </a>). YFP-positive fibers in the striatal complex were detected in the OT ( <a href="#F5">Fig. 5 <i>E–E’</i> <i>’</i> </a>). NEX-Cre mice showed a low number of YFP cells in the VTA ( <a href="#F5">Fig. 5 <i>F–F’</i> <i>’</i> </a>), in accordance with the modest distribution of endogenous NeuroD6 mRNA described above. Weak YFP fluorescence was detected in fibers throughout the NAcSh and OT ( <a href="#F5">Fig. 5 <i>F–F’</i> <i>’</i> </a>). The distribution pattern of YFP-positive cells in the VTA of NEX-Cre mice was similar as the distribution of endogenous NeuroD6 mRNA. However, the YFP appeared more abundant than the above analyzed NeuroD6 mRNA. Quantification was performed to address the overlap between YFP and TH. The majority of NEX-Cre/YFP and Calb2-Cre/YFP neurons showed TH immunoreactivity; however, for both Cre-lines, a number of YFP cells were negative for TH (NEX-Cre/ChR2: TH+: 4013 ± 21.72, eYFP+ 965 ± 4.17, double: 715 ± 3.24; Calb2-Cre/ChR2: TH+: 4187 ± 18.9, eYFP+: 1396 ± 6.04, double: 939 ± 4.69). In total, 74% of NEX-Cre and 67% of Calb2-Cre neurons showed overlap between YFP and TH ( <a href="#F5">Fig. 5 <i>G</i>, <i>H</i> </a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5.</span> </a> </div>   <p>Spatially restricted striatal innervation by NeuroD6 and Calb2 VTA neurons. <b> <i>A</i> </b>, Schematic illustration of stereotaxic injection into VTA of Cre-dependent <i>DIO-ChR2-eYFP</i> DNA construct packaged into AAV. <b> <i>B</i> </b>, Representative VTA neurons immunopositive for TH (red), YFP (green), or both (yellow; <i>DIO-ChR2-eYFP</i>-injected NEX-Cre mice). <b> <i>C–F</i> </b>, Representative pictures of VTA (left panels) and striatal complex (right panels) in <i>DIO-ChR2-eYFP</i>-injected DAT-Cre ( <b> <i>C–C’’</i> </b>), Vglut2-Cre ( <b> <i>D–D’’</i> </b>), Calb2-Cre ( <b> <i>E–E’’</i> </b>), and NEX-Cre ( <b> <i>F–F’’</i> </b>) mice. Panel far right, Schematic summary of striatal innervation pattern. Additional target areas listed in <a href="#T2">Table 2</a>. Quantification of YFP and TH immunofluorescent overlap: schematic illustration of four representative VTA areas selected for counting, shown as squares and labeled VTA 1–4 ( <b> <i>G</i> </b>). Results of quantifications shown in histograms for each VTA area and the total sum ( <b> <i>H</i> </b>). PBP, parabrachial pigmented nucleus; PN, paranigral nucleus; PIF, parainterfascicular nucleus; RLi, rostral linear nucleus; IF, interfascicular nucleus; SNc, substantia nigra pars compacta; SNr, substantia nigra pars reticulata; IPR, interpeduncular nucleus, rostral subnucleus; IPC, interpeduncular nucleus, caudal subnucleus; DMStr, dorsomedial striatum; NAcC, nucleus accumbens core; NAcSh, nucleus accumbens shell; aca; anterior commissure, anterior part; OT, olfactory tubercle. DAT, Dopamine transporter, Calb2, Calbindin 2 (Calretinin); NEX, NeuroD6; Vglut2; Vesicular glutamate transporter 2; Th, Tyrosine hydroxylase; ChR2; Channelrhodopsin 2; eYFP, enhanced Yellow fluorescent protein. </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2.</span>  <p>Projection areas of VTA neurons represented in NEX-Cre and Calb2-Cre mice compared with DAT-Cre and Vglut2-Cre mice </p>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Area</th> <th>Cre-driver</th> </tr> <tr> <th /> <th>DAT</th> <th>Vglut2</th> <th>Calb2</th> <th>NEX</th> </tr> </thead> <tbody> <tr> <td>Anterior olfactory area</td> <td>+</td> <td>+</td> <td>+</td> <td>+</td> </tr> <tr> <td>Medial prefrontal cortex (infralimbic, prelimbic, and anterior cingulate cortices)</td> <td>+</td> <td>+</td> <td>+</td> <td>+</td> </tr> <tr> <td>(Medial) orbital cortex</td> <td>+</td> <td>+</td> <td>+</td> <td>+</td> </tr> <tr> <td>Nucleus accumbens shell</td> <td>+</td> <td>+</td> <td>(+)</td> <td>+</td> </tr> <tr> <td>Nucleus accumbens core</td> <td>+</td> <td>-</td> <td>-</td> <td>+</td> </tr> <tr> <td>Dorsomedial Striatum</td> <td>+</td> <td>-</td> <td>-</td> <td>-</td> </tr> <tr> <td>Olfactory tubercle</td> <td>+</td> <td>+</td> <td>+</td> <td>+</td> </tr> <tr> <td>Cingulate cortex</td> <td>+</td> <td>+</td> <td>+</td> <td>+</td> </tr> <tr> <td>Septum/septal nuclei</td> <td>+</td> <td>+</td> <td>-</td> <td>+</td> </tr> <tr> <td>Diagonal band of Broca</td> <td>+</td> <td>+</td> <td>+</td> <td>+</td> </tr> <tr> <td>Ventral pallidum</td> <td>+</td> <td>+</td> <td>+</td> <td>+</td> </tr> <tr> <td>Bed nuclei of the stria terminalis</td> <td>+</td> <td>+</td> <td>+</td> <td>+</td> </tr> <tr> <td>Preoptic area</td> <td>+</td> <td>+</td> <td>+</td> <td>+</td> </tr> <tr> <td>Lateral habenula</td> <td>+</td> <td>+</td> <td>-</td> <td>-</td> </tr> <tr> <td>Medial habenula</td> <td>-</td> <td>-</td> <td>+</td> <td>+</td> </tr> <tr> <td>Hippocampus</td> <td>-</td> <td>-</td> <td>-</td> <td>+</td> </tr> <tr> <td>Dentate gyrus</td> <td>-</td> <td>-</td> <td>-</td> <td>+</td> </tr> <tr> <td>Amygdala</td> <td>+</td> <td>+</td> <td>+</td> <td>+</td> </tr> <tr> <td>Hypothalamic area</td> <td>+</td> <td>+</td> <td>-</td> <td>+</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Summary of projection areas for VTA neurons virally injected with optogenetic constructs ( <i>DIO-ChR2-eYFP</i>) in DAT-Cre, Vglut2-Cre, Calb2-Cre, and NEX-Cre mice, respectively, and detected as YFP-positive fibers. + indicates presence of YFP-positive fibers; - indicates absence of YFP-positive fibers; (+) indicates low presence of fibers. </p> </div> </div> </div> </div> <div class="optogeneticstimulationinstriataltargetareasofneurod6andcalb2vtaneuronsverifiesdarelease" title="sec"> <div class="title" tagx="title" title="title">
Optogenetic stimulation in striatal target areas of NeuroD6 and Calb2 VTA neurons verifies DA release </div> <p>To address neurotransmitter release, extracellular DA concentration upon optogenetic stimulation was recorded using FSCV in slice preparations. DAT-Cre, NEX-Cre, and Calb2-Cre mice injected with the same <i>DIO-ChR2-eYFP</i> construct as above ( <a href="#F6">Fig. 6 <i>A</i> </a>) were analyzed upon photostimulation and subsequent recording within the NAcSh and OT ( <a href="#F6">Fig. 6 <i>B</i> </a>). Cre-mice injected with <i>DIO-eYFP</i> were used as controls. DA levels (∼1 μM) were readily recorded upon photostimulation in both the NAcSh of DIO-ChR2 injected DAT-Cre (0.9699 ± 0.1471 μM) and NEX-Cre mice (0.4701 ± 0.08043 μM), while a lower signal was obtained in the NAcSh of Calb2-Cre/ChR2 mice (0.01509 ± 0.002845 μM; <a href="#F6">Fig. 6 <i>C</i>, <i>D</i> </a>). Upon photostimulation and recording in the OT, lower DA levels (∼200 nM) than those measured in the NAcSh were obtained in DAT-Cre/ChR2 mice (0.2129 ± 0.01291 μM) while even smaller levels were detected in both Calb2-Cre/ChR2 (0.02097 ± 0.002712 μM) and NEX-Cre/ChR2 mice (0.01362 ± 0.002304 μM; <a href="#F6">Fig. 6 <i>C</i>, <i>D</i> </a>). Despite comparably low in size, all DA levels recorded in mice expressing the ChR2-YFP were significantly larger than in mice injected with the control virus (DAT-Cre, NAcSh ChR2: 0.9699 ± 0.1471 μM, eYFP: 0.006802 ± 0.0008813 μM, <i>t</i> <sub>(9)</sub> = 6.55 <i>p</i> &amp;lt; 0.0001, OT ChR2 0.2129 ± 0.01291 μM vs eYFP 0.004649 ± 0.0009871 μM, <i>t</i> <sub>(9)</sub> = 16.08 <i>p</i> &amp;lt; 0.0001; NEX-Cre, NAcSh ChR2: 0.4701 ± 0.08043 μM, eYFP: 0.0102 ± 0.001682 μM, <i>t</i> <sub>(9)</sub> = 5.716 <i>p</i> &amp;lt; 0.0001, OT ChR2: 0.01362 ± 0.002304 μM, eYFP: 0.005791 ± 0.0008003 μM, <i>t</i> <sub>(9)</sub> = 3.209 <i>p</i> = 0.0049; Calb2-Cre, NacSh ChR2: 0.01509 ± 0.002845 μM, eYFP: 0.006087 ± 0.001746 μM, <i>t</i> <sub>(9)</sub> = 2.696 <i>p</i> = 0.0148, OT ChR2: 0.02097 ± 0.002712 μM, eYFP 0.007081 ± 0.001315 μM, <i>t</i> <sub>(9)</sub> = 4.607 <i>p</i> = 0.0002; <a href="#F6">Fig. 6 <i>C</i>, <i>D</i> </a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>Optogenetic stimulation in striatal target areas of NeuroD6 and Calb2 VTA neurons verifies DA release. <b> <i>A</i> </b>, Schematic representation of stereotaxic injection into VTA of Cre-dependent <i>DIO-ChR2-eYFP</i> and <i>DIO-eYFP</i> (Ctrl); FSCV recording sites within NAcSh and OT (red dots). <b> <i>B</i> </b>, Illustration of the experimental setup. <b> <i>C</i> </b>, Representative light-evoked DA recordings from injected DAT-Cre (left), NEX-Cre (middle), and Calb2-Cre (right) mice in the NAcSh (top) and the OT (bottom). <b> <i>D</i> </b>, Quantification of photostimulation-evoked DA release in the NAc shell (left) and OT (right); <i>N</i> = 10 recording sites per group for each region. Mice used for the recordings: DAT-Cre/ChR2 <i>N</i> = 2, DAT-Cre/eYFP <i>N</i> = 2, NEX-Cre/ChR2 <i>N</i> = 3, NEX-Cre/eYFP <i>N</i> = 2 Calb2-Cre/ChR2 <i>N</i> = 3, and Calb2-Cre/eYFP <i>N</i> = 2. Box and whisker plots, Center lines indicate medians, box edges represent the interquartile range, whiskers extend to the minimal and maximal values (* <i>p</i> &amp;lt; 0.05, ** <i>p</i> &amp;lt; 0.01, *** <i>p</i> &amp;lt; 0.001 ChR2 vs ctrl). DStr, dorsal striatum; NAcSh, nucleus accumbens shell; OT, olfactory tubercle. DAT, Dopamine transporter; Calb2, Calbindin 2 (Calretinin); ChR2; Channelrhodopsin 2; eYFP, enhanced Yellow fluorescent protein; NEX, NeuroD6. </p>   </div> </div> <div class="optogeneticstimulationinstriataltargetareasofneurod6andcalb2vtaneuronsrevealsaglutamatergicpostsynapticresponse" title="sec"> <div class="title" tagx="title" title="title">
Optogenetic stimulation in striatal target areas of NeuroD6 and Calb2 VTA neurons reveals a glutamatergic postsynaptic response </div> <p>To address the presence of postsynaptic currents in NAcSh and OT neurons upon optogenetic activation, patch clamp electrophysiology was implemented in NEX-Cre and Calb2-Cre injected with <i>DIO-ChR2-eYFP</i> ( <a href="#F7">Fig. 7</a>). Upon optogenetic stimulation, 82% of neurons in the NAcSh NEX-Cre mice (18 out of 22 cells) and 87% of OT neurons in Calb2-Cre mice (13 out of 15 cells) showed EPSCs (NEX-Cre NAcSh, mean amplitude 28 ± 6.8 pA; Calb2-Cre OT, mean amplitude 39 ± 7.7 pA; <a href="#F7">Fig. 7 <i>B</i>, <i>C</i> </a>). In both cases, EPSCs were almost completely abolished after bath application of 10 μM the AMPA receptor antagonist DNQX, demonstrating that the recorded currents are AMPA receptor-mediated (NEX-Cre NAcSh mean amplitude, control: 33 ± 13 pA, DNQX 1.5 ± 0.96 pA <i>t</i> <sub>(5)</sub> = 2.602 <i>p</i> = 0.0481; Calb2-Cre OT, mean amplitude, control: 46 ± 16 pA, DNQX: 0.74 ± 0.74 pA <i>t</i> <sub>(4)</sub> = 2.867 <i>p</i> = 0.0456; <a href="#F7">Fig. 7 <i>D</i> </a>). The synaptic delay of the EPSCs was short (NEX-Cre NAcSh 3.3 ± 0.25 ms; Calb2-Cre OT 3.6 ± 0.21 ms). In contrast, the mean decay time was longer in the OT than in NAcSh (NEX-Cre NAcSh 5.3 ± 0.5 ms; Calb2-Cre OT 7.8 ± 0.62 ms). No inhibitory/GABA-receptor-mediated currents were observed during recordings in either NEX-Cre or Calb2-Cre mice ( <a href="#F7">Fig. 7 <i>B</i> </a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7.</span> </a> </div>   <p>Optogenetic stimulation in striatal target areas of NeuroD6 and Calb2 VTA neurons reveals glutamatergic postsynaptic responses. <b> <i>A</i> </b>, Representative picture from patch-clamp slice electrophysiology in NAcSh of NEX-Cre mice and OT of Calb2-Cre mice injected with <i>DIO-ChR2-eYFP</i>. <b> <i>B</i> </b>, Representative traces of photostimulation-evoked postsynaptic currents recorded from NAcSh cells from NEX-Cre and OT cells from Calb2-Cre mice injected with <i>DIO-ChR2-eYFP</i>. <b> <i>C</i> </b>, Pie charts represent the percentage of cells showing EPSCs (white) versus negative (black) upon photostimulation of terminals in the NAcSh ( <i>N</i> = 18 cells from four mice) of NEX-Cre mice, and OT ( <i>N</i> = 14 cells from 4 mice) of Calb2-Cre mice. The <i>y</i>-axis shows amplitude in pA; each circle represents one cell, and bold lines the mean amplitude ± SEM. <b> <i>D</i> </b>, Patch-clamp recordings pre-bath (control) and post-bath application of DNQX upon photostimulation in NAcSh of NEX-Cre/ChR2 (left, <i>N</i> = 6 cells from three mice) and OT of Calb2-Cre/ChR2 mice (right, <i>N</i> = 5 cells from three mice). Each circle represents one cell (* <i>p</i> &amp;lt; 0.05 control vs DNQX). NAcSh, nucleus accumbens shell; OT, olfactory tubercle. Calb2, Calbindin 2 (Calretinin); NEX; NeuroD6; ChR2; Channelrhodopsin 2; eYFP, enhanced Yellow fluorescent protein. </p>   </div> </div> <div class="optogeneticactivationofneurod6vtaneurons,butnotcalb2vtaneurons,inducesplacepreference" title="sec"> <div class="title" tagx="title" title="title">
Optogenetic activation of NeuroD6 VTA neurons, but not Calb2 VTA neurons, induces place preference </div> <p>Finally, <i>in vivo</i> optogenetic stimulation in the VTA of NEX-Cre and Calb2-Cre mice was applied to assess whether this would induce place preference behavior. Again, DAT-Cre and Vglut2-Cre mice were used as references for comparison to Calb2-Cre and NEX-Cre mice. Mice received <i>DIO-ChR2-eYFP</i> or <i>DIO-eYFP</i> (control) injection and implantation of optic fibers above the VTA ( <a href="#F8">Fig. 8 <i>A</i>, <i>G</i> </a>), and were analyzed for RT-PP and CR ( <a href="#F8">Fig. 8 <i>A</i> </a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 8.</span> </a> </div>   <p>Optogenetic activation of NeuroD6 VTA neurons, but not Calb2 VTA neurons, induces place preference. <b> <i>A</i> </b>, Schematic drawing of stereotaxic injection into VTA of Cre-dependent <i>DIO-ChR2-eYFP</i> and of experimental setup for RT-PP analysis. <b> <i>B–F</i> </b>, Time spent in light-paired (blue), unpaired (white during phase 1, black during reversal phase), and neutral (gray) compartments shown as mean percentage of time spent in each compartment ± SEM (left; * <i>p</i> &amp;lt; 0.05, *** <i>p</i> &amp;lt; 0.001 paired vs unpaired compartment); average percentage of time spent in each compartment during days 3, 4, 6, and 7 ± SEM (bar graphs; right; * <i>p</i> &amp;lt; 0.05, *** <i>p</i> &amp;lt; 0.001 vs light-paired compartment; # <i>p</i> &amp;lt; 0.05, ## <i>p</i> &amp;lt; 0.01, ### <i>p</i> &amp;lt; 0.001 vs unpaired compartment). DAT-Cre <i>N</i> = 10; Vglut2-Cre <i>N</i> = 7; Calb2-Cre <i>N</i> = 7; NEX-Cre <i>N</i> = 5. <b> <i>F</i> </b>, High-power stimulation of bilaterally injected NEX-Cre mice ( <i>N</i> = 4). <b> <i>G</i> </b>, Schematic illustration of optical fiber placement in mice analyzed in RT-PP analysis. NS, non-significant. DAT, Dopamine transporter; Calb2, Calbindin 2 (Calretinin); NEX, NeuroD6; Vglut2; Vesicular glutamate transporter 2; ChR2; Channelrhodopsin 2; eYFP, enhanced Yellow fluorescent protein. </p>   </div> <div class="analysisofrt-ppandcrindat-creandvglut2-cremice" title="sec"> <div class="title" tagx="title" title="title">
Analysis of RT-PP and CR in DAT-Cre and Vglut2-Cre mice</div> <p>DAT-Cre mice displayed a significant preference to the light-paired compartment on every day of stimulation (effect of compartment <i>F</i> <sub>(2,18)</sub> = 51.8, <i>p</i> &amp;lt; 0.001; day × compartment interaction <i>F</i> <sub>(12,108)</sub> = 33, <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 paired vs unpaired compartment; <a href="#F8">Fig. 8 <i>B</i> </a>, left). This place preference was also evident when the effect of stimulation was averaged for the four experimental days (effect of compartment <i>F</i> <sub>(2,6)</sub> = 166, <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 vs paired compartment; <a href="#F8">Fig. 8 <i>B</i> </a>, right). In the absence of stimulation, on days 5 and 8, DAT-Cre mice demonstrated a CR for the previous light-paired compartment (*** <i>p</i> &amp;lt; 0.001 paired vs unpaired; <a href="#F8">Fig. 8 <i>B</i> </a>). Control mice (DAT-Cre negative or DAT-Cre injected with <i>DIO-eYFP</i>) did not display any preference toward the stimulation [effect of compartment <i>F</i> <sub>(2,4)</sub> = 4.26, <i>p</i> = 0.102; day × compartment interaction, <i>F</i> <sub>(12,24)</sub> = 0.898 <i>p</i> = 0.562, * <i>p</i> &amp;lt; 0.05 paired vs unpaired compartment (Extended Data <a href="#fig8-1">Fig. 8 <i>-</i>1 <i>A</i> </a>, left); effect of compartment <i>F</i> <sub>(2,6)</sub> = 48.7 <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 ### <i>p</i> &amp;lt; 0.001 neutral versus paired and unpaired, respectively (Extended Data <a href="#fig8-1">Fig. 8 <i>-</i>1 <i>A</i> </a>, right); effect of compartment <i>F</i> <sub>(2,4)</sub> = 27.9, <i>p</i> = 0.004, day × compartment interaction <i>F</i> <sub>(12,24)</sub> = 0.767 <i>p</i> = 0.677, *** <i>p</i> &amp;lt; 0.001 paired vs unpaired compartment; right: effect of compartment <i>F</i> <sub>(2,6)</sub> = 2.97, <i>p</i> = 0.127 (Extended Data <a href="#fig8-1">Fig. 8-1 <i>B</i> </a>, left); effect of compartment <i>F</i> <sub>(2,10)</sub> = 18.6, <i>p</i> &amp;lt; 0.001, day × compartment interaction <i>F</i> <sub>(12,60)</sub> = 0.963, <i>p</i> = 0.494 (Extended Data <a href="#fig8-1">Fig. 8-1 <i>C</i> </a>, left); effect of compartment <i>F</i> <sub>(2,6)</sub> = 9.27, <i>p</i> = 0.015, * <i>p</i> &amp;lt; 0.05 # <i>p</i> &amp;lt; 0.05 neutral vs paired and unpaired, respectively (Extended Data <a href="#fig8-1">Fig. 8-1 <i>C</i> </a>, right)]. These results were in accordance with the literature ( <a href="#B53">Yoo et al., 2016</a>) and thereby validated the experimental setup. In contrast to the strong place preference induced by stimulation in DAT-Cre mice, Vglut2-Cre mice analyzed in the same setup displayed a preference for the unpaired compartment [effect of compartment <i>F</i> <sub>(2,12)</sub> = 40.9, <i>p</i> &amp;lt; 0.001 and day × compartment interaction <i>F</i> <sub>(12,72)</sub> = 16.1, <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 paired vs unpaired ( <a href="#F8">Fig. 8 <i>C</i> </a>, left); effect of compartment <i>F</i> <sub>(2,6)</sub> = 162, <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 versus paired, ### <i>p</i> &amp;lt; 0.001 vs unpaired ( <a href="#F8">Fig. 8 <i>C</i> </a>, right)]. To further verify this observation, the protocol was modified so that the mice would receive photostimulation upon entry to either one of the main compartments but not upon entry into the interconnecting neutral compartment (NCP; Extended Data <a href="#fig8-1">Fig. 8-1 <i>Ii</i> </a>). Once again, Vglut2-Cre mice preferred to spend time in the area lacking stimulation [effect of compartment <i>F</i> <sub>(2,8)</sub> = 70.9, <i>p</i> &amp;lt; 0.001 and day × compartment interaction <i>F</i> <sub>(4,16)</sub> = 6.90 <i>p</i> = 0.002, ** <i>p</i> &amp;lt; 0.01, *** <i>p</i> &amp;lt; 0.001 neutral vs paired compartments (Extended Data <a href="#fig8-1">Fig. 8-1 <i>Iii</i> </a>); effect of compartment <i>F</i> <sub>(2,2)</sub> = 54.2, <i>p</i> = 0.018, * <i>p</i> &amp;lt; 0.05 neutral vs paired compartments (Extended Data <a href="#fig8-1">Fig. 8-1 <i>Iiii</i> </a>)]. In the current setups, optogenetic VTA-stimulation of DAT-Cre mice thus leads to place preference while same stimulation of Vglut2-Cre mice causes an avoidance to any compartment that activates photostimulation within the VTA. </p> <div class="local-data" title="local-data"> <a href="https://dx.doi.org/10.1523/ENEURO.0066-19.2019.f8-1">10.1523/ENEURO.0066-19.2019.f8-1</a> <span class="label" tagx="label" title="label">Extended Data Figure 8-1</span>  <p>Complementary data from behavioral optogenetics. DAT-Cre-negative mice injected with <i>DIO-ChR2-eYFP</i> ( <b> <i>A</i> </b>) and DAT-Cre-positive mice injected with <i>DIO-eYFP</i> mice ( <b> <i>B</i> </b>) did not show preference to the light paired compartment so their results were pooled together ( <b> <i>C</i> </b>). <b> <i>D</i> </b>, Schematic illustration of optical fiber placement for mice analyzed in RT-PP analysis. <b> <i>E–G</i> </b>, RT-PP testing under high power (20 mW, 5 ms, 20 Hz) stimulation for DAT-Cre/ChR2 ( <b> <i>E</i> </b>), Calb2-Cre/ChR2 ( <b> <i>F</i> </b>) and NEX-Cre/ChR2 ( <b> <i>G</i> </b>). <b> <i>H</i> </b>, RT-PP results for bilaterally injected NEX-Cre/ChR2 mice. <b> <i>A–C</i> </b>, <b> <i>E–H</i> </b>, Graphs: * <i>p</i> &amp;lt; 0.05, ** <i>p</i> &amp;lt; 0.01, *** <i>p</i> &amp;lt; 0.001 paired versus unpaired; bar graphs: * <i>p</i> &amp;lt; 0.05, *** <i>p</i> &amp;lt; 0.001 versus paired, # <i>p</i> &amp;lt; 0.05, ### <i>p</i> &amp;lt; 0.001 versus unpaired. <b> <i>I</i> </b>, Vglut2-Cre/ChR2 tested in the NCP test. <b> <i>Ii</i> </b>, Schematic of the experimental setup. <b> <i>Iii–Iiii</i> </b>, Mice spent significantly more time in the light-unpaired neutral compartment (* <i>p</i> &amp;lt; 0.05, ** <i>p</i> &amp;lt; 0.01, *** <i>p</i> &amp;lt; 0.001 neutral vs Paired1 and Paired2 compartments). DAT-Cre negative <i>N</i> = 3, DAT-Cre/eYFP = 3, DAT-Cre/ChR2 high power <i>N</i> = 4, Calb2-Cre/ChR2 high power <i>N</i> = 7, NEX-Cre/ChR2 <i>N</i> = 4, NEX-Cre/ChR2 bilateral <i>N</i> = 4. NS, non-significant. Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">Figure 8-1, EPS file</span>. </p>  </div> </div> <div class="analysisofrt-ppandcrincalb2-creandnex-cremice" title="sec"> <div class="title" tagx="title" title="title">
Analysis of RT-PP and CR in Calb2-Cre and NEX-Cre mice</div> <p>Using these behaviors as references and for comparison in the place preference setup, Calb2-Cre mice showed a strikingly different behavior. Neither preference nor avoidance was detected but instead, mice spent equal amount of time in both main compartments [effect of compartment <i>F</i> <sub>(2,12)</sub> = 27, <i>p</i> &amp;lt; 0.001, day × compartment interaction, <i>F</i> <sub>(12,72)</sub> = 1.45, <i>p</i> = 0.163 and no differences between paired versus unpaired across days ( <a href="#F8">Fig. 8 <i>D</i> </a>, left); effect of compartment <i>F</i> <sub>(2,6)</sub> = 90.1, <i>p</i> &amp;lt; 0.001, no differences between paired vs unpaired, *** <i>p</i> &amp;lt; 0.001, ### <i>p</i> &amp;lt; 0.001 neutral vs paired and unpaired, respectively ( <a href="#F8">Fig. 8 <i>D</i> </a>, right)]. When analyzing whether optogenetic activation of NEX-Cre VTA neurons would cause place preference, a significant behavioral response toward the photostimulation was observed (effect of compartment <i>F</i> <sub>(2,8)</sub> = 76.8, <i>p</i> &amp;lt; 0.001, day × compartment interaction, <i>F</i> <sub>(12,48)</sub> = 4.63, <i>p</i> &amp;lt; 0.001; <a href="#F8">Fig. 8 <i>E</i> </a>, left). NEX-Cre mice responded weakly to VTA-photostimulation on days 3 and 4, but on days 6 and 7, NEX-Cre mice preferred the light-paired compartment (* <i>p</i> = 0.02, *** <i>p</i> &amp;lt; 0.001 paired vs unpaired). However, no CR was observed on either day 5 or 8 ( <a href="#F8">Fig. 8 <i>E</i> </a>, left). By averaging the results of all four RT-PP days, NEX-Cre mice showed a significant preference for paired over unpaired and neutral compartments (effect of compartment <i>F</i> <sub>(2,6)</sub> = 39.7, <i>p</i> &amp;lt; 0.001 * <i>p</i> = 0.013 *** <i>p</i> &amp;lt; 0.001 vs paired, ## <i>p</i> = 0.008 neutral vs unpaired; <a href="#F8">Fig. 8 <i>E</i> </a>, right). </p> </div> <div class="analysisofrt-ppandcrindat-cre,calb2-cre,andnex-cremiceusinghigherpowerstimulation" title="sec"> <div class="title" tagx="title" title="title">
Analysis of RT-PP and CR in DAT-Cre, Calb2-Cre, and NEX-Cre mice using higher power stimulation </div> <p>While the result above demonstrated that activation of NEX-Cre VTA neurons induced place preference behavior, higher power stimulation (5-ms pulse width, 20 Hz, 20 mW) was subsequently used to test whether these laser parameters would boost the observed behavioral response. Again, DAT-Cre mice showed a strong preference for the light-paired chamber [effect of compartment <i>F</i> <sub>(2,6)</sub> = 105, <i>p</i> &amp;lt; 0.001, day × compartment interaction <i>F</i> <sub>(12,36)</sub> = 22.6, <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 paired vs unpaired (Extended Data <a href="#fig8-1">Fig. 8-1 <i>E</i> </a>, left); effect of compartment <i>F</i> <sub>(2,6)</sub> = 404, <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 unpaired and neutral vs paired (Extended Data <a href="#fig8-1">Fig. 8-1 <i>E</i> </a>, right)], while Calb2-Cre mice continued not to respond to the VTA photostimulation [effect of compartment <i>F</i> <sub>(2,12)</sub> = 12.5, <i>p</i> = 0.001, day × compartment interaction <i>F</i> <sub>(12,72)</sub> = 0.469, <i>p</i> = 0.927 (Extended Data <a href="#fig8-1">Fig. 8-1 <i>F</i> </a>, left); effect of compartment <i>F</i> <sub>(2,6)</sub> = 47.3, <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 and ### <i>p</i> &amp;lt; 0.001 neutral versus unpaired and paired (Extended Data <a href="#fig8-1">Fig. 8-1 <i>F</i> </a>, right)]. In contrast, NEX-Cre mice showed a significant preference also with this higher power stimulation [effect of compartment <i>F</i> <sub>(2,6)</sub> = 48.3, <i>p</i> &amp;lt; 0.001, day × compartment interaction <i>F</i> <sub>(12,36)</sub> = 8.58, <i>p</i> &amp;lt; 0.001, ** <i>p</i> = 0.003, *** <i>p</i> &amp;lt; 0.001 paired vs unpaired (Extended Data <a href="#fig8-1">Fig. 8-1 <i>G</i> </a>, left); effect of compartment <i>F</i> <sub>(2,6)</sub> = 178, <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 paired vs unpaired and neutral, ### <i>p</i> &amp;lt; 0.001 unpaired vs neutral (Extended Data <a href="#fig8-1">Fig. 8-1 <i>G</i> </a>, right)]. Finally, to further validate the role of NEX-Cre VTA neurons in place preference, a subset of NEX-Cre mice was bilaterally injected with <i>DIO-ChR2-eYFP</i> and tested in the same protocol under normal and high-power light stimulation (Extended Data <a href="#fig8-1">Fig. 8-1 <i>H</i> </a>; <a href="#F8">Fig. 8 <i>F</i> </a>). Mice preferred the light-paired side over the unpaired under both conditions, and high-power stimulation accentuated the preference toward the light paired compartment which reached a 3-fold increase compared to the unpaired [standard power: effect of compartment <i>F</i> <sub>(2,6)</sub> = 43.3, <i>p</i> &amp;lt; 0.001, day × compartment interaction <i>F</i> <sub>(12,36)</sub> = 2.13, <i>p</i> = 0.04 (Extended Data <a href="#fig8-1">Fig. 8-1 <i>H</i> </a>, left); effect of compartment <i>F</i> <sub>(2,6)</sub> = 331, <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 vs paired ### <i>p</i> &amp;lt; 0.001 vs unpaired (Extended Data <a href="#fig8-1">Fig. 8-1 <i>H</i> </a>, right); high power: effect of compartment <i>F</i> <sub>(2,6)</sub> = 36.5, <i>p</i> &amp;lt; 0.001, day × compartment interaction <i>F</i> <sub>(12,36)</sub> = 9.03, <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 paired vs unpaired ( <a href="#F8">Fig. 8 <i>F</i> </a>, left); effect of compartment <i>F</i> <sub>(2,6)</sub> = 106 <i>p</i> &amp;lt; 0.001, *** <i>p</i> &amp;lt; 0.001 paired vs unpaired and neutral, # <i>p</i> = 0.011 unpaired vs neutral ( <a href="#F8">Fig. 8 <i>F</i> </a>, right)]. However, unlike DAT-Cre mice, NEX-Cre mice did not show any CR in any RT-PP experiment [day 5 paired vs unpaired <i>p</i> &amp;gt; 0.999, day 8 paired vs unpaired <i>p</i> = 0.937 ( <a href="#F8">Fig. 8 <i>E</i> </a>); day 5 paired vs unpaired <i>p</i> &amp;gt; 0.999, day 8 paired vs unpaired <i>p</i> = 0.989 ( <a href="#F8">Fig. 8 <i>F</i> </a>)]. </p> </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>It is well established that the VTA is involved in a range of functions, including behavioral reinforcement, reward, aversion, motivation and incentive salience ( <a href="#B34">Morales and Margolis, 2017</a>). However, an area of active investigation is how the VTA can possess the ability to contribute to all of these diverse functions, some even contrasting. It is now becoming increasingly clear that functional diversity within the mDA system might be matched by molecular and anatomic heterogeneity ( <a href="#B29">Lammel et al., 2011</a>, <a href="#B30">2012</a>; <a href="#B43">Roeper, 2013</a>; <a href="#B34">Morales and Margolis, 2017</a>; <a href="#B39">Poulin et al., 2018</a>). Why is this important? The possibility to determine the exact identity of neurons that contribute to a particular behavior opens up entirely new perspectives in the opportunity to to selectively target only those neurons that contribute to clinical symptoms without causing side-effects by affecting adjacent neuronal populations. In this study, we used Cre-driven mouse genetics and optogenetics to begin to disentangle the contribution of the newly described NeuroD6 VTA subtype ( <a href="#B50">Viereckel et al., 2016</a>; <a href="#B25">Khan et al., 2017</a>; <a href="#B27">Kramer et al., 2018</a>) in reward-related behaviors commonly ascribed to the VTA DA system. The main finding of our study is that despite their restricted number, NeuroD6 VTA neurons contribute to psychostimulant-induced hyperlocomotion and that their activation induces place preference behavior. </p> <div class="neurod6vtaneuronsrepresentamodestneuronalpopulationwithinthevtawithmolecularcapacityfordopaminergicandglutamatergicneurotransmission" title="sec"> <div class="title" tagx="title" title="title">
NeuroD6 VTA neurons represent a modest neuronal population within the VTA with molecular capacity for dopaminergic and glutamatergic neurotransmission </div> <p>In the current study, we showed that NeuroD6 VTA neurons constitute a modest proportion (circa 12%) of all VTA neurons expressing the gene encoding TH within the PN, PIF, PBP, IF, and RLi subareas. Within these VTA subareas, all NeuroD6-positive neurons were positive for both Th and Dat mRNAs, markers of dopaminergic neurons. In addition, while no or very few NeuroD6 neurons were positive for Viaat mRNA, a marker of GABAergic neurons, 12% of the NeuroD6/Th double-positive neurons within the VTA were positive for Vglut2 mRNA, suggesting a capacity for dual dopaminergic/glutamatergic neurotransmission. Indeed, DA/glutamate co-release has in several studies been shown as a property of certain mDA neurons where it has been proposed to play a role in reward-related behavior reinforced by DA (for recent review, see <a href="#B48">Trudeau and El Mestikawy, 2018</a>). The identification of co-labeling of NeuroD6 mRNA with Th, Dat and Vglut2 mRNAs within distinct neurons was partly in accordance with our analysis of a NEX-Cre transgenic mouse line, implemented here to achieve manipulation of the NeuroD6 VTA neurons, which identified substantial co-localization between NEX-Cre-driven reporter gene expression (YFP) and TH immunofluorescence. However, lack of TH/YFP co-localization was also identified. The findings showing that the majority of NeuroD6 VTA neurons expressed DA markers were in accordance with our electrophysiological data in which optogenetic VTA stimulation of NEX-Cre neurons enabled the identification of DA release, as further discussed below. Further, optogenetic stimulation also gave rise to EPSCs of glutamatergic nature, while no GABAergic currents were detected, in agreement with the co-localization of NeuroD6 mRNA with Vglut2 mRNA but lack of significant co-localization with Viaat mRNA. </p> <p>In the context of transgenic mice, it is noteworthy that our result showing non-complete overlap between NEX-Cre-driven reporter gene expression and TH, which contrasts the parallel finding that all VTA neurons positive for endogenous NeuroD6 mRNA also label for Th mRNA, are in accordance with a recent study in which a substantial number of non-dopaminergic NEX-Cre VTA neurons were identified ( <a href="#B27">Kramer et al., 2018</a>). Collectively, these findings propose that interpretation of VTA-data originating from the current NEX-Cre mouse line should be considered with awareness of complex downstream neurocircuitry. Further, as extensively discussed in the literature, regulatory promoters implemented experimentally to drive Cre expression may give rise to transient and/or ectopic Cre activity that fails to mimic endogenous gene expression due to gene regulatory events, not least during developmental phases. Indeed, patterns of ectopic Cre activity have been described for other transgenic mouse lines, including DAT-Cre and TH-Cre transgenic mouse lines commonly implemented for the study of DA neurons ( <a href="#B32">Lindeberg et al., 2004</a>; <a href="#B45">Stamatakis et al., 2013</a>; <a href="#B31">Lammel et al., 2015</a>; <a href="#B36">Nordenankar et al., 2015</a>; <a href="#B40">Pupe and Wallén-Mackenzie, 2015</a>; <a href="#B47">Stuber et al., 2015</a>; <a href="#B34">Morales and Margolis, 2017</a>). While the current NEX-Cre transgenic line has been thoroughly validated recently for the study of VTA neurons ( <a href="#B25">Khan et al., 2017</a>; <a href="#B27">Kramer et al., 2018</a>), to direct selectivity to VTA DA neurons, we here implemented a conditional genetic approach to specifically abrogate vesicular packaging of DA in NEX-Cre neurons. Further, we used optogenetically driven neuronal activation to study effects upon direct stimulation of NEX-Cre VTA neurons. </p> </div> <div class="targetingofthevmat2geneinnex-crevtadaneuronsallowedidentificationofaroleinpsychostimulant-mediatedresponse" title="sec"> <div class="title" tagx="title" title="title">
Targeting of the <i>Vmat2</i> gene in NEX-Cre VTA DA neurons allowed identification of a role in psychostimulant-mediated response </div> <p>To enable the study of how reward-related behaviors classically associated with the mDA system would be affected if the NEX-Cre DA neuron subtype lost its ability for dopaminergic function, a conditional gene-targeting approach was implemented in which VMAT2 was ablated specifically from NEX-Cre neurons. Since we could show that NeuroD6 and Vmat2 mRNAs only co-localized within the VTA, no other monoaminergic population should suffer from loss of VMAT2 by this approach. Indeed, the results confirmed that Vmat2 mRNA was selectively knocked out within the VTA, while all other monoaminergic neurons maintained normal Vmat2 mRNA. Thus, the <i>Vmat2 <sup>lox/lox;NEX-Cre</sup> </i> mouse line forms a new mouse model of DA-release deficiency from a restricted group of VTA DA neurons characterized by NeuroD6 promoter activity. Based on the importance of mDA system in processing natural and drug rewards ( <a href="#B24">Kalivas et al., 1992</a>; <a href="#B9">Di Chiara and Bassareo, 2007</a>; <a href="#B20">Ikemoto, 2007</a>; <a href="#B1">Baik, 2013</a>; <a href="#B42">Robinson and Berridge, 1993</a>), we addressed the behavioral responses of <i>Vmat2 <sup>lox/lox;NEX-Cre-tg</sup> </i> cKO mice and <i>Vmat2 <sup>lox/lox;NEX-Cre-wt</sup> </i> control mice to sugar, ethanol and the psychostimulants amphetamine and cocaine. cKO mice displayed higher locomotor activation on repeated administration of psychostimulants than control mice. In contrast, sugar preference and CPP to cocaine and amphetamine were similar between cKO and control mice, and both genotype groups showed a preference for increasing dose of ethanol, albeit in different patterns. </p> <p>While acute administration of cocaine failed to cause differences in locomotor responses between cKO and control mice, repeated administration caused exaggerated locomotor behavior in cKO mice when measured in the CPP paradigm. In contrast, with repeated amphetamine injections, the locomotor response was elevated above control levels in the open field, but not in the CPP. The tests implemented were designed to study different behavioral parameters, and results obtained in different setups and by different drugs are therefore not directly comparable. What may seem as apparent discrepancies might be related to several different properties. Firstly, the size and properties of the test environment were substantially different between setups. The open field test took place in an environment that resembled the home cage. Locomotion was recorded during the conditioning phase when the mice were confined to a much smaller compartment with specific patterns and no bedding. Secondly, the injection regime differed between tests. In the open field, mice received acute injections of cocaine or were sensitized to amphetamine by receiving daily injections after a 30-min habituation period. In contrast, in the CPP experiment, the mice received in total four injections of the drug in two non-consecutive days without any previous habituation period. Finally, the recording period was shorter in the CPP compared to the open field (30 min vs 1.5 h), a parameter that could mask the long-lasting effects of amphetamine on locomotion. Further, the observation of heightened, rather than reduced, psychostimulant-induced locomotion might seem counter-intuitive: Loss of VMAT2 should lead to decreased packaging and release of DA which might be expected to cause reduced locomotion compared to control levels. However, the results obtained from our spatially selective cKO mice are similar to the heightened amphetamine-induced hyperlocomotion observed in a study of mice heterozygous for <i>Vmat2</i> in all DAT-Cre neurons ( <a href="#B23">Isingrini et al., 2016</a>). Thus, lowering the level of VMAT2 throughout all DAT-Cre neurons or ablating it within the NEX-Cre VTA DA population give rise to similar behavioral consequences. Further analyses focused around VMAT2 in psychomotor behavior will be necessary to pin-point this matter, however, developmental adaptations, a common feature of KO strategies induced during embryonal development, may underlie the heightened locomotor response. </p> </div> <div class="striataloptogeneticstimulationinnex-cremiceinduceddareleaseandglutamatergicepscs" title="sec"> <div class="title" tagx="title" title="title">
Striatal optogenetic stimulation in NEX-Cre mice induced DA release and glutamatergic EPSCs </div> <p>Complementary to the cKO approach, we used optogenetics-based experiments in which the NEX-Cre VTA population could be directly stimulated. This type of manipulation provides high spatial and temporal resolution ( <a href="#B8">Deisseroth, 2015</a>) and thus has the advantage of enabling selective stimulation of Cre-driven neurons in real time with the benefit of directly pin-pointing the role of molecularly defined neurons in measurable behavior. By analysis of optogenetic reporter gene (eYFP) expression upon injection into the VTA of NEX-Cre mice, we showed that NeuroD6 VTA neurons projected mainly to the NAcSh of the striatal complex, with substantially lower density than observed upon similar injection in DAT-Cre and Vglut2-Cre mice used here as controls ( <a href="#B46">Stuber et al., 2010</a>; <a href="#B17">Hnasko et al., 2012</a>; <a href="#B37">Pascoli et al., 2015</a>; <a href="#B41">Qi et al., 2016</a>; <a href="#B53">Yoo et al., 2016</a>). NEX-Cre VTA projections also reached several additional areas, but with even lower density than seen in the NAcSh, including the OT, medial habenula and ventral pallidum. In accordance with the co-localization of eYFP with TH immunoreactivity, we could verify that NEX-Cre VTA neurons released DA in both the NAcSh and OT upon striatal optogenetic stimulation. Although the levels were lower than those observed upon similar stimulation of DAT-Cre-positive VTA neurons, they were significantly higher than those observed in control experiments, demonstrating that the NEX-Cre VTA population indeed releases measurable amounts of DA in their target areas. To investigate whether the TH-negative cellular population, present most profoundly in the medial VTA, was of glutamatergic or GABAergic nature, patch-clamp electrophysiology was performed which showed that optogenetic stimulation of NEX-Cre terminals induced EPSCs, but not IPSCs, in NAcSh, thus verifying glutamatergic neurotransmission. While glutamatergic postsynaptic currents were evidently a result of the optogenetic stimulation of NEX-Cre VTA neurons, it remains to be established if the rare endogenous NeuroD6+/Th+/NeuroD6+ triple-positive neurons observed in our histologic analysis are sufficiently potent to drive a similar postsynaptic response in the natural situation, that is, upon excitation of the NeuroD6 VTA neurons in a non-transgenic context. Finally, the current setup did not allow us to conclude if the EPSCs were of monosynaptic or polysynaptic nature. The short onset of EPSCs was suggestive of monosynaptic transmission, however, electrophysiological approaches combined with pharmacological agents will be necessary to fully define the signaling properties. </p> </div> <div class="optogeneticstimulationofnex-crevtaneuronsrevealsaroleinplacepreferencebehavior" title="sec"> <div class="title" tagx="title" title="title">
Optogenetic stimulation of NEX-Cre VTA neurons reveals a role in place preference behavior </div> <p>Optogenetic stimulation of the mDA system of TH-Cre and DAT-Cre mice has been demonstrated to potently induce DA release and real time place preference ( <a href="#B49">Tsai et al., 2009</a>; <a href="#B46">Stuber et al., 2010</a>; <a href="#B53">Yoo et al., 2016</a>). The same type of activation of VTA in Vglut2-Cre mice has been described to cause postsynaptic glutamatergic currents and to induce either place preference or place avoidance, depending on stimulation parameters ( <a href="#B17">Hnasko et al., 2012</a>; <a href="#B51">Wang et al., 2015</a>; <a href="#B41">Qi et al., 2016</a>; <a href="#B53">Yoo et al., 2016</a>). Using DAT-Cre and Vglut2-Cre mice as references, we could show here that optogenetic stimulation within the VTA of NEX-Cre mice induced a significant preference for the light-paired compartment. The magnitude of the preference observed was, however, smaller in NEX-Cre than in DAT-Cre mice. This difference is likely related to the substantially smaller population of VTA neurons activated upon photostimulation in the NEX-Cre compared to DAT-Cre VTA and the different projection patterns of these neuronal populations. This is supported by the analysis of YFP-positive fibers, which differ substantially between DAT-Cre and NEX-Cre mice. VTA-injection of ChR2-YFP in DAT-Cre mice results in strong YFP-fluorescence in all innervation areas ascribed to the mDA system. In contrast, the same injection into the VTA of NEX-Cre mice results in substantially lower YFP-derived fluorescence in the VTA and sparse fluorescence in target areas. </p> <p>Despite smaller magnitude, the ability of NEX-Cre VTA neurons to induce real time place preference is an important finding as it demonstrates the possibility of identifying spatially restricted groups of VTA neurons that are sufficient to induce a measurable behavior. Further arguing for the importance of this result, the optogenetically induced preference behavior displayed by NEX-Cre mice was strengthened by viral injections in bilateral, rather than unilateral, manner as well as by increased laser power. The results of these experimental manipulations suggest that the enhanced recruitment of NEX-Cre neurons strengthened the behavioral output. While additional studies will be required to completely disentangle the behavioral role of NeuroD6 VTA neurons, the current optogenetics-based setup already enabled us to demonstrate that VTA activation in NEX-Cre mice could induce place preference in real-time, but that it failed to result in CR, defined as significant place preference even in absence of actual optogenetic stimulation. This contrasts the strong CR observed in the DAT-Cre mice, and hence, activation of VTA populations in NEX-Cre mice and DAT-Cre mice differ in more than one parameter: Magnitude in real time place preference and presence of a detectable CR. In contrast to the preference behavior displayed by NEX-Cre and DAT-Cre mice, optogenetic stimulation of VTA Vglut2-Cre neurons led to real time place avoidance defined here as reduced time spent in the stimulation-paired compartment. This result is consistent with a recent study which found that real time avoidance coincided with a frequency-dependent increase in entries to the light-paired compartment and robust self-stimulation in an operant task ( <a href="#B53">Yoo et al., 2016</a>). In contrast, another study found that photostimulation of Vglut2-Cre neurons in VTA induced modest real time place preference and self-stimulation ( <a href="#B51">Wang et al., 2015</a>). These data show that the behavioral effects of VTA glutamate neuron stimulation are sensitive to the task, including the design of the apparatus and stimulus parameters. In this context, it is noteworthy that VTA neurons of the NEX-Cre transgenic mouse line, with their mixture of dopaminergic and glutamatergic signaling properties, might have shown lower level of place preference than DAT-Cre mice not only due to the smaller number of neurons and sparser projections, but also as their activation might have caused a glutamate-mediated avoidance behavior that counterbalanced the behavioral preference for light stimulation. </p> </div> <div class="neurod6andcalb2mrnasshowpartialoverlap,butnex-creandcalb2-crevtaneuronshavedistinctprojectionsandroleinbehavior" title="sec"> <div class="title" tagx="title" title="title">
NeuroD6 and Calb2 mRNAs show partial overlap, but NEX-Cre and Calb2-Cre VTA neurons have distinct projections and role in behavior </div> <p>Parallel to the focus on NeuroD6 VTA neurons in neurocircuitry and behavioral regulation, our histological analysis enabled us to identify a degree of co-localization between NeuroD6 and Calb2 mRNAs. While NeuroD6 mRNA was uniquely found in the VTA and excluded from the SNc, Calb2 mRNA was found distributed throughout these dopaminergic areas. However, histological analysis showed a degree of co-localization between NeuroD6 and Calb2 mRNAs, a finding which adds to the recent molecular description of NeuroD6 as co-localized with gastrin-releasing peptide (GRP) and additional markers ( <a href="#B25">Khan et al., 2017</a>; <a href="#B27">Kramer et al., 2018</a>; <a href="#B39">Poulin et al., 2018</a>). Beyond the partial co-localization of NeuroD6 and Calb2 mRNAs, the results demonstrate that Calb2 VTA neurons constitute a substantially larger proportion within the mDA population, show considerable expression of the gene encoding VIAAT, and are present in the SNc, an area devoid of NeuroD6 neurons. Our neurophysiological circuitry analyses of Calb2-Cre mice showed that Calb2-Cre VTA neurons belong to the category of VTA/SNc neurons that projects to the OT where their stimulation resulted in DA release and glutamatergic postsynaptic currents. While it was recently described that activation of dopaminergic fibers from VTA to the medial OT can induce place preference in DAT-Cre mice ( <a href="#B54">Zhang et al., 2017</a>), a similar response was not observed here on Calb2-Cre VTA stimulation. These differences might be explained by the difference in density of the innervation patterns in the OT between the DAT-Cre and Calb2-Cre mice. The difference in preference behavior between NeuroD6-Cre and Calb2-Cre mice shows that distinct VTA neurocircuitry is crucial for the behavioral output. </p> </div> <div class="unravelingthebehavioralrolesofneurod6vtaneuronsstandstobenefitcurrentdecodingofvta-relateddisorders" title="sec"> <div class="title" tagx="title" title="title">
Unraveling the behavioral roles of NeuroD6 VTA neurons stands to benefit current decoding of VTA-related disorders </div> <p>The behavioral complexity mediated by the VTA is implicated in a range of neuropsychiatric conditions including substance use disorder, schizophrenia, and ADHD for which clinical interventions based on increasing, decreasing, stabilizing or modulating the mDA system are commonly prescribed. In addition, since VTA DA neurons are less susceptible to degeneration in PD than SNc DA neurons, molecular differences are intensively searched for. GRP, in several studies identified as a marker for VTA DA neurons ( <a href="#B7">Chung et al., 2005</a>; <a href="#B16">Greene et al., 2005</a>; <a href="#B28">La Manno et al., 2016</a>; <a href="#B50">Viereckel et al., 2016</a>) was recently shown to co-localize with NeuroD6 ( <a href="#B27">Kramer et al., 2018</a>). Several lines of evidence suggest that a discrete NeuroD6/GRP VTA subtype should be of specific interest. Overexpression of the gene encoding GRP increased the survival rate of cultured DA neurons in a parkinsonian experimental model ( <a href="#B7">Chung et al., 2005</a>) and GRP-positive mDA neurons remain in biopsies from deceased PD patients ( <a href="#B50">Viereckel et al., 2016</a>). Further, NeuroD6 increases neuronal survival in a toxin model of PD ( <a href="#B27">Kramer et al., 2018</a>). The NeuroD6/GRP VTA subtype might thereby possess resistance to PD. Our current results show that, despite their modest representation within the VTA, NeuroD6-expressing VTA neurons are implicated in distinct aspects of reward-related behavior. Their resistance to PD may thus contribute to the cause of behavioral dysfunction observed in the non-motor symptom domain of PD, including treatment-induced complications that resemble aspects of neuropsychiatric diseases, such as behavioral addictions ( <a href="#B6">Cenci et al., 2015</a>). </p> <p>Current molecular profiling of DA neuron subtypes should prove valuable for prospects of selective treatment in conditions related to VTA dysfunction. Of essence to achieve such selectivity is the systematic decoding of the explicit behavioral roles mediated by distinct VTA neurons. In this study, we initiated such analysis and now propose that NeuroD6 VTA neurons are of particular interest for further analysis of motivated and addictive behavior as they are here implicated in reward-related behavior measured as real time place preference and as their controlled dysregulation alters the responsiveness to psychostimulants. Our findings should prove useful for future investigations aimed at advancing the knowledge of VTA neurocircuitry in healthy conditions and in neuropsychiatric illness implicating the VTA. </p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>Acknowledgements: We thank Professors Lars Olson and Nils-Göran Larsson (Karolinska Institutet, Sweden) for the DAT-Cre transgenic mice, Professor Ole Kiehn and Dr. Lotta Borgius (Karolinska Institutet, Sweden and University of Copenhagen, Denmark) for the Vglut2-Cre mice, and Professor Klaus-Armin Nave and Dr. Sandra Goebbels (Max Planck Institute of Experimental Medicine, Göttingen, Germany) for the NEX-Cre mice. We also thank BioVis (Uppsala University) and Marie-Laure Niepon at the Image platform at Institute de la Vision (Paris, France) for slide scanning, as well as previous PhD student Dr. Nadine Schweizer and all current members of the Mackenzie lab for constructive feedback. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Baik</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Dopamine signaling in reward-related behaviors</span>. <span class="source" tagx="source" title="source">Front Neural Circuits</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">16</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23440175">23440175</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Beier</span> <span class="given-names" tagx="given-names" title="given-names">KT</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Steinberg</span> <span class="given-names" tagx="given-names" title="given-names">EE</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Deloach</span> <span class="given-names" tagx="given-names" title="given-names">KE</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Miyamichi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Schwarz</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">XJ</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kremer</span> <span class="given-names" tagx="given-names" title="given-names">EJ</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Malenka</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Circuit architecture of VTA dopamine neurons revealed by systematic input-output mapping</span>. <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">162</span>: <span class="fpage" tagx="fpage" title="fpage">622</span>– <span class="lpage" tagx="lpage" title="lpage">634</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2015.07.015">10.1016/j.cell.2015.07.015</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26232228">26232228</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Björklund</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Dunnett</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Dopamine neuron systems in the brain: an update</span>. <span class="source" tagx="source" title="source">Trends Neurosci</span> <span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">194</span>– <span class="lpage" tagx="lpage" title="lpage">202</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tins.2007.03.006">10.1016/j.tins.2007.03.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17408759">17408759</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Borgius</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Restrepo</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Leao</span> <span class="given-names" tagx="given-names" title="given-names">RN</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Saleh</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kiehn</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">A transgenic mouse line for molecular genetic analysis of excitatory glutamatergic neurons </span>. <span class="source" tagx="source" title="source">Mol Cell Neurosci</span> <span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">245</span>– <span class="lpage" tagx="lpage" title="lpage">257</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.mcn.2010.06.016">10.1016/j.mcn.2010.06.016</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20600924">20600924</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Brichta</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Greengard</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Molecular determinants of selective dopaminergic vulnerability in Parkinson’s disease: an update </span>. <span class="source" tagx="source" title="source">Front Neuroanat</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">152</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25565977">25565977</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Cenci</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Francardo</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">O’Sullivan</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lindgren</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Rodent models of impulsive-compulsive behaviors in Parkinson’s disease: how far have we reached? </span> <span class="source" tagx="source" title="source">Neurobiol Dis</span> <span class="volume" tagx="volume" title="volume">82</span>: <span class="fpage" tagx="fpage" title="fpage">561</span>– <span class="lpage" tagx="lpage" title="lpage">573</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.nbd.2015.08.026">10.1016/j.nbd.2015.08.026</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26325219">26325219</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chung</span> <span class="given-names" tagx="given-names" title="given-names">CY</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Seo</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Sonntag</span> <span class="given-names" tagx="given-names" title="given-names">KC</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Brooks</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Isacson</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Cell type-specific gene expression of midbrain dopaminergic neurons reveals molecules involved in their vulnerability and protection </span>. <span class="source" tagx="source" title="source">Hum Mol Genet</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">1709</span>– <span class="lpage" tagx="lpage" title="lpage">1725</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/hmg/ddi178">10.1093/hmg/ddi178</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15888489">15888489</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Deisseroth</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Optogenetics: 10 years of microbial opsins in neuroscience</span>. <span class="source" tagx="source" title="source">Nat Neurosci</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">1213</span>– <span class="lpage" tagx="lpage" title="lpage">1225</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nn.4091">10.1038/nn.4091</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26308982">26308982</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Di Chiara</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bassareo</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Reward system and addiction: what dopamine does and doesn’t do</span>. <span class="source" tagx="source" title="source">Curr Opin Pharmacol</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">69</span>– <span class="lpage" tagx="lpage" title="lpage">76</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.coph.2006.11.003">10.1016/j.coph.2006.11.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17174602">17174602</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Divac</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Prostran</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Jakovcevski</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Cerovac</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Second-generation antipsychotics and extrapyramidal adverse effects</span>. <span class="source" tagx="source" title="source">Biomed Res Int</span> <span class="volume" tagx="volume" title="volume">2014</span>: <span class="fpage" tagx="fpage" title="fpage">656370</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1155/2014/656370">10.1155/2014/656370</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24995318">24995318</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ekstrand</span> <span class="given-names" tagx="given-names" title="given-names">MI</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Terzioglu</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Galter</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hofstetter</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lindqvist</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Thams</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bergstrand</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hansson</span> <span class="given-names" tagx="given-names" title="given-names">FS</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Trifunovic</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hoffer</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Cullheim</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mohammed</span> <span class="given-names" tagx="given-names" title="given-names">AH</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Olson</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Larsson</span> <span class="given-names" tagx="given-names" title="given-names">NG</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Progressive parkinsonism in mice with respiratory-chain-deficient dopamine neurons</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">104</span>: <span class="fpage" tagx="fpage" title="fpage">1325</span>– <span class="lpage" tagx="lpage" title="lpage">1330</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0605208103">10.1073/pnas.0605208103</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17227870">17227870</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Faget</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Osakada</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Duan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ressler</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">AB</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Proudfoot</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Yoo</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Callaway</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hnasko</span> <span class="given-names" tagx="given-names" title="given-names">TS</span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Afferent inputs to neurotransmitter-defined cell types in the ventral tegmental area</span>. <span class="source" tagx="source" title="source">Cell Rep</span> <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">2796</span>– <span class="lpage" tagx="lpage" title="lpage">2808</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.celrep.2016.05.057">10.1016/j.celrep.2016.05.057</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27292633">27292633</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Franklin</span> <span class="given-names" tagx="given-names" title="given-names">KBJ</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Paxinos</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="source" tagx="source" title="source">The mouse brain in stereotaxic coordinates</span>, <span class="edition" tagx="edition" title="edition">Ed 3</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Boston</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press</span>. </span> </li> <li tag="ref"> <a name="B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">YH</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Halliday</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Rusznák</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Watson</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Paxinos</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">A cytoarchitectonic and chemoarchitectonic analysis of the dopamine cell groups in the substantia nigra, ventral tegmental area, and retrorubral field in the mouse </span>. <span class="source" tagx="source" title="source">Brain Struct Funct</span> <span class="volume" tagx="volume" title="volume">217</span>: <span class="fpage" tagx="fpage" title="fpage">591</span>– <span class="lpage" tagx="lpage" title="lpage">612</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00429-011-0349-2">10.1007/s00429-011-0349-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21935672">21935672</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Goebbels</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bormuth</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bode</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hermanson</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Schwab</span> <span class="given-names" tagx="given-names" title="given-names">MH</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Nave</span> <span class="given-names" tagx="given-names" title="given-names">K-A</span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Genetic targeting of principal neurons in neocortex and hippocampus of NEX-Cre mice</span>. <span class="source" tagx="source" title="source">Genesis</span> <span class="volume" tagx="volume" title="volume">44</span>: <span class="fpage" tagx="fpage" title="fpage">611</span>– <span class="lpage" tagx="lpage" title="lpage">621</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1002/dvg.20256">10.1002/dvg.20256</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17146780">17146780</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Greene</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Dingledine</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Greenamyre</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Gene expression profiling of rat midbrain dopamine neurons: implications for selective vulnerability in parkinsonism </span>. <span class="source" tagx="source" title="source">Neurobiol Dis</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">19</span>– <span class="lpage" tagx="lpage" title="lpage">31</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.nbd.2004.10.003">10.1016/j.nbd.2004.10.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15649693">15649693</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hnasko</span> <span class="given-names" tagx="given-names" title="given-names">TS</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hjelmstad</span> <span class="given-names" tagx="given-names" title="given-names">GO</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Fields</span> <span class="given-names" tagx="given-names" title="given-names">HL</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Edwards</span> <span class="given-names" tagx="given-names" title="given-names">RH</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Ventral tegmental area glutamate neurons: electrophysiological properties and projections</span>. <span class="source" tagx="source" title="source">J Neurosci</span> <span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">15076</span>– <span class="lpage" tagx="lpage" title="lpage">15085</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1523/JNEUROSCI.3128-12.2012">10.1523/JNEUROSCI.3128-12.2012</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23100428">23100428</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hook</span> <span class="given-names" tagx="given-names" title="given-names">PW</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">McClymont</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Cannon</span> <span class="given-names" tagx="given-names" title="given-names">GH</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Law</span> <span class="given-names" tagx="given-names" title="given-names">WD</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Morton</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Goff</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">McCallion</span> <span class="given-names" tagx="given-names" title="given-names">AS</span> </span> ( <span class="year" tagx="year" title="year">2018</span>) <span class="mixed-article-title" title="mixed-article-title">Single-cell RNA-Seq of mouse dopaminergic neurons informs candidate gene selection for sporadic Parkinson disease </span>. <span class="source" tagx="source" title="source">Am J Hum Genet</span> <span class="volume" tagx="volume" title="volume">102</span>: <span class="fpage" tagx="fpage" title="fpage">427</span>– <span class="lpage" tagx="lpage" title="lpage">446</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ajhg.2018.02.001">10.1016/j.ajhg.2018.02.001</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29499164">29499164</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hopman</span> <span class="given-names" tagx="given-names" title="given-names">AHN</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ramaekers</span> <span class="given-names" tagx="given-names" title="given-names">FCS</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Speel</span> <span class="given-names" tagx="given-names" title="given-names">EJM</span> </span> ( <span class="year" tagx="year" title="year">1998</span>) <span class="mixed-article-title" title="mixed-article-title">Rapid synthesis of biotin-, digoxigenin-, trinitrophenyl-, and fluorochrome-labeled tyramides and their application for in situ hybridization using CARD amplification </span>. <span class="source" tagx="source" title="source">J Histochem Cytochem</span> <span class="volume" tagx="volume" title="volume">46</span>: <span class="fpage" tagx="fpage" title="fpage">771</span>– <span class="lpage" tagx="lpage" title="lpage">777</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1177/002215549804600611">10.1177/002215549804600611</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9603790">9603790</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ikemoto</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Dopamine reward circuitry: two projection systems from the ventral midbrain to the nucleus accumbens-olfactory tubercle complex </span>. <span class="source" tagx="source" title="source">Brain Res Rev</span> <span class="volume" tagx="volume" title="volume">56</span>: <span class="fpage" tagx="fpage" title="fpage">27</span>– <span class="lpage" tagx="lpage" title="lpage">78</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.brainresrev.2007.05.004">10.1016/j.brainresrev.2007.05.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17574681">17574681</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ikemoto</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bonci</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Neurocircuitry of drug reward</span>. <span class="source" tagx="source" title="source">Neuropharmacology</span> <span class="volume" tagx="volume" title="volume">76</span>: <span class="fpage" tagx="fpage" title="fpage">329</span>– <span class="lpage" tagx="lpage" title="lpage">341</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.neuropharm.2013.04.031">10.1016/j.neuropharm.2013.04.031</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23664810">23664810</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ilango</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kesner</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Broker</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">DV</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ikemoto</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Phasic excitation of ventral tegmental dopamine neurons potentiates the initiation of conditioned approach behavior: parametric and reinforcement-schedule analyses </span>. <span class="source" tagx="source" title="source">Front Behav Neurosci</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">155</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24834037">24834037</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Isingrini</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Perret</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Rainer</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Sagueby</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Moquin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gratton</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Giros</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Selective genetic disruption of dopaminergic, serotonergic and noradrenergic neurotransmission: insights into motor, emotional and addictive behaviour </span>. <span class="source" tagx="source" title="source">J Psychiatry Neurosci</span> <span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">169</span>– <span class="lpage" tagx="lpage" title="lpage">181</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1503/jpn.150028">10.1503/jpn.150028</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26505143">26505143</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kalivas</span> <span class="given-names" tagx="given-names" title="given-names">PW</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Striplin</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Steketee</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Klitenick</span> <span class="given-names" tagx="given-names" title="given-names">M. a</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Duffy</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> ( <span class="year" tagx="year" title="year">1992</span>) <span class="mixed-article-title" title="mixed-article-title">Cellular mechanisms of behavioral sensitization to drugs of abuse</span>. <span class="source" tagx="source" title="source">Ann NY Acad Sci</span> <span class="volume" tagx="volume" title="volume">654</span>: <span class="fpage" tagx="fpage" title="fpage">128</span>– <span class="lpage" tagx="lpage" title="lpage">135</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1632577">1632577</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Stott</span> <span class="given-names" tagx="given-names" title="given-names">SRW</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chabrat</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Truckenbrodt</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Spencer-Dene</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Nave</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Guillemot</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Levesque</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ang</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> ( <span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Survival of a novel subset of midbrain dopaminergic neurons projecting to the lateral septum is dependent on NeuroD proteins </span>. <span class="source" tagx="source" title="source">J Neurosci</span> <span class="volume" tagx="volume" title="volume">37</span>: <span class="fpage" tagx="fpage" title="fpage">2305</span>– <span class="lpage" tagx="lpage" title="lpage">2316</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1523/JNEUROSCI.2414-16.2016">10.1523/JNEUROSCI.2414-16.2016</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28130357">28130357</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Baratta</span> <span class="given-names" tagx="given-names" title="given-names">MV</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Boyden</span> <span class="given-names" tagx="given-names" title="given-names">ES</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Fiorillo</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Optogenetic mimicry of the transient activation of dopamine neurons by natural reward is sufficient for operant reinforcement </span>. <span class="source" tagx="source" title="source">PLoS One</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">8</span>. </span> </li> <li tag="ref"> <a name="B27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kramer</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Risso</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kosillo</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ngai</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bateup</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> ( <span class="year" tagx="year" title="year">2018</span>) <span class="mixed-article-title" title="mixed-article-title">Combinatorial expression of Grp and Neurod6 defines dopamine neuron populations with distinct projection patterns and disease vulnerability </span>. <span class="source" tagx="source" title="source">eNeuro</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="comment">ENEURO.0152-18.2018</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1523/ENEURO.0152-18.2018">10.1523/ENEURO.0152-18.2018</a> </span> </span> </li> <li tag="ref"> <a name="B28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">La Manno</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gyllborg</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Codeluppi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Nishimura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Salto</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zeisel</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Borm</span> <span class="given-names" tagx="given-names" title="given-names">LE</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Stott</span> <span class="given-names" tagx="given-names" title="given-names">SRW</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Toledo</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Villaescusa</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lönnerberg</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ryge</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Barker</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Arenas</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Linnarsson</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Molecular diversity of midbrain development in mouse, human, and stem cells</span>. <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">167</span>: <span class="fpage" tagx="fpage" title="fpage">566</span>– <span class="lpage" tagx="lpage" title="lpage">580</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2016.09.027">10.1016/j.cell.2016.09.027</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27716510">27716510</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lammel</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ion</span> <span class="given-names" tagx="given-names" title="given-names">DI</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Roeper</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Malenka</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Projection-specific modulation of dopamine neuron synapses by aversive and rewarding stimuli </span>. <span class="source" tagx="source" title="source">Neuron</span> <span class="volume" tagx="volume" title="volume">70</span>: <span class="fpage" tagx="fpage" title="fpage">855</span>– <span class="lpage" tagx="lpage" title="lpage">862</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.neuron.2011.03.025">10.1016/j.neuron.2011.03.025</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21658580">21658580</a> </span> </span> </li> <li tag="ref"> <a name="B30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lammel</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">BK</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ran</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">KW</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Betley</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Tye</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Deisseroth</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Malenka</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Input-specific control of reward and aversion in the ventral tegmental area</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">491</span>: <span class="fpage" tagx="fpage" title="fpage">212</span>– <span class="lpage" tagx="lpage" title="lpage">217</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature11527">10.1038/nature11527</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23064228">23064228</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lammel</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Steinberg</span> <span class="given-names" tagx="given-names" title="given-names">EE</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Földy</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wall</span> <span class="given-names" tagx="given-names" title="given-names">NR</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Beier</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Malenka</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Diversity of transgenic mouse models for selective targeting of midbrain dopamine neurons </span>. <span class="source" tagx="source" title="source">Neuron</span> <span class="volume" tagx="volume" title="volume">85</span>: <span class="fpage" tagx="fpage" title="fpage">429</span>– <span class="lpage" tagx="lpage" title="lpage">438</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.neuron.2014.12.036">10.1016/j.neuron.2014.12.036</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25611513">25611513</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lindeberg</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Usoskin</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bengtsson</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gustafsson</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kylberg</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Söderström</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ebendal</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Transgenic expression of Cre recombinase from the tyrosine hydroxylase locus</span>. <span class="source" tagx="source" title="source">Genesis</span> <span class="volume" tagx="volume" title="volume">40</span>: <span class="fpage" tagx="fpage" title="fpage">67</span>– <span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1002/gene.20065">10.1002/gene.20065</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15452869">15452869</a> </span> </span> </li> <li tag="ref"> <a name="B33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Menegas</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bergan</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ogawa</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Isogai</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Venkataraju</span> <span class="given-names" tagx="given-names" title="given-names">KU</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Osten</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Uchida</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Watabe-Uchida</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Dopamine neurons projecting to the posterior striatum form an anatomically distinct subclass </span>. <span class="source" tagx="source" title="source">Elife</span> <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">30</span>. </span> </li> <li tag="ref"> <a name="B34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Morales</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Margolis</span> <span class="given-names" tagx="given-names" title="given-names">EB</span> </span> ( <span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Ventral tegmental area: cellular heterogeneity, connectivity and behaviour</span>. <span class="source" tagx="source" title="source">Nat Rev Neurosci</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">73</span>– <span class="lpage" tagx="lpage" title="lpage">85</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrn.2016.165">10.1038/nrn.2016.165</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28053327">28053327</a> </span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Narboux-Nême</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Sagné</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Doly</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Diaz</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">CBP</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Angenard</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Martres</span> <span class="given-names" tagx="given-names" title="given-names">MP</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Giros</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hamon</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lanfumey</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gaspar</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mongeau</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Severe serotonin depletion after conditional deletion of the vesicular monoamine transporter 2 gene in serotonin neurons: neural and behavioral consequences </span>. <span class="source" tagx="source" title="source">Neuropsychopharmacology</span> <span class="volume" tagx="volume" title="volume">36</span>: <span class="fpage" tagx="fpage" title="fpage">2538</span>– <span class="lpage" tagx="lpage" title="lpage">2550</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/npp.2011.142">10.1038/npp.2011.142</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21814181">21814181</a> </span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Nordenankar</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Smith-Anttila</span> <span class="given-names" tagx="given-names" title="given-names">CJA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Schweizer</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Viereckel</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Birgner</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mejia-Toiber</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Morales</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Leao</span> <span class="given-names" tagx="given-names" title="given-names">RN</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wallén-Mackenzie</span> <span class="given-names" tagx="given-names" title="given-names">Å</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Increased hippocampal excitability and impaired spatial memory function in mice lacking VGLUT2 selectively in neurons defined by tyrosine hydroxylase promoter activity </span>. <span class="source" tagx="source" title="source">Brain Struct Funct</span> <span class="volume" tagx="volume" title="volume">220</span>: <span class="fpage" tagx="fpage" title="fpage">2171</span>– <span class="lpage" tagx="lpage" title="lpage">2190</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00429-014-0778-9">10.1007/s00429-014-0778-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24802380">24802380</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Pascoli</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Terrier</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hiver</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lüscher</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Sufficiency of mesolimbic dopamine neuron stimulation for the progression to addiction</span>. <span class="source" tagx="source" title="source">Neuron</span> <span class="volume" tagx="volume" title="volume">88</span>: <span class="fpage" tagx="fpage" title="fpage">1054</span>– <span class="lpage" tagx="lpage" title="lpage">1066</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.neuron.2015.10.017">10.1016/j.neuron.2015.10.017</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26586182">26586182</a> </span> </span> </li> <li tag="ref"> <a name="B38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Poulin</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zou</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Drouin-Ouellet</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">KYA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Cicchetti</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Awatramani</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Defining midbrain dopaminergic neuron diversity by single-cell gene expression profiling</span>. <span class="source" tagx="source" title="source">Cell Rep</span> <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">930</span>– <span class="lpage" tagx="lpage" title="lpage">943</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.celrep.2014.10.008">10.1016/j.celrep.2014.10.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25437550">25437550</a> </span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Poulin</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Caronia</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hofer</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Helm</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ramakrishnan</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chan</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Dombeck</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Deisseroth</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Awatramani</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> ( <span class="year" tagx="year" title="year">2018</span>) <span class="mixed-article-title" title="mixed-article-title">Mapping projections of molecularly defined dopamine neuron subtypes using intersectional genetic approaches </span>. <span class="source" tagx="source" title="source">Nat Neurosci</span> <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">1260</span>– <span class="lpage" tagx="lpage" title="lpage">1271</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/s41593-018-0203-4">10.1038/s41593-018-0203-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30104732">30104732</a> </span> </span> </li> <li tag="ref"> <a name="B40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Pupe</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wallén-Mackenzie</span> <span class="given-names" tagx="given-names" title="given-names">Å</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Cre-driven optogenetics in the heterogeneous genetic panorama of the VTA</span>. <span class="source" tagx="source" title="source">Trends Neurosci</span> <span class="volume" tagx="volume" title="volume">38</span>: <span class="fpage" tagx="fpage" title="fpage">375</span>– <span class="lpage" tagx="lpage" title="lpage">386</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tins.2015.04.005">10.1016/j.tins.2015.04.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25962754">25962754</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H-L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Barker</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Miranda-Barrientos</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Morales</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">VTA glutamatergic inputs to nucleus accumbens drive aversion by acting on GABAergic interneurons </span>. <span class="source" tagx="source" title="source">Nat Neurosci</span> <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">725</span>– <span class="lpage" tagx="lpage" title="lpage">733</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nn.4281">10.1038/nn.4281</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27019014">27019014</a> </span> </span> </li> <li tag="ref"> <a name="B42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">TE</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Berridge</span> <span class="given-names" tagx="given-names" title="given-names">KC</span> </span> ( <span class="year" tagx="year" title="year">1993</span>) <span class="mixed-article-title" title="mixed-article-title">The neural basis of drug craving: an incentive-sensitization theory of addiction</span>. <span class="source" tagx="source" title="source">Brain Res Rev</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">247</span>– <span class="lpage" tagx="lpage" title="lpage">291</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0165-0173(93)90013-P">10.1016/0165-0173(93)90013-P</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8401595">8401595</a> </span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Roeper</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Dissecting the diversity of midbrain dopamine neurons</span>. <span class="source" tagx="source" title="source">Trends Neurosci</span> <span class="volume" tagx="volume" title="volume">36</span>: <span class="fpage" tagx="fpage" title="fpage">336</span>– <span class="lpage" tagx="lpage" title="lpage">342</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tins.2013.03.003">10.1016/j.tins.2013.03.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23582338">23582338</a> </span> </span> </li> <li tag="ref"> <a name="B44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Root</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mejias-Aponte</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Morales</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Role of glutamatergic projections from ventral tegmental area to lateral habenula in aversive conditioning </span>. <span class="source" tagx="source" title="source">J Neurosci</span> <span class="volume" tagx="volume" title="volume">34</span>: <span class="fpage" tagx="fpage" title="fpage">13906</span>– <span class="lpage" tagx="lpage" title="lpage">13910</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1523/JNEUROSCI.2029-14.2014">10.1523/JNEUROSCI.2029-14.2014</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25319687">25319687</a> </span> </span> </li> <li tag="ref"> <a name="B45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Stamatakis</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Jennings</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ung</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Blair</span> <span class="given-names" tagx="given-names" title="given-names">GA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Weinberg</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Neve</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Boyce</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mattis</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ramakrishnan</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Deisseroth</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Stuber</span> <span class="given-names" tagx="given-names" title="given-names">GD</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">A unique population of ventral tegmental area neurons inhibits the lateral habenula to promote reward </span>. <span class="source" tagx="source" title="source">Neuron</span> <span class="volume" tagx="volume" title="volume">80</span>: <span class="fpage" tagx="fpage" title="fpage">1039</span>– <span class="lpage" tagx="lpage" title="lpage">1053</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.neuron.2013.08.023">10.1016/j.neuron.2013.08.023</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24267654">24267654</a> </span> </span> </li> <li tag="ref"> <a name="B46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Stuber</span> <span class="given-names" tagx="given-names" title="given-names">GD</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hnasko</span> <span class="given-names" tagx="given-names" title="given-names">TS</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Britt</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Edwards</span> <span class="given-names" tagx="given-names" title="given-names">RH</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bonci</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Dopaminergic terminals in the nucleus accumbens but not the dorsal striatum corelease glutamate </span>. <span class="source" tagx="source" title="source">J Neurosci</span> <span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">8229</span>– <span class="lpage" tagx="lpage" title="lpage">8233</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1523/JNEUROSCI.1754-10.2010">10.1523/JNEUROSCI.1754-10.2010</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20554874">20554874</a> </span> </span> </li> <li tag="ref"> <a name="B47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Stuber</span> <span class="given-names" tagx="given-names" title="given-names">GD</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Stamatakis</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kantak</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Considerations when using cre-driver rodent lines for studying ventral tegmental area circuitry </span>. <span class="source" tagx="source" title="source">Neuron</span> <span class="volume" tagx="volume" title="volume">85</span>: <span class="fpage" tagx="fpage" title="fpage">439</span>– <span class="lpage" tagx="lpage" title="lpage">445</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.neuron.2014.12.034">10.1016/j.neuron.2014.12.034</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25611514">25611514</a> </span> </span> </li> <li tag="ref"> <a name="B48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Trudeau</span> <span class="given-names" tagx="given-names" title="given-names">LE</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">El Mestikawy</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> ( <span class="year" tagx="year" title="year">2018</span>) <span class="mixed-article-title" title="mixed-article-title">Glutamate cotransmission in cholinergic, GABAergic and monoamine systems: contrasts and commonalities </span>. <span class="source" tagx="source" title="source">Front Neural Circuits</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">113</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30618649">30618649</a> </span> </span> </li> <li tag="ref"> <a name="B49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Tsai</span> <span class="given-names" tagx="given-names" title="given-names">HC</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Adamantidis</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Stuber</span> <span class="given-names" tagx="given-names" title="given-names">GD</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bonci</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">de Lecea</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Deisseroth</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Phasic firing in dopaminergic neurons is sufficient for behavioral conditioning</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">324</span>: <span class="fpage" tagx="fpage" title="fpage">1080</span>– <span class="lpage" tagx="lpage" title="lpage">1084</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1168878">10.1126/science.1168878</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19389999">19389999</a> </span> </span> </li> <li tag="ref"> <a name="B50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Viereckel</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Dumas</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Smith-Anttila</span> <span class="given-names" tagx="given-names" title="given-names">CJA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Vlcek</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bimpisidis</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lagerström</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Konradsson-Geuken</span> <span class="given-names" tagx="given-names" title="given-names">Å</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wallén-Mackenzie</span> <span class="given-names" tagx="given-names" title="given-names">Å</span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Midbrain gene screening identifies a new mesoaccumbal glutamatergic pathway and a marker for dopamine cells neuroprotected in Parkinson’s disease </span>. <span class="source" tagx="source" title="source">Sci Rep</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">16</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28442746">28442746</a> </span> </span> </li> <li tag="ref"> <a name="B51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">HL</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Morales</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Rewarding effects of optical stimulation of ventral tegmental area glutamatergic neurons</span>. <span class="source" tagx="source" title="source">J Neurosci</span> <span class="volume" tagx="volume" title="volume">35</span>: <span class="fpage" tagx="fpage" title="fpage">15948</span>– <span class="lpage" tagx="lpage" title="lpage">15954</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1523/JNEUROSCI.3428-15.2015">10.1523/JNEUROSCI.3428-15.2015</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26631475">26631475</a> </span> </span> </li> <li tag="ref"> <a name="B52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Weintraub</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Dopamine and impulse control disorders in Parkinson’s disease</span>. <span class="source" tagx="source" title="source">Ann Neurol</span> <span class="volume" tagx="volume" title="volume">64</span>: <span class="fpage" tagx="fpage" title="fpage">93</span>– <span class="lpage" tagx="lpage" title="lpage">100</span>. </span> </li> <li tag="ref"> <a name="B53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Yoo</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zell</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gutierrez-Reed</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ressler</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Shenasa</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">AB</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Fife</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Faget</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hnasko</span> <span class="given-names" tagx="given-names" title="given-names">TS</span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Ventral tegmental area glutamate neurons co-release GABA and promote positive reinforcement</span>. <span class="source" tagx="source" title="source">Nat Commun</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. </span> </li> <li tag="ref"> <a name="B54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wen</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Rao</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lv</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Cao</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> ( <span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Activation of the dopaminergic pathway from VTA to the medial olfactory tubercle generates odor-preference and reward </span>. <span class="source" tagx="source" title="source">Elife</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">24</span>. </span> </li> </ul> </div> <div class="synthesis-author-response" title="synthesis-author-response"> <div class="title" tagx="title" title="title">
Synthesis</div> <div class="boxed-text" tagx="boxed-text" title="boxed-text"> <p>Reviewing Editor: Yavin Shaham, NIDA-IRP/NIH</p> <p>Decisions are customarily a result of the Reviewing Editor and the peer reviewers coming together and discussing their recommendations until a consensus is reached. When revisions are invited, a fact-based synthesis statement explaining their decision and outlining what is needed to prepare a revision will be listed below. The following reviewer(s) agreed to reveal their identity: David Root. </p> </div> <p>The reviewers found merit in the study but also have raised several significant concerns that should be addressed in the revision. It is particularly important to satisfactory address comments 1-2 of review 2. Note that while the reviewers made suggestions for additional behavioral experiments, these are not mandatory for a successful revision. I enclose below the individual comments of the reviewers and in addition to their comments, also address the comments below: </p> <p>1. Based on the reviewers' comments, it is important to tone down the different conclusions and statements in the text regarding the role of NeuroD6 in reward. </p> <p>2. Lines 7 and 29: change "not least addiction" to "including drug addiction"</p> <p>3. Line 20-24: please describe in concrete terms what "altered locomotor response" means. Did the experimental manipulation decrease or increase the locomotor response to the drugs? Also, change "is sufficient to drive significant" to ‘caused’ and ‘fails’ to ‘failed’ (in general the findings described in the paper should be described in past tense). </p> <p>4. Abstract: please rewrite this section to also include the negative results in the study </p> <p>5. Methods: please provide RRID numbers of the different antibodies and other reagents when possible. Also, change ‘animals’ to ‘mice’ when referring to the experimental animals </p> <p>6. Results: please add F or t values and their degrees of freedom to all p values reported in the Results section. Also, please describe the different behavioral results under different sub-headings in different paragraphs. Finally, please remove background text with literature citations and Methods-related text from this section. The background text belongs to either the Introduction or Discussion and the Methods text belongs to the Methods section. </p> <p>7. Results, DA release section and subsequent text below it: please add F values, degrees of freedom, and p values for descriptions of group differences </p> <p>8. Line 59 and other places in the text: Change "is sufficient to drive place preference" to ‘induced’ or ‘caused’ </p> <p>9. Discussion: please add 2-3 informative sub-headings to orient the readers of the topics being discussed. In line 741, change "strikingly heterogeneous" to "heterogeneous"; in general, avoid using subjective terms to describe experimental findings, these are not necessary </p> <p>10. Figure legends 6-7: add the n of mice per group</p> <p>11. Figure 1c-d: change the y-axis to start from 0%</p> <p>REVIEWER 1</p> <p>NeuroD6-expressing DA neurons were recently identified as neuroprotected in experimental Parkinsons. However, the circuits and functions of NeuroD6 neurons are not entirely known. The authors used multiple mouse lines to identify the circuits and functions of NeuroD6 VTA neurons. Many experiments are conducted, most with null effects. However, it is identified that NeuroD6 neurons release dopamine into accumbens and can drive conditioned reward when optogenetically stimulated. The experiments identify important insights into VTA heterogeneity and function. However, as written, there are a few inconsistencies in the data that require further explanation prior to publication. These major concerns are listed below: </p> <p>Major concerns: </p> <p>1. The removal of the capacity to vesicularly release dopamine from NeuroD6 neurons enhanced sensitization to amphetamine but not to cocaine. In addition, the enhanced locomotion to amphetamine was not replicated at the same dose with the CPP procedure. This raises the question of how reliable the original amphetamine finding is. </p> <p>2. The authors write that a few NeuroD6 cells co-expressed VGluT2 but in Fig 1 it is stated that 100% of NeuroD6 cells lack VGluT2. In addition, how would it be possible for VGluT2-lacking neurons to cause glutamatergic currents (Calb and NEX)? </p> <p>3. Calb and NEX release glutamate in OT and shell. Are the currents monosynaptic? Testing under TTX and 4-AP could test this and determine if glutamate release is polysynaptic. </p> <p>4. It was unclear if NEX::Cre-labeled neurons express NeuroD6. 74% of labeled neurons have TH protein, but all NeuroD6 neurons have TH mRNA. This dichotomy should be explained. </p> <p>5. It is a little puzzling why Calb2::cre mice did not show dopamine release when most of these cells are TH+. Do they lack VMAT2? </p> <p>6. Some NEX-eYFP labeling was observed in interpeduncular nucleus. Was this a projection or cell body labeling outside VTA? This would be consistent with medial habenula labeling (Table 2) </p> <p>7. Stimulation of VTA VGluT2 cell bodies caused aversion, which differs from previous reports of reward (though aversion at terminal regions). The authors write that there were differences in procedure but were not fully explained that might mediate the observed difference. </p> <p>Minor Concerns: </p> <p>1 (Line 59)...omit "over the past couple of years" </p> <p>2 (Line 10)...omit ‘e.g’ </p> <p>3 Missing figure label for Figure 2 (No description for E) </p> <p>4 Line 401: that, not than </p> <p>5 It should be noted that the ChR2-eYFP outputs identified in the paper are plausible outputs as axons are shown but not necessarily synapses. </p> <p>REVIEWER 2</p> <p>The authors use a combination of in situ hybridization, as well as transgenic and viral approaches to characterize and manipulate selective populations of midbrain dopamine neurons with a particular focus on NeuroD6-expressing neurons. They then employ voltammetry, slice electrophysiology, and behavioral analysis to interrogate the functional properties of this neuron subpopulation. The manuscript is well written and the purpose of the study - to further tease apart functional roles of molecularly distinct groups of dopamine neurons - has great merit. However, I have concerns about several of the technical approaches used and don't feel that the data currently support the claims that the manuscript advances regarding the role of NeuroD6+ dopamine neurons in reward-related behaviors. The conclusions presented need to be much more conservative in their wording. </p> <p>Major concerns: </p> <p>1) Although several published studies have characterized the abundance and spatial distribution of NeuroD6 neurons, additional confirmation would still be useful and it's good that the authors carry out their own analysis of the NeuroD6+ neuron abundance and distribution. However, the in situ images they present are not convincing. Figure 1A is labeled and described in the figure legend as showing TH mRNA in green. But there many red cells visible and some that appear violet as though there is also colocalization with a third marker or DAPI. Then in panel 1B, which is supposed to show NeuroD6 (red) and TH (green), the same green TH signal from panel 1A is shown, but a different population of red cells is visible. The red signals that are visible are also readily apparent in the SNc and many of them are TH-negative. This does not match with the statement that cells expressing NeuroD6 mRNA are only located in the medial VTA and that 100% of those cells are TH+. Panel 1C is supposed to show NeuroD6 only, but this is a different set of red signals than those that were shown in panel 1B and the signal is very dim. The authors should double check to see if there has been some mix up in which images were included in figures. The authors should also show higher magnification images of the medial VTA that clearly illustrate 100% overlap of NeuroD6 mRNA with TH mRNA in many cells rather than one tiny inset showing one cell. I have similar concerns about panel 1E where there are appear to be abundant red cells not colocalized with DAT mRNA, yet the claim is that 100% of NeuroD6 cells are DAT+. In panel 1G, it looks like there are many yellow cells positive for both NeuroD6 and Viaat, but the claim is that all NeuroD6 cells are Viaat negative. Higher magnification images of the medial VTA rather than tiny insets could help to clarify all of these concerns. Because these data are the foundation for most other analyses and interpretation that follows in the manuscript, it is critical for this piece of the dataset to be absolutely convincing. </p> <p>2) The approaches used to confirm Vmat2 knockout in NeuroD6 neurons seem unnecessarily indirect and thus raise questions. Why not just quantify overlap between NeuroD6 mRNA and probe 2 for Vmat2 mRNA? With the current data, all we know is that Vmat2 is eliminated from some neurons in the medial VTA. We don't have any information about whether this Vmat2 elimination was specific to NeuroD6 neurons (and it likely isn't based on their findings with ChR2-EYFP expression in Nex-Cre mice in Figure 5). We also don't know from what percentage of NeuroD6 neurons was Vmat2 eliminated. If for some reason the authors can't look directly at overlap between NeuroD6 mRNA and probe 2 for Vmat2, they could at least quantify the number of Vmat2 probe2+probe1- cells in the different sub regions of the VTA and see how closely this matches the number and density of NeuroD6+ cells in those same sub regions. Again, this is part of the foundation for claiming that NeuroD6 neurons play a unique role in reward-related behaviors, so these data should be absolutely convincing for clear interpretation of behavioral experiments that follow. </p> <p>3) There seems to be enough non-specific expression of ChR2 in the Nex-Cre mice that this complicates interpretation of the behavioral experiments. There were numerous EYFP+TH- cells in Nex-Cre mice. Further, although there was virtually no overlap between NeuroD6 mRNA and VGlut2 mRNA in figure 1 in situ experiments, clear glutamatergic currents were recorded in the NAc shell upon stimulation of ChR2+ terminals in these mice. VTA optogenetic stimulation in these mice during behavior experiments is very likely resulting in activation of NeuroD6 dopamine neurons and some VTA glutamatergic neurons. For this reason, the authors should be more cautious in their claims about potential roles for NeuroD6 dopamine neurons in reward-related behaviors. They can say that their experiments suggest roles for these neurons in reward-related behaviors but need to be very clear that additional experiments are needed to unequivocally establish which behavioral effects are due to NeuroD6 neurons alone. Perhaps using a viral construct with DAT promoter (or some other dopamine neuron-specific promoter) driving DIO-ChR2-EYFP could be a viable approach? </p> <p>Minor concerns: </p> <p>1) A number of the behavioral results are inconsistent or counter-intuitive. For example, there isn't a consistent impact of Nex-Cre Vmat2 knockout on psychostimulant induced locomotor behavior (different effects for cocaine versus amphetamine and different effects during open field analysis compared to during CPP). The authors delve into this a bit in the discussion, but this is another reason why the claims advanced in the abstract and title need to be softened. It seems a bit surprising to me that you could see a clear effect of ChR2 stimulation in Nex-Cre mice on real time place preference but no impact on conditioned place preference and no impact of Vmat2 deletion on conditioned place preference. I would have liked to see more discussion of how the behavioral analysis in Vmat2 knockout and ChR2 stimulation in Nex-Cre mice relate to one another. Behavioral analysis is not my main area of expertise so hopefully other reviewers can add additional comments to this portion of the study. </p> <p>2) Many of the higher magnification insets throughout the manuscript are quite pixelated. Increased image resolution will make the images easier to interpret. </p> <p>3) FIGURE 4 - why are 83% of Calb2 neurons positive for TH and only 51% are positive for DAT. This seems a little strange. </p> <p>4) Fig4E - where is the location of the inset? </p> <p>5) Text line 522 states that the percent overlap between YFP and TH exceeds that of the quantification from mRNA and so they moved forward with analysis of neurotransmitter release; but my understanding was that fewer YFP+ cells were positive for TH than NeuroD6 cells were positive for TH (NEX-Cre: 74% YFP+ are TH+ versus 100%NeuroD6 are TH+ at messenger RNA level; Calb2-Cre: 67% YFP+ are TH+ versus 81%Calb2 are TH+ at messenger RNA level). </p> <p>6) Fig7A - ideally show the recording site for OT as well otherwise this is confusing and makes you think that recording site is NAc shell for both Cre lines. </p> <p>7) Fig7C - is a bit confusing - the y axis is showing amplitude of EPSCs but then x axis says % connectivity and two numbers are shown. I am assuming these represent a range of % of cells that showed currents? It might be more clear to make a separate graph for percent connectivity or just state the percent connectivity in the results. </p> </div> </div>  </body></html>