<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Commun Biol</span> <span class="iso-abbrev" title="iso-abbrev">Commun Biol</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Communications Biology</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2399-3642</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6325061">6325061</a> </span> <span class="publisher-id" title="publisher-id">264</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s42003-018-0264-7">10.1038/s42003-018-0264-7</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Optogenetic control of integrin-matrix interaction</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baaske</span> <span class="given-names" tagx="given-names" title="given-names">Julia</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mühlhäuser</span> <span class="given-names" tagx="given-names" title="given-names">Wignand W. D.</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-5304-729X</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yousefi</span> <span class="given-names" tagx="given-names" title="given-names">O. Sascha</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> <a href="#Aff3">3</a> <a href="#Aff4">4</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zanner</span> <span class="given-names" tagx="given-names" title="given-names">Sebastian</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Radziwill</span> <span class="given-names" tagx="given-names" title="given-names">Gerald</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hörner</span> <span class="given-names" tagx="given-names" title="given-names">Maximilian</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> <a href="#Aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schamel</span> <span class="given-names" tagx="given-names" title="given-names">Wolfgang W. A.</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> <a href="#Aff3">3</a> <a href="#Aff4">4</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">Wilfried</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">wilfried.weber@biologie.uni-freiburg.de</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> <a href="#Aff3">3</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">grid.5963.9</span> <span class="institution" title="institution">Faculty of Biology, </span> <span class="institution" title="institution">University of Freiburg, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">grid.5963.9</span> <span class="institution" title="institution">Signalling Research Centers BIOSS and CIBSS, </span> <span class="institution" title="institution">University of Freiburg, </span> </div> </span> <span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">grid.5963.9</span> <span class="institution" title="institution">Spemann Graduate School of Biology and Medicine (SGBM), </span> <span class="institution" title="institution">University of Freiburg, </span> </div> </span> <span class="citation_author_institution" id="Aff4">[4], </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-1-1</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-1-1</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span> </span> <span class="volume" tagx="volume" title="volume">2</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">15</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-4-6</span> <span class="accepted" title="accepted">accepted: 2018-12-12</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Optogenetic approaches have gathered momentum in precisely modulating and interrogating cellular signalling and gene expression. The use of optogenetics on the outer cell surface to interrogate how cells receive stimuli from their environment, however, has so far not reached its full potential. Here we demonstrate the development of an optogenetically regulated membrane receptor-ligand pair exemplified by the optically responsive interaction of an integrin receptor with the extracellular matrix. The system is based on an integrin engineered with a phytochrome-interacting factor domain (OptoIntegrin) and a red light-switchable phytochrome B-functionalized matrix (OptoMatrix). This optogenetic receptor-ligand pair enables light-inducible and -reversible cell-matrix interaction, as well as the controlled activation of downstream mechanosensory signalling pathways. Pioneering the application of optogenetic switches in the extracellular environment of cells, this OptoMatrix–OptoIntegrin system may serve as a blueprint for rendering matrix–receptor interactions amendable to precise control with light. </p> </div> <div class="abstract" title="abstract"> <p>Julia Baaske et al. present an optogenetically-regulated membrane receptor-ligand pair that enables light-inducible and light-reversible cell-matrix interaction and controlled activation of downstream mechanosensory signalling pathways. They show that optogenetics can be used to control receptor activation using light. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">https://doi.org/10.13039/501100001659</span> <span class="institution" title="institution">Deutsche Forschungsgemeinschaft (German Research Foundation)</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">EXC-294, GSC-4</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">Wilfried</span> </span> </span> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Optogenetics uses light to control protein function in living cells, and has gathered momentum in the analysis and control of biological processes. The use of light to manipulate signalling processes has the major advantage of being minimally invasive and of enabling precise spatiotemporal control. A vast array of optogenetic switches has been designed to control the activity, localization, interaction or degradation of intracellular proteins to steer biological function <sup> <a href="#CR1">1</a>– <a href="#CR4">4</a> </sup>. However, important extracellular processes such as the interaction of cell surface receptors with ligands or with the extracellular matrix represent a field of intensive study <sup> <a href="#CR5">5</a>– <a href="#CR10">10</a> </sup>. We present here a strategy to optically control the interaction of integrin cell surface receptors with the extracellular matrix. </p> <p>Integrins are a major group of transmembrane, heterodimeric cell adhesion receptors that link the actin cytoskeleton to the extracellular matrix. Upon ligand binding, integrins undergo a change in conformation that triggers recruitment of signalling and cytoskeletal adaptor proteins such as talin, kindlins and paxillin, which ultimately leads to the formation of focal adhesions and the activation of mechanosensory pathways <sup> <a href="#CR11">11</a> </sup>. This conformational change in canonical integrin signalling relies on the ligand binding which in turn leads to a transmission of conformational changes initially from the metal ion-dependent adhesion site, over the swing out of the hybrid domain, to the extension of the ectodomain and subsequent spatial separation of both tails. This induced conformational change exposes effector binding site and hence activates downstream signalling processes <sup> <a href="#CR12">12</a> </sup>. </p> <p>Integrins have been found to be involved in the regulation of invasion, proliferation and survival of tumour cells, making them prime targets for anti-tumour therapy <sup> <a href="#CR13">13</a> </sup>. In mammals, the integrin family comprises 24 heterodimers formed by the combination of 18 α and 8 β subunits. One of the most extensively studied members of this receptor family is the αVβ3 integrin, which belongs to the RGD subset of integrins that bind extracellular ligands with Arg-Gly-Asp motifs. Integrin αVβ3 has a critical role in angiogenesis during tumour development, and is associated with tumour growth and metastasis <sup> <a href="#CR13">13</a>, <a href="#CR14">14</a> </sup>. The dynamics of integrin engagement plays a key role in determining the biological outcome of integrin activation, but so far there is a paucity of tools that allow for precise control of integrin activation. In order to provide means for analysing integrin signalling with superior precision, we set out to develop an optogenetically-regulated αVβ3-based integrin–ligand pair. To render the interaction between integrin αVβ3 and the extracellular matrix light-inducible we use the red light-responsive phytochrome B (PhyB) and the phytochrome-interacting factor 6 (PIF6) of <i>Arabidopsis thaliana</i> <sup> <a href="#CR15">15</a> </sup>. Light sensing by PhyB is mediated by the photoisomerization of a covalently bound tetrapyrrole chromophore such as phycocyanobilin. Exposure to 660 nm red light leads to an isomerization of the chromophore and the allosteric transition of PhyB from the Pr (inactive) to the Pfr (active) form <sup> <a href="#CR16">16</a>– <a href="#CR18">18</a> </sup>. In its active Pfr form, PhyB binds phytochrome-interacting factors such as PIF6. Illumination with 740 nm far-red light triggers photoconversion back to the inactive Pr form, leading to dissociation from PIF6. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="designofalight-induciblemembranereceptor–ligandpair" title="sec"> <div class="title" tagx="title" title="title">
Design of a light-inducible membrane receptor–ligand pair</div> <p>We postulated that integrin–matrix interactions could be rendered light-switchable by immobilizing PhyB (amino acids 1–651 <sup> <a href="#CR19">19</a> </sup>) to a matrix and inserting a PIF6 variant into the extracellular domain of the β subunit of integrin αVβ3 (Fig. <a href="#Fig1">1a</a>). Since PIF is inserted in the extracellular integrin domain which is folded in the endoplasmic reticulum and transported to the cell surface via the secretory pathway, we used a secretion-optimized PIF6 variant (termed PIF <sup>S</sup>) <sup> <a href="#CR20">20</a> </sup> comprising amino acids 1–100 of PIF6 <sup> <a href="#CR15">15</a>, <a href="#CR16">16</a> </sup> in which a glycosylation motif, as well as two cysteines, have been removed by site-directed mutagenesis (C9S/C10S/S37A). We evaluated four sites for PIF <sup>S</sup> insertion in the β subunit of the human integrin αVβ3: N-terminally (construct pJB003); between Glu108 and Asp109 (pJB004), which is between the hybrid domain and the β-I domain; between Met180 and Lys181 (pJB005), which is in close proximity to the RGD-binding site; and between Ser77 and Ser78 (pJB006), which is in the hybrid domain (Fig. <a href="#Fig1">1b, c</a>, and Supplementary Table <a href="#MOESM5">1</a>; all indicated amino acids refer to the crystal structure: 1L5G (Protein data bank accession number)) <sup> <a href="#CR12">12</a>, <a href="#CR14">14</a>, <a href="#CR21">21</a> </sup>. In each construct, PIF <sup>S</sup> was flanked on both sides by a flexible Ser-Ala-Gly linker. Furthermore, the β subunit was linked via a self-cleaving P2A peptide to an αV subunit to ensure equimolar production of each subunit <sup> <a href="#CR22">22</a> </sup>. To facilitate detection, we added an HA tag to the N-terminus and a FLAG tag to the C-terminus of the β subunit. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>Design and construction of light-inducible receptor–ligand interactions. <b>a</b> (3-glycidyloxypropyl)trimethoxysilane (GLYMO)-functionalized glass slides are coated with NeutrAvidin and subsequently with biotinylated PhyB <sub>1–651</sub>. Cells expressing PIF <sup>S</sup>-coupled OptoIntegrins are seeded on top of glass slides. Upon illumination with 660 nm light, PhyB <sub>1–651</sub> converts from its inactive Pr form to the active Pfr form. The active PhyB <sub>1–651</sub> form interacts with the OptoIntegrin which then activates mechanosensory pathways. <b>b</b> Schematic view of the OptoIntegrin-encoding sequence with PIF <sup>S</sup> insertion sites. Amino acids numbers under plasmid names refer to insertion sites in the crystal structure (Protein data bank accession number: 1L5G <sup> <a href="#CR21">21</a> </sup>). <b>c</b> Crystal structure of αVβ3 extracellular domain with RGD ligand (1L5G <sup> <a href="#CR21">21</a> </sup>). ITGB3 subunit in blue and ITGAV subunit in purple. PIF <sup>S</sup> insertion sites for different constructs are marked in red. <b>d</b> Expression of different OptoIntegrin constructs in HEK-293T cells. Western blot analysis for presence of HA-tagged β subunit in cell lysate expressing different integrin constructs. <b>e</b> PhyB <sub>1–651</sub> binding to surface of untransfected (Mock) HEK-293T cells and HEK-293T cells transiently expressing OptoIntegrin constructs under illumination with 660 nm or 740 nm light. Analysis of stained cells with flow cytometry. Data of two replicates ( <i>n</i> &amp;gt; 5000 cells) per condition is presented in box and whisker plot format </p>   </div> <p /> <p>We first evaluated expression of the integrin constructs. To this end, human embryonic kidney (HEK-293T) cells were transfected with each construct, lysed and analysed by immunoblotting for the presence of the modified β subunit using anti-HA western blotting (Fig. <a href="#Fig1">1d</a>). The presence of the HA tag confirmed that each β subunit variant is produced in the cells. To further analyse the constructs, it is important to know whether the chimeric PIF <sup>S</sup> constructs can fold properly and translocate to the plasma membrane, and whether PIF <sup>S</sup> is accessible for the interaction with PhyB <sub>1–651</sub>. Previous work has shown that only correctly formed integrin heterodimers are transported to the plasma membrane <sup> <a href="#CR23">23</a> </sup>. We stained living HEK-293T cells expressing the integrin variants with anti-HA antibodies and analysed the cells by flow cytometry. All engineered αVβ3 constructs were expressed on the cell surface (Supplementary Figure <a href="#MOESM5">1a</a>). To evaluate whether PIF <sup>S</sup> was accessible for interaction with PhyB <sub>1–651</sub>, we took advantage of the fluorescence of PhyB <sub>1–651</sub> (Supplementary Figure <a href="#MOESM5">2</a>). We incubated transiently transfected HEK-293T cells with PhyB <sub>1–651</sub> under 660 nm red light or 740 nm far-red light and subsequently measured the PhyB <sub>1–651</sub> fluorescence intensity using flow cytometry (Fig. <a href="#Fig1">1e</a> and Supplementary Figure <a href="#MOESM5">1b</a>). Interestingly, only cells expressing construct pJB005 showed increased fluorescence intensity after 660 nm illumination, indicating that even though all constructs were expressed, only integrin variant from pJB005 had PIF <sup>S</sup> inserted in a way that it could interact with PhyB <sub>1–651</sub>. Here, we name the αVβ3-PIF <sup>S</sup> variant of pJB005 OptoIntegrin. Next, we generated HEK-293T, human cervical cancer cell (HeLa) and human breast cancer cell (MCF7) lines which stably express OptoIntegrin αVβ3 in order to further characterize its functionality (Supplementary Figure <a href="#MOESM5">3</a>). As control, we further constructed a HEK-293T-based cell line expressing a moxGFP-PIF <sup>S</sup> fusion on the cell surface in order to differentiate between effects mediated by mere cell–matrix attachment or by functional integrin–matrix interaction (Supplementary Figure <a href="#MOESM5">4</a>). We showed that functional PIF <sup>S</sup> was displayed on the cell surface and interacted with PhyB <sub>1–651</sub> in a light-dependent manner (Supplementary Figure <a href="#MOESM5">4b, c</a>). </p> </div> <div class="lightcontrolledcell–matrixinteraction" title="sec"> <div class="title" tagx="title" title="title">
Light controlled cell–matrix interaction</div> <p>To evaluate whether the interaction of cells expressing OptoIntegrin αVβ3 with a synthetic matrix can be controlled by light, we synthesized PhyB <sub>1–651</sub>-coated glass slides (‘OptoMatrix’). Glass slides were first functionalized with (3-glycidyloxypropyl)trimethoxysilane (GLYMO) and subsequently with NeutrAvidin <sup> <a href="#CR24">24</a> </sup>. Biotinylated PhyB <sub>1–651</sub> was then added to couple to NeutrAvidin. This treatment prevented wild type cells from attaching and spreading on the glass slide (Supplementary Figure <a href="#MOESM5">5a</a>). </p> <p>To analyse light-inducible receptor–ligand interactions, we seeded HEK-293T, HeLa and MCF-7 cells stably expressing the OptoIntegrin onto the OptoMatrix under 660 nm or 740 nm light. Under 660 nm light, the cells adhered and formed protrusions, whereas cells kept in 740 nm light did not attach to the slide, but rather formed large aggregates (Fig. <a href="#Fig2">2a</a>). When removing the PhyB surface of OptoMatrix by scratching, however, OptoIntegrin-expressing cells also attached under 740 nm light (Supplementary Figure <a href="#MOESM5">5b</a>). The control cells expressing moxGFP-PIF <sup>S</sup> showed attachment to the OptoMatrix under 660 nm light, yet the cells mainly kept a round shape and did not start to spread. This suggests that mere cell attachment is not sufficient for triggering processes required for efficient spreading. We next compared cell spreading of wildtype cells grown on fibronectin-coated glass slides to OptoIntegrin-expressing cells growing on OptoMatrix (Supplementary Figure <a href="#MOESM5">5c</a>). This analysis revealed no significant difference in cell area (p-values obtained using Welch-test: <i>p</i> = 0.184 for HEK-293T, <i>p</i> = 0.878 for HeLa and <i>p</i> = 0.124 for MCF7, see Supplementary Figure <a href="#MOESM5">5c</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>Validation of cell–matrix interaction with different cell lines stably expressing OptoIntegrin. <b>a</b> HEK-293T, HeLa and MCF7 cells stably expressing OptoIntegrin or moxGFP-PIF <sup>S</sup> were seeded on PhyB <sub>1–651</sub>-coated glass slides and incubated under 660 nm or 740 nm light (I = 20 µmol m <sup>−2</sup> s <sup>−1</sup>) for 5 h and subsequently imaged using a transmission light microscope (scale bar = 200 µm). <b>b</b> Live cell imaging of cell–matrix interaction under 660 nm light and then switch to 740 nm light after 35 min with binary images of cell shapes to illustrate the cells spreading. For this, HeLa cells stably expressing OptoIntegrin were seeded on OptoMatrix and subsequently imaged. Micrographs were taken at indicated time points (scale bar = 10 µm). <b>c</b> Spatial control of cell attachment with OptoIntegrin-expressing HEK-293T cells. OptoIntegrin-expressing cells were cultivated on OptoMatrix, locally illuminated with 660 nm or 740 nm for 3 min and then left in darkness for 4 h. Afterwards, cells were fixed and imaged (scale bar = 200 µm) </p>   </div> <p /> <p>A strong advantage of using the PhyB-PIF system is the ability to instantaneously reverse an interaction using 740 nm light. In live cell imaging experiments, stably OptoIntegrin-expressing HeLa cells exposed to 660 nm light attached, formed filopodia and spread on the matrix. Upon switching to 740 nm far-red light, however, cells immediately retracted and almost completely detached from OptoMatrix (Fig. <a href="#Fig2">2b</a>, Supplementary Movies <a href="#MOESM2">1</a> (HeLa), 2 (MCF7), 3 (HEK-293T)). Besides temporal control, light also enables high spatial control. To analyse spatially controlled cell attachment, we illuminated one half of the OptoMatrix surface with 660 nm and the other one with 740 nm light (Supplementary Figure <a href="#MOESM5">6</a>). Cultivating cells in this configuration revealed attachment and spreading exclusively in the 660 nm-illuminated area (Fig. <a href="#Fig2">2c</a>). </p> </div> <div class="adhesionviaoptointegrintriggersmechanosensingpathways" title="sec"> <div class="title" tagx="title" title="title">
Adhesion via OptoIntegrin triggers mechanosensing pathways</div> <p>Ligand binding of integrins leads to subsequent clustering and the formation of focal adhesions, which not only mechanically links the matrix-bound integrins to the cytoskeleton, but also engages mechanosensory signalling pathways <sup> <a href="#CR11">11</a> </sup>. These macromolecular assemblies are stabilized through the recruitment of several proteins such as paxillin, which is an important scaffolding protein containing several protein interaction domains <sup> <a href="#CR25">25</a> </sup>. We next assessed whether the OptoIntegrin-OptoMatrix-mediated interaction could trigger such intracellular mechanosensory pathways. To this aim, HEK-293T cells stably expressing OptoIntegrin were allowed to attach to PhyB <sub>1–651</sub>-coated glass slides for 1.5 h under 660 nm light. Two light treatments were then applied: one group of cells was switched from 660 nm to 740 nm for 0.5 h, while the other group was kept under constant 660 nm light. Cells were subsequently fixed, stained for paxillin and imaged (Fig. <a href="#Fig3">3a</a> and Supplementary Figure <a href="#MOESM5">7a</a>). In cells grown under constant 660 nm light, we observed in approximately 55% of the cells paxillin clusters at the edges. After switching to 740 nm light, cells were less spread and only 40% of cells showed paxillin cluster formation, which was significantly different compared to constant 660 nm light condition. Similar results were further obtained using HeLa cells stably expressing OptoIntegrin (Supplementary Figure <a href="#MOESM5">7b–d</a>). In approximately 80% of the HeLa cells illuminated with constant 660 nm light, we observed clustering of paxillin at the cellular periphery. In contrast, only approximately 20% of cells treated with far-red light showed this distinct paxillin localization pattern. These observations suggest that engineered OptoIntegrin−OptoMatrix interactions, and the ensuing cellular formation of focal adhesions, can be induced and reversed by light. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>Transducing signals from the extracellular to the intracellular with cells expressing OptoIntegrin. Cells were seeded on PhyB <sub>1–651</sub>-coated glass surface and illuminated as indicated. <b>a</b> Paxillin staining with anti-paxillin antibody of fixed stably OptoIntegrin-expressing HEK-293T cells. The intensities of the images were inverted to better visualize clusters at cell edges. The data of three independent experiments where at least 35 cells per condition were analysed in a blinded manner is presented in box and whisker plot format ( <i>p</i> = 0.000989 using Cochran–Mantel–Haenszel-test with one degree of freedom, showing a consistent difference in the populations across several repeats). <b>b</b> Phosphorylation of ERK1/2 in stably OptoIntegrin-expressing HEK-293T cells in response to different light conditions analysed by western blot. Biological triplicates are shown for each condition using anti-pERK1/2 antibodies and anti-HA antibodies to detect pERK1/2 and OptoIntegrin HA tagged β subunit, respectively. <b>c</b> YAP1 (green) and nucleus (DAPI, blue) staining of OptoIntegrin-expressing HEK-293T cells. Chart shows the percentage of cells from a single experiment where YAP1 is localized mainly in the cytoplasm, mainly in the nucleus or in an equal distributions between cytoplasm and nucleus ( <i>p</i> &amp;lt; 0.0001, with a Chi-square value of 87.56 and two degrees of freedom, using Chi-square test of independence shows dependent variables). For ( <b>a</b>), ( <b>b</b>) and ( <b>c</b>) λ <sub>1/2</sub> = 660/740 nm, I <sub>1/2</sub> = 20 µmol m <sup>−2</sup> s <sup>−1</sup> and for <b>a</b> and <b>b</b> scale bar = 50 µm </p>   </div> <p /> <p>After demonstrating the recruitment of direct integrin interaction proteins, we next examined whether mechanosensory signalling events further downstream could be controlled using this light-inducible OptoIntegrin–OptoMatrix interaction. First, we examined the activation of the Ras-Raf-MEK-ERK cascade, which is activated downstream of integrins <sup> <a href="#CR26">26</a> </sup>. For this we set up two constant light conditions in which we incubated cells stably expressing OptoIntegrin under constant 660 nm or 740 nm light and two conditions where the light colour was switched 30 min before cell lysis (Fig. <a href="#Fig3">3b</a>). Cells were analysed by immunoblotting for the presence of active, phosphorylated form of ERK1/2 (pERK1/2). Cells stably expressing OptoIntegrins showed higher abundance of pERK1/2 under 660 nm illumination compared to 740 nm, indicating that OptoIntegrins activate intracellular signalling upon ligand binding. Disruption of the OptoIntegrin–PhyB <sub>1-651</sub> interaction induced by switching from 660 nm to 740 nm led to a reduction of pERK1/2 levels. Cells expressing moxGFP-PIF <sup>S</sup> grown on the OptoMatrix, as well as wildtype cells and OptoIntegrin-expressing cells grown on fibronectin-coated glass slides did not show such difference under the different illumination conditions (Supplementary Figure <a href="#MOESM5">8a</a>). </p> <p>Further, we speculated that by triggering OptoIntegrin activation, we could mimic the mechanical stimuli that activate the transcriptional regulator YAP1 (Yes-associated protein 1). Activation of YAP1 is regulated by cytoskeletal tension and substrate stiffness and results in the nuclear translocation of the protein <sup> <a href="#CR27">27</a>– <a href="#CR30">30</a> </sup>. To this end, stably OptoIntegrin-expressing HEK-293T cells were allowed to interact with the OptoMatrix for 1.5 h under 660 nm red light, and were subsequently exposed to 740 nm far-red light or kept under 660 nm light for additional 30 min. Cells kept under 660 nm light showed higher abundance of nuclear YAP1, whereas cells that were switched to 740 nm light displayed a more diffuse localization of YAP1 (Fig. <a href="#Fig3">3c</a> and Supplementary Figure <a href="#MOESM5">8b</a>). Cells grown on fibronectin-coated glass slides did not show light-dependent differential YAP1 localisation (Supplementary Figure <a href="#MOESM5">8c</a>). Analysis of YAP1 localization in HeLa cells expressing the OptoIntegrin showed results similar to HEK-293T cells (Supplementary Figure <a href="#MOESM5">8d</a>). These results suggest that OptoIntegrin is able to induce cell–matrix interactions and transduce mechanical stimuli. </p> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>The activation of our OptoIntegrin is likely regulated through a pulling force created by the interaction of PhyB <sub>1–651</sub> and PIF <sup>S</sup>. We hypothesize that this interaction mimics the natural ligand binding by creating an adhesion force that pulls the integrin into the extended, active conformation. Through this force generation, we suggest, that we shortcut the canonical adhesion mechanism to provoke an adhesion signal. This light-controlled OptoIntegrin can thus, in the future, be used to study more in detail the integrin activation mechanisms and shed further light on the role of mechanical forces. </p> <p>Optogenetic methods enable precise control of cellular processes, and are powerful tools to study biological signalling. Here, we expanded the application of optogenetics to the extracellular environment by generating the red light-switchable PhyB OptoMatrix for activation of a PIF <sup>S</sup>-containing OptoIntegrin. Using this system, we showed for the first time, to our knowledge, that it is possible to use optogenetics to faithfully recapitulate an extracellular ligand–receptor interaction that transduces signals from the matrix to the intracellular mechanosensory machinery of cells. The extension of this system to other ligand–receptor interactions, as well as the design of specialised OptoMatrices and OptoLigands, could provide a straightforward, widely applicable method to exert precise spatiotemporal control over receptor activation using light. This technology therefore has the potential to provide unprecedented insight into the dynamics and signalling sequelae of receptor activation. </p> </div> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plasmids" title="sec"> <div class="title" tagx="title" title="title">
Plasmids</div> <p>The plasmids designed in this study are described in Supplementary Table <a href="#MOESM5">1</a>. </p> </div> <div class="sampleillumination" title="sec"> <div class="title" tagx="title" title="title">
Sample illumination</div> <p>Illumination was performed with light boxes similar to those used before (for further information see <a href="#MOESM5">supplementary methods</a>) <sup> <a href="#CR31">31</a> </sup>, with LED panels emitting at 660 nm (LED660N-03, Roithner LaserTechnik; LH W5Am, Osram Opto Semiconductors; L-53SR, Kingbright Electronic) or 740 nm (H2A1-H740, Roithner LaserTechnik; LED740-01AU, Roithner LaserTechnik; LZ4-00R308, LED Engin). LEDs for live imaging experiments were pE-4000 from CoolLED. LED-stand was 3D printed in house and equipped with 740 nm (LED740-01AU) and 660 nm LEDs (LED660N-03) with intensities of 2000 µmol m <sup>−2</sup> s <sup>−1</sup> and 300 µmol m <sup>−2</sup> s <sup>−1</sup>, respectively. Light intensities and spectra were measured using an AvaSpec-ULS2048 fiber optic spectrometer (Avantes BV). All cell handling involving the PhyB-based systems was done under safe 520 nm green light. </p> </div> <div class="phyb1–651production" title="sec"> <div class="title" tagx="title" title="title">
PhyB <sub>1–651</sub> production </div> <p>Biotinylated PhyB <sub>1–651</sub> was produced in <i>Escherichia coli</i> BL21 (DE3) (Invitrogen) together with the enzymes Ho and PcyA for biosynthesis of phycocyanobilin <sup> <a href="#CR32">32</a> </sup>. PhyB <sub>1–651</sub> was purified by IMAC. Protein purity and chromophore incorporation were verified by Coomassie staining and 1 mM zinc acetate staining (Supplementary Figure <a href="#MOESM5">2a</a>). Protein concentration was determined by Bradford assay (Bio-Rad, cat. no.: 500-0006) using bovine serum albumin (BSA, Sigma Aldrich, cat. no.: 05479) as standard. For flow cytometry staining of cells, PhyB <sub>1–651</sub> was dialysed against PBS (2.7 mM KCl, 1.5 mM KH <sub>2</sub>PO <sub>4</sub>, 8.1 mM Na <sub>2</sub>HPO <sub>4</sub>, 137 mM NaCl) using SnakeSkin Dialysis Tubes (Fisher Scientific, cat. no.: 10005743). </p> </div> <div class="coatingglasswithphyb1–651" title="sec"> <div class="title" tagx="title" title="title">
Coating glass with PhyB <sub>1–651</sub> </div> <p>Glass slides (Carl Roth, cat. no.: P231.1) were washed in methanol (MeOH), dried and immersed in a 1:3 (v:v) solution of 30% (w/w) hydrogen peroxide and 95% (w/w) sulfuric acid for 40 min at 70 °C. After washing three times in deionized water and two times in MeOH, the glass slides were immediately immersed in a 5% (v/v) (3-glycidyloxypropyl)trimethoxysilane (GLYMO, Sigma, cat. no.: 440167) solution in MeOH for 16 h at room temperature. The glass slides were subsequently washed three times in 2-propanol and dried at 105 °C for 1 h (after this, glass could be stored in a dry place) <sup> <a href="#CR24">24</a> </sup>. The slides were then coated with 5 mg mL <sup>−1</sup> NeutrAvidin (Thermo Scientific, cat. no.: 31050, dissolved in PBS) overnight at room temperature (RT), washed three times with PBS, and before final incubation in 3 mg ml <sup>−1</sup> PhyB <sub>1–651</sub> (in elution buffer (50 mM NaH <sub>2</sub>PO <sub>4</sub>, 300 mM NaCl, 250 mM imidazole, pH 8.0)) for 2 h at RT in darkness. </p> </div> <div class="coatingglasswithfibronectin" title="sec"> <div class="title" tagx="title" title="title">
Coating glass with fibronectin</div> <p>Glass slides (Carl Roth, cat. no.: P231.1) were incubated with 200 µl of 25 mg mL <sup>−1</sup> fibronectin in PBS for 2 h at RT. After incubation slides were directly transferred to cell culture medium for further use. </p> </div> <div class="cellcultureandtransienttransfections" title="sec"> <div class="title" tagx="title" title="title">
Cell culture and transient transfections</div> <p>Human embryonic kidney cells (HEK-293T), human cervical cancer cells (HeLa), human breast cancer cells (MCF7) and cell lines stably expressing OptoIntegrin were maintained in Dulbecco’s modified Eagle’s medium (DMEM) complete (PAN, cat. no.: P04-03550) supplemented with 10% (v/v) fetal calf serum (FCS, PAN, cat. no.: P30-3306, lot no.: P140204), 100 U mL <sup>−1</sup> penicillin and 100 μg mL <sup>−1</sup> streptomycin (PAN) at 37 °C in a humidified atmosphere containing 5% CO <sub>2</sub>. For passaging, cells were detached enzymatically using a trypsin/ethylenediaminetetraacetic acid (EDTA) solution (PAN, cat. no.: P10-023500), for attachment experiments, cells were detached using non-enzymatic cell dissociation buffer (Sigma Aldrich, cat. no.: C5914) and placed in DMEM supplemented with 1% FCS, 100 U mL <sup>−1</sup> penicillin and 100 μg mL <sup>−1</sup> streptomycin (PAN). For transient transfections, HEK-293T and HeLa cells were seeded in DMEM complete to obtain 70–80% confluency at time of transfection. After 24 h, cells were transfected using polyethylenimine (PEI, linear, MW: 25 kDa, Polyscience, Warrington, PA, cat. no.: 23966‐2), as described elsewhere <sup> <a href="#CR33">33</a> </sup>. After 6 h the medium was exchanged with fresh DMEM complete. Following incubation for 24–48 h in the dark, the cells were used for experiments. </p> </div> <div class="generationofstableoptointegrincelllines" title="sec"> <div class="title" tagx="title" title="title">
Generation of stable OptoIntegrin cell lines</div> <p>For production of VSV-G-pseudotyped lentiviral particles, 9 × 10 <sup>4</sup> HEK-293T cells cm <sup>−2</sup> were seeded in DMEM complete. After 24 h, the cells were transfected using PEI with the packaging plasmids pLTR-G <sup> <a href="#CR34">34</a> </sup>, pCD/NL-BH* ΔΔΔ <sup> <a href="#CR35">35</a> </sup> and the OptoIntegrin-encoding plasmid pJB021 in a mass ratio of 1:1:2. 5 h after transfection, the medium was replaced by DMEM lenti (Advanced DMEM (Thermo Fisher, cat. no.: 12491015), 2% FCS, 100 U mL <sup>−1</sup> penicillin and 100 μg mL <sup>−</sup>1 streptomycin, 10 μM cholesterol, 10 μM egg lecithin (Serva Electrophoresis, cat. no.: 27608), 1 × chemical defined lipid concentrate (Thermo Fisher, cat. no.: 11905031)) <sup> <a href="#CR36">36</a> </sup>. The supernatant containing the viral particles was harvested after 48 h and filtered through a 0.45 μm C-AS filter (GE Healthcare, cat. no.: 514-1113). Clarified supernatant was added to each well of a 6-well plate containing HEK-293T, HeLa or MCF7 cells at 70–80% confluency. After 72 h, the transduced cells were harvested and stained with primary anti-HA antibody (Sigma, cat. no.: H9658, diluted 1:100 in PBS supplemented with 2% FCS) and subsequently with Alexa Fluor 488-labelled donkey anti-mouse IgG (Thermo Fisher, cat. no.: A2102, diluted 1:200 in PBS containing 2% FCS) for fluorescence-activated cell sorting (S3e Cell Sorter, Bio-Rad). For this, the cells were washed in PBS supplemented with 2% FCS and incubated with anti-HA for 25 min on ice. Subsequently, cells were washed twice with PBS containing 2% FCS and incubated with secondary Alexa Fluor 488-labelled donkey anti-mouse IgG for 25 min on ice. The cells were then washed three times and resuspended in PBS containing 2% FCS for sorting of high expressing population by fluorescence-activated cell sorting (Supplementary Figure <a href="#MOESM5">3</a>). For the control cell line expressing pJB031 the same procedure was applied, but cells were not sorted after transduction. </p> </div> <div class="flowcytometrystainingusingphyb1–651" title="sec"> <div class="title" tagx="title" title="title">
Flow cytometry staining using PhyB <sub>1–651</sub> </div> <p>Cells were detached using non-enzymatic cell dissociation buffer. Cells were subsequently washed with PBS supplemented with 10% FCS before incubation in PhyB <sub>1–651</sub> (1.5 mg mL <sup>−1</sup>) in PBS supplemented with 2% FCS for 1 h at 4 °C. This incubation step was performed under 660 nm or 740 nm light (20 µmol m <sup>−2</sup> s <sup>−1</sup>). Then cells were washed twice with ice-cold PBS and resuspended in PBS supplemented with 2% FCS. PhyB <sub>1–651</sub> fluorescence was measured with a GALLIOS flow cytometer (Beckman Coulter) using 638 nm red excitation laser and a 660/20 emission filter. </p> </div> <div class="attachmentassays" title="sec"> <div class="title" tagx="title" title="title">
Attachment assays</div> <p>For attachment assays, cells were detached using non-enzymatic cell dissociation solution and seeded on PhyB <sub>1–651</sub>- or fibronectin-coated glass slides and illuminated as indicated. For live cell imaging and spatial control experiments, cells were seeded on PhyB <sub>1–651</sub> coated glass slides placed in an IBIDI glass bottom dish (cat. no.: 81158), illuminated with 740 nm until imaging started, where cells were subsequently illuminated as indicated and kept at 37 °C and 5% CO <sub>2</sub>. Illumination intensities for live cell imaging ranged for 660 nm between 30 and 60 µmol m <sup>−2</sup> s <sup>−1</sup> and for 740 nm from 90–120 µmol m <sup>−2</sup> s <sup>−1</sup>. Images were taken using either an Axio Observer Z1/7 microscope (Zeiss) with a 20 × objective or an EVOS Xl transmitted light microscope (Fisher Scientific). Binary images of cell shapes were obtained with Fiji <sup> <a href="#CR37">37</a> </sup>. Cell spreading area was determined by dividing pictures into smaller tiles (HEK-293T: 100 × 100 PxI; HeLa/MCF7: 300 × 300 PxI) from which 30 random tiles for HEK-293T and 10 for HeLa/MCF7 were manually segmented to determine cell spreading area. Area was determined using Green formula from OpenCV <sup> <a href="#CR38">38</a> </sup>. </p> </div> <div class="immunostaining" title="sec"> <div class="title" tagx="title" title="title">
Immunostaining</div> <p>For paxillin and YAP1 staining, untransfected or OptoIntegrin-expressing HEK-293T cells were seeded at a density of ~1 × 10 <sup>5</sup> cells cm <sup>−2</sup> on glass coverslips coated with PhyB <sub>1–651</sub> or fibronectin, and subsequently illuminated as indicated. HeLa cells were transfected (as described above) with pCMV-LifeAct-TagGFP2 (Ibidi, cat. no.: 60101) 24 h before the experiment and subsequently handled in the same manner. Cells were fixed with 4% (w/v) paraformaldehyde in PBS for 15 min under green light conditions. Following a washing step with PBS, samples were permeabilized with PBS containing 0.5% (v/v) Triton X-100 for 10 min and blocked for 1 h with PBS containing 1% (w/v) BSA before being incubated for 1 h with the anti-paxillin primary antibody (Atlas Antibodies, cat. no.: HPA051309, diluted 1:100 in blocking buffer) or anti-YAP1 primary antibody (Santa Cruz, cat. no.: sc-101199, diluted 1:100 in blocking buffer). After washing with PBS-T (PBS with 0.05% (v/v) Tween-20), samples were incubated for 1 h with an Alexa Fluor 647-labelled goat anti-rabbit IgG (Thermo Scientific, cat. no.: A-21244, diluted 1:200 in blocking buffer) or Alexa Fluor 488-labelled donkey anti-mouse IgG (Thermo Fisher, cat. no.: A2102, diluted 1:200 in blocking buffer) and washed again with PBS-T. Finally, the stained coverslips were mounted on microscope slides with Mowiol 4-88 (Carl Roth, cat. no.: 0713), and confocal images were acquired on an upright confocal microscope (Nikon Instruments Eclipse Ni-E with a C2 confocal laser scanner, 100 × oil objective NA = 1.45 or 60× oil objective NA = 1.40). TagGFP2 and Alexa Fluor 488 or Alexa Fluor 647 were excited with a 488 or 647 nm laser, respectively. Image analysis was performed with Fiji software <sup> <a href="#CR37">37</a> </sup>. Subcellular localization of YAP1 or paxillin was manually evaluated. </p> </div> <div class="westernblot" title="sec"> <div class="title" tagx="title" title="title">
Western blot</div> <p>Cells were seeded on PhyB <sub>1–651</sub> coated glass slides at a density of 1.3 × 10 <sup>5</sup> cells cm <sup>−2</sup> and treated with indicated light conditions. Cells were then immediately lysed using ice-cold lysis buffer (20 mM Tris-HCl, 100 mM NaCl, 1 mM EDTA, 0.5% (v/v) Triton X-100, 0.1% (w/v) SDS) supplemented with protease inhibitors (complete protease inhibitor cocktail, Roche, cat. no.: 04693116001) and phosphatase inhibitors (1 mM sodium orthovanadate, 10 mM sodium pyrophosphate, 50 mM sodium fluoride, 10 mM β-glycerophosphate). For expression study of integrin β subunit cells were lysed 24 h after transfection without any further treatments. After 10 min incubation on ice, lysates were transferred to −80 °C for further lysis. Lysates were thawed on ice and centrifuged at 10,000 ×  <i>g</i> at 4 °C for 10 min. The supernatants were mixed with 5× SDS loading buffer (50% (v/v) glycerol, 312.5 mM Tris, 0.05% (w/v) bromphenol blue, 10% (w/v) SDS, 12.5% (v/v) 2-mercaptoethanol), boiled at 95 °C for 5 min, separated by sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) and transferred to polyvinylidene fluoride (PVDF) membranes. The membranes were blocked in blocking buffer (TBS-T (TBS (20 mM Tris, 150 mM NaCl, pH 7.4) containing 0.05% (v/v) Tween-20) supplemented with 5% (w/v) BSA (Sigma-Aldrich, cat. no.: 05479)) at RT for 1 h. Primary antibody incubation was conducted at 4 °C overnight or for 1 h at RT in blocking buffer. After three washing steps with TBS-T, the blots were incubated with the corresponding horseradish peroxidase-coupled secondary antibody in blocking buffer at RT for 1 h. Following three washing steps with TBS-T, chemiluminescence was detected with ECL substrate using the ImageQuant LAS-4000 mini system (GE Healthcare, cat. no.: 28-9558-13). For Western blotting the following primary antibodies (diluted 1:1000 in blocking buffer) were used for Western blotting: mouse anti-HA (Sigma, cat. no.: H9658), rabbit anti-pERK1/2 (CST, cat. no.: 9101S), rabbit anti-GAPDH (CST, cat. no.: 5174S), rabbit anti-pFAK Y397 (CST, cat. no.:3283). The following secondary antibodies were used for detection: goat anti-rabbit-HRP (CST, cat. no.: 70745; 1:3000) and goat anti-mouse-HRP (Santa Cruz, cat. no.: sc-2005; 1:5000). All uncropped blots are presented in Supplementary Figures <a href="#MOESM5">9</a> and <a href="#MOESM5">10</a>. </p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>Cochran–Mantel–Haenszel-Test (continuity correction: 0.5, χ <sup>2</sup> with one degree of freedom) was performed to analyse multiple 2 × 2 tables for paxillin staining. The Chi-Square test of independence with two degrees of freedom was used to analyse independent 2 × 3 tables from YAP1 staining. Welch-test was used to determine two test null hypothesis for difference of cell surface areas. Results were considered as significant when <i>p</i> &amp;lt; 0.01. For statistical analysis Excel 2013, Python 3.6.6 and GraphPad Prism7 were used. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="42003_2018_264_MOESM1_ESM.docx">LINK</a>  <p>Description of Supplementary Video</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="42003_2018_264_MOESM2_ESM.avi">LINK</a>  <p>Supplementary Movie 1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="42003_2018_264_MOESM3_ESM.avi">LINK</a>  <p>Supplementary Movie 2</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="42003_2018_264_MOESM4_ESM.avi">LINK</a>  <p>Supplementary Movie 3</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="42003_2018_264_MOESM5_ESM.pdf">LINK</a>  <p>Supplementary Information</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Publisher’s note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p> <b>Supplementary Information</b> accompanies this paper at 10.1038/s42003-018-0264-7. </p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank the Life Imaging Center (LIC) for providing equipment and assistance and the BIOSS Signalling Factory for technical support. We are grateful to Dr. Christina Groß for critically reading the manuscript and providing valuable input on the manuscript. We thank Prof. Matias Zurbriggen for constructive discussions and Dr. Hannes Beyer for technical advice. We thank Prof. Ingo Ahrens for kindly providing the plasmids pcDNA3.1- <i>ITGAV</i> and pZeoSV- <i>ITGB3</i>. We thank Katja Kolar for providing purified fibronectin and Laura Börnert for supporting the data analysis. This work was supported by the Excellence Initiative of the German Federal and State Governments (BIOSS-EXC-294 and SGBM-GSC-4). </p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>J.B., S.Z., W.W.D.M. and O.S.Y. performed research. M.H. provided technical advice. J.B., W.W., M.H., G.R. and W.W.A.S. planned the study. W.W. and W.W.A.S. supervised research and/or discussed results. J.B. and W.W. wrote the manuscript. </p> <div class="title" tagx="title" title="title">
Data availability</div> <p>The datasets generated during and/or analysed during the current study are available in the Figshare repository: <a href="https://figshare.com/s/fc63580c3f69355423cc">https://figshare.com/s/fc63580c3f69355423cc</a>. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolar</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Synthetic biological approaches to optogenetically control cell signaling</span> <span class="source" tagx="source" title="source">Curr. Opin. Biotechnol.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">112</span> <span class="lpage" tagx="lpage" title="lpage">119</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.copbio.2017.06.010">10.1016/j.copbio.2017.06.010</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28715701">28715701</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Leopold, A. V., Chernov, K. G. &amp;amp; Verkhusha, V. V. Optogenetically controlled protein kinases for regulation of cellular signaling. <i>Chem. Soc. Rev</i>. (2018). 10.1039/C7CS00404D. </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nihongaki</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otabe</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Emerging approaches for spatiotemporal control of targeted genome with inducible CRISPR-Cas9</span> <span class="source" tagx="source" title="source">Anal. Chem.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">90</span> <span class="fpage" tagx="fpage" title="fpage">429</span> <span class="lpage" tagx="lpage" title="lpage">439</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acs.analchem.7b04757">10.1021/acs.analchem.7b04757</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29161010">29161010</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolar</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knobloch</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stork</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Žnidarič</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">OptoBase: A web platform for molecular optogenetics</span> <span class="source" tagx="source" title="source">ACS Synth. Biol.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">1825</span> <span class="lpage" tagx="lpage" title="lpage">1828</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acssynbio.8b00120">10.1021/acssynbio.8b00120</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29913065">29913065</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kadem</span> <span class="given-names" tagx="given-names" title="given-names">LF</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High-frequency mechanostimulation of cell adhesion</span> <span class="source" tagx="source" title="source">Angew. Chem. - Int. Ed.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">225</span> <span class="lpage" tagx="lpage" title="lpage">229</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/anie.201609483">10.1002/anie.201609483</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nanoscale optomechanical actuators for controlling mechanotransduction in living cells</span> <span class="source" tagx="source" title="source">Nat. Methods</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">143</span> <span class="lpage" tagx="lpage" title="lpage">146</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.3689">10.1038/nmeth.3689</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26657558">26657558</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rosales</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anseth</span> <span class="given-names" tagx="given-names" title="given-names">KS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The design of reversible hydrogels to capture extracellular matrix dynamics</span> <span class="source" tagx="source" title="source">Nat. Rev. Mater.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">15</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/natrevmats.2015.12">10.1038/natrevmats.2015.12</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valon</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marín-Llauradó</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyatt</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Charras</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trepat</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optogenetic control of cellular forces and mechanotransduction</span> <span class="source" tagx="source" title="source">Nat. Commun.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">14396</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms14396">10.1038/ncomms14396</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28186127">28186127</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Yüz, S. G., Ricken, J. &amp;amp; Wegner, S. V. Independent control over multiple cell types in space and time using orthogonal blue and red light switchable cell interactions. <i>Adv. Sci</i>. 1800446 (2018). 10.1002/advs.201800446. </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tillberg</span> <span class="given-names" tagx="given-names" title="given-names">PW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boyden</span> <span class="given-names" tagx="given-names" title="given-names">ES</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A fully genetically encoded protein architecture for optical control of peptide ligand concentration </span> <span class="source" tagx="source" title="source">Nat. Commun.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">3019</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms4019">10.1038/ncomms4019</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24407101">24407101</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fässler</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Integrin-mediated mechanotransduction</span> <span class="source" tagx="source" title="source">J. Cell Biol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">215</span> <span class="fpage" tagx="fpage" title="fpage">445</span> <span class="lpage" tagx="lpage" title="lpage">456</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1083/jcb.201609037">10.1083/jcb.201609037</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27872252">27872252</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campbell</span> <span class="given-names" tagx="given-names" title="given-names">ID</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Humphries</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Integrin structure, activation, and interactions. <i>Cold Spring Harb</i> </span> <span class="source" tagx="source" title="source">Prespectives Biol.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">15</span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Desgrosellier</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheresh</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Integrins in cancer: biological implications and therapeutic opportunities</span> <span class="source" tagx="source" title="source">Nat. Rev. Cancer</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">9</span> <span class="lpage" tagx="lpage" title="lpage">22</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrc2748">10.1038/nrc2748</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20029421">20029421</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">αvβ3 integrin crystal structures and their functional implications</span> <span class="source" tagx="source" title="source">Biochemistry</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">8814</span> <span class="lpage" tagx="lpage" title="lpage">8828</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/bi300734n">10.1021/bi300734n</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23106217">23106217</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khanna</span> <span class="given-names" tagx="given-names" title="given-names">WR</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A novel molecular recognition motif necessary for targeting photoactivated phytochrome signaling to specific basic helix-loop-helix transcription factors </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">3033</span> <span class="lpage" tagx="lpage" title="lpage">3044</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.104.025643">10.1105/tpc.104.025643</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15486100">15486100</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levskaya</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiner</span> <span class="given-names" tagx="given-names" title="given-names">OD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">WA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voigt</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Spatiotemporal control of cell signalling using a light-switchable protein interaction</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">461</span> <span class="fpage" tagx="fpage" title="fpage">997</span> <span class="lpage" tagx="lpage" title="lpage">1001</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature08446">10.1038/nature08446</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19749742">19749742</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quail</span> <span class="given-names" tagx="given-names" title="given-names">PH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phytochrome photosensory signalling networks</span> <span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">85</span> <span class="lpage" tagx="lpage" title="lpage">93</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrm728">10.1038/nrm728</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11836510">11836510</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burgie</span> <span class="given-names" tagx="given-names" title="given-names">ES</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bussell</span> <span class="given-names" tagx="given-names" title="given-names">AN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walker</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dubiel</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vierstra</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Crystal structure of the photosensing module from a red/far-red light-absorbing plant phytochrome </span> <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">111</span> <span class="fpage" tagx="fpage" title="fpage">10179</span> <span class="lpage" tagx="lpage" title="lpage">10184</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1403096111">10.1073/pnas.1403096111</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24982198">24982198</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsushita</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mochizuki</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagatani</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Dimers of the N-terminal domain of phytochrome B are functional in the nucleus</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">424</span> <span class="fpage" tagx="fpage" title="fpage">571</span> <span class="lpage" tagx="lpage" title="lpage">574</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature01837">10.1038/nature01837</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12891362">12891362</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Yousefi, O. S. <i>et al</i>. Optogenetic control shows that kinetic proofreading regulates the activity of the T cell receptor. <i>bioRxiv</i>. 432740 (2018). 10.1101/432740. </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiong</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Crystal structure of the extracellular segment of integrin αVβ3 in complex with an Arg-Gly-Asp ligand </span> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">296</span> <span class="fpage" tagx="fpage" title="fpage">151</span> <span class="lpage" tagx="lpage" title="lpage">155</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1069040">10.1126/science.1069040</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11884718">11884718</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">2A self-cleaving peptide-based multi-gene expression system in the silkworm Bombyx mori </span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">16273</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep16273">10.1038/srep16273</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26537835">26537835</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barczyk</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carracedo</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gullberg</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Integrins</span> <span class="source" tagx="source" title="source">Cell Tissue Res.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">339</span> <span class="fpage" tagx="fpage" title="fpage">269</span> <span class="lpage" tagx="lpage" title="lpage">280</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00441-009-0834-6">10.1007/s00441-009-0834-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19693543">19693543</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orlov</span> <span class="given-names" tagx="given-names" title="given-names">AV</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Development of immunoassays using interferometric real-time registration of their kinetics </span> <span class="source" tagx="source" title="source">Acta Nat.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">85</span> <span class="lpage" tagx="lpage" title="lpage">95</span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turner</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Paxillin and focal adhesion signalling</span> <span class="source" tagx="source" title="source">Nat. Cell Biol.</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">231</span> <span class="lpage" tagx="lpage" title="lpage">236</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/35046659">10.1038/35046659</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Renshaw</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">XD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwartz</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Growth factor activation of MAP kinase requires cell adhesion</span> <span class="source" tagx="source" title="source">EMBO J.</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">5592</span> <span class="lpage" tagx="lpage" title="lpage">5599</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/emboj/16.18.5592">10.1093/emboj/16.18.5592</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9312018">9312018</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dupont</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Role of YAP/TAZ in mechanotransduction</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">474</span> <span class="fpage" tagx="fpage" title="fpage">179</span> <span class="lpage" tagx="lpage" title="lpage">184</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature10137">10.1038/nature10137</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21654799">21654799</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elosegui-Artola</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mechanical regulation of a molecular clutch defines force transmission and transduction in response to matrix rigidity </span> <span class="source" tagx="source" title="source">Nat. Cell Biol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">540</span> <span class="lpage" tagx="lpage" title="lpage">548</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncb3336">10.1038/ncb3336</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27065098">27065098</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chaudhuri</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Substrate stress relaxation regulates cell spreading</span> <span class="source" tagx="source" title="source">Nat. Commun.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">6365</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms7365">10.1038/ncomms7365</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">NG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gumbiner</span> <span class="given-names" tagx="given-names" title="given-names">BM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Adhesion to fibronectin regulates Hippo signaling via the FAK-Src-PI3K pathway</span> <span class="source" tagx="source" title="source">J. Cell Biol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">210</span> <span class="fpage" tagx="fpage" title="fpage">503</span> <span class="lpage" tagx="lpage" title="lpage">515</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1083/jcb.201501025">10.1083/jcb.201501025</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26216901">26216901</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Müller</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zurbriggen</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Control of gene expression using a red- and far-red light-responsive bi-stable toggle switch </span> <span class="source" tagx="source" title="source">Nat. Protoc.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">622</span> <span class="lpage" tagx="lpage" title="lpage">632</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nprot.2014.038">10.1038/nprot.2014.038</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24556785">24556785</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">RW</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Unearthing the transition rates between photoreceptor conformers</span> <span class="source" tagx="source" title="source">BMC Syst. Biol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">110</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12918-016-0368-y">10.1186/s12918-016-0368-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27884151">27884151</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Müller</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Multi-chromatic control of mammalian gene expression and signaling</span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">e124</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt340">10.1093/nar/gkt340</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23625964">23625964</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reiser</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transduction of nondividing cells using pseudotyped defective high-titer HIV type 1 particles </span> <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">93</span> <span class="fpage" tagx="fpage" title="fpage">15266</span> <span class="lpage" tagx="lpage" title="lpage">15271</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.93.26.15266">10.1073/pnas.93.26.15266</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8986799">8986799</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">XY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">La Russa</span> <span class="given-names" tagx="given-names" title="given-names">VF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reiser</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transduction of bone-marrow-derived mesenchymal stem cells by using lentivirus vectors pseudotyped with modified RD114 envelope glycoproteins </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">78</span> <span class="fpage" tagx="fpage" title="fpage">1219</span> <span class="lpage" tagx="lpage" title="lpage">1229</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.78.3.1219-1229.2004">10.1128/JVI.78.3.1219-1229.2004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14722277">14722277</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitta</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rimann</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fussenegger</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Detailed design and comparative analysis of protocols for optimized production of high-performance HIV-1-derived lentiviral particles </span> <span class="source" tagx="source" title="source">Metab. Eng.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">426</span> <span class="lpage" tagx="lpage" title="lpage">436</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ymben.2005.06.006">10.1016/j.ymben.2005.06.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16102993">16102993</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schindelin</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fiji: an open-source platform for biological-image analysis</span> <span class="source" tagx="source" title="source">Nat. Methods</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">676</span> <span class="lpage" tagx="lpage" title="lpage">682</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.2019">10.1038/nmeth.2019</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22743772">22743772</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bradski, G. The OpenCV Library. <i>Dr. Dobb’s J. Softw. Tools</i> (2000). </span> </li> </ul> </div> </div>  </body></html>