<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Nat Commun</span> <span class="iso-abbrev" title="iso-abbrev">Nat Commun</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Nature Communications</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2041-1723</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6240110">6240110</a> </span> <span class="publisher-id" title="publisher-id">7286</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41467-018-07286-8">10.1038/s41467-018-07286-8</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Optogenetic dissection of Rac1 and Cdc42 gradient shaping</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Beco</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaidžiulytė</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manzi</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dalier</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">di Federico</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cornilleau</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dahan</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coppey</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">mathieu.coppey@curie.fr</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 2308 1657</span> <span class="institution-id" title="institution-id">grid.462844.8</span> <span class="institution" title="institution">PASTEUR, Département de chimie, École normale supérieure, CNRS UMR 8640, </span> <span class="institution" title="institution">PSL Research University, Sorbonne Université, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-11-11</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-11-11</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">4816</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-4-3</span> <span class="accepted" title="accepted">accepted: 2018-10-19</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>During cell migration, Rho GTPases spontaneously form spatial gradients that define the front and back of cells. At the front, active Cdc42 forms a steep gradient whereas active Rac1 forms a more extended pattern peaking a few microns away. What are the mechanisms shaping these gradients, and what is the functional role of the shape of these gradients? Here we report, using a combination of optogenetics and micropatterning, that Cdc42 and Rac1 gradients are set by spatial patterns of activators and deactivators and not directly by transport mechanisms. Cdc42 simply follows the distribution of Guanine nucleotide Exchange Factors, whereas Rac1 shaping requires the activity of a GTPase-Activating Protein, β2-chimaerin, which is sharply localized at the tip of the cell through feedbacks from Cdc42 and Rac1. Functionally, the spatial extent of Rho GTPases gradients governs cell migration, a sharp Cdc42 gradient maximizes directionality while an extended Rac1 gradient controls the speed. </p> </div> <div class="abstract" title="abstract"> <p>A steep gradient of Cdc42 is at the front of migrating cells, whereas the active Rac1 gradient is graded. Here the authors show that Cdc42 gradients follow the distribution of GEFs and govern direction of migration, while Rac1 gradients require the activity of the GAP β2-chimaerin and control cell speed. </p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Cell migration plays a major role in various biological functions, including embryonic development, immune response, wound closure, and cancer invasion. Cells, either isolated or in cohesive groups, are able to respond to many types of spatially distributed environmental cues, including gradients of chemoattractants <sup> <a href="#CR1">1</a>, <a href="#CR2">2</a> </sup>, of tissue stiffness (durotaxis) <sup> <a href="#CR3">3</a>– <a href="#CR5">5</a> </sup>, and of adhesion (haptotaxis) <sup> <a href="#CR6">6</a>, <a href="#CR7">7</a> </sup>. To sense and orient their migration accordingly, cells need to integrate complex and noisy signals and to polarize along the selected direction. A simple explanation for such directed migration would be to consider that external gradients are directly translated into internal gradients. However, recent works <sup> <a href="#CR8">8</a>– <a href="#CR10">10</a> </sup> point to a two-tiered mechanism. First, a set of signaling proteins (Rho GTPases and Ras) behave as an excitable system that spontaneously establish intracellular membrane-bound gradients, conferring the ability of cells to polarize even in the absence of external stimuli. Second, a sensing machinery based on membrane receptors aligns the polarization axis along the direction of external gradient cues. In the present work, we address the mechanisms shaping the Rho GTPases gradients at the front of randomly migrating cells. </p> <p>Rho GTPases are known to play a key role in orchestrating the spatially segregated activities that define the polarity axis of migrating cells. At the cell front, membrane protrusions fueled by actin polymerization push the cell forward, while retraction of the cell back depends on acto-myosin contractility <sup> <a href="#CR11">11</a>– <a href="#CR13">13</a> </sup>. The schematic view is that front-to-back gradients of Cdc42 and Rac1 define the cellular front, while RhoA is mostly active at the back. Cdc42 is known to be required for filopodia formation, through N-WASP-mediated activation of the ARP2/3 complex as well as F-actin bundling proteins such as fascin and formin <sup> <a href="#CR11">11</a>, <a href="#CR14">14</a> </sup>. Conversely, Rac1 is involved in branched actin polymerization and lamellipodia formation, through WAVE-mediated activation of the ARP2/3 complex <sup> <a href="#CR15">15</a> </sup>. RhoA is responsible for stress fiber formation and retraction of the cellular tail through Rho kinase-mediated contraction of myosin II <sup> <a href="#CR16">16</a>, <a href="#CR17">17</a> </sup>. In reality the situation is more complex since RhoA is also active at the very front of migrating mouse embryonic fibroblasts <sup> <a href="#CR18">18</a>, <a href="#CR19">19</a> </sup> and is involved in actin polymerization through Diaphanous-related formins as well as focal adhesions <sup> <a href="#CR20">20</a>, <a href="#CR21">21</a> </sup>. In addition, the Rho GTPase family contains more than the three members aforementioned, with more than 20 proteins having been discovered <sup> <a href="#CR20">20</a>, <a href="#CR22">22</a> </sup>. Despite the fact that these other members are classified in the three Cdc42, Rac1, and RhoA sub-families, they present overlapping activities. </p> <p>Three main classes of proteins regulate the activity of Rho GTPases. Guanine Exchange Factors (GEFs) activate Rho GTPases by promoting the exchange from GDP to GTP, whereas GTPase-activating proteins (GAPs) inhibit Rho GTPases by catalyzing the hydrolysis of GTP <sup> <a href="#CR23">23</a> </sup>. A multitude of GEFs and GAPs ensure signaling specificity and fine-tuned regulation. In addition, guanine-nucleotide dissociation inhibitors (GDIs) are negative regulators of Rho GTPases, extracting them from the plasma membrane and blocking their interactions with GEFs <sup> <a href="#CR24">24</a>, <a href="#CR25">25</a> </sup>. GEFs and GAPs can be localized and activated by upstream factors such as receptor tyrosine kinases or interaction with lipids such as PIP3 <sup> <a href="#CR26">26</a>, <a href="#CR27">27</a> </sup>, hereby connecting the polarization machinery with the sensing one. Moreover, complex crosstalks connect Rho GTPases and their interactors, resulting in a signaling network that finely regulates Rho GTPases activities. Although many molecular interactions defining this signaling network have been characterized, we currently have little insight on how these interactions are orchestrated in space to shape Rho GTPase activity patterns. </p> <p>Positive feedbacks acting on Rac1, Cdc42, and RhoA have been proposed to account for their ability to form gradients spontaneously. Rho GTPase activity pulses would be generated thanks to an excitable system <sup> <a href="#CR9">9</a> </sup> and specific activators like GEFs would orient and stabilize them <sup> <a href="#CR8">8</a>, <a href="#CR28">28</a> </sup>. Yet, activity patterns governed by excitable systems have a propensity to propagate through the whole cell, and inhibitory mechanisms are required to limit their expansion <sup> <a href="#CR29">29</a>– <a href="#CR31">31</a> </sup>. Three mechanisms could confine Rho GTPases activities. First, Rho GTPase cycles can be locally regulated by GEF and GAP concentrations, whose distributions along the cell would shape Rho GTPase intracellular gradients <sup> <a href="#CR31">31</a>– <a href="#CR34">34</a> </sup>. Second, anchoring or trapping in the cortical acto-myosin network can decrease diffusion considerably. Since Rho GTPases trigger actin polymerization and branching, this mechanism could act as a negative feedback restricting their activity zones. Third, Rho GTPase extraction from the plasma membrane by GDIs can be locally regulated <sup> <a href="#CR25">25</a> </sup>, such that deactivation regions could be set by the activity of GDIs. It is unclear which of these mechanisms is responsible for the formation of Rho GTPase intracellular spatial patterns. </p> <p>In this work, we show that Cdc42 and Rac1 gradients are formed thanks to a combination of distributed GEFs and GAPs and not directly by diffusion or actin retrograde flow from a localized source. A combination of experimental approaches and minimal mathematical model suggests that: (i) the amount of active Cdc42 simply follows its GEFs distribution thanks to a uniform GAP activity, (ii) the Rac1 gradient requires an additional inhibition at the front by the β2-chimaerin GAP that shifts its peak of activity and hereby increases its spatial extent. We show that the localized activity of β2-chimaerin depends on both Cdc42 and Rac1, forming a negative feedback on Rac1, and that the actin retrograde flow is required for β2-chimaerin enrichment. Finally, we show that the resulting spatial properties of Cdc42 and Rac1 gradients govern the directionality and the speed of cell movement, respectively. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="cdc42andrac1gradientsshowtwodistinctshapesatthefrontofmigratingcells" title="sec"> <div class="title" tagx="title" title="title">
Cdc42 and Rac1 gradients show two distinct shapes at the front of migrating cells</div> <p>We investigated the spatial activity gradients of Cdc42 and Rac1 Rho GTPases at the basal plasma membrane by imaging FRET biosensors based on an intramolecular fusion between Rac1 and a PAK1 binding domain <sup> <a href="#CR35">35</a> </sup>. HeLa cells stably expressing FRET reporters were left to migrate randomly on glass coverslips, and were imaged using total internal reflection fluorescence (TIRF) microscopy. The FRET ratio was calculated as a proxy for GTPase activity. Front-to-back gradients of either Cdc42 or Rac1 activity were measured from the cell protruding edge to the nucleus (Fig. <a href="#Fig1">1a</a>). As previously reported in neutrophils <sup> <a href="#CR9">9</a> </sup>, we observed gradients that differed both in shape and in spatial extent. Cdc42 gradient was steep and monotonous, peaking at the protruding edge, and presenting an exponentially decaying profile of characteristic length <i>d</i> = 8.3 µm ± 0.6 µm (SEM, <i>n</i> = 19). In contrast, Rac1 gradient peaked at a distance <i>d</i> = 5.8 ± 0.5 µm from the cell edge, and then decayed with a characteristic length <i>d</i> = 9.6 µm ± 0.7 µm (characteristic length of the exponentially decaying part, <i>n</i> = 31). We defined the extent of the gradient by the distance between the tip of the cell and the point where the signal reaches half-amplitude. The extent for Rac1 was <i>d</i> = 14.6 ± 0.7 µm, compared to <i>d</i> = 8.9 ± 0.6 µm for Cdc42 (Fig. <a href="#Fig1">1b</a>). Interestingly, these observations match those reported for gradients in other cell lines <sup> <a href="#CR9">9</a>, <a href="#CR36">36</a> </sup>. We thus questioned what could be the mechanisms generating these gradients and accounting for their distinct shapes. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>Rac1 and Cdc42 activity gradients have different shapes. <b>a</b> FRET biosensors were used to monitor Rac1 (top) or Cdc42 (bottom) activity in freely migrating HeLa cells. GTPase activity is measured by the FRET ratio, and represented with a color scale. Several representative cells are shown. Scale bar: 20 µm. <b>b</b> Mean normalized FRET ratio of Rac1 (red) and Cdc42 (blue) is plotted as a function of the distance from the cell edge. The error bars indicate the standard deviation (s.d.) of <i>n</i> = 31 (Rac1) or <i>n</i>=19 (Cdc42) cells. Black segments at the top show positions at which the curves are statistically different ( <i>p</i> &amp;lt; 0.05, Wilcoxon’s rank sum test). <b>c</b> Rho GTPase cycle, where the protein switches between an inactive and active state thanks to activators (GEFs) and deactivators (GAPs). <b>d</b>, <b>e</b> Two simplified mechanisms can explain the formation of cellular-scale Rho GTPase gradients. <b>d</b> A sharply localized GEF (blue profile) acts as a punctual source of active Rho GTPases (red) that are further transported by diffusion or flow (dashed gray arrows) until they reverse to the inactive state thanks to a GAP (black). <b>e</b> A cellular-scale distributed GEF locally activates the Rho GTPase such that both have the same profile </p>   </div> <p /> <p>Two generic classes of models can account for the patterning of spatially graded distributions <sup> <a href="#CR29">29</a> </sup>. The first class relies on transport mechanisms (diffusion, flow) to establish gradients from a localized source (Fig. <a href="#Fig1">1c, d</a>). A canonical example is the synthesis−diffusion−degradation model, which has been heavily discussed in the context of the Bicoid morphogen gradient <sup> <a href="#CR37">37</a> </sup>. The second class of models assumes a graded distribution of activators and deactivators (Fig. <a href="#Fig1">1c, e</a>). In this context, the local concentration is set by the local balance between activation and deactivation. This second class of model has also been proposed to explain the establishment of morphogen gradients, e.g. for the formation of the bone morphogenetic protein gradient that patterns the dorso-ventral axis of the early <i>Xenopus</i> embryo <sup> <a href="#CR38">38</a>, <a href="#CR39">39</a> </sup>. </p> </div> <div class="cdc42andrac1gradientsareshapedbyspatiallydistributedgefsandgapsbutnotbydiffusion" title="sec"> <div class="title" tagx="title" title="title">
Cdc42 and Rac1 gradients are shaped by spatially distributed GEFs and GAPs but not by diffusion </div> <p>In order to distinguish between these two classes of models, we opted for an input−output relationship approach. We used optogenetics <sup> <a href="#CR40">40</a>, <a href="#CR41">41</a> </sup> to impose activation gradients of either Intersectin-1 (ITSN) or T-Cell Lymphoma Invasion and Metastasis 1 (TIAM1), two GEFs specifically activating Cdc42 or Rac1, respectively. We used fusions of CRY2 with the DHPH catalytic domain of ITSN or TIAM to activate specifically Cdc42 or Rac1 <sup> <a href="#CR41">41</a> </sup> (Fig. <a href="#Fig2">2c</a>). A home-made illumination setup using a DMD (Digital Micromirror Device <sup> <a href="#CR42">42</a> </sup>) allowed us to shine spatial gradients of light with an 8-bit gray level resolution. Cells were confined on round micropatterns to prevent cell shape polarity <sup> <a href="#CR43">43</a> </sup> and gradients of light with slopes ranging from 1× to 4× were applied (Fig. <a href="#Fig2">2a</a>). As we could predict in a previous work <sup> <a href="#CR44">44</a> </sup>, recruitment of the cytoplasmic optogenetic partner CRY2 to the basal plasma membrane followed the stimulation signal with the addition of an exponential decaying tail of 5 µm characteristic length due to the diffusion of CIBN-CRY2 dimers at the membrane (Fig. <a href="#Fig2">2b</a>). This allowed us to tune precisely the spatial distribution of desired GEFs and test the relationship between the activation input and the output in terms of GTPase activity. If any transport mechanism (model 1) was taking place, we would expect a difference in the spatial distribution of the output compared to the input. For example, diffusion would give rise to a more extended distribution of the output by the addition of a length scale <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\ell _{{\mathrm{diff}}} = \sqrt {\tau D},$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ℓ</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
diff</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msqrt_UNKNOWN"> <div class="mi_UNKNOWN">
τ</div> <div class="mi_UNKNOWN">
D</div> </div> <div class="mo_UNKNOWN">
,</div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  where <i>τ</i> is the lifetime of the Rho GTPase in its active GTP-bound state and <i>D</i> is its lateral diffusion coefficient. Contrarily, model 2 predicts that the spatial distribution of the output would mirror the distribution of the input, given that deactivators are uniform. Indeed, we reasoned that the optogenetic activation would dominate the other sources of activation such that the input−output relationship would reveal the distribution of the deactivators. <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>Cdc42 and Rac1 have different responses to GEF activation. <b>a</b> We imposed GEF activity gradients of different slopes using optogenetics. Patterned illumination with grayscale levels (left) was shone onto the samples, imposing linear light gradients of the same amplitude but different spatial extents on cells attached on round micropatterns of 35 µm in diameter (top right). The reference gradient called 1× spans over the whole diameter of the cell. The gradient 2× spans over a half cell diameter, and therefore has a slope twice as sharp as for the gradient 1× (bottom right). <b>b</b> Membrane recruitment of the optogenetic partner CRY2-mCherry to the basal side of cells on round micro-patterns following 30 min of illumination with 4× (top left) and 2× (bottom left) gradients. mCherry fluorescence (solid lines) was measured along the cell diameter following illumination with 1× (light gray), 2× (medium gray) and 4× (black) gradients (dashed lines). Error bars indicate the s.d. of <i>n</i> = 15 (1×), <i>n</i> = 24 (2×) and <i>n</i> = 15 (4×) cells. <b>c</b> We activated GEFs of Cdc42 (ITSN) or Rac1 (TIAM) with light gradients and measured the fluorescence pattern of PAK1-iRFP. Schemes at the top represent the fusion proteins used. <b>d</b> PAK1-iRFP recruitment following 4x (top) or 2× (bottom) activation gradients of ITSN. Fluorescence was recorded using TIRFM in HeLa cells on round micro-patterns, and initial fluorescence was subtracted for normalization. Micrographs represent the averaged fluorescence of <i>n</i> = 15 (4×, top) or <i>n</i> = 12 (2×, bottom) cells. Insets show the illumination patterns (not to scale). <b>e</b> Normalized fluorescence of ITSN-CRY2-mCherry (blue) and PAK1-iRFP (purple) was measured along the cell diameter and averaged (solid lines). Error bars: s.d. Gray lines at the top show positions at which the curves are statistically different ( <i>p</i> &amp;lt; 0.05, Wilcoxon’s rank sum test). <b>f</b> PAK1-iRFP recruitment following 4× (top, <i>n</i> = 15) or 2× (bottom, <i>n</i> = 19) activation gradients of TIAM. <b>g</b> Normalized fluorescence of TIAM-CRY2-mCherry (red) and PAK1-iRFP (purple) along the cell diameter of <i>n</i> = 15 (4×, top) or <i>n</i> = 19 (2×, bottom) cells, Error bars: s.d. Gray: Wilcoxon’s rank sum test ( <i>p</i> &amp;lt; 0.05). Single cell data for <b>e</b> and <b>g</b> are presented in Supplementary Figure <a href="#MOESM2">3</a>. Scale bars: 20 µm </p>   </div> <p /> <p>To determine whether Cdc42 and Rac1 followed the imposed activation pattern, we used downstream effectors as reporters of GTPase activity. The protein PAK1 is activated downstream of both Cdc42 and Rac1. We monitored the basal membrane recruitment of a PAK1-iRFP fluorescent reporter following gradient activation of each one of the GTPases (Fig. <a href="#Fig2">2c</a>). As controls, we verified that the observed recruitment of PAK1-iRFP was not due to fluorescence bleed-through or nonspecific activity of CRY2-mCherry (Supplementary Figure <a href="#MOESM2">1a</a>), nor to volume effects or cell deformation (Supplementary Figure <a href="#MOESM2">1b</a>). Importantly, we also verified that the GEF DHPH domains used in our optogenetic approach were truly specific (Supplementary Figure <a href="#MOESM2">2</a>). PAK1-iRFP recruitment patterns followed the activation gradients of ITSN-CRY2 remarkably well, independently of their spatial extents (Fig. <a href="#Fig2">2d, e</a>, Supplementary Movie <a href="#MOESM4">1</a>). We could not detect any significant difference between the PAK1 response and the activating ITSN gradients, independently of their spatial extents (Fig. <a href="#Fig2">2d, e</a>, Supplementary Movie <a href="#MOESM4">1</a>), up to the resolution of our measurement estimated as ~2 µm (two standard deviations of the spatial extent). This result suggests that GEF activity levels are sufficient to shape Cdc42 activity patterns without the requirement of other mechanisms. Conversely, PAK1-iRFP spatial recruitment was independent of the shape of the activating TIAM-CRY2 gradient. It did not follow the sharpest activation gradient (4×), and the peak at 6 µm from the protrusion edge was present from the beginning of the stimulation (Supplementary Figure <a href="#MOESM2">4</a>, Supplementary Movie <a href="#MOESM5">2</a>) despite its absence from the gradients of TIAM-CRY2 (Fig. <a href="#Fig2">2f, g</a>). Interestingly, the PAK1-iRFP gradient obtained with our synthetic approach matched the Rac1 gradient observed in native cells (Fig. <a href="#Fig1">1b</a>). We thus sought to discriminate between two possibilities explaining how the Rac1 gradient is shaped: whether shaping involves transport or nonuniformly distributed deactivators. </p> </div> <div class="acrosstalkbetweencdc42andrac1throughgefsandgapscontributestorac1gradientshaping" title="sec"> <div class="title" tagx="title" title="title">
A crosstalk between Cdc42 and Rac1 through GEFs and GAPs contributes to Rac1 gradient shaping </div> <p>A complex crosstalk between the Cdc42 and Rac1 pathways has been shown previously <sup> <a href="#CR12">12</a>, <a href="#CR45">45</a> </sup>. We questioned whether such network could explain the complex pattern of Rac1 activity we observed. We used the Abi1-iRFP fusion protein as a reporter of Rac1 activity. Abi1 is part of the WAVE complex that has been shown to be activated specifically by Rac1 but not by Cdc42 <sup> <a href="#CR46">46</a> </sup> (Fig. <a href="#Fig3">3a</a>). We observed that Abi1 is activated at the cell edge following TIAM but also ITSN optogenetic activation (Fig. <a href="#Fig3">3b</a>), suggesting that Cdc42 directly or indirectly activates Rac1. Interestingly, in both cases the induced Abi1 recruitment was more restricted to the cell border than the activating gradients (Supplementary Figure <a href="#MOESM2">5</a>). This observation is in accordance with the known distribution of the WAVE complex at the tip of the lamellipodia <sup> <a href="#CR47">47</a> </sup>, suggesting a compartmentalization independent of the immediate Rho GTPase activation. Yet, in addition to the positive crosstalk, we also observed a cross inactivation of Rac1 by Cdc42. When we inhibited Cdc42 by siRNA (Supplementary Figure <a href="#MOESM2">6a</a>), we observed an increase of Rac1 activity at the cell front as measured by the non-normalized FRET profile (Fig. <a href="#Fig3">3c</a>). Strikingly, the bump of Rac1 activity 6 µm from the cell edge was abolished in this condition. Since the overall effect of Cdc42 depletion is to increase Rac1 activity, we reasoned that the dominant role of Cdc42 on Rac1 is to specifically activate a GAP inhibiting Rac1. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>Cdc42 and β2-chimaerin are involved in shaping the activity gradient of Rac1. <b>a</b> We activated GEFs of Cdc42 (ITSN) or Rac1 (TIAM) with light gradients and measured the fluorescence pattern of Abi1-iRFP. <b>b</b> Averaged Abi1-iRFP recruitment (right column) following 4× activation gradients (left column) of TIAM ( <i>n</i> = 10, top) or ITSN ( <i>n</i> = 11, bottom) visualized using TIRFM on round micro-patterns. The averaging procedure is explained in the Methods section. Insets show the illumination patterns (not to scale). <b>c</b> Non-normalized Rac1 FRET ratio profiles along cell diameters of cells treated with control siRNA ( <i>n</i> = 38, red) or Cdc42-directed siRNA ( <i>n</i> = 37, black). Error bars: s.d. Gray lines at the top show positions at which the curves are statistically different ( <i>p</i> &amp;lt; 0.05, Wilcoxon’s rank sum test). <b>d</b>, <b>e</b> β2-chimaerin-iRFP (right) recruitment following 4× activation gradients (left) of TIAM ( <i>n</i> = 11, top) or ITSN ( <i>n</i> = 12, bottom), imaged in TIRFM on round micro-patterns. <b>d</b> Micrographs represent the averaged fluorescence (see Methods). <b>e</b> Normalized fluorescence of β2-chimaerin-iRFP was measured along the cell diameter following the activation of ITSN-CRY2-mCherry (blue, <i>n</i> = 12) or TIAM-CRY2-mCherry (red, <i>n</i> = 11). Error bars: s.d. <b>f</b> Non-normalized Rac1 FRET ratio profiles along cell diameters of cells treated with control siRNA ( <i>n</i> = 38, red) or β2-chimaerin-directed siRNA ( <i>n</i> = 25, black). Error bars: s.d. Gray: Wilcoxon rank sum test ( <i>p</i> &amp;lt; 0.05). <b>g</b> β2-chimaerin staining (left, Gamma correction was applied to images in order to visualize the full dynamics) compared to normalized Rac1 FRET (middle) in the same cells. Insets show zoomed regions of the cell edge. Levels of β2-chimaerin (black) and Rac1 activity (red) are anticorrelated at the cell front (right panel, <i>n</i> = 8, Error bars: s.d). <b>h</b> PAK1-iRFP (purple) recruitment following 4× activation gradients of TIAM (red) after treatment with β2-chimaerin-directed siRNA ( <i>n</i> = 14). Curves were found not significantly different on their whole length (Wilcoxon, <i>p</i> &amp;gt; 0.05). <b>i</b> PAK1-iRFP (purple) recruitment following 4× activation gradients of TIAM (red) after treatment with JLY cocktail ( <i>n</i> = 23). n.s., nonsignificant. <b>j</b> β2-chimaerin staining after DMSO (left) or JLY cocktail (right) treatment. <b>k</b> Fraction of cell perimeter showing β2-chimaerin signal at the cell edge larger than in the cytosol (DMSO: <i>n</i> = 11, JLY: <i>n</i> = 10). Fluorescence at the cell edge was measured along a 1-μm-thick line obtained from the thresholding-based segmentation of the cell shape. The signal in the cytosol was evaluated from a 1 μm-thick line outlining that cell edge on its cytosolic side. Box plots represent the median, interquartile (box), 1.5 IQR (whiskers), and outliers (red crosses). Statistical significance was evaluated using Wilcoxon’s rank sum test. *** <i>p</i> ≤ 0.001. Scale bars: 20 µm </p>   </div> <p /> <p>β2-chimaerin is a GAP of Rac1 that was shown to be activated at the protrusion edge downstream of chemotactic signals <sup> <a href="#CR48">48</a> </sup>. We monitored the recruitment of the β2-chimaerin-iRFP reporter following the optogenetic activation of TIAM or ITSN. We could observe that both pathways could recruit β2-chimaerin at the cell edge, in a very localized fashion similar to the WAVE recruitment (Fig <a href="#Fig3">3d, e</a>), suggesting that this GAP recruitment is conditioned by other signaling components belonging to the tip of the lamellipodia, in accordance with the colocalization between β2-chimaerin and F-actin observed in the lamellipodia of unstimulated cells (Supplementary Figure <a href="#MOESM2">7</a>). Accordingly, inhibiting β2-chimaerin using siRNA (Supplementary Figure <a href="#MOESM2">6b</a>) led to a strong increase of Rac1 activity especially at the cell front such that the bump was abolished (Fig. <a href="#Fig3">3f</a>). This result suggests that β2-chimaerin might act downstream of Cdc42 and Rac1 to inhibit Rac1 locally at the cell front. Indeed, at the front of randomly migrating cells we observed an anticorrelation between β2-chimaerin and Rac1 activities measured by FRET (Fig. <a href="#Fig3">3g</a>). We could verify that the observed localization of β2-chimaerin at the cell edge was not due to volume effects related to the local membrane ruffling activity (Supplementary Figure <a href="#MOESM2">8</a>). We further confirmed the direct role of β2-chimaerin in shaping the Rac1 gradient by inducing the sharp Rac1 activation (4×) using optogenetics in β2-chimaerin-depleted cells, which resulted in a PAK1 gradient that now matched the activating profile (Fig. <a href="#Fig3">3h</a>). </p> <p>Orthogonally to the previous experiments, we also tested the role of transport in shaping the Rac1 gradient. Given that we observed the same PAK1 spatial profile for two distinct TIAM-CRY2 activating gradients (Fig. <a href="#Fig2">2g</a>), we excluded diffusion as it would have smoothened both input distributions. Conversely, the retrograde flow of actin in the lamellipodia can give rise to two similar outputs if the distribution of the flow velocities is ultimately limiting the spatial expansion of the gradient. When cells were treated with the Jasplakinolide-LatrunculinB-Y27632 (JLY) drug cocktail that freezes actin dynamics <sup> <a href="#CR49">49</a> </sup>, we indeed observed a Rac1 activity gradient that matched the sharp (4×) TIAM-CRY2 input gradient (Fig. <a href="#Fig3">3i</a>). Surprisingly, this result shows that the actin retrograde flow can also account for the bump observed in the endogenous Rac1 gradient besides our previously found role for β2-chimaerin. However, this effect could be indirect if the retrograde flow was acting not on Rac1 itself but on the machinery required for proper β2-chimaerin localized distribution. To test this hypothesis, we compared the distribution of β2-chimaerin in control and JLY-treated cells (Fig <a href="#Fig3">3j)</a>. β2-chimaerin localization disappeared from the tip of migrating cells in JLY-treated cells, confirming the indirect role of actin dynamics (Fig. <a href="#Fig3">3k</a>). </p> </div> <div class="aminimalmodeloflocalreactionsrecapitulatescdc42andrac1gradientshaping" title="sec"> <div class="title" tagx="title" title="title">
A minimal model of local reactions recapitulates Cdc42 and Rac1 gradient shaping</div> <p>Given the numerous layers of interactions that we identified experimentally, we sought for a minimal model that would capture the main mechanisms giving rise to the cellular-scale properties of the Cdc42 and Rac1 gradients. To this end, we built a one-dimensional model, where the <i>x</i>-axis spanned across the cell from <i>x</i>=0 to <i>x</i> = 35 μm. We assumed that the Rho GTPases were activated and deactivated with first-order kinetics, and that levels of Rho GTPases equilibrated on a fast time scale. We assumed that the total amount of Rho GTPase <i>R</i> <sub>tot</sub> was not limiting. Eventually, we excluded diffusion and flow, such that the model was purely local. Thus, the local concentration of active Rho GTPase <i>R</i> <sup>*</sup>( <i>x</i>) at steady-state is of the form: </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">1</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\frac{{R^ \ast \left( x \right)}}{{R_{{\mathrm{tot}}}}} = \frac{{\mathop {\sum }\nolimits_i \alpha _i[{\mathrm{GEF}}]_i\left( x \right)}}{{\mathop {\sum }\nolimits_i \beta _i[{\mathrm{GAP}}]_i\left( x \right)}},$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
*</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
tot</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
GEF</div> </div> <div class="mo_UNKNOWN">
]</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
GAP</div> </div> <div class="mo_UNKNOWN">
]</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where [GEF] <sub> <i>i</i> </sub> ( <i>x</i>) and [GAP] <sub> <i>i</i> </sub> ( <i>x</i>) are the concentration profiles of GEFs and GAPs, and <i>α</i> <sub> <i>i</i> </sub> and <i>β</i> <sub> <i>i</i> </sub> their associated effective activation and deactivation rates, which can be a function of the concentration of the Rho GTPases themselves in the case of crosstalks. From the full set of identified interactions (Fig. <a href="#Fig4">4a</a>), we could extract a minimal model explaining the formation of Cdc42 and Rac1 gradients (Fig. <a href="#Fig4">4b</a>). For Cdc42, the shape of the gradient can be simply given by an exponentially distributed GEF and uniform GAP (Fig. <a href="#Fig4">4c</a>): <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">2</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\mathrm{Cdc}}42^ \ast \left( x \right) \propto \frac{{\alpha _{\mathrm C}{\mathrm e}^{ - \frac{x}{\lambda }}}}{{\beta _{\mathrm C}}},$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Cdc</div> <div class="mn_UNKNOWN">
4</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
*</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> </div> <div class="mo_UNKNOWN">
∝</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> </div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <i>λ</i> is the decay length measured for Cdc42 itself (about 10 µm). Note that in the case of optogenetic activation, the optogenetic term <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M7">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\alpha _{{\mathrm{opto}}}{\mathrm e}^{ - \frac{x}{{\lambda _{\mathrm{o}}}}}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
opto</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> </div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> most probably dominates the endogenous GEF activity ( <i>α</i> <sub>opto</sub>&amp;gt;&amp;gt; <i>α</i> <sub>C</sub>) such that the induced gradient follows the activating one. For Rac1, our model contains an exponentially distributed GEF of 10 µm decay length and uniform GAP, similarly to Cdc42, but also a second GAP (β2-chimaerin) exponentially distributed with its own characteristic length <i>γ</i> = 5 µm: <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">3</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M9">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\mathrm{Rac}}1^ \ast \left( x \right) \propto \frac{{\alpha _{\mathrm R}{\mathrm e}^{ - \frac{x}{\lambda }}}}{{\beta _{\mathrm R} + \beta _{\mathrm b}{\mathrm e}^{ - \frac{x}{\gamma }}}},$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Rac</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
*</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> </div> <div class="mo_UNKNOWN">
∝</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> </div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> </div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <i>β</i> <sub>b</sub> is the effective rate constant for β2-chimaerin GAP activity on Rac1. This expression for Rac1 is sufficient to explain the bump (Fig. <a href="#Fig4">4c</a>), the position of which is determined by the ratio <i>r</i> =  <i>β</i> <sub>R</sub>/ <i>β</i> <sub>b</sub> between the strength of the uniform GAP over the strength of the localized β2-chimaerin, and by the characteristic lengths of the decaying profiles: <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">4</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M11">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$x_{{\mathrm{bump}}} = \gamma {\mathrm{ln}}\left( {\frac{{\lambda - \gamma }}{{r.\gamma }}} \right).$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
bump</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
ln</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
γ</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
γ</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>From this relationship, we can see that a bump will be present if <i>r</i> &amp;lt; ( <i>λ</i> −  <i>γ</i>)/ <i>γ</i>, which reduces to <i>r</i> &amp;lt; 1 using the experimental numbers for the decay lengths, or equivalently <i>β</i> <sub>R</sub>&amp;lt; <i>β</i> <sub>b</sub>. This means that the strength of the uniform GAP has to be less than the strength of β2-chimaerin to observe a Rac1 bump. The evolution of the bump position as a function of <i>r</i> is presented in Fig. <a href="#Fig4">4d</a>. From the bump position observed in our experiment, we could predict that β2-chimaerin dominates the uniform GAPs by a factor of ~2. This minimal model for Rac1 can be refined to account for the respective roles of Cdc42 and Rac1 in mediating β2-chimaerin activity at the tip (Fig. <a href="#Fig4">4e</a>). Assuming that <i>β</i> <sub>b</sub> is a linear function of Cdc42 and Rac1 concentrations: <i>β</i> <sub>b</sub> =  <i>β</i> <sub>Cb</sub>Cdc42( <i>x</i>) +  <i>β</i> <sub>Rb</sub>Racl( <i>x</i>), the model shows that Rac1 self-inhibition is required to account for the observed differences in the Rac1 gradient between cells depleted for Cdc42 and cells depleted for β2-chimaerin. Altogether, our minimal modeling approach suggests a simple mechanism of distributed activators and deactivators that shape Cdc42 and Rac1 gradients such that their spatial extents are ultimately different. We thus anticipated that the spatial extent of these Rho GTPases would play a functional role. <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 4</span> </a> </div>   <p>A minimal model for Cdc42 and Rac1 gradient formation. <b>a</b> Full model based on our experimental findings. Activation rates are denoted by <i>α</i> and deactivation rates by <i>β</i>. <b>b</b> A minimal model that recapitulates the formation of Cdc42 and Rac1 gradients. <b>c</b> Gradient shaping of Cdc42 and Rac1. Since the absolute amplitude of Rho GTPases are unknown, we assigned the following arbitrary values to the rates: <i>α</i> <sub>C</sub>= <i>α</i> <sub>R</sub>=1; <i>β</i> <sub>C</sub>= <i>β</i> <sub>R</sub>=0.5; and <i>β</i> <sub>b</sub>=1. Left: Cdc42 (blue) is set by an exponentially distributed GEF (green) with a characteristic length <i>λ</i>=10 µm and uniform GAP (black). Middle: Rac1 (red) requires an additional GAP, β2-chimaerin (dashed black, characteristic length <i>γ</i>=5 µm), that is localized more sharply at the cell edge than Rac1 GEF (green). The overall GAP activity (plain black) is the sum of β2-chimaerin and a uniform GAP (dashed black). As a result, the putative Rac1 gradient without β2-chimaerin (dashed red) is chopped off at the cell edge resulting in a bell-shaped gradient (plain red). Right: once normalized to 1, Cdc42 and Rac1 gradients present distributions that are similar to the ones measured in cells. <b>d</b> Effect of the relative ratio <i>r</i>= <i>β</i> <sub>R</sub>/ <i>β</i> <sub>b</sub> between uniformly distributed GAPs and the localized gradient of β2-chimaerin on the position of the Rac1 bump. Left: Rac1 gradients obtained with decreasing values of <i>r</i> ( <i>r</i> = 2, 1, 0.6, 0.3, 0.2, 0.1 respectively from dark to light red). Right: exponentially distributed β2-chimaerin (dashed line) and uniformly distributed GAPs ( <i>β</i> <sub>R</sub>=2, 1, 0.6, 0.3, 0.2, 0.1 from dark to light gray, solid lines) corresponding to the values used for the left plot. <b>e</b> Effects of Cdc42 or β2-chimaerin inhibition in silico on the Rac1 gradient ( <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M13">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\alpha _{{\mathrm C}_{\mathrm b}} = 0.4$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.4</div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> , and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M15">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\alpha _{{\mathrm R}_{\mathrm b}} = 0.3$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.3</div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>). The profiles are normalized (by the same factor) to match the FRET signal values measured experimentally (Fig. <a href="#Fig2">2c, d</a>) <p />   </div> <p /> </div> <div class="acontrolledassaytomonitorthedependenceofcellmigrationonrac1andcdc42gradients" title="sec"> <div class="title" tagx="title" title="title">
A controlled assay to monitor the dependence of cell migration on Rac1 and Cdc42 gradients</div> <p>We next questioned whether the different properties of Cdc42 and Rac1 gradients had an impact on migration properties. For this purpose, we imposed optogenetic gradients of ITSN or TIAM with increasing slopes (Fig. <a href="#Fig2">2a</a>). In order to control the experimental initial conditions, i.e. to prevent initial cell polarity prior to the optogenetic stimulation but still be able to monitor cell movement following it, we opted for a switchable micropatterning technique <sup> <a href="#CR50">50</a> </sup>. Cells were plated on round micropatterns, and would then keep an isometric shape until the surrounding repelling surface was rendered adhesive by coupling a fibronectin-mimicking chemical compound (BCN-RGD) that binds to the modified PLL-PEG repellent (APP). After addition of this reagent, cells were released from patterns and free to migrate on the coverslip (Fig. <a href="#Fig5">5a, b</a>, top row). Optogenetic stimulation with gradients of light concomitantly with the release of adhesion allowed us to study cell migration with one changing parameter, namely the extent of blue light gradients (Fig. <a href="#Fig5">5a, b</a>, bottom row). From <i>n</i> ~ 20 cells per each condition, we quantified the cell edge morphodynamics (see Methods) and averaged them for each activating gradient slope (Fig. <a href="#Fig5">5c</a>). As expected, both Rac1 and Cdc42 biased the membrane protruding activity toward the direction of the gradient. Rac1 led to an immediate cell movement while Cdc42 led to slightly delayed cell movement (Fig. <a href="#Fig5">5c</a>). We observed that cells shifted from an oriented spreading (when the back of the cell kept steady) to a directed migration (when the back of the cell moved together with the front) by increasing the gradient slope (Supplementary Movies <a href="#MOESM6">3</a>, <a href="#MOESM7">4</a>). Yet, the center of mass of cells monotonously increases its movement toward the gradient as the gradient slope increased (Fig. <a href="#Fig5">5d</a>) suggesting that the quantitative properties of the gradients have a differential role in migration. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 5</span> </a> </div>   <p>Scheme of the quantitative migration assay. <b>a</b> Cells are seeded on 35 µm round patterns. After complete adhesion, the adhesive reagent BCN-RGD is added and binds to the coverslip’s surface, allowing free 2D cell migration (top). Directed migration can be triggered by optogenetic activation of GEFs through light gradients at the same time as cell adhesion is released (bottom). <b>b</b> Examples of cells expressing CIBN-GFP-CAAX and TIAM-CRY2-mCherry with (3× gradient, bottom) or without (top) photo-activation (visualized: TIAM-CRY2-mCherry). Time indicates the duration after addition of BCN-RGD and concomitant blue light illumination. The dashed orange line corresponds to the initial position of the cell center. <b>c</b>, <b>d</b> HeLa cells expressing CIBN-GFP-CAAX and ITSN-CRY2-mCherry or CIBN-GFP-CAAX and TIAM-CRY2-mCherry were illuminated with various gradients of light as the adhesive patterns were released. <b>c</b> Average morphodynamic maps for each condition (ITSN: top, TIAM: bottom). The vertical axis corresponds to the coordinate along the cell contour (centered on the direction of the light gradient) and the horizontal axis corresponds to time. The local velocity of the edge of the cell membrane is color coded accordingly to the bar on the right side. Gradient extents are schemed on the left side of each map. ITSN: <i>n</i>=25 (control with uniform illumination), <i>n</i>=16 (1× gradient), <i>n</i>=20 (2×), <i>n</i> = 19 (3×) or <i>n</i> = 16 (4×). TIAM: <i>n</i> = 19 (ctrl), <i>n</i> = 18 (1×), <i>n</i> = 18 (2×), <i>n</i> = 17 (3×), <i>n</i> = 18 (4×). <b>d</b> We tracked the position of the centroid of individual cells. Top: Trajectories of cells stimulated with various gradients of ITSN. Bottom: The angles between the displacement vector (initial to final centroid position) and the stimulation axis for each cell are represented in polar coordinates. Scale bars: 20 µm </p>   </div> <p /> </div> <div class="cdc42providesdirectionalitywhilerac1providesspeed" title="sec"> <div class="title" tagx="title" title="title">
Cdc42 provides directionality while Rac1 provides speed</div> <p>In order to assess the quantitative effect of gradient on motility, we focused on two coarse-grained parameters: the maximal instantaneous velocity and the precision of the migration orientation. Sharper gradients of either ITSN or TIAM both increased cellular speed. However, activating Rac1 through TIAM had a stronger effect on speed than activating Cdc42 through ITSN (Fig. <a href="#Fig6">6a</a>), consistent with the known effect of Rac1 as a critical factor for lamellipodium formation <sup> <a href="#CR15">15</a> </sup> (Supplementary Figure <a href="#MOESM2">9</a>). In fact, even shallow gradients of TIAM induced an enhanced migration speed. In comparison, only sharp gradients of ITSN (3× and 4×) induced an increased cellular speed, but even in these conditions speed was lower than for equivalent TIAM gradients. Conversely, ITSN gradients had a stronger effect on orientation precision. While 1× to 4× TIAM gradients had a similar effect on orientation, increasingly sharp gradients of ITSN induced an increasing precision of migration (Fig. <a href="#Fig6">6b</a>). Indeed, the 4× ITSN gradient induced the most oriented response (with a remarkable angular precision), consistent with the known role of Cdc42 as a regulator of directed migration <sup> <a href="#CR51">51</a>, <a href="#CR52">52</a> </sup>, even though this role seems to be cell dependent <sup> <a href="#CR53">53</a> </sup>. We show here that directed migration is better achieved with sharp Cdc42 gradients similar to the ones measured endogenously in cells (Cdc42 gradient extent measured in migrating cells <i>d</i> = 8.9 ± 0.6 µm, Fig. <a href="#Fig1">1</a>, 4× Cdc42 gradient extent imposed and measured through PAK-iRFP <i>d</i> = 6.1 ± 0.9 µm, Fig. <a href="#Fig2">2</a>). Thus, in our experimental model, Cdc42 provides directionality while Rac1 provides speed of movement. These functions appear to be specific of each GTPase, since inhibition of Rac1 abolishes cell speed but not orientation for Cdc42 activation (Fig. <a href="#Fig6">6c, d</a>). Consequently, crossed activities (speed induction by Cdc42, orientation by Rac1) seem to be due to crosstalks between these Rho GTPases. Along this line, a possible functional role for β2-chimaerin is to spatially segregate Rac1 and Cdc42 activities to avoid competition between their functional roles. Indeed, as seen in supplementary figure <a href="#MOESM2">10</a>, β2-chimaerin suppression has no effect on cell speed but leads to a significant reduction in angular precision. This suggests that β2-chimaerin limits Rac1 protrusive activity at the very cell front to allow Cdc42 activity to steer cell migration. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 6</span> </a> </div>   <p>Cdc42 and Rac1 drive different cellular responses. <b>a</b>, <b>b</b> Trajectories of cells stimulated as represented in Fig. <a href="#Fig5">5</a> were analyzed quantitatively. <b>a</b> Cell speed defined as the instantaneous velocity of the cell displacement averaged over five consecutive time frames (top scheme). Box plots show instantaneous velocity of cells expressing CIBN-GFP-CAAX together with ITSN-CRY2-mCherry (blue) or TIAM-CRY2-mCherry (red) stimulated with various gradients of light (ITSN: <i>n</i> = 25 (ctrl), <i>n</i> = 16 (1×), <i>n</i> = 20 (2×), <i>n</i> = 19 (3×), <i>n</i> = 16 (4×), TIAM: <i>n</i> = 19 (ctrl), <i>n</i> = 18 (1×), <i>n</i> = 18 (2×), <i>n</i> = 17 (3×), <i>n</i> = 18 (4×)). Box plots represent the median, interquartile (box), 1.5 IQR (whiskers). <b>b</b> Directionality defined as the angular precision of cell displacement: the angle of displacement was measured using the initial position averaged over the first three frames and the final position averaged over the last three frames (top scheme). Angles were then bootstrapped and angular precision was calculated with the formula <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M17">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$r = \sqrt {( {1}/{n}^{\ast} \sum _{i = 1}^{n} \sin\theta _{i} )^{2} + ( {1}/{n}^{\ast} \sum _{i = 1}^{n} \cos\theta _{i} )^{2}} .$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
=</div> <div class="msqrt_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
∕</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
*</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
 sin</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
∕</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
*</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
 cos</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  Box plots represent the median, interquartile (box), 1.5 IQR (whiskers). Statistical significance between consecutive conditions was evaluated using Wilcoxon’s rank sum test. * <i>p</i> ≤ 0.05. n.s., nonsignificant ( <i>p</i> &amp;gt; 0.05). <b>c</b> Speed and <b>d</b> directionality measurements on HeLa cells expressing CIBN-GFP-CAAX and ITSN-CRY2-mCherry after Rac1 inhibition with 100 µM NSC 23766 and stimulated with various Cdc42 gradients. <i>n</i> = 8 (1×), <i>n</i> = 16 (2×) or <i>n</i> = 17 (4×) <b>. e</b> Effects of slope, amplitude, and spatial extent of Cdc42 gradients on cell velocity and angular precision. HeLa cells expressing CIBN-GFP-CAAX and ITSN-CRY2-mCherry were stimulated with varying gradients of light. Gradients in blue ( <b>a</b>, <b>b</b>) and red ( <b>c</b>, <b>d</b>) have two distinct spatial extents. Gradients in light ( <b>a</b>, <b>c</b>) and dark ( <b>b</b>, <b>d</b>) color have two distinct amplitudes. Two gradients ( <b>b</b>, <b>c</b>) have the same slope. <i>n</i> = 13 ( <b>a</b>), <i>n</i> = 16 ( <b>b</b>), <i>n</i> = 22 ( <b>c</b>), <i>n</i> = 16 ( <b>d</b>). * <i>p</i> ≤ 0.05, ** <i>p</i> ≤ 0.01, n.s., nonsignificant ( <i>p</i> &amp;gt; 0.05) (Wilcoxon rank sum test) <p />   </div> <p /> </div> <div class="thespatialextentbutnottheamplitudeorslopeofthecdc42gradientmattersfordirectionality" title="sec"> <div class="title" tagx="title" title="title">
The spatial extent but not the amplitude or slope of the Cdc42 gradient matters for directionality </div> <p>Since we showed that the shape of Rho GTPase activation gradients directly influence the outcome of cell migration, we thus questioned whether cells are actually sensitive to the slope or to the spatial extent of Rho GTPase activation gradients. In fact, in the previous experiments, both parameters varied concomitantly. It is known that cells can sense and process various extra- and intracellular signaling gradients that can hence influence cell polarity and migration <sup> <a href="#CR54">54</a>– <a href="#CR56">56</a> </sup>. However, it is not known to which quantitative properties of Rho GTPases intracellular signaling gradients cells are sensitive. Using the experimental setup detailed above, we could independently test the effect of gradient slope or spatial extent. When we applied gradients of ITSN activation with different slopes but the same spatial extent, we could not detect any difference in cell motility (Fig. <a href="#Fig6">6e</a>). This also confirms that the amplitude of the imposed gradient itself does not affect the cellular response. Instead, when we imposed gradients of similar slope or amplitude but different extents, we could observe that cells stimulated with the shorter gradient of ITSN activation migrated with higher velocity and better orientation (Fig. <a href="#Fig6">6e</a>). These results indicate that the spatial extent is the critical parameter of Rho GTPase gradients read by cells. </p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>In this work, we observed that the front of randomly migrating cells presents an exponentially decaying Cdc42 activity gradient whereas Rac1 shows a complex shape peaking at approximately 6 µm from the cell edge, similarly to what has been observed before in other cell types <sup> <a href="#CR9">9</a>, <a href="#CR36">36</a> </sup>. Combining experimental and model-based approaches, we could identify a network topology and map it spatially, allowing us to explain how these two distinct intracellular patterns are formed. By quantitatively tweaking the spatial patterns of specific GEF activity for either Cdc42 or Rac1 using optogenetics while quantifying the downstream recruitment of effectors, we showed that Cdc42 patterning can be simply explained by the combination of a localized GEF and a uniform GAP, but that Rac1 required a more complex circuitry. </p> <p>We found that two mechanisms could account for Rac1 patterning. Combining one exponentially decaying GEF with either a GAP with a shorter exponential decay (like β2-chimaerin) or a directed transport from the cell front due to the actin retrograde flow was sufficient to recapitulate the observed Rac1 gradient. Yet, we showed that the effect of the actin retrograde flow does not act directly on Rac1 itself but is required for the front-most localization of β2-chimaerin. It has been previously demonstrated that the actin retrograde flow is coupled to cell polarity, by transporting various proteins away from the cell front <sup> <a href="#CR54">54</a> </sup>. The actin flow could act on an inhibitor of β2-chimaerin. Another possibility is that β2-chimaerin localizes at the barbed end of actin filaments thanks to its interaction with the adaptor protein Nck1 <sup> <a href="#CR48">48</a> </sup>. Nck1 is also localized at the tip of migrating cells by the Gab1-NWASP complex <sup> <a href="#CR57">57</a> </sup>. Since we additionally showed that a feedback from Rac1 leads to β2-chimaerin enrichment, β2-chimaerin recruitment would depend on two concomitant signals: a Rac1-dependent signal likely going through Rac1-dependent PKC-DAG production <sup> <a href="#CR58">58</a> </sup>, and an actin polymerizing signal through the adaptor protein Nck1. This would also explain the crosstalk from Cdc42 to β2-chimaerin through N-WASP and an increase of Nck1-mediated β2-chimaerin recruitment. </p> <p>Interestingly, Cdc42 and Rac1 gradients have similar exponential decays but different spatial extents due to the local inhibition of Rac1 activity at the cell front. This observation raises important questions about the way cells interpret signaling gradients. Using quantitative optogenetics, we could directly control the spatial extent, slope or amplitude of intracellular activity gradients. We showed that cell migration is not determined by the amplitude or slope of Rho GTPase gradients, but rather by their spatial extent, similarly to what was proposed in a recent work on ERK morphogen gradients in <i>Drosophila</i> embryos <sup> <a href="#CR59">59</a> </sup>. The spatial extent of Cdc42 needs to be small to ensure fine directionality in cell movement, in accordance with the previously shown role of Cdc42 as the primary conductor of chemotactic steering and cell polarity <sup> <a href="#CR9">9</a> </sup>. The spatial extent of Rac1 is larger, providing speed to the cell. Yet, since we could not effectively apply sharper Rac1 gradients without disrupting the network topology, we do not know if the spatial extent of Rac1 presents a functional optimum as for Cdc42. Our approach can appear similar to the recent work of Zimmerman et al. who used optogenetic activations of Cdc42 and Rac1 to guide cell migration <sup> <a href="#CR60">60</a> </sup>. However, in their work they imposed long-range light gradients to mimic external chemo-attractant gradients, whereas in our work we imposed subcellular light gradients to keep the cell and not the environment as the relevant spatial referential of Rho GTPase gradients. </p> <p>In this study, we did not consider the temporal dynamics of Rho GTPase activities. While it is very likely that spatial and temporal dynamics are connected in freely migrating cells and while Rho GTPases patterns evolve on timescales of ~100 s <sup> <a href="#CR19">19</a> </sup>, the response functions we measured under our steady optogenetic activations did not show evolving spatiotemporal patterns (Supplementary Figure <a href="#MOESM2">4</a>). Thus, even if our synthetic approach does not recapitulate the full spatiotemporal complexity seen in native cells, we can consider our results as an example of the signaling network capacity to respond to spatially modulated inputs. Given the similarity between the native and induced gradients of Rac1/Cdc42, we can be confident that the mechanisms we propose for gradient shaping are biologically relevant, at least at the coarse-grained cellular scale. </p> <p>Following a correlative approach, Yamao et al. recently studied in time and space the patterns of Rac1 and Cdc42 activities and their link with the membrane dynamics in randomly migrating cells <sup> <a href="#CR61">61</a> </sup>. They concluded that Cdc42 induces random cell migration and Rac1 is responsible for persistent movement. While it might sound different from our results, the discrepancies might be explained by the scales and parameters observed in each case. We measure local and instantaneous quantities (speed and directionality), and Yamao and colleagues measure integrated and macroscopic ones (persistence and randomness). We were not able to measure those integrated quantities, since our optogenetic activations were not following the cells as they moved out of the adhesive micro-patterns. However, these different scales can be reconciled. As we conclude that Rac1 provides cells with higher speed, it also means long-term movement is more persistent <sup> <a href="#CR54">54</a>, <a href="#CR62">62</a> </sup>. Similarly, since we show that sharp Cdc42 gradients can fine-tune directionality, local and transient Cdc42 pulses could steer cells randomly in complex trajectories. </p> <p>The minimal circuitry that we identified as sufficient to shape Cdc42 and Rac1 gradients raises new unanswered questions. In particular, it is unclear how gradients of GEFs and GAPs are shaped throughout the cell, beside the formation of the β2-chimaerin gradient we identified. Our results suggest a role for the cytoskeleton itself and its dynamics to enrich β2-chimaerin at the cell border. More generally, actin networks and actin-regulating complexes can act as scaffolding complexes in protrusive regions where they localize. For example, the WAVE Complex, a downstream effector of Rac1, recruits WRP, a GAP inhibiting Rac1 <sup> <a href="#CR63">63</a> </sup>. Similarly, N-WASP, a downstream effector of Cdc42, associates with the GEF ITSN. More mechanisms are probably involved. In particular, membranes could play a direct role in the localization of these regulators of Rho GTPase activity. The local lipid composition, and in particular the concentration of PIP3, has been shown to control the activity of Rac1 and Cdc42 <sup> <a href="#CR26">26</a>, <a href="#CR28">28</a>, <a href="#CR64">64</a> </sup>. In addition, membrane curvature-sensing BAR proteins localize at highly bent membranes, including cell edges. Several BAR proteins are known to bind Rho GTPAses or their regulators. IRSp53, a member of the I-BAR family found in lamellipodia and filopodia has been shown to bind Cdc42, Rac1 and WAVE2 <sup> <a href="#CR65">65</a>, <a href="#CR66">66</a> </sup>. Even if β2-chimaerin was sufficient to explain Rac1 shaping in the present work, other known GAPs, such as ARHGAP22, ARHGAP24 (FILGAP) and SH3BP1, interact with the proteins involved in cell protrusion and could play a similar role as β2-chimaerin. In particular, it was previously shown that depletion of SH3BP1 results in a high activity of Rac1 at the front <sup> <a href="#CR67">67</a> </sup>. Also, it remains to be explored if the Cdc42 and Rac1 positive feedbacks and crosstalks, as previously suggested <sup> <a href="#CR9">9</a> </sup> and observed in our work (Fig. <a href="#Fig3">3b</a>), play a role in shaping GEF distributions. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plasmidsandmolecularconstructs" title="sec"> <div class="title" tagx="title" title="title">
Plasmids and molecular constructs</div> <p>ITSN-CRY2-mCherry was constructed as detailed previously <sup> <a href="#CR44">44</a> </sup>. The TIAM DH-PH domain was similarly amplified from TIAM(DHPH)-Linker-YFP-PIF (gift from O. Weiner, University of California, San Francisco) and cloned into CRY2PHR-mCherry. Both ITSN-CRY2-mCherry and TIAM-CRY2-mCherry were cloned in pHR lentiviral vectors (gift from O. Weiner) by Genscript (Nanjing, China) using <i>Mlu</i>I and <i>Bst</i>BI cloning sites. N-WASP-iRFP, PAK1-iRFP and β2-chimaerin-iRFP fusion genes were constructed by Genscript (Nanjing, China) by cloning the corresponding human cDNAs upstream the iRFP713 gene sequence <sup> <a href="#CR68">68</a> </sup>, separated by a PVAT sequencer. The Abi1-iRFP plasmid was kindly provided by Maria Carla Parrini. Rac1BS and Cdc42BS plasmids were kindly provided by Dr. Louis Hodgson <sup> <a href="#CR35">35</a> </sup>, and were subcloned into the lentiviral pLVX vector (Clontech, Mountain View, CA USA) between <i>Xma</i>I and <i>Xba</i>I cloning sites. </p> </div> <div class="cellcultureandreagents" title="sec"> <div class="title" tagx="title" title="title">
Cell culture and reagents</div> <p>HeLa cells (CCL-2 strain, bought from ATCC) were cultured at 37 °C with 5% CO <sub>2</sub> in Dulbecco’s modified Eagle’s medium supplemented with 10% fetal bovine serum and penicillin-streptomycin (100 U/mL). Transfections were performed using X-tremeGENE 9 (Roche Applied Science, Penzburg, Bavaria, Germany) according to the manufacturer’s instructions using an equal amount of plasmid DNA for each construct (1 µg). Stable cell lines were obtained using lentiviral infections: all lentiviruses were produced by transfecting pHR- or pLVX-based plasmids along with the vectors encoding packaging proteins (pMD2.G and psPax2) using HEK-293T cells. Viral supernatants were collected 2 days after transfection and HeLa cells were transduced at an MOI of 2. Gene expression knockdown was achieved using pooled siRNA with the following sequences. Cdc42: 5′- CGAUGGUGCUGUUGGUAAA-3′ and 5′-CUAUGCAGUCACAGUUAUG-3′, β2-chimaerin: 5′- AUUGAAGCAAGAGGAUUAA-3′ and 5′-CCACUUCAAUUAUGAGAAG-3′, Rac1: 5′-UUUACCUACAGCUCCGUCUUU-3′ and 5′-UACAGCACCAAUCUCCUUAUU-3′, ctrl: 5′-AGGUAGUGUAAUCGCCUUG-3′ and 5′-GCGGGATATTTCGGTCAAT-3′. siRNA transfection was done following the manufacturer’s protocol (Lipofectamine RNAiMax, Thermo Fischer Scientific), and cells were imaged 48 h after transfection. The JLY cocktail (8 µM jasplakinolide, 5 µM Latrunculin B, 20 µM Y27632) was applied 15 min before image acquisition. </p> </div> <div class="livecellimagingandoptogenetics" title="sec"> <div class="title" tagx="title" title="title">
Live cell imaging and optogenetics</div> <p>Micropatterned coverslips were prepared as described by Azioune et al. <sup> <a href="#CR69">69</a> </sup>: O <sub>2</sub> plasma-cleaned coverslips were incubated with 0.1 mg/ml of PLL-g-PEG (Surface Solutions, Switzerland) in 10 mM HEPES, pH 7.4 for 1 h. They were then exposed to deep UV through micropatterned quartz/chrome photomasks (Toppan, Round Rock, TX) for 5 min, and incubated with fibronectin in 100 mM NaHCO <sub>3</sub> (pH 8.5) for 1 h. Releasable micropatterns were prepared similarly, with PLL-PEG being replaced by azido-PLL-g-PEG (APP) at 100 µg/ml. Migration was released by addition of 20 µM BCN-RGD for 10 min. Before imaging, cells were dissociated using Versene (Life Technologies) and seeded for adhesion on the previously mentioned coverslips for at least 2 h. Experiments were performed at 37 °C in 5% CO <sub>2</sub> in a heating chamber (Pecon, Meyer Instruments, Houston, TX) placed on an inverted microscope model No. IX71 equipped with a ×60 objective with NA 1.45 (Olympus, Melville, NY) and a Luca R camera (Andor, Belfast, UK). The microscope was controlled with the Metamorph software (Molecular Devices, Eugene, OR). TIRF images were acquired using an azimuthal TIRF module (iLas2; Roper Scientific, Tucson, AZ). Optogenetics stimulations were performed every 30−40 s with a DMD in epi-mode (DLP Light Crafter, Texas Instruments) illuminated with a SPECTRA Light Engine (Lumencor, Beaverton, OR USA) at 440 ± 10 nm. </p> </div> <div class="fret" title="sec"> <div class="title" tagx="title" title="title">
FRET</div> <p>HeLa cells were lentivirus-infected with a Cdc42-FRET-biosensor or a Rac1-FRET-biosensor (kindly provided by Louis Hodgson) and sorted for intermediary fluorescence using fluorescence-activated cell sorting (FACS). Twenty-four hours after plating them on glass coverslips, cells were imaged by TIRF microscopy. Excitation was done with a laser at 405 nm, dichroic mirrors stayed the same (BS: FF-458-DiO2, Semrock) while a filterwheel allowed for the switching of appropriate emission filters to acquire sequentially donor (mCerulean, Em: FF01-483/32) and FRET (Em: FF01-542/27) emissions. Image processing included registration, flat-field correction, background subtraction, segmentation, and FRET/donor ratio calculations. FRET profiles measured from the FRET images were normalized between 0 and 1 when comparing the two Cdc42 and Rac1 FRET reporters or when comparing a FRET reporter with another fluorescence signal. We did not normalize FRET profiles from the same reporter when comparing two different experimental conditions. </p> </div> <div class="immunofluorescencemicroscopy" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescence microscopy</div> <p>HeLa cells stably expressing a Rac1-FRET-biosensor were fixed in phosphate buffer saline (PBS) with 4% paraformaldehyde for 20 min at 25 °C. After permeabilization in PBS + 0.1% triton X-100 for 15 min and blocking in PBS + 1% BSA + 1% FBS for 20 min, stainings were performed in PBS with 0.05% triton + 1% BSA 1 h at room temperature. Antibodies were used as follows: β2-chimaerin primary antibody: 1 /100 (Orb182594, Biorbyt), anti-rabbit-Alexa594 antibody: 1 /400 (ThermoFisher). Phalloidin-488 was used at 300 nM. Acquisitions were made in HiLo mode using an azimuthal TIRF module as described above). </p> </div> <div class="imageprocessingandquantificationofintracellulargradients" title="sec"> <div class="title" tagx="title" title="title">
Image processing and quantification of intracellular gradients</div> <p>Images were analyzed with custom-built Matlab routines. For the images obtained in our optogenetic experiments, we subtracted the initial pre-optogenetics signal from all subsequent images in order to measure solely the recruitment of fluorescent proteins to the basal membrane and to avoid volume artifacts. The resulting differential images were normalized between 0 and 1 using the same normalizing factors as the gradients quantified from each image (see below). Normalized images were then averaged over ten time points and over all cells to produce the averaged images shown in Figs. <a href="#Fig2">2d, f</a> and <a href="#Fig3">3b, d</a>. The associated Fire color scale was defined as the average temporal fold change of fluorescence that we measured from the raw images after background subtraction (taken outside the cell mask). For the quantification of the gradients presented in Figs. <a href="#Fig1">1b</a>, and <a href="#Fig3">3c, f, g</a>, we measured the FRET ratio along two linescans per cell, drawn manually perpendicular to the cell edge in protrusive regions with a line width of 10 pixels. The gradients were first averaged for each cell, and then averaged over all cells. For the quantification of the gradients presented in Figs. <a href="#Fig2">2b, e, g</a> and <a href="#Fig3">3e, h, i</a>, fluorescence was quantified along a line of 10 pixels in width spanning across the cell diameter in the direction of the optogenetic gradients. The curves in Figs. <a href="#Fig2">2b, e, g</a> and <a href="#Fig3">3e, h, i</a> were normalized between 0 and 1 where 0 stands for the average of the five minimal values and 1 stands for the average of the five maximal fluorescence values. </p> </div> <div class="processingofthemigrationmovies" title="sec"> <div class="title" tagx="title" title="title">
Processing of the migration movies</div> <p>Movies were analyzed with custom-built Matlab routines. The segmentation of cell borders was performed on fluorescence images using the Matlab function Graythresh. Cell centroid positions were determined using the Matlab function Regionprops and used to quantify cell movement. To measure cell velocity, we computed instantaneous speed of cell centroids at each time frame, and then averaged it over several time frames. Cells stimulated through TIAM activation reached maximum speed soon after the beginning of illumination, so instantaneous speed was averaged between <i>t</i> = 15 min to <i>t</i> = 45 min. Cells stimulated through ITSN activation reached maximum speed at later stages, and instantaneous speed was thus averaged between <i>t</i> = 60 min to <i>t</i> = 90 min. Angular precision was computed as follows: for each cell, the displacement vector was computed between the initial cell centroid (averaged over the three first time frames) and the final cell centroid (averaged over the three last time frames), and we measured the angle between this vector and the axis of stimulation gradients. These angles were bootstrapped over 1000 replications, and angular precision was estimated with the formula <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M19">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$p = \sqrt {\left( {( {{1}/{n}^\ast \mathop {\sum }\nolimits_{i = 1}^n \sin\theta _i} )^2 + \left( {{1}/{n}^\ast \mathop {\sum }\nolimits_{i = 1}^n \cos\theta _i} \right)^2} \right.} .$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mo_UNKNOWN">
=</div> <div class="msqrt_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
∕</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
*</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
 sin</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
∕</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
*</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
 cos</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  The morphodynamics maps (Fig. <a href="#Fig5">5c</a>) were obtained using a routine adapted from Yang <sup> <a href="#CR9">9</a> </sup>. The cell contour was aligned such that the middle of the map was centered on the direction imposed by the optogenetic gradient. <p /> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2018_7286_MOESM1_ESM.pdf">LINK</a>  <p>Peer Review File</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2018_7286_MOESM2_ESM.pdf">LINK</a>  <p>Supplementary Information</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2018_7286_MOESM3_ESM.docx">LINK</a>  <p>Description of Additional Supplementary Files</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2018_7286_MOESM4_ESM.mp4">LINK</a>  <p>Supplementary Movie 1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2018_7286_MOESM5_ESM.mp4">LINK</a>  <p>Supplementary Movie 2</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2018_7286_MOESM6_ESM.mp4">LINK</a>  <p>Supplementary Movie 3</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2018_7286_MOESM7_ESM.mp4">LINK</a>  <p>Supplementary Movie 4</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Publisher’s note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p> <b>Supplementary Information</b> accompanies this paper at 10.1038/s41467-018-07286-8. </p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Louis Hodgson for sharing the Rho GTPase sensors; Maria Carla Parrini, Kristine Schauer, and Christophe Tribet for critical reading of the manuscript and helpful discussions; Remy Fert and Eric Nicolau for their technical assistance; The Institut Curie Shared FACS Facility for cell sorting. M.C. acknowledges financial support from French National Research Agency (LICOP no. ANR-12-JSV5-0002-01). M.C. and M.D. acknowledge funding from French National Research Agency (ANR) Paris-Science-Lettres Program (ANR-10-IDEX-0001-02 PSL), Labex CelTisPhyBio (No. ANR-10-LBX-0038), the France-BioImaging infrastructure supported by ANR Grant ANR-10-INSB-04 (Investments for the Future), and Institut Pierre-Gilles de Gennes (laboratoire d’excellence, Investissements d’Avenir program ANR-10-IDEX-0001-02 PSL and ANR-10-LABX-31). </p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>Conceptualization, S.D.B., M.D. and M.C.; Methodology, S.D.B., M.D. and M.C.; Software, S.D.B. and M.C.; Formal analysis, S.D.B., K.V. and M.C.; Investigation S.D.B., K.V. and J.M.; Resources, F.d.F., G.C., and F.D.; Writing—original draft, S.D.B. and M.C.; Writing—review and editing, S.D.B., M.C., M.D. and K.V.; Visualization, S.D.B. and K.V.; Supervision, M.C.; Funding acquisition, M.C. </p> <div class="title" tagx="title" title="title">
Data availability</div> <p>The data that support the findings of this study and all custom codes used for analysis are available from the corresponding author upon reasonable request. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Haastert</span> <span class="given-names" tagx="given-names" title="given-names">PJM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Devreotes</span> <span class="given-names" tagx="given-names" title="given-names">PN</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chemotaxis: signalling the way forward</span> <span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">626</span> <span class="lpage" tagx="lpage" title="lpage">634</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrm1435">10.1038/nrm1435</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15366706">15366706</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiner</span> <span class="given-names" tagx="given-names" title="given-names">OD</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Hem-1 complexes are essential for Rac activation, actin polymerization, and myosin regulation during neutrophil chemotaxis </span> <span class="source" tagx="source" title="source">PLoS Biol.</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">186</span> <span class="lpage" tagx="lpage" title="lpage">199</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pbio.0040038">10.1371/journal.pbio.0040038</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bray</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Axonal growth in response to experimentally applied mechanical tension</span> <span class="source" tagx="source" title="source">Dev. Biol.</span> <span class="year" tagx="year" title="year">1984</span> <span class="volume" tagx="volume" title="volume">102</span> <span class="fpage" tagx="fpage" title="fpage">379</span> <span class="lpage" tagx="lpage" title="lpage">389</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0012-1606(84)90202-1">10.1016/0012-1606(84)90202-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/6706005">6706005</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choquet</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Felsenfeld</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheetz</span> <span class="given-names" tagx="given-names" title="given-names">MP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Extracellular matrix rigidity causes strengthening of integrin–cytoskeleton linkages</span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">88</span> <span class="fpage" tagx="fpage" title="fpage">39</span> <span class="lpage" tagx="lpage" title="lpage">48</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0092-8674(00)81856-5">10.1016/S0092-8674(00)81856-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9019403">9019403</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pelham</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cell locomotion and focal adhesions are regulated by the mechanical properties of the substrate </span> <span class="source" tagx="source" title="source">Biol. Bull.</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">194</span> <span class="fpage" tagx="fpage" title="fpage">348</span> <span class="lpage" tagx="lpage" title="lpage">350</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2307/1543109">10.2307/1543109</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11536880">11536880</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCarthy</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palm</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Furcht</span> <span class="given-names" tagx="given-names" title="given-names">LT</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Migration by haptotaxis of a schwann-cell tumor line to the basement-membrane glycoprotein laminin </span> <span class="source" tagx="source" title="source">J. Cell. Biol.</span> <span class="year" tagx="year" title="year">1983</span> <span class="volume" tagx="volume" title="volume">97</span> <span class="fpage" tagx="fpage" title="fpage">772</span> <span class="lpage" tagx="lpage" title="lpage">777</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1083/jcb.97.3.772">10.1083/jcb.97.3.772</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/6885918">6885918</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Interstitial dendritic cell guidance by haptotactic chemokine gradients</span> <span class="source" tagx="source" title="source">Science (80-.).</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">339</span> <span class="fpage" tagx="fpage" title="fpage">328</span> <span class="lpage" tagx="lpage" title="lpage">332</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1228456">10.1126/science.1228456</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iglesias</span> <span class="given-names" tagx="given-names" title="given-names">Pa</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Devreotes</span> <span class="given-names" tagx="given-names" title="given-names">PN</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An excitable signal integrator couples to an idling cytoskeletal oscillator to drive cell migration </span> <span class="source" tagx="source" title="source">Nat. Cell Biol.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">1307</span> <span class="lpage" tagx="lpage" title="lpage">1316</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncb2859">10.1038/ncb2859</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24142103">24142103</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">HW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Collins</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyer</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Locally excitable Cdc42 signals steer cells during chemotaxis</span> <span class="source" tagx="source" title="source">Nat. Cell Biol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">191</span> <span class="lpage" tagx="lpage" title="lpage">201</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncb3292">10.1038/ncb3292</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26689677">26689677</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Haastert</span> <span class="given-names" tagx="given-names" title="given-names">PJM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keizer-Gunnink</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kortholt</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Coupled excitable Ras and F-actin activation mediates spontaneous pseudopod formation and directed cell movement </span> <span class="source" tagx="source" title="source">Mol. Biol. Cell</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">922</span> <span class="lpage" tagx="lpage" title="lpage">934</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1091/mbc.e16-10-0733">10.1091/mbc.e16-10-0733</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28148648">28148648</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nobes</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rho, rac, and cdc42 GTPases regulate the assembly of multimolecular focal complexes associated with actin stress fibers, lamellipodia, and filopodia </span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">81</span> <span class="fpage" tagx="fpage" title="fpage">53</span> <span class="lpage" tagx="lpage" title="lpage">62</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0092-8674(95)90370-4">10.1016/0092-8674(95)90370-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7536630">7536630</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nobes</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rho GTPases control polarity, protrusion, and adhesion during cell movement</span> <span class="source" tagx="source" title="source">J. Cell Biol.</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">144</span> <span class="fpage" tagx="fpage" title="fpage">1235</span> <span class="lpage" tagx="lpage" title="lpage">1244</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1083/jcb.144.6.1235">10.1083/jcb.144.6.1235</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10087266">10087266</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ridley</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cell migration: integrating signals from front to back</span> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">302</span> <span class="fpage" tagx="fpage" title="fpage">1704</span> <span class="lpage" tagx="lpage" title="lpage">1709</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1092053">10.1126/science.1092053</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14657486">14657486</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wallar</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flanders</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swiatek</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alberts</span> <span class="given-names" tagx="given-names" title="given-names">AS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Disruption of the Diaphanous-related formin Drf1 gene encoding mDia1 reveals a role for Drf3 as an effector for Cdc42 </span> <span class="source" tagx="source" title="source">Curr. Biol.</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">534</span> <span class="lpage" tagx="lpage" title="lpage">545</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0960-9822(03)00170-2">10.1016/S0960-9822(03)00170-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12676083">12676083</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Svitkina</span> <span class="given-names" tagx="given-names" title="given-names">TM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borisy</span> <span class="given-names" tagx="given-names" title="given-names">GG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Progress in protrusion: the tell-tale scar</span> <span class="source" tagx="source" title="source">Trends Biochem. Sci.</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">432</span> <span class="lpage" tagx="lpage" title="lpage">436</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0968-0004(99)01461-9">10.1016/S0968-0004(99)01461-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10542409">10542409</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raftopoulou</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cell migration: rho GTPases lead the way</span> <span class="source" tagx="source" title="source">Dev. Biol.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">265</span> <span class="fpage" tagx="fpage" title="fpage">23</span> <span class="lpage" tagx="lpage" title="lpage">32</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ydbio.2003.06.003">10.1016/j.ydbio.2003.06.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14697350">14697350</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burridge</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wennerberg</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rho and Rac take center stage</span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">116</span> <span class="fpage" tagx="fpage" title="fpage">167</span> <span class="lpage" tagx="lpage" title="lpage">179</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0092-8674(04)00003-0">10.1016/S0092-8674(04)00003-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14744429">14744429</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pertz</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hodgson</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klemke</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hahn</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Spatiotemporal dynamics of RhoA activity in migrating cells</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">440</span> <span class="fpage" tagx="fpage" title="fpage">1069</span> <span class="lpage" tagx="lpage" title="lpage">1072</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature04665">10.1038/nature04665</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16547516">16547516</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Machacek</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Coordination of Rho GTPase activities during cell protrusion</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">461</span> <span class="fpage" tagx="fpage" title="fpage">99</span> <span class="lpage" tagx="lpage" title="lpage">103</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature08242">10.1038/nature08242</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19693013">19693013</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspenstrom</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fransson</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saras</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rho GTPases have diverse effects on the organization of the actin filament system</span> <span class="source" tagx="source" title="source">Biochem. J.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">377</span> <span class="fpage" tagx="fpage" title="fpage">327</span> <span class="lpage" tagx="lpage" title="lpage">337</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1042/bj20031041">10.1042/bj20031041</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14521508">14521508</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kitzing</span> <span class="given-names" tagx="given-names" title="given-names">TM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Positive feedback between Dia1, LARG, and RhoA regulates cell morphology and invasion</span> <span class="source" tagx="source" title="source">Genes Dev.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">1478</span> <span class="lpage" tagx="lpage" title="lpage">1483</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gad.424807">10.1101/gad.424807</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17575049">17575049</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bishop</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rho GTPases and their effector proteins</span> <span class="source" tagx="source" title="source">Biochem. J.</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">348</span> <span class="issue" tagx="issue" title="issue">Pt 2</span> <span class="fpage" tagx="fpage" title="fpage">241</span> <span class="lpage" tagx="lpage" title="lpage">255</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1042/bj3480241">10.1042/bj3480241</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10816416">10816416</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bos</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rehmann</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wittinghofer</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">GEFs and GAPs: critical elements in the control of small G proteins</span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">129</span> <span class="fpage" tagx="fpage" title="fpage">865</span> <span class="lpage" tagx="lpage" title="lpage">877</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2007.05.018">10.1016/j.cell.2007.05.018</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17540168">17540168</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dovas</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Couchman</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">RhoGDI: multiple functions in the regulation of Rho family GTPase activities</span> <span class="source" tagx="source" title="source">Biochem. J.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">390</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">9</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1042/BJ20050104">10.1042/BJ20050104</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16083425">16083425</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hodgson</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">FRET binding antenna reports spatiotemporal dynamics of GDI-Cdc42 GTPase interactions</span> <span class="source" tagx="source" title="source">Nat. Chem. Biol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">802</span> <span class="lpage" tagx="lpage" title="lpage">809</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nchembio.2145">10.1038/nchembio.2145</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27501396">27501396</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aoki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakamura</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujikawa</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsuda</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Local phosphatidylinositol 3,4,5-trisphosphate accumulation recruits Vav2 and Vav3 to activate Rac1/Cdc42 and initiate neurite outgrowth in nerve growth factor-stimulated PC12 cells </span> <span class="source" tagx="source" title="source">Mol. Biol. Cell</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">2207</span> <span class="lpage" tagx="lpage" title="lpage">2217</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1091/mbc.e04-10-0904">10.1091/mbc.e04-10-0904</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15728722">15728722</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karunarathne</span> <span class="given-names" tagx="given-names" title="given-names">WKA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giri</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venkatesh</span> <span class="given-names" tagx="given-names" title="given-names">KV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gautam</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optical control demonstrates switch-like PIP3 dynamics underlying the initiation of immune cell migration </span> <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">110</span> <span class="fpage" tagx="fpage" title="fpage">E1575</span> <span class="lpage" tagx="lpage" title="lpage">E1583</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1220755110">10.1073/pnas.1220755110</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23569254">23569254</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiner</span> <span class="given-names" tagx="given-names" title="given-names">OD</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A PtdInsP(3)- and Rho GTPase-mediated positive feedback loop regulates neutrophil polarity </span> <span class="source" tagx="source" title="source">Nat. Cell Biol.</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">509</span> <span class="lpage" tagx="lpage" title="lpage">513</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncb811">10.1038/ncb811</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12080346">12080346</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">GC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kholodenko</span> <span class="given-names" tagx="given-names" title="given-names">BN</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Spatial gradients of cellular phospho-proteins</span> <span class="source" tagx="source" title="source">FEBS Lett.</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">457</span> <span class="fpage" tagx="fpage" title="fpage">452</span> <span class="lpage" tagx="lpage" title="lpage">454</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0014-5793(99)01058-3">10.1016/S0014-5793(99)01058-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10471827">10471827</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bement</span> <span class="given-names" tagx="given-names" title="given-names">WM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Von Dassow</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rho GTPase activity zones and transient contractile arrays</span> <span class="source" tagx="source" title="source">Bioessays</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">983</span> <span class="lpage" tagx="lpage" title="lpage">993</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bies.20477">10.1002/bies.20477</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16998826">16998826</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fritz</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pertz</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The dynamics of spatio-temporal Rho GTPase signaling: formation of signaling patterns</span> <span class="source" tagx="source" title="source">F1000Res.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">749</span> <span class="lpage" tagx="lpage" title="lpage">761</span> <span class="pub-id"> <a href="https://dx.doi.org/10.12688/f1000research.7370.1">10.12688/f1000research.7370.1</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fusco</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Computer vision profiling of neurite outgrowth dynamics reveals spatiotemporal modularity of Rho GTPase signaling </span> <span class="source" tagx="source" title="source">J. Cell Biol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">212</span> <span class="fpage" tagx="fpage" title="fpage">91</span> <span class="lpage" tagx="lpage" title="lpage">111</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1083/jcb.201506018">10.1083/jcb.201506018</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26728857">26728857</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pertz</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Spatio-temporal Rho GTPase signaling—where are we now?</span> <span class="source" tagx="source" title="source">J. Cell Sci.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">123</span> <span class="fpage" tagx="fpage" title="fpage">1841</span> <span class="lpage" tagx="lpage" title="lpage">1850</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1242/jcs.064345">10.1242/jcs.064345</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20484664">20484664</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vicente-Manzanares</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Newell-Litwa</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bachir</span> <span class="given-names" tagx="given-names" title="given-names">AI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whitmore</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horwitz</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Myosin IIA/IIB restrict adhesive and protrusive signaling to generate front-back polarity in migrating cells </span> <span class="source" tagx="source" title="source">J. Cell Biol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">193</span> <span class="fpage" tagx="fpage" title="fpage">381</span> <span class="lpage" tagx="lpage" title="lpage">396</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1083/jcb.201012159">10.1083/jcb.201012159</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21482721">21482721</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanna</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miskolci</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cox</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hodgson</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A new genetically encoded single-chain biosensor for Cdc42 based on FRET, useful for live-cell imaging </span> <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">e96469</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0096469">10.1371/journal.pone.0096469</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24798463">24798463</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnsson</span> <span class="given-names" tagx="given-names" title="given-names">AKE</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Rac-FRET mouse reveals tight spatiotemporal control of Rac activity in primary cells and tissues </span> <span class="source" tagx="source" title="source">Cell Rep.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">1153</span> <span class="lpage" tagx="lpage" title="lpage">1164</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.celrep.2014.02.024">10.1016/j.celrep.2014.02.024</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24630994">24630994</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grimm</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coppey</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wieschaus</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Modelling the Bicoid gradient</span> <span class="source" tagx="source" title="source">Development</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">137</span> <span class="fpage" tagx="fpage" title="fpage">2253</span> <span class="lpage" tagx="lpage" title="lpage">2264</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1242/dev.032409">10.1242/dev.032409</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20570935">20570935</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ben-Zvi</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shilo</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fainsod</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barkai</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Scaling of the BMP activation gradient in Xenopus embryos</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">453</span> <span class="fpage" tagx="fpage" title="fpage">1205</span> <span class="lpage" tagx="lpage" title="lpage">1211</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature07059">10.1038/nature07059</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18580943">18580943</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plouhinec</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zakin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moriyama</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Robertis</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chordin forms a self-organizing morphogen gradient in the extracellular space between ectoderm and mesoderm in the Xenopus embryo </span> <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">110</span> <span class="fpage" tagx="fpage" title="fpage">20372</span> <span class="lpage" tagx="lpage" title="lpage">20379</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1319745110">10.1073/pnas.1319745110</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24284174">24284174</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kennedy</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rapid blue-light-mediated induction of protein interactions in living cells</span> <span class="source" tagx="source" title="source">Nat. Methods</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">973</span> <span class="lpage" tagx="lpage" title="lpage">975</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.1524">10.1038/nmeth.1524</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21037589">21037589</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levskaya</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiner</span> <span class="given-names" tagx="given-names" title="given-names">OD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">WA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voigt</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Spatiotemporal control of cell signalling using a light-switchable protein interaction</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">461</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">5</span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakai</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ueno</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishizuka</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yawo</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Parallel and patterned optogenetic manipulation of neurons in the brain slice using a DMD-based projector </span> <span class="source" tagx="source" title="source">Neurosci. Res.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">75</span> <span class="fpage" tagx="fpage" title="fpage">59</span> <span class="lpage" tagx="lpage" title="lpage">64</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.neures.2012.03.009">10.1016/j.neures.2012.03.009</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22469653">22469653</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thery</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Anisotropy of cell adhesive microenvironment governs cell internal organization and orientation of polarity </span> <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">103</span> <span class="fpage" tagx="fpage" title="fpage">19771</span> <span class="lpage" tagx="lpage" title="lpage">19776</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0609267103">10.1073/pnas.0609267103</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17179050">17179050</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valon</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Predictive spatiotemporal manipulation of signaling perturbations using optogenetics</span> <span class="source" tagx="source" title="source">Biophys. J.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">109</span> <span class="fpage" tagx="fpage" title="fpage">1785</span> <span class="lpage" tagx="lpage" title="lpage">1797</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bpj.2015.08.042">10.1016/j.bpj.2015.08.042</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26536256">26536256</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guilluy</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia-Mata</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burridge</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rho protein crosstalk: another social network?</span> <span class="source" tagx="source" title="source">Trends Cell Biol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">718</span> <span class="lpage" tagx="lpage" title="lpage">726</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tcb.2011.08.002">10.1016/j.tcb.2011.08.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21924908">21924908</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Innocenti</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Abi1 regulates the activity of N-WASP and WAVE in distinct actin-based processes</span> <span class="source" tagx="source" title="source">Nat. Cell Biol.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">969</span> <span class="lpage" tagx="lpage" title="lpage">976</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncb1304">10.1038/ncb1304</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16155590">16155590</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dang</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Inhibitory signalling to the Arp2/3 complex steers cell migration</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">503</span> <span class="fpage" tagx="fpage" title="fpage">281</span> <span class="lpage" tagx="lpage" title="lpage">284</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature12611">10.1038/nature12611</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24132237">24132237</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gutierrez-Uzquiza</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Coordinated activation of the Rac-GAP β2-chimaerin by an atypical proline-rich domain and diacylglycerol </span> <span class="source" tagx="source" title="source">Nat. Commun.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">1849</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms2834">10.1038/ncomms2834</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23673634">23673634</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">GE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiner</span> <span class="given-names" tagx="given-names" title="given-names">OD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A pharmacological cocktail for arresting actin dynamics in living cells</span> <span class="source" tagx="source" title="source">Mol. Biol. Cell</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">3986</span> <span class="lpage" tagx="lpage" title="lpage">3994</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1091/mbc.e11-04-0379">10.1091/mbc.e11-04-0379</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21880897">21880897</a> </span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Dongen</span> <span class="given-names" tagx="given-names" title="given-names">SFM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maiuri</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marie</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tribet</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Piel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Triggering cell adhesion, migration or shape change with a dynamic surface coating</span> <span class="source" tagx="source" title="source">Adv. Mater.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">1687</span> <span class="lpage" tagx="lpage" title="lpage">1691</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/adma.201204474">10.1002/adma.201204474</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23355329">23355329</a> </span> </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Etienne-Manneville</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Integrin-mediated activation of Cdc42 controls cell polarity in migrating astrocytes through PKCzeta </span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">106</span> <span class="fpage" tagx="fpage" title="fpage">489</span> <span class="lpage" tagx="lpage" title="lpage">498</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0092-8674(01)00471-8">10.1016/S0092-8674(01)00471-8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11525734">11525734</a> </span> </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene targeting of Cdc42 and Cdc42GAP affirms the critical involvement of Cdc42 in filopodia induction, directed migration, and proliferation in primary mouse embryonic fibroblasts </span> <span class="source" tagx="source" title="source">Mol. Biol. Cell</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">4675</span> <span class="lpage" tagx="lpage" title="lpage">4685</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1091/mbc.e06-05-0466">10.1091/mbc.e06-05-0466</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16914516">16914516</a> </span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pankov</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A Rac switch regulates random versus directionally persistent cell migration</span> <span class="source" tagx="source" title="source">J. Cell Biol.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">170</span> <span class="fpage" tagx="fpage" title="fpage">793</span> <span class="lpage" tagx="lpage" title="lpage">802</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1083/jcb.200503152">10.1083/jcb.200503152</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16129786">16129786</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maiuri</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Actin flows mediate a universal coupling between cell speed and cell persistence</span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">161</span> <span class="fpage" tagx="fpage" title="fpage">374</span> <span class="lpage" tagx="lpage" title="lpage">386</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2015.01.056">10.1016/j.cell.2015.01.056</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25799384">25799384</a> </span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Amplification and temporal filtering during gradient sensing by nerve growth cones probed with a microfluidic assay </span> <span class="source" tagx="source" title="source">Biophys. J.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">103</span> <span class="fpage" tagx="fpage" title="fpage">1648</span> <span class="lpage" tagx="lpage" title="lpage">1656</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bpj.2012.08.040">10.1016/j.bpj.2012.08.040</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23083707">23083707</a> </span> </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andrew</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Insall</span> <span class="given-names" tagx="given-names" title="given-names">RH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chemotaxis in shallow gradients is mediated independently of PtdIns 3-kinase by biased choices between random protrusions </span> <span class="source" tagx="source" title="source">Nat. Cell Biol.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">193</span> <span class="lpage" tagx="lpage" title="lpage">200</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncb1536">10.1038/ncb1536</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17220879">17220879</a> </span> </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abella</span> <span class="given-names" tagx="given-names" title="given-names">JV</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Gab1 scaffold regulates RTK-dependent dorsal ruffle formation through the adaptor Nck </span> <span class="source" tagx="source" title="source">J. Cell Sci.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">123</span> <span class="fpage" tagx="fpage" title="fpage">1306</span> <span class="lpage" tagx="lpage" title="lpage">1319</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1242/jcs.062570">10.1242/jcs.062570</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20332103">20332103</a> </span> </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kazanietz</span> <span class="given-names" tagx="given-names" title="given-names">MG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chimaerins: GAPs that bridge diacylglycerol signalling and the small G-protein Rac</span> <span class="source" tagx="source" title="source">Biochem. J.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">403</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">12</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1042/BJ20061750">10.1042/BJ20061750</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17346241">17346241</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">HE</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The spatiotemporal limits of developmental Erk signaling</span> <span class="source" tagx="source" title="source">Dev. Cell</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">185</span> <span class="lpage" tagx="lpage" title="lpage">192</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.devcel.2016.12.002">10.1016/j.devcel.2016.12.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28118601">28118601</a> </span> </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zimmerman</span> <span class="given-names" tagx="given-names" title="given-names">SP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asokan</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuhlman</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bear</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cells lay their own tracks—optogenetic Cdc42 activation stimulates fibronectin deposition supporting directed migration </span> <span class="source" tagx="source" title="source">J. Cell Sci.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">130</span> <span class="fpage" tagx="fpage" title="fpage">2971</span> <span class="lpage" tagx="lpage" title="lpage">2983</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1242/jcs.205948">10.1242/jcs.205948</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28754687">28754687</a> </span> </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamao</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Distinct predictive performance of Rac1 and Cdc42 in cell migration</span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">17527</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep17527">10.1038/srep17527</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26634649">26634649</a> </span> </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maiuri</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The first world cell race</span> <span class="source" tagx="source" title="source">Curr. Biol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">R673</span> <span class="lpage" tagx="lpage" title="lpage">R675</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cub.2012.07.052">10.1016/j.cub.2012.07.052</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22974990">22974990</a> </span> </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soderling</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The WRP component of the WAVE-1 complex attenuates Rac-mediated signalling</span> <span class="source" tagx="source" title="source">Nat. Cell Biol.</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">970</span> <span class="lpage" tagx="lpage" title="lpage">975</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncb886">10.1038/ncb886</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12447388">12447388</a> </span> </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gradient sensing during chemotaxis §</span> <span class="source" tagx="source" title="source">Curr. Opin. Cell Biol.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">532</span> <span class="lpage" tagx="lpage" title="lpage">537</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ceb.2013.06.007">10.1016/j.ceb.2013.06.007</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23880435">23880435</a> </span> </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Connolly</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rice</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feig</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buchsbaum</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Tiam1-IRSp53 complex formation directs specificity of rac-mediated actin cytoskeleton regulation </span> <span class="source" tagx="source" title="source">Mol. Cell. Biol.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">4602</span> <span class="lpage" tagx="lpage" title="lpage">4614</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/MCB.25.11.4602-4614.2005">10.1128/MCB.25.11.4602-4614.2005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15899863">15899863</a> </span> </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pykäläinen</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lappalainen</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">I-BAR domain proteins: linking actin and plasma membrane dynamics</span> <span class="source" tagx="source" title="source">Curr. Opin. Cell Biol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">14</span> <span class="lpage" tagx="lpage" title="lpage">21</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ceb.2010.10.005">10.1016/j.ceb.2010.10.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21093245">21093245</a> </span> </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parrini</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">SH3BP1, an exocyst-associated RhoGAP, inactivates Rac1 at the front to drive cell motility </span> <span class="source" tagx="source" title="source">Mol. Cell</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">42</span> <span class="fpage" tagx="fpage" title="fpage">650</span> <span class="lpage" tagx="lpage" title="lpage">661</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.molcel.2011.03.032">10.1016/j.molcel.2011.03.032</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21658605">21658605</a> </span> </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Filonov</span> <span class="given-names" tagx="given-names" title="given-names">GS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Bright and stable near-infrared fluorescent protein for in vivo imaging</span> <span class="source" tagx="source" title="source">Nat. Biotechnol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">757</span> <span class="lpage" tagx="lpage" title="lpage">761</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nbt.1918">10.1038/nbt.1918</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21765402">21765402</a> </span> </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Azioune</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Storch</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bornens</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thery</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Piel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Simple and rapid process for single cell micro-patterning</span> <span class="source" tagx="source" title="source">Lab. Chip</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">1640</span> <span class="lpage" tagx="lpage" title="lpage">1642</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/b821581m">10.1039/b821581m</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19458875">19458875</a> </span> </span> </li> </ul> </div> </div>  </body></html>