<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-journal-id" title="nlm-journal-id">0410462</span> <span class="pubmed-jr-id" title="pubmed-jr-id">6011</span> <span class="nlm-ta" title="nlm-ta">Nature</span> <span class="iso-abbrev" title="iso-abbrev">Nature</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Nature</span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">0028-0836</span> <span class="issn-epub" tagx="issn" title="issn-epub">1476-4687</span> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5876151">5876151</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29562237">29562237</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/nature26141">10.1038/nature26141</a> </span> <span class="manuscript" title="manuscript">nihpa942948</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Optogenetic regulation of engineered cellular metabolism for microbial chemical production</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Evan M.</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Yanfei</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mehl</span> <span class="given-names" tagx="given-names" title="given-names">Justin</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">Helen</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lalwani</span> <span class="given-names" tagx="given-names" title="given-names">Makoto A.</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toettcher</span> <span class="given-names" tagx="given-names" title="given-names">Jared E.</span> </span> <a href="#A2">2</a> <a href="#FN1">*</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Avalos</span> <span class="given-names" tagx="given-names" title="given-names">Jos√© L.</span> </span> <a href="#A1">1</a> <a href="#A3">3</a> <a href="#FN1">*</a> </span> </div> <span class="citation_author_institution" id="A1">[1], </span> <span class="citation_author_institution" id="A2">[2], </span> <span class="citation_author_institution" id="A3">[3], </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">javalos@princeton.edu</span> (J.L.A.); <span class="email" tagx="email" title="email">toettcher@princeton.edu</span> (J.E.T.) </div> </div> <span class="pub-date-nihms-submitted" title="pub-date-nihms-submitted">nihms-submitted: <span>2018-2-2</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-3-3</span> </span> <span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2018-3-3</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-9-9</span> </span> <span class="volume" tagx="volume" title="volume">555</span> <span class="issue" tagx="issue" title="issue">7698</span> <span class="fpage" tagx="fpage" title="fpage">683</span> <span class="lpage" tagx="lpage" title="lpage">687</span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p">Users may view, print, copy, and download text and data-mine the content in such documents, for the purposes of academic research, subject always to the full Conditions of use: <span class="uri" title="uri"> <a href="http://www.nature.com/authors/editorial_policies/license.html#terms">http://www.nature.com/authors/editorial_policies/license.html#terms</a> </span> </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The optimization of engineered metabolic pathways requires careful control over the levels and timing of metabolic enzyme expression <sup> <a href="#R1">1</a>- <a href="#R4">4</a> </sup>. Optogenetic tools are ideal for achieving such precise control, as light can be applied and removed instantly without complex media changes. Here we show that light-controlled transcription can be used to enhance the biosynthesis of valuable products in engineered <i>Saccharomyces cerevisiae</i>. We introduce new optogenetic circuits to shift cells from a light-induced growth phase to a darkness-induced production phase, which allows us to control fermentation purely with light. Furthermore, optogenetic control of engineered pathways enables a new mode of bioreactor operation using periodic light pulses to tune enzyme expression during the production phase of fermentation to increase yields. Using these advances, we control the mitochondrial isobutanol pathway to produce up to 8.49 ¬± 0.31 g/L of isobutanol and 2.38 ¬± 0.06 g/L of 2-methyl-1-butanol micro-aerobically from glucose. These results make a compelling case for the application of optogenetics to metabolic engineering for valuable products. </p> </div> </div> </div> <div class="body" title="body"> <p>Metabolic engineering aims to rewire the metabolism of organisms for efficient conversion of inexpensive substrates into valuable products such as chemicals, fuels, or drugs <sup> <a href="#R1">1</a>, <a href="#R2">2</a> </sup>. Fine-tuning the timing and expression levels of enzymes involved in both natural and engineered pathways can relieve bottlenecks and minimize the metabolic burden of chemical production <sup> <a href="#R3">3</a>, <a href="#R4">4</a> </sup>. This is especially critical when the product of interest or its precursors are toxic, or when the biosynthetic pathway of interest competes with endogenous pathways that are essential for cell growth. </p> <p>To address these challenges, metabolic engineers frequently use inducible systems to control metabolic enzyme expression <sup> <a href="#R5">5</a>‚Äì <a href="#R7">7</a> </sup> (see Supplementary Discussion). This approach makes it possible to separate bioreactor operation into two phases: a growth phase, during which product biosynthesis is repressed, and a production phase, when flux through the engineered pathway is maximized. Essential pathways that compete with product formation can be controlled with "metabolic valves": genetic programs that express essential enzymes during the growth phase to build biomass, and repress them during the production phase to redirect metabolism towards desired products <sup> <a href="#R5">5</a>, <a href="#R8">8</a> </sup>. </p> <p>Light is an attractive strategy to control gene expression in yeast for metabolic engineering applications. It is inexpensive and compatible with any carbon source or nutrient composition. Furthermore, light can be applied or removed instantaneously; this precise control over the level or duration of enzyme expression could simplify the screening of optimal metabolic pathway enzyme proportions and enable new time-varying modes of control <sup> <a href="#R9">9</a>, <a href="#R10">10</a> </sup>. In recent years, light-switchable transcription modules have been shown to enable non-toxic, tunable gene expression in a variety of organisms <sup> <a href="#R11">11</a>, <a href="#R12">12</a> </sup>, including yeast <sup> <a href="#R13">13</a>‚Äì <a href="#R15">15</a> </sup>. We thus sought to test whether optogenetics could be used to control rewired cellular metabolisms to overproduce valuable chemicals. </p> <p>Here, we describe two powerful optogenetic gene expression systems for yeast, <i>OptoEXP</i> and <i>OptoINVRT</i>, based on the blue light-activated EL222 eukaryotic gene expression system <sup> <a href="#R12">12</a> </sup>. Using these systems, we show that it is possible to strongly activate and repress distinct sets of genes in a light-dependent manner. We apply our approach to control endogenous and engineered metabolic pathways to define the growth and production phases of fermentation, enabling production of three valuable chemicals (lactate or isobutanol and 2-methyl-1-butanol) whose biosyntheses directly compete with ethanol production. Using a time-varying illumination schedule, we achieve titers of 8.49 ¬± 0.31 g/L of isobutanol and 2.38 ¬± 0.06 g/L of 2-methyl-1-butanol (2-MBOH). Our results thus reveal that a simple technology ‚Äì bidirectional light-controlled enzyme expression ‚Äì offers a rich set of tools for metabolic engineering. </p> <p>Our first goal was to construct bidirectional gene circuits in yeast to either induce or repress genes of interest with light. We used the EL222 optogenetic transcription system, which consists of a light-sensitive transcription factor from <i>Erythrobacter litoralis</i> (EL222) and its corresponding promoter, P <sub>C120</sub>, to drive expression of a gene of interest ( <a href="#F11">Fig. 1a</a>) <sup> <a href="#R16">16</a>‚Äì <a href="#R18">18</a> </sup>. The EL222 system is a robust and versatile gene expression platform, previously applied in <i>E. coli</i>, mammalian cells, and zebrafish <sup> <a href="#R12">12</a>, <a href="#R19">19</a>, <a href="#R20">20</a> </sup>. We first constructed a yeast strain, YEZ139, in which expression of VP16-EL222 was driven by the strong constitutive P <sub>PGK1</sub> promoter (P <sub>PGK1</sub>-VP16-EL222) and green fluorescent protein (GFP) was driven by the P <sub>C120</sub> promoter (P <sub>C120</sub>-GFP; see Methods, <a href="#SD1">Supplementary Tables 1 and 2</a>, <a href="#F1">Extended Data Fig. 1</a>). We called this system (and variants with different promoters driving EL222) <i>OptoEXP</i>. </p> <p> <i>OptoEXP</i> enables strong and titratable light-inducible gene expression. In both glucose and glycerol media, cells with <i>OptoEXP</i> controlling GFP expression show a 43-fold increase in GFP expression when exposed to constant light compared to dark-incubated cells, while intermittent light pulses produce intermediate expression levels ( <a href="#F11">Fig. 1b</a>, <a href="#F2">Extended Data Fig. 2</a>). We tested that all light sources used were sufficiently bright to maximally activate the EL222 system; thus, varying duty cycle was a reliable and reproducible method for achieving intermediate gene expression output ( <a href="#F3">Extended Data Fig. 3</a>, Online Methods). The maximum activation levels of <i>OptoEXP</i> are comparable to those reached by the <i>ADH1</i> promoter (P <sub>ADH1</sub>), a constitutive promoter commonly used in metabolic engineering ( <a href="#F11">Fig. 1c</a>). </p> <p>To construct a light-repressible gene circuit, we inverted the response of the <i>OptoEXP</i> system in a manner akin to the NOT logical gate used in digital processes. We harnessed the yeast galactose (GAL) regulon <sup> <a href="#R21">21</a> </sup>, in which Gal80p binds to and inhibits the Gal4p transcription factor, blocking its ability to induce expression from the <i>GAL1</i> promoter (P <sub>GAL1</sub>). We reasoned that engineering yeast cells with constitutive <i>GAL4</i> and EL222-regulated <i>GAL80</i> would lead to constitutive expression from the P <sub>GAL1</sub> promoter in the dark and repression of P <sub>GAL1</sub> in the light ( <a href="#F11">Fig. 1d</a>, <a href="#F4">Extended Data Fig. 4a, b</a>). </p> <p>Starting from a strain in which both <i>GAL80</i> and <i>GAL4</i> are deleted, YEZ44, we constructed three expressed variants of this core inverter topology, which we termed <i>OptoINVRT1-3</i> ( <a href="#SD1">Supplementary Tables 1 and 2</a>). These variants differed in the strength of the promoter driving <i>GAL4</i> and the fusion of a photosensitive degron domain <sup> <a href="#R22">22</a> </sup> to the C-terminus of Gal4p to induce faster and more complete light-dependent repression ( <a href="#F4">Extended Data Fig. 4a, b</a> and <a href="#SD1">Supplementary Table 3</a>). Using P <sub>GAL1</sub>-GFP as a reporter, we found that all three <i>OptoINVRT</i> circuits exhibit robust light-induced gene repression ( <a href="#F11">Fig. 1e</a>, <a href="#SD1">Supplementary Table 4</a>). <i>OptoINVRT2</i> has the highest maximum expression in the dark (almost 85% of P <sub>TEF1</sub> levels), while <i>OptoINVRT3</i> has the highest fold of repression (more than 70-fold) and lowest expression in full-light. All three <i>OptoINVRT</i> circuits show similar responses in a second yeast strain, Y202, that is relevant for metabolic engineering due to its deletion of all three pyruvate decarboxylase genes (S288C, <i>gal80-Œî pdc1-Œî, pdc5-Œî</i>, and <i>pdc6-Œî</i>; <a href="#F11">Fig. 1f</a>). Our results thus demonstrate that the <i>OptoINVRT</i> platform can achieve a wide range of expression levels, light sensitivities, and fold-change responses in different strain backgrounds for flexible incorporation in diverse metabolic engineering applications. </p> <p>To reduce ethanol byproduct formation, while still allowing cell growth on glucose, we used <i>OptoEXP</i> to control pyruvate decarboxylation (PDC), an essential step in ethanol biosynthesis. Completely removing PDC activity in <i>S. cerevisiae</i> (by deleting <i>PDC1</i>, <i>PDC5</i> and <i>PDC6</i>) does not suffice, because the triple deletion renders yeast unable to grow on glucose due to the essential role of these genes in NAD <sup>+</sup> recycling for glycolysis <sup> <a href="#R23">23</a> </sup>, as well as a lack of effective alternatives for ATP generation due to glucose-mediated repression of respiration <sup> <a href="#R24">24</a> </sup>. Thus, we used our <i>OptoEXP</i> circuit to build a light-dependent metabolic valve for <i>PDC1</i> expression ( <a href="#F12">Fig. 2a</a>). Light stimulation "opens" the valve, enabling robust cell growth and ethanol production; dark incubation "closes" the valve to limit the metabolism of pyruvate into ethanol, thereby inhibiting cell growth on glucose, and enhancing the biosynthesis of alternative pyruvate-derived products. </p> <p>A strain with all <i>PDC</i> genes deleted and with <i>OptoEXP</i> driving <i>PDC1</i> expression, YEZ61-23 ( <a href="#SD1">Supplementary Table 2</a>), exhibits light-dependent growth on glucose. YEZ61-23 grows on glycerol/ethanol (YPGE) independently of light because PDC is dispensable for respiration. However, it is only able to grow on glucose plates (YPD) in the presence of blue light ( <a href="#F12">Fig. 2b</a>), with its growth rate in liquid media reaching 91.9 ¬± 4.9% of the wild type (BY4741) at full light ( <a href="#F12">Fig. 2c and d</a>). Significant growth is achieved even at low light doses (43.7 ¬± 2.7% of wild-type growth rate with only 12.5% light duty cycle), suggesting that light-dependent growth may be achieved at cell densities relevant for metabolic engineering applications. </p> <p>We then tested whether the bidirectional control afforded by combining our <i>OptoEXP</i> and <i>OptoINVRT</i> circuits could be used to drive two phases of cellular metabolism ‚Äì a growth phase with mainly ethanol fermentation, and a production phase where carbon instead accumulates as a desired valuable product: either lactate or isobutanol. Production of both lactate and isobutanol compete directly with ethanol production as distinct enzymes modify pyruvate to produce each product (ethanol: Pdc1p; lactate: LDH; isobutanol: Ilv2p and subsequent enzymes) ( <a href="#F13">Fig. 3a</a>, <a href="#F5">Extended Data Fig. 5a</a>). We reasoned that by controlling the expression of <i>PDC1</i> with <i>OptoEXP</i> and LDH/ <i>ILV2</i> with <i>OptoINVRT</i>, moving cells from light to dark would shift the metabolism from ethanol to lactate/isobutanol production ( <a href="#F13">Fig. 3b</a>, <a href="#F5">Extended Data Fig. 5b</a>). </p> <p>We first tested our strategy in the simpler case of lactate biosynthesis, which requires overexpression of only one enzyme (LDH) and whose presence in the media is well-tolerated by yeast. We integrated multiple copies of P <sub>GAL1</sub>-LDH and P <sub>C120</sub>- <i>PDC1</i> into our Y202 <i>OptoINVRT</i> strains ( <a href="#F6">Extended Data Fig. 6a</a>) and observed a pronounced increase in lactate biosynthesis upon a shift from dark to light, consistent with light-dependent changes in <i>PDC1</i> and LDH expression levels (see Methods, <a href="#F4">Extended Data Fig. 4c</a>, <a href="#F5">5</a>). Based on these promising results, we next tested our optogenetic circuits on the more complex biosynthesis of isobutanol ( <a href="#F13">Fig. 3a, b</a>), an advanced biofuel, with much higher toxicity than lactate. Our approach was to optogenetically control only the first enzyme in the isobutanol pathway, acetolactate synthase (encoded by <i>ILV2</i>), reasoning that if subsequent enzymes were constitutively expressed, overall pathway flux would still be light-controlled. </p> <p>We transformed Y202 with <i>OptoEXP</i>, each one of the <i>OptoINVRT</i> circuits, multiple copies of P <sub>GAL1</sub>- <i>ILV2</i> and P <sub>C120</sub>- <i>PDC1</i> and the rest of the mitochondrial isobutanol biosynthetic pathway under strong constitutive promoters <sup> <a href="#R25">25</a> </sup>, producing YEZ159, YEZ156, and, HPY6 ( <a href="#SD1">Supplementary Tables 1 and 2</a>, see Methods). We found that colonies from YEZ159, containing <i>OptoINVRT1</i>, produced the highest isobutanol titers from 4% glucose ( <a href="#F4">Extended Data Fig. 4d, e</a>). To further enhance isobutanol production, we deleted the mitochondrial branched chain amino acid aminotransferase, <i>BAT1</i>, which competes for the Œ±-ketoisovalerate precursor <sup> <a href="#R26">26</a>, <a href="#R27">27</a> </sup>, resulting in strain YEZ167-4 ( <a href="#SD1">Supplementary Table 2</a>). YEZ167-4 contains six copies of P <sub>C120</sub>- <i>PDC1</i> ( <a href="#F6">Extended Data Fig. 6a</a>), leading to fast cell growth under full light. </p> <p>Varying the cell density at which cultures are switched from dark to light and the incubation time in the dark before starting the fermentation ( <a href="#F5">Extended Data Fig. 5c</a>), we found that YEZ167-4 can produce as much as 735 ¬± 15 mg/L of isobutanol from 2 % glucose over 48 h (a yield of 34.2 ¬± 0.7 mg isobutanol/g glucose, <a href="#F7">Extended Data Fig. 7</a>). Switching to 15% glucose and 80 hour-long fermentations increases production to only 1.22 ¬± 0.11 g/L of isobutanol ( <a href="#F13">Fig. 3c</a>). However, under these conditions YEZ167-4 is unable to consume all the glucose in the medium ( <a href="#F8">Extended Data Fig. 8a</a>), indicating a stalled fermentation in which cells have undergone premature metabolic arrest. </p> <p>We hypothesized that during prolonged dark incubation, Pdc1p becomes limiting to a point where cellular metabolism arrests due to NAD <sup>+</sup> depletion. We reasoned that periodic pulses of light during the production phase of fermentation could transiently induce <i>PDC1</i> expression, thus increasing NAD <sup>+</sup> pools and restoring cellular metabolism, glucose consumption, and isobutanol production. We tested different light schedules, applying periodic illumination bouts (30 minutes of blue light at a duty cycle of 15s/65s) once every 5, 10, or 20 hours throughout the 80-hour-long production phase of the fermentation. When cells are exposed to one light bout every 10 hours, isobutanol production nearly triples to 3.37 ¬± 0.17 g/L ( <a href="#F13">Fig. 3c</a>), and 2-MBOH, another desirable advanced biofuel co-produced with the mitochondrial isobutanol pathway <sup> <a href="#R25">25</a> </sup>, is produced at 433 ¬± 69 mg/L. These isobutanol yields are more than 2.5 times higher than if cells are kept in the dark throughout the production phase of the fermentation, and 4 times higher than YZy335, a strain that contains the entire isobutanol pathway and <i>BAT1</i> deletion but lacks light-inducible metabolic control. We directly confirmed that our light pulses partially replenish the NAD <sup>+</sup> metabolite pool under these fermentation conditions (see Methods; <a href="#F8">Extended Data Fig. 8b</a>). </p> <p>To determine the applicability of optogenetics to metabolic engineering in lab-scale bioreactors, we measured the growth of the <i>OptoEXP</i>-driven <i>PDC1</i> strain, YEZ167-4, in a 2-L fermentor with 15% glucose and constant blue light ( <a href="#F9">Extended Data Fig. 9a, b</a>). Under these conditions, YEZ167-4 is able to reach the same optical density (OD <sub>600</sub> = 17.1) as a light-independent, PDC <sup>+</sup> strain, YZy335 ( <a href="#SD1">Supplementary Table 2</a>), indicating that light penetration into the bioreactor is sufficient to drive robust <i>PDC1</i> expression and growth ( <a href="#F13">Fig. 3d</a>). We then measured isobutanol and 2-MBOH production in fed-batch 0.5 L bioreactors under microaerobic conditions and pH control ( <a href="#F9">Extended Data Fig. 9c, d</a>). We used constant illumination to build YEZ167-4 biomass during a batch growth phase, and then switched to a fed-batch production phase with periodic bouts of light to maintain sufficient NAD <sup>+</sup> levels to preserve metabolic activity. This fermentation strategy yielded 8.49 ¬± 0.31 g/L of isobutanol and 2.38 ¬± .06 g/L of 2-MBOH, with post-induction average yields of 53.5 ¬± 8.4 mg/g-glucose for isobutanol (13% of theoretical maximum) and 14.17 ¬± 2.57 mg/g-glucose of 2-MBOH, with measured instantaneous isobutanol yields of up to 270.6 mg/g (66% theoretical yield) ( <a href="#F13">Fig. 3e</a>, <a href="#SD1">Supplementary Table 5, see Methods and Supplementary Results</a>). Our mechanistic predictions of a darkness-induced metabolic shift to isobutanol biosynthesis is supported by qPCR measurements of <i>PDC1</i> and <i>ILV2</i> mRNA levels ( <a href="#F6">Extended Data Fig. 6b</a>). </p> <p>Optogenetics is a powerful way to control cellular physiology. Photosensitive proteins have had an enormous impact in neuroscience, cell biology, and developmental biology by controlling ion channels, enzyme activity, and gene expression <sup> <a href="#R9">9</a>, <a href="#R28">28</a>‚Äì <a href="#R30">30</a> </sup>. Our work demonstrates that optogenetics also holds promise in metabolic engineering, enabling the reversible control and fine-tuning of engineered metabolic pathways. </p> <p>Nevertheless, combining optogenetics and metabolic engineering has its challenges. The high cell densities usually associated with microbial fermentations might be predicted to severely limit light penetration, which is one reason we initially adopted our design of a darkness-induced production phase. However, additional experiments showed robust light-stimulated gene expression could be achieved even at cell densities as high as OD <sub>600</sub> = 50 in 5-L bioreactors ( <a href="#F10">Extended Data Fig. 10</a>), suggesting that the high light sensitivity and strong gene expression achieved by the current generation of optogenetic tools may be sufficient even in these challenging conditions. Even more potent responses to dim light could potentially be achieved by using well-characterized LOV mutants with longer lit-state lifetimes <sup> <a href="#R18">18</a> </sup> (our current VP16-EL222 remains in the photoactivated state for ~30 sec <sup> <a href="#R12">12</a> </sup>), or other photoactive proteins with highly stable light-switchable conformations <sup> <a href="#R13">13</a> </sup> (see <a href="#SD1">Supplementary Discussion</a>). For fermentation conditions that pose a greater challenge to light penetration than in this study, <i>OptoINVRT</i> circuits offer a solution to gene expression control as they do not require light during the fermentation. </p> <p>We utilized optogenetic regulation of engineered metabolic pathways to address the long-standing challenge of ethanol competition in branched-chain alcohol production. We and others have developed triple PDC deletion strains ( <i>pdc1Œî, pdc5Œî, pdc6Œî</i>) that recovered their ability to grow on glucose by directed evolution <sup> <a href="#R23">23</a> </sup>. However, efforts to produce isobutanol in these strains from mitochondrial (unpublished results) or cytosolic <sup> <a href="#R31">31</a> </sup> biosynthetic pathways have thus far been unsuccessful. Our light-controlled metabolic valve offers an effective alternative to genetically deleting essential pathways that compete with a pathway of interest <sup> <a href="#R5">5</a> </sup>. This allowed us to surpass the highest titers of isobutanol and 2-MBOH reported for yeast in the peer-reviewed literature by 5-fold ( <a href="#SD1">Supplementary Table 6</a>) and 20-fold <sup> <a href="#R25">25</a> </sup>, respectively, making a strong case for the application of optogenetics to metabolic engineering. </p> <p>Optogenetic regulation of metabolic pathways opens the door to new strategies for optimizing engineered pathways and fermentation conditions using periodic light pulses. Although further development for industrial applications is needed, in the future, light inputs delivered to a bioreactor could be automatically controlled in response to feedback from different fermentation outputs (e.g. optical density or biosensors readout), providing unprecedented capabilities for operating, optimizing, and automating fermentations for valuable product biosynthesis. </p> <div class="onlinemethods" title="sec"> <div class="title" tagx="title" title="title">
ONLINE METHODS</div> <div class="assemblyofdnaconstructs" title="sec"> <div class="title" tagx="title" title="title">
Assembly of DNA constructs</div> <p>We cloned promoter-gene-terminator sequences into previously described standardized vector series (pJLA vectors) <sup> <a href="#R25">25</a> </sup>. This allows for easy manipulation and generation of multi-gene plasmids. All genes were designed to have NheI and XhoI restriction sites at the 5‚Ä≤ and 3‚Ä≤ ends, respectively, which were used to insert the genes into pJLA vectors. Each promoter-gene-terminator construct is flanked by XmaI and AgeI restriction sites at their 5‚Ä≤ ends, and MreI, AscI and BspEI sites at their 3‚Ä≤ ends, which we used for easy assembly of multi-gene plasmids, as previously described <sup> <a href="#R25">25</a> </sup> ( <a href="#SD1">Supplementary Table 1</a>). </p> <p>Qiagen Miniprep, Qiagen Gel Extraction, and Qiagen PCR purification kits were used to extract and purify plasmids and DNA fragments. Most genes and promoters ( <i>ILV2</i>, <i>ILV3</i>, <i>ILV5</i>, <i>ARO10</i>, LlAdhA <sup>RE1,32</sup>, <i>GAL4</i>, <i>GAL80</i>, GFP, P <sub>GAL1</sub>, P <sub>TEF</sub>, P <sub>TDH3</sub>, P <sub>PGK1</sub>, P <sub>CYC1</sub>, P <sub>ADH1</sub>) were amplified from yeast genomic DNA or lab plasmids, using Phusion Polymerase from NEB, following manufacturer‚Äôs instructions. Other genes were amplified from plasmids kindly provided by other groups: PsLDH from plasmid pET28a, LDH <sup>33,34</sup> from Dr. Jinsuk J. Lee; and the photosensitive degron derived from the fusion of phototropin1 LOV2-V19L domain from <i>Arabidopsis thaliana</i> and a synthetic degradation sequence derived from the murine ornithine decarboxylase (ODC) from plasmid pDS143 from Dr. Christof Taxis <sup> <a href="#R22">22</a> </sup>. The codon-optimized sequence for VP16-EL222 was purchased as a gBlock from IDT. The sequence for the P <sub>C120</sub> promoter was synthesized by Bio Basic‚Äôs gene synthesis service. When pJLA vectors were not available, we used Gibson isothermal assembly to produce our constructs, based on published protocols <sup>35</sup>. Enzymes were purchased from NEB (XmaI, AscI, NheI, XhoI, BspEI, AgeI, T4 DNA ligase, Phusion Polymerase) and Thermo Fisher (MreI). </p> <p>We modified the single copy integration plasmid pNH603 <sup>36</sup> to make a plasmid compatible with the pJLA vectors that can be used to introduce gene cassettes into the <i>HIS3</i> locus. We first removed the AscI site of pNH603 and replaced the T <sub>ADH1</sub> sequence between PtsI and SacI with a fragment containing an XmaI restriction site. Then we introduced a cloning sequence array consisting of XmaI, MreI, AscI, and BspEI between the XmaI site (which replaced the T <sub>ADH1</sub>) and KpnI restrictions sites, to make pYZ12-B ( <a href="#F1">Extended Data Fig. 1a</a>, <a href="#SD1">Supplementary Table 1</a>). This addition makes pYZ12-B compatible with the pJLA platform of vectors, and allowed for easy transfer of gene cassettes from pJLA 2Œº plasmids. Gene constructs in pYZ12-B were integrated into the <i>HIS3</i> locus of the genome by linearizing the plasmid with PmeI. We also used pRSII416 <sup>37</sup> to introduce genes in a single copy episomal plasmid (CEN); this was created by inserting promoter-gene-terminator constructs cut from pJLA vectors with XmaI and MreI and inserting at the XmaI site of pRSII416. </p> <p>Similar to pYZ12-B, we developed pYZ23 ( <a href="#SD1">Supplementary Table 1</a>, <a href="#F1">Extended Data Fig. 1b</a>), a pJLA vector-compatible plasmid, to integrate multiple copies of gene constructs into the Œ¥-sites of the yeast genome. The pYZ23 plasmid targets the YARCdelta5, the 337 bp long terminal repeat (LTR) of <i>Saccharomyces cerevisiae</i> Ty1 retrotransposons (YARCTy1-1, SGD ID: S000006792) <sup>38</sup>. We constructed pYZ23 with four overlapping DNA fragments using Gibson isothermal assembly method <sup>35</sup>. The four fragments are: (1) the PmeI-linearized backbone fragment from pYZ12-B containing the AMP-resistance gene; (2) the first 207 bp of the YARCdelta5 LTR and (3) the last 218 bp of the YARCdelta5 LTR, both of which were amplified from the BY4741 genome using primer pairs Yfz_Oli39 &amp;amp; Yfz_Oli40, and Yfz_Oli43 &amp;amp; Yfz_Oli44, respectively ( <a href="#SD1">Supplementary Table 7</a>); and (4) the BleMX6 gene cassette from pCY 3090-07 (Addgene plasmid # 36232), amplified with primers Yfz_Oli41 and Yfz_Oli42 ( <a href="#SD1">Supplementary Table 7</a>), which add flanking loxP sites (lox66 and lox71) to the BleMX6 gene. Additional restriction sites, including MreI, AscI, and BspEI, were introduced for subcloning ( <a href="#F1">Extended Data Fig. 1b</a>, <a href="#SD1">Supplementary Table 7</a>). </p> <p>All vectors were sequenced with Sanger Sequencing from GENEWIZ before using them to transform yeast. We avoid using tandem repeats to prevent recombination after yeast transformation ( <a href="#F1">Extended Data Fig. 1c</a>), and thus do not observe instability of plasmids. </p> </div> <div class="yeasttransformations" title="sec"> <div class="title" tagx="title" title="title">
Yeast transformations</div> <p>Yeast transformations were carried out using standard lithium acetate protocols, and the resulting strains are cataloged in <a href="#SD1">Supplementary Table 2</a>. Gene constructs derived from pYZ12-B and pYZ23 vectors were genomically integrated into the <i>HIS3</i> locus and Œ¥-sites (YARCdelta5), respectively. These vectors were first linearized with PmeI, followed by purification of the DNA fragments using the Qiagen PCR purification kit before using them for yeast transformation. Gene deletions were carried out by homologous recombination. DNA fragments containing antibiotic resistance cassettes flanked with Lox-P sites were amplified with PCR from pAG26 <sup>39</sup> (containing the hygromycin resistance gene HygB-PT), pUG6 <sup>40</sup> (containing the G418 resistance gene KanMX), or pAG36 <sup>39</sup> (containing the nourseothricin resistance gene NAT1), using primers with 40 base pairs of homology to the promoter and terminator regions of the gene targeted for deletion. Antibiotic resistance markers were subsequently removed by expressing Cre recombinase from the pSH62 (AF298785) vector <sup>41</sup>. After transformation, cells were plated on synthetic complete (SC) drop out media depending on the autotrophy restored by the construct. In the case of antibiotic selection, cells were plated onto nonselective YPD plates for 16 hours, then replica plated onto YPD plates with 300 Œºg/mL hygromycin (purchased from Invitrogen), 200 Œºg/mL nourseothricin (purchased from WERNER BioAgents), or 200 Œºg/mL G418, purchased from Gibco by Life Technologies). Zeocin was used to select for Œ¥-integration ranging from 800 to 1200 Œºg/mL (purchased from Thermo Fisher Scientific). </p> <p>All strains with genomic integrations or gene deletions were genotyped with PCR to confirm their accuracy. We integrated constructs in the <i>HIS3</i> locus or Œ¥-sites to promote strain stability. </p> </div> <div class="yeastcellculturegrowth,centrifugation,andopticalmeasurements" title="sec"> <div class="title" tagx="title" title="title">
Yeast cell culture growth, centrifugation, and optical measurements</div> <p>Unless otherwise specified, liquid yeast cultures were grown in 24-well plates, at 30 ¬∞C and shaken at 200 rpm, in either YPD or SC-dropout media with 2% glucose. SC-dropout media includes 94.8 mg/L of valine, 94.8 mg/L of isoleucine, and 189.6 mg/L of leucine along with all other necessary standard nutrients unless otherwise specified. To stimulate cells with light, we used blue LED panels (HQRP New Square 12‚Ä≥ Grow Light Blue LED 14W), placed 40 cm from cell cultures. To control light duty cycles, the LED panels were regulated with a Nearpow Multifunctional Infinite Loop Programmable Plug-in Digital Timer Switch (purchased from Amazon). Cell cultures were centrifuged in a table-top centrifuge, with 24-well plate rotor adaptors. Unless otherwise specified, plates were centrifuged at 1000 rpm for 10 min. </p> <p>Fluorescence and optical density (OD <sub>600</sub>) measurements were taken using a TECAN plate reader (infinite M200PRO). The excitation and emission wavelengths used for GFP fluorescence measurements were 485 nm and 535 nm, respectively, using an optimal gain for all measurements. To process fluorescence data, the background fluorescence from the media was first subtracted from values. Then, the GFP/OD <sub>600</sub> values of cells lacking a GFP construct were subtracted from the fluorescence values (GFP/OD <sub>600</sub>) of each sample to normalize for light bleaching of the media and cell contents. Thus, reported values were calculated according to the following formula. </p><div class="disp-formula" title="disp-formula"> <div class="math_UNKNOWN"> <div class="mtext_UNKNOWN">
GFP</div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
OD</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Strain</div> <div class="mo_UNKNOWN">
,</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
Condition</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
GFP</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Strain</div> <div class="mo_UNKNOWN">
,</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
Condition</div> </div> </div> <div class="mo_UNKNOWN">
‚àí</div> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
GFP</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Media</div> <div class="mo_UNKNOWN">
,</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
Condition</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
OD</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Strain</div> <div class="mo_UNKNOWN">
,</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
Condition</div> </div> </div> <div class="mo_UNKNOWN">
‚àí</div> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
OD</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Media</div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
Condition</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
‚àí</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
GFP</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
No GFP Control Strain</div> <div class="mo_UNKNOWN">
,</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
Condition</div> </div> </div> <div class="mo_UNKNOWN">
‚àí</div> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
GFP</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Media</div> <div class="mo_UNKNOWN">
,</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
Condition</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
OD</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
No GFP Control Strain</div> <div class="mo_UNKNOWN">
,</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
Condition</div> </div> </div> <div class="mo_UNKNOWN">
‚àí</div> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
OD</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Media</div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
Condition</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> <p /> <p>All fluorescence measurements were done at the end of experiments or on samples taken from experimental cultures, such that potential activation of VP16-EL222 by the light used to excite GFP did not affect our experiments or results. </p> <p>To measure cell concentration, optical density measurements were taken at 600 nm, using media (exposed to the same conditions as the yeast) as blank. Measurements were done with the TECAN plate reader (infinite M200PRO) or Eppendorf spectrophotometer (BioSpectrometer basic), from samples diluted to a range of OD <sub>600</sub> of 0.1 to 1.0. </p> <p>All experiments involving light-inducible strains were done under minimal ambient light, unless otherwise specified, in order to avoid unwanted activation of optogenetic systems. </p> </div> <div class="constructionofoptoexpsystem" title="sec"> <div class="title" tagx="title" title="title">
Construction of <i>OptoEXP</i> system </div> <p>We purchased a gBlock (IDT) containing the yeast codon-optimized sequence of VP16-EL222, flanked by NheI and XhoI restriction sites, which we inserted into plasmid pJLA121 <sup>0301</sup> (containing P <sub>PGK1</sub>_T <sub>CYC1</sub>). We then used XmaI and AscI to subclone P <sub>PGK1</sub>_VP16-EL222_T <sub>CYC1</sub> into pYZ12-B to make EZ-L105 ( <a href="#SD1">Supplementary Table 1</a>), which allows single genomic integration of gene cassettes into the <i>HIS3</i> locus. We then changed the promoter to P <sub>TEF1</sub> using XmaI and NheI restriction site cutting and subsequent ligation to make EZ-158, used for <i>OptoEXP</i> expression in glycerol media. In addition, we synthesized the C120 and minimal promoter sequence (TAGAGGGTATATAATGGAAGCTCGACTTCCAG), otherwise known as P <sub>C120</sub>, using Bio Basic‚Äôs gene synthesis service and used it to develop new pJLA vectors with the P <sub>C120</sub> promoter and either an <i>ADH1</i> or <i>ACT1</i> terminator, making pJLA121 <sup>0803</sup> and pJLA121 <sup>0802</sup>, respectively ( <a href="#SD1">Supplementary Table 1</a>). </p> </div> <div class="characterizationofoptoexp" title="sec"> <div class="title" tagx="title" title="title">
Characterization of <i>OptoEXP</i> </div> <p>To test the <i>OptoEXP</i> system, we built plasmid EZ-L83 (pJLA111-GFP <sup>0803</sup>), which places GFP under P <sub>C120</sub> transcriptional control in a CEN/ARS plasmid with a <i>URA3</i> marker ( <a href="#SD1">Supplementary Table 1</a>). We then used EZ-L105 to integrate a single copy of P <sub>PGK1</sub>_VP16-EL222_T <sub>CYC1</sub> construct into the <i>HIS3</i> locus of CENPK2-1C, selecting strain YEZ24 from a SC-His + 2% glucose plate. Subsequently, we transformed YEZ24 with EZ-L83, and selected strain YEZ32 from a SC-ura + 2% glucose plate. We also transformed YEZ24 with empty pRSII416 to make control strain YEZ32C, which has no GFP. </p> <p>To characterize light induction by <i>OptoEXP</i>, we tested four different colonies of YEZ32. We grew cells from each colony in liquid SC-ura overnight in the dark (tin foiled). The next morning, we diluted the cells to 0.1 OD <sub>600</sub> in fresh SC-ura, to a final volume of 1mL each, incubated in individual wells of a 24-well clear Costar plate. We prepared 5 identical plates and tin foiled one of them. The plates were then placed under either dark, constant blue light, or blue light under a duty cycles of 5s ON/75s OFF, 8s ON/72s OFF, and 11s ON/69s OFF, with blue LED panel at 40 cm from each plate. We used duty cycles instead of light intensity to better control light dose and reproducibility. Cell cultures were grown for 8 hours under blue light panels or in the dark, then their GFP fluorescence and optical density (OD <sub>600</sub>) were measured using the TECAN plate reader. Error bars represent one standard deviation from biological replicates (n=4). </p> <p>To benchmark the combination of P <sub>PGK1</sub>_VP16-EL222_T <sub>CYC1</sub>, and a single copy of P <sub>C120</sub>, which we refer to as Opto <i>EXP</i>, we compared it to several constitutive promoters. To achieve this, we made pJLA111-GFP <sup>0X0X</sup> constructs (CEN/ARS with URA3 markers) containing P <sub>CYC1</sub> (EZ-L64), P <sub>ADH1</sub> (EZ-L63), P <sub>PGK1</sub> (EZ-L67), P <sub>TDH3</sub> (EZ-L65), and P <sub>TEF1</sub> (EZ-L66), which we used to transform YEZ24 to make yeast stains YEZ27 (EZ-L64), YEZ28 (EZ-L63), YEZ29 (EZ-L67), YEZ30 (EZ-L65), and YEZ31 (EZ-L66). </p> <p>YEZ32C was used as a control for no GFP production for strains with pJLA111-GFP <sup>0X0X</sup> plasmids. YEZ27, YEZ28, YEZ29, YEZ30, YEZ31, and YEZ32 were tested similarly to the <i>OptoEXP</i> testing above (YEZ32). </p> </div> <div class="flowcytometryexperiments" title="sec"> <div class="title" tagx="title" title="title">
Flow cytometry experiments</div> <p>We used flow cytometry to determine whether the Opto <i>EXP</i> system produces a homogeneous response in the cell population. To construct strains for these experiments, we transformed CEN.PK2-1C with PmeI-linearized pYZ12-B (a non-fluorescent control), EZ-L136, EZ-L516, or EZ-L350 to make YEZ140 (CEN.PK2-1C with histidine prototrophy restored), YEZ139 (CEN.PK2-1C with Opto <i>EXP</i> driving GFP, and P <sub>PGK1</sub> driving VP16-EL222, which works best in glucose), YEZ243 (CEN.PK2-1C with Opto <i>EXP</i> driving GFP, and P <sub>TEF1</sub> driving VP16-EL222, which works well in glucose or glycerol), and YEZ186 (CEN.PK-2C with <i>HIS3</i>::P <sub>TEF1</sub>_GFP_T <sub>ACT1</sub>, for constitutive expression of GFP as control), respectively. </p> <p>To test the homogeneity of gene expression of <i>OptoEXP</i> we grew overnight cultures of YEZ139, YEZ140, YEZ186 in SC-his + 2% glucose media, in the dark. The next morning, we diluted 20 ŒºL of these cultures into 980 ŒºL of fresh media in two 24-well plates. Both plates were placed in dark and shaken at 200 rpm at 30¬∞C for 3 hours. Then, one plate was placed 40 cm under a blue light panel and the other was tin-foiled and kept in the dark. Both plates were shaken at 200 rpm at 30¬∞C for 3 hours. Then, 5 ŒºL of culture was diluted into 995 ŒºL of phosphate-buffered saline media and used for flow cytometry ( <a href="#F2">Extended Data Fig. 2a and b</a>). Samples were run in triplicates from three different cultures separated after the overnight stage. Representative samples from these triplicates were chosen for the figures. </p> <p>To test <i>OptoEXP</i> performance in non-fermentative conditions, we grew 10 mL overnight cultures of YEZ243, YEZ140, YEZ186 in SC-his + 3% glycerol + 2% ethanol media, in tin-foiled tubes (in the dark) in the roller drum. The next morning, we diluted 30 ŒºL of these cultures into 980 ŒºL of fresh media to an OD <sub>600</sub> of 0.05-0.1 in four 24-well plates. Both plates were placed in the dark (tin foiled) and shaken at 200 rpm at 30¬∞C for 6 hours (until the yeasts were in the exponential growth phase, OD <sub>600</sub> ~ 3). Then, one plate was placed 40 cm under a blue light panel and the others were kept tin-foiled (in the dark). After one hour, another plate was moved into the light. After another half hour more, another plate was moved into the light. All plates were kept shaken at 200 rpm at 30¬∞C for 2 hours after the initial 6 hours of growth. Then, 5 ŒºL of each culture was diluted into 995 ŒºL of phosphate-buffered saline media and used for flow cytometry ( <a href="#F2">Extended Data Fig. 2c</a>). Samples were run in triplicates from three different cultures separated after the overnight stage. Representative samples from these triplicates were chosen for the figures. </p> <p>GFP expression was quantified by flow cytometry using a BD LSR II flow cytometer (BD Biosciences, San Jose, CA, USA) with the excitation wavelength of 488 nm and the emission wavelength of 530 nm. The gating used in our analyses was defined to include positive (YEZ186) and negative (YEZ140) control cells based on GFP fluorescence, but exclude particles that are either too small or too large to be single living yeast cells, based on the side scatter (SSC-A) vs forward scatter (FSC-A) plots ( <a href="#F2">Extended Data Fig. 2d</a>). Mean fluorescence values were determined from 20,000 cells. Data were analyzed with the FlowJo Version 10 software (Tree Star, Ashland, OR, USA). </p> <p>All fluorescence measurements were done at the end of experiments or on samples taken from experimental cultures, such that potential activation of VP16-EL222 by the light used to excite GFP did not affect our experiments or results. </p> </div> <div class="constructionoftheoptoinvrtcircuits" title="sec"> <div class="title" tagx="title" title="title">
Construction of the <i>OptoINVRT</i> circuits </div> <p> <i>OptoINVRT</i> circuits were initially developed and characterized in two yeast strain backgrounds: YEZ44 (CENPK.2-1C, <i>gal80-Œî</i>, <i>gal4-Œî</i>) and Y202 (S288C, <i>pdc1</i>Œî, <i>pdc5</i>Œî, <i>pdc6</i>Œî, <i>gal80Œî</i>, containing pJLA121PDC1 <sup>0202</sup>), ( <a href="#SD1">Supplementary Table 2</a>). </p> <p>Gene circuits were assembled using restriction enzyme digests and ligations afforded by pJLA vectors, in which ORFs were inserted using NheI and XhoI sites, and multiple cassettes assembled using XmaI (or AgeI), MreI (or BspEI), and AscI <sup> <a href="#R25">25</a> </sup>. Each gene circuit was constructed by assembling five promoter-gene-terminator sequences into single integration vectors targeting the <i>HIS3</i> locus (EZ-L259, EZ-L260, and EZ-L266; <a href="#SD1">Supplementary Tables 1 and 3</a>). The photosensitive degron (PSD) used in <i>OptoINVRT</i>3 is derived from the fusion of phototropin1 LOV2 domain (V19L mutant) from <i>Arabidopsis thaliana</i> and a synthetic degradation sequence derived from the murine ornithine decarboxylase (ODC) <sup> <a href="#R22">22</a> </sup>. This PSD was fused to the C-terminus of <i>GAL4</i> using Gibson assembly of pJLA121 <sup>0303</sup> cut with XhoI and PSD amplified from pDS143 <sup> <a href="#R22">22</a> </sup> with 30 bp-overhang sequences homologous to the multiple cloning sequence and T <sub>ADH1</sub> of pJLA121 <sup>0303</sup> to make EZ-L247, to which we then inserted <i>GAL4</i> using NheI and XhoI through ligation to make the precursor plasmid of EZ-L266 (EZ-L251). </p> </div> <div class="characterizationofoptoinvrtcircuits" title="sec"> <div class="title" tagx="title" title="title">
Characterization of <i>OptoINVRT</i> circuits </div> <p>The three <i>OptoINVRT</i> circuits, controlling GFP expression, were initially characterized by transforming YEZ44 to produce YEZ100 ( <i>OptoINVRT1</i>), YEZ101 ( <i>OptoINVRT2</i>), and YEZ102 ( <i>OptoINVRT3</i>) and Y202 to produce YEZ115 ( <i>OptoINVRT1</i>), YEZ116 ( <i>OptoINVRT2</i>), and YEZ117 ( <i>OptoINVRT3</i>) ( <a href="#SD1">Supplementary Table 2</a>). <i>OptoINVRT</i> circuits were characterized using strains YEZ100-102 and YEZ115-117 by performing the same experiment done on YEZ32, described above, to characterize the <i>OptoEXP</i> system. In this case, we exposed cells to full light, complete darkness, or light pulses of 8s ON/72s OFF. YEZ186 and YEZ140 were used as controls for strains based in YEZ44, and YEZ171 and YEZ94 were used as controls for strains based in Y202. </p> </div> <div class="characterizationoflightpanels" title="sec"> <div class="title" tagx="title" title="title">
Characterization of light panels</div> <p>The light intensity of panels was measured with a Quantum meter with separate sensor (Model MQ-510 from Apogee Instruments), placed at 40 cm (the same distance used for all 24-well plate experiments). The resulting intensities ranged from 53 to 134 Œºmoles/m <sup>2</sup>/s with an average of 77.5 Œºmoles/m <sup>2</sup>/s (with 465 nm max peak spectra). Ambient light under these conditions ranged from 4-5 Œºmoles/m <sup>2</sup>/s. We then measured the levels of GFP expression in strains YEZ243 and YEZ186 (as well as YEZ140, as negative control) induced by three different panels with 53, 82, or 134 Œºmoles/m <sup>2</sup>/s intensities, placed 40 cm from the 24-well plates, using the same protocol for experiments shown in <a href="#F11">Fig. 1b</a>. Overnight cultures grown in the dark where diluted to OD of 0.1, and incubated for 8 hours in either full light, or light pulses of 10s ON/70s OFF. The levels of GFP expression were indistinguishable across the three panels tested (including the brightest and dimmest panels used in this study), and responded uniformly to changes in light duty cycle ( <a href="#F3">Extended Data Fig. 3</a>). This demonstrates that there is no detectable difference between variations in the intensities of panels used in this study and that variations in duty cycle is an effective way to control gene expression using our optogenetic transcriptional controls. </p> </div> <div class="developmentofanoptoexp-pdcstrain" title="sec"> <div class="title" tagx="title" title="title">
Development of an <i>OptoEXP</i>-PDC strain </div> <p>Strain Y200 contains a triple gene-deletion of <i>pdc1</i>Œî, <i>pdc5</i>Œî, and <i>pdc6</i>Œî, as well as a 2Œº- <i>URA3</i> plasmid pJLA121PDC1 <sup>0202</sup> with P <sub>TEF1</sub>- <i>PDC1</i>-T <sub>ACT1</sub>, which allows it to grow robustly in glucose. Y200 was transformed with a PmeI-linearized EZ-L165 plasmid to insert a cassette composed of P <sub>TEF1</sub>_VP16-EL222 _T <sub>CYC1</sub> and P <sub>C120</sub>_ <i>PDC1</i>_T <sub>ADH1</sub> into its <i>HIS3</i> locus, resulting in strain YEZ50 ( <a href="#SD1">Supplementary Tables 1 and 2</a>). As a control, we also transformed Y200 with PmeI-linearized EZ-L158, a vector containing P <sub>TEF1</sub>_VP16-EL222_T <sub>CYC1</sub> but lacking P <sub>C120</sub>_ <i>PDC1</i>_T <sub>ADH1</sub>, and then counter-selected against the pJLA121- <i>PDC1</i> <sup>0202</sup> plasmid by growing on 5-FOA (described below) to produce the control strain YEZ50C. Strain YEZ50 was then transformed with PmeI-linearized EZ-L143, which inserts multiple copies of P <sub>C120</sub>_ <i>PDC1</i>_T <sub>ADH1</sub> into Œ¥-integration sites of the yeast genome. Colonies able to grow on YPD plates containing 800 Œºg/mL of Zeocin were replica plated on plates containing SC‚Äìhis + 3% Glycerol + 2% Ethanol twice. The resulting plates were then replica plated on SC‚Äìhis + 3% Glycerol + 2% Ethanol + 1mg/mL 5FOA, and then finally back onto plates containing SC-his + 3% Glycerol + 2% Ethanol. This treatment efficiently counter-selects against pJLA121- <i>PDC1</i> <sup>0202</sup>, due to its <i>URA3</i> marker <sup>42</sup>. From this plate, we isolated YEZ61-23, a strain that can grow on SC-his + 2% glucose plates only when exposed to blue light, which is consistent with having <i>PDC1</i> expression controlled by P <sub>C120</sub>, and VP16-EL222. As a control, we also transformed Y200 with PmeI-linearized EZ-L143, which contains P <sub>C120</sub>_ <i>PDC1</i>_T <sub>ADH1</sub> but lacks P <sub>TEF1</sub>_VP16-EL222_T <sub>CYC1</sub>, and counter-selected against in pJLA121- <i>PDC1</i> <sup>0202</sup> in 5FOA to produce YEZ61C. This control strain has multiple copies of P <sub>C120</sub>_ <i>PDC</i>1_T <sub>ADH1</sub> in Œ¥-sites, but lacks the VP16-EL222 needed to transcribe them. </p> </div> <div class="light-dependentgrowthofanoptoexp-pdcstrain" title="sec"> <div class="title" tagx="title" title="title">
Light-dependent growth of an <i>OptoEXP</i>-PDC strain </div> <div class="solidmedia" title="sec"> <div class="title" tagx="title" title="title">
Solid media</div> <p>Cells from strains BY4741, YEZ61-23, YEZ50C, and YEZ61C were patched onto an agar plate containing yeast extract, peptone and 3% glycerol, and 2% ethanol (YPGE plate), and grown over night in ambient light. This plate was then replica plated onto a fresh YPGE plate and two YPD plates. One of the YPD plates was covered in tin foil and the other exposed to constant blue light, while the YPGE was left at ambient lighting. Replica plating was done such that the YPD plate ultimately covered in tin foil was replicated first, then the YPD plate ultimately exposed to constant blue light, and the YPGE plate last. All plates were incubated at 30¬∞C for 48h ( <a href="#F12">Fig. 2b</a>). </p> </div> <div class="liquidmedia" title="sec"> <div class="title" tagx="title" title="title">
Liquid media</div> <p>Single colonies of YEZ61-23 and BY4741 were used to inoculate liquid SC-his + 2% glucose medium. The cells were grown for 24 h on a roller drum at 200 rpm, 30¬∞C, and 40 cm away from a blue light from a HQRP New Square 12‚Ä≥ LED Grow Light System 225 Blue LED 14W. Subsequently, the OD <sub>600</sub> was measured with a spectrophotometer, and the cells were diluted to 0.1 OD <sub>600</sub> in fresh SC-his 2% glucose media in three 1 mL-replicates in four 24-well plates (Celltreat non-treated sterile flat bottom plates). Each plate was centered 40 cm under an HQRP blue light panel. One plate was exposed to full blue light. The other three plates were exposed to 10s ON/70s OFF of blue light, 20s ON/60s OFF of blue light, and 40s ON/40s OFF of blue light. A fifth plate was wrapped in aluminum foil to incubate cells in the absence of light. OD readings were taken using a TECAN plate reader at 0 h, 10 h, 12.5 h, 13.5 h, 15 h, 17 h, 19 h, 20.5 h, and 31.75 h. Readings were taken under minimal light conditions to prevent unwanted activation of EL222. Data is available in a supplementary spreadsheet. </p> <p>The exponential growth phase of YEZ61-23 (identified as the most linear portion of the plot of Log <sub>e</sub>(OD) vs time), was used to find the specific growth rates at different light doses. This was done by fitting the data to Log <sub>e</sub>(OD)=Log <sub>e</sub>(OD <sub>0</sub>) + Œº*t, using least squares linear regression, where OD <sub>0</sub> is a constant corresponding to the initial OD, and Œº corresponds to the specific growth rate constant. The Œº constants were calculated for each independent experiment and then averaged, with error bars representing standard deviations (n = 3). Data is available in a supplementary spreadsheet. </p> </div> </div> <div class="developmentofchemicalproductionstrains" title="sec"> <div class="title" tagx="title" title="title">
Development of chemical production strains</div> <p>To develop light-dependent lactic acid producing strains, we transformed Y202 with PmeI-linearized EZ-L259, EZ-L260, and EZ-L266 ( <i>OptoINVRT</i>1, <i>OptoINVRT</i>2, and <i>OptoINVRT</i>3 respectively) yielding YEZ115, YEZ116, and YEZ117. We then integrated into Œ¥-sites multiple copies of PmeI-linearized EZ-L235, ( <a href="#SD1">Supplementary Table 1</a>) containing P <sub>C120</sub> driving <i>PDC1</i> and P <sub>GAL1</sub> driving the Lactic Acid Dehydrogenase (LDH) from <i>Pelodiscus sinensis</i> (kindly provided by Dr. Lee from the Samsung research center). Then we counter-selected against pJLA121PDC1 <sup>0202</sup> to produce YEZ144 ( <i>OptoINVRT1</i>), YEZ145 ( <i>OptoINVRT2</i>), and YEZ146 ( <i>OptoINVRT3</i>) ( <a href="#SD1">Supplementary Table 2</a>). These strains induce <i>PDC1</i> and repress LDH expression in the light; while in the dark they stop inducing <i>PDC1</i> and induce LDH instead. </p> <p>To develop light-dependent isobutanol producing strains, we transformed YEZ115, YEZ116, and YEZ117 with PmeI-linearized EZ-L316, which integrates multiple copies of P <sub>C120</sub>-driven <i>PDC1</i> and P <sub>GAL1</sub>-driven <i>ILV2</i> in genomic Œ¥-integration sites. We then counter-selected the transformants against pJLA121PDC1 <sup>0202</sup> with 5-FOA to produce strains YEZ131 (Opto <i>INVRT1</i>), YEZ149 (Opto <i>INVRT2</i>), and YEZ133 (Opto <i>INVRT3</i>), respectively. Subsequently, we transformed these strains with plasmid EZ-L310 ( <a href="#SD1">Supplementary Table 1</a>), which contains five genes of the mitochondrial isobutanol biosynthetic pathway: <i>ILV2</i>, <i>ILV5</i>, <i>ILV3</i>, CoxIV- <i>ARO10</i> and CoxIV-LlAdhA <sup>RE1,33</sup>. The last two genes in EZ-L310 are fused on their N-termini to the mitochondrial localization signal of <i>COXIV</i>, ensuring that all five genes are targeted to the mitochondria <sup> <a href="#R25">25</a> </sup>. In addition, P <sub>GAL1</sub> drives <i>ILV2</i> expression, which places this gene under the control of <i>OptoINVRT</i> circuits. The resulting strains are YEZ159, YEZ156, HPY6 for <i>OptoINVRT1</i>, <i>OptoINVRT2</i>, and <i>OptoINVRT3</i>, respectively ( <a href="#SD1">Supplementary Table 2</a>). </p> </div> <div class="screensforlacticacid-andisobutanol-producingstrains" title="sec"> <div class="title" tagx="title" title="title">
Screens for lactic acid- and isobutanol-producing strains</div> <p>Colonies from each transformation plate (grown in glucose and under blue light) were screened for lactic acid (7 colonies from each of YEZ144, YEZ145, and YEZ146) or isobutanol (12 colonies from each of YEZ159, YEZ156, and HPY6) production ( <a href="#SD1">Supplementary Table 2</a>). Each colony was used to inoculate 1mL of SC-his + 2% media (for lactic acid producing strains) or SC-ura + 2% glucose media (for isobutanol producing strains) in 24 well plates and grown overnight at 30 ¬∞C, 200 RPM, and blue light. The next morning, each culture was diluted to 0.1 OD <sub>600</sub> for lactic acid producing strains, or 0.15 OD <sub>600</sub> for isobutanol producing strains, in fresh media, and grown for 12 hours (for lactic acid production) or 18 hours (for isobutanol production), at 30 ¬∞C, 200 RPM, and blue light. After these incubation periods, the cultures reached OD <sub>600</sub> values of 5 (for lactic acid-producing strains, œÅ = 5) and 5 and 8 (for isobutanol-producing strains, œÅ = 8, œÅ = 5); at which point they were moved into the dark for 6 hours for lactic acid producing strains (Œ∏ = 6 hours) and 3 hours for isobutanol producing strains (Œ∏ = 3 hours). The cultures were then centrifuged and re-suspended in fresh media, in plates that were then sealed with Nunc Sealing Tape (Thermo Scientific) to begin the fermentations. The plates were incubated in the dark at 30 ¬∞C and shaken at 200 RPM for 48 hours during fermentation. Subsequently, the cultures were centrifuged, and the supernatants collected for HPLC analysis. For lactic acid-producing strains, most of the 7 colonies screened for each <i>OptoINVRT</i> circuit produced similar amounts of lactic acid, with colonies of strain YEZ145 (containing <i>OptoINVRT2</i>) producing the highest titers ( <a href="#F4">Extended Data Fig. 4c</a>). On the other hand, out of the 12 colonies screened for each isobutanol-producing strain, only 2 or 3 were high producing strains for each <i>OptoINVRT</i> circuit, with YEZ159 (containing <i>OptoINVRT1</i>) producing the highest titers ( <a href="#F4">Extended Data Fig. 4d, e</a>). </p> </div> <div class="constructionofahighisobutanolproducingstrain" title="sec"> <div class="title" tagx="title" title="title">
Construction of a high isobutanol producing strain</div> <p>We deleted <i>BAT1</i> from YEZ131 via homologous recombination, using the HygB Hygromycin resistance marker, resulting in strain YEZ158. Subsequently, we transformed YEZ158 with EZ-L310, resulting in transformants YEZ167, from which we screened 7 colonies, as described above, and identified YEZ167-4 as the strain with highest isobutanol production. </p> </div> <div class="optimizingexperimentalparametersforlight-dependentfermentation" title="sec"> <div class="title" tagx="title" title="title">
Optimizing experimental parameters for light-dependent fermentation</div> <p>The highest producing colonies of YEZ144, YEZ145, and YEZ146 (for lactic acid) and of YEZ167 (specifically YEZ167-4 for isobutanol), were used to optimize the pre-growth parameters of fermentation for lactic acid or isobutanol production. For each strain, an overnight culture was grown in blue light, 30¬∞C and shaking at 200 RPM, in 2% glucose-containing SC media (SC-his for lactic acid producing strains and SC-ura for isobutanol producing strains). To find the optimal cell density at which to switch cultures from light to dark, we diluted the overnight cultures into 1mL of the SC dropout medium to different OD <sub>600</sub> values, ranging from 0.04 to 0.32. The lactic acid-producing strains were then grown for 16 hours under 15s ON/65s OFF blue light. The isobutanol-producing strains were grown for 18 hours under 15s ON/65s OFF blue light. We then measured the OD <sub>600</sub> of each culture (these values correspond to variations in œÅ), and incubated them in the dark for 6 hours for lactic acid-producing strains (Œ∏ = 6 hours) and 3 hours for isobutanol-producing strains (Œ∏ = 3 hours). After this dark incubation period, the cultures were centrifuged at 1000 rpm for 5 minutes and suspended in fresh SC dropout media containing glucose at 26.5 g/L (for lactic-acid producing strains) or 21.5 g/L (for isobutanol-producing strains). The plates were sealed with Nunc Sealing Tape, and incubated in the dark for fermentation at 30¬∞C and 200 RPM. Control cultures were grown under 15s ON/65s OFF blue light during the growth phase, the dark incubation period, and the fermentation. Cultures producing lactic acid were harvested after 48 hours, while samples of cultures producing isobutanol were taken after 24, 48, and 72 hours. Cultures were centrifuged at 1000 rpm for 10 minutes, and supernatants analyzed with HPLC. </p> <p>To optimize the dark incubation period immediately before fermentation, Œ∏, the best isobutanol-producing strain, YEZ167-4, was grown overnight under blue light in SC-ura, 2% glucose. The overnight culture was then diluted into seven different plates in quadruplicate samples in fresh media to a starting OD <sub>600</sub> of 0.1. The cultures were then grown to an OD <sub>600</sub> of 8.5 (which was found to be the optimal OD <sub>600</sub> in our previous experiment). At that point, the plates were tin foiled to ensure complete darkness; after every hour, one of the plates was centrifuged, and the cells suspended in fresh SC-ura medium with 20.8 g/L glucose media and subjected to a 48-hour fermentation in the dark. </p> </div> <div class="optimizingthefrequencyoflightboutsduringtheproductionphaseofbatchfermentation" title="sec"> <div class="title" tagx="title" title="title">
Optimizing the frequency of light bouts during the production phase of batch fermentation</div> <p>A single colony of the best isobutanol producing strain, YEZ167-4, was used to inoculate 5 mL of SC-ura + 4% glucose media and grown overnight under light. The next morning, the culture was diluted in 1mL of fresh media to an OD <sub>600</sub> of 0.2 (in quadruplicates) and grown under light for 20 hours to an OD <sub>600</sub> of 9.5. Subsequently, the cultures were incubated for 3 hours in the dark. To start the fermentations, the cultures were centrifuged again, and suspended in fresh SC-ura + 15% glucose (precisely 157.0 g/L glucose, as measured with HPLC) media, and kept in the dark. During the fermentation, the cultures were first exposed to 4 hours of light and then pulsed every 5, 10, or 20 hours for 30 minutes, at a duty cycle of 15s ON/65s OFF. As controls, some plates were always kept in the dark or in full light. Fermentations lasted for 80 hours, after which, the cultures were centrifuged, and the supernatants analyzed with HPLC. </p> </div> <div class="nad+/nadhratiomeasurements" title="sec"> <div class="title" tagx="title" title="title"> <i>NAD <sup>+</sup>/NADH </i> ratio measurements </div> <p>YEZ167-4 was inoculated into 3mL of SC-ura + 2% glucose media and grown overnight under light at 30¬∞C and 200 RPM. The next morning, the culture was diluted into 1mL of fresh media in 6 different 24-well plates to an OD <sub>600</sub> of 0.2 (in quadruplicates) and grown at 30¬∞C and shaken at 200 rpm under light for 20 hours to an OD <sub>600</sub> of 9.5. Subsequently, the cultures were incubated under the same conditions for 3 hours in the dark. The cultures were then centrifuged at 1,000 rpm for 5 minutes and resuspended in fresh SC-ura + 15% glucose media (precisely 157.0 g/L glucose, as measured with HPLC), and kept in the dark. As controls, one plate was always kept in the dark and one plate was kept in full light. The remaining four plates were pulsed every 10 hours for 30 minutes, at a duty cycle of 15s/65s. One plate was exposed to an extra 30-minute pulse of light at 15s/65s precisely 4 hours before harvest and placed back into the dark for the remainder of the fermentation. One plate was exposed to an extra 30-minute pulse of light at 15s/65s precisely 2 hours before harvest and placed back into the dark for the remainder of the fermentation. One plate was exposed to an extra 30-minute pulse of light at 15s/65s precisely 30 minutes before harvesting to test how the light pulses affect the NAD <sup>+</sup>/NADH ratio. After 48 hours, cells were harvested by centrifuging at 9,000 g for 5 minutes and removing all supernatant. The NAD <sup>+</sup>/NADH ratios of the pelleted cells was measured as previously described <sup>43</sup>. </p> </div> <div class="light-dependentgrowthina2lbioreactor" title="sec"> <div class="title" tagx="title" title="title">
Light-dependent growth in a 2L bioreactor</div> <p>To test if light penetration becomes limiting to a strain with light-dependent growth, we used a 2-L photobioreactor to compare the growth of YEZ167-4 (which is an isobutanol producing strain with light-dependent growth) to that of YZy335 (which is a strain that constitutively makes isobutanol and is PDC <sup>+</sup>, and thus its growth and isobutanol production is independent of light). </p> <p>We used a single colony of YEZ167-4 or YZy335 to inoculate 5 mL of SC-ura + 2% glucose media under light, overnight. The next morning, we diluted the culture in 0.5 L of SC-ura + 15% glucose media to an OD <sub>600</sub> of 0.1 in a UTEX 2-Liter glass Photobioreactor surrounded by three blue light panels, placed at 1 cm from its glass wall ( <a href="#F9">Extended Data Fig. 9a and b</a>), instead of the UTEX LED lighting platform. The culture was grown for 84 hours at 30¬∞C and constant blue illumination, with cells gently mixed with a magnetic stir bar and air sparging. Samples were taken every 12 hours to measure the OD <sub>600</sub> of the cell cultures ( <a href="#F13">Fig. 3d</a>). We report the average measurements of three independent fermentations for each strain, and the error bars correspond to the standard deviations of those three measurements. </p> </div> <div class="fed-batchfermentation" title="sec"> <div class="title" tagx="title" title="title">
Fed-batch fermentation</div> <p>To test the capability of these systems to utilize higher amounts of glucose in a lab-scale fermentor, we conducted fed-bath experiments using the Sixfors INFORS AG CH-4103 Boltmingen/Switzerland based on a previously described protocol <sup>44</sup>. We did not sparge air to keep the fermentation in microaerobic conditions. The pH was set to 5.5 and maintained with a 0.5 M KOH feed. We autoclaved the 500 mL fermentor with 250 mL of ddH <sub>2</sub>O and exchanged the ddH <sub>2</sub>O (using the air pump) with filtered 250mL of SC-ura + 10% glucose media. We used a single colony of YEZ167-4 to inoculate 5 mL of SC-ura + 2% glucose media under light, overnight. The next day, we added the inoculum to the fermentor to an OD <sub>600</sub> of 0.2. The fermentation was kept at 30¬∞C and mixed at 200 rpms. Two light panels were placed 0.5 cm away from the vessel walls ( <a href="#F9">Extended Data Fig. 9c and d</a>), and kept on for the first 46 hours (growth phase of the fermentation), in which the culture grew to an OD <sub>600</sub> of 8.2 ¬± 0.1. The culture was then covered with black cloth and the light was turned off for 4 hours. We then started a glucose feed of SC-ura + 50% glucose at a rate of 2.9 mL//hour for 90 hours (run 1) or 120 hours (runs 2 and 3) and turned on the light (continuously) for the first 4 hours of the glucose feed. After these initial 4 hours, the light was switched to a duty cycle of 45 minutes ON/ 7 hours 15 minutes OFF for the remainder of the fermentation (production phase). Samples of 1mL were taken at hours 0, 24, 48, 72, 96, 120, 144, 168, 192, 216, 240, 264, (after inoculation), for both OD <sub>600</sub> measurements, and HPLC analysis. Samples collected at 48, 96, 144, 192, and 264 hours were also analyzed with qPCR. </p> </div> <div class="analysisofchemicalconcentrations" title="sec"> <div class="title" tagx="title" title="title">
Analysis of chemical concentrations</div> <p>The concentrations of glucose, lactic acid, ethanol, isobutanol, and 2-methyl-1-butanol were quantified with high-performance liquid chromatography (HPLC), using an Agilent 1260 Infinity instrument (Agilent Technologies, Santa Clara, CA, USA). Samples were centrifuged to remove cells and other solid debris, and analyzed using an Aminex HPX-87H ion-exchange column (Bio-Rad, Richmond, CA, USA). The column was eluted with a mobile phase of 5 mM sulfuric acid at 55 ¬∞C and a flow rate of 0.6 ml/min. Glucose, lactic acid, ethanol, isobutanol, and 2-methyl-1-butanol were monitored with a refractive index detector (RID). To determine their concentration, the peak areas were measured and compared to those of standard solutions for quantification. </p> </div> <div class="quantificationofgeneexpressionandŒ¥-integrationcopynumber" title="sec"> <div class="title" tagx="title" title="title">
Quantification of gene expression and Œ¥-integration copy number</div> <p>To measure the number of copies of P <sub>C120</sub>- <i>PDC1</i> integrated into the genome of YEZ50lost, YEZ61-23, YEZ144, YEZ145, YEZ146 and YEZ167-4, we ran quantitative PCR (qPCR) experiments on their genomic DNA, extracted with phenol-chloroform <sup>45</sup>. As a control strain containing a single copy of <i>PDC1</i>, we developed YEZ50lost by counter-selecting against plasmid pJLA121 <i>PDC1</i> <sup>0202</sup> from strain YEZ50, using 5-FOA as described above. </p> <p>To quantify the expression of <i>PDC1</i> and <i>ILV2</i> during the fed-batch fermentation experiments, we measured transcript levels using qPCR. We collected 5 mL samples over the course of the fermentation and centrifuged them at 3,000 rpm for 1.5 minutes. The mRNA was then extracted with phenol-chloroform <sup>45</sup>. Concentrations of samples were measured using the absorbance at 260 nm/280 nm. The QuantiTect Reverse Transcription Kit (Qiagen) was used to remove genomic DNA and synthesize cDNA. </p> <p>Quantitative PCR was performed using the Bio-Rad Mini Opticon and the iTaq Universal SYBR Green Supermix (Bio-Rad). Reactions contained the appropriate amounts of reagents as specified by kit protocols. Threshold cycle values were set by StepOne software (ThermoFisher Scientific) and melting curve analysis following amplification was used to verify the quantity of single PCR products. Primers specific for <i>PDC1</i>, <i>ILV2</i>, and <i>ACT1</i> are listed in <a href="#SD1">Supplementary Table 7</a>. The expression level of each gene was normalized to that of <i>ACT1</i> from the same samples. </p> </div> <div class="testinggeneexpressionofoptoexpin5lfermentorandhighercelldensities" title="sec"> <div class="title" tagx="title" title="title">
Testing gene expression of <i>OptoEXP</i> in 5L fermentor and higher cell densities </div> <p>To test <i>OptoEXP</i> in higher cell density conditions, we inoculated YEZ243 into SC-His + 3% glycerol + 2% ethanol media and grew in the dark for 24 hours. We then set up a BioFlo120 system with a 5L bioreactor (Catalog No. B120110002) from Eppendorf and added 2.4 liters of SC-His+ 3% glycerol+2% ethanol media after autoclaving. The reactor was set to 30¬∞C, pH of 6.5, and a minimum dissolved oxygen (DO) percentage of 40. Blue LED strips (SMD3528 from Shenzhen Shinesky Optoelectronics Co., LTD) were wrapped around the reactor, covering 73% of the available bulk surface area of the fermentation ( <a href="#F10">Extended Data Fig. 10a and b</a>). These LED strips emitted light at an intensity of 129 Œºmoles/m <sup>2</sup>/s (with 465nm max peak spectra) measured in the same condition as the light panels. The reactor was then inoculated to an OD <sub>600</sub> of 1 and the cells grown in the dark until an OD <sub>600</sub> of 15 (maintained by covering the reactor with black fabric). At OD <sub>600</sub> of 15, the lights were turned on and samples were taken at OD <sub>600</sub> of 16 (1 hour after induction), 19(6 hours after induction), 41(24 hours after induction), 46(32 hours after induction), and 50 (40 hours after induction). Cells were fixed by diluting them to an OD <sub>600</sub> of 1 in SC-his+ 3% glycerol+2% ethanol media, adding paraformaldehyde to 3.7%, and incubating for 1 hour at 25¬∞C. To prepare samples for flow cytometry, cells were washed twice with DPBS, and re-suspended again in DPBS to an OD <sub>600</sub> of 0.5. </p> </div> <div class="statistics" title="sec"> <div class="title" tagx="title" title="title">
Statistics</div> <p>Statistical significance was determined using a standard t test for p values. T scores were calculated by the formula: <span class="inline-formula" title="inline-formula"> <div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mtext_UNKNOWN">
Mea</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Condition</div> <div class="mo_UNKNOWN"> </div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
‚àí</div> <div class="mo_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
Mea</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Condition</div> <div class="mo_UNKNOWN"> </div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="msqrt_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Number</div> <div class="mo_UNKNOWN"> </div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
samples</div> </div> </div> <div class="mo_UNKNOWN"> </div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Standard</div> <div class="mo_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
Deviatio</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Condition</div> <div class="mo_UNKNOWN"> </div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> </div> </span>. P values were calculated using a degree of freedom of 2 and a one-sided t-test calculator. </p> </div> </div> <div class="extendeddata" title="sec"> <div class="title" tagx="title" title="title">
Extended Data</div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Maps of key vectors used in this study</div> <p> <b>(a)</b> pYZ12-B vector used to integrate genes or circuits into the <i>HIS3</i> locus. Constructs are usually transferred from pJLA vectors using XmaI and AscI sites. Final constructs are linearized with PmeI prior to yeast transformation. <b>(b)</b> pYZ23 vector used to integrate genes or circuits into Œ¥-5 sites of yeast. Constructs are usually transferred from pJLA vectors using MreI and AscI sites. Final constructs are linearized with PmeI prior to yeast transformation. <b>(c)</b> General vector map shows the relative orientation of the five positions listed in <a href="#SD1">Supplementary Table 1</a>, in which different genes (including promoters and terminators) were assembled, using a previously described multiple gene insertion strategy <sup> <a href="#R25">25</a> </sup>. All vectors have an ampicillin resistance marker ( <i>AMPR</i>) for cloning in <i>E. coli</i> and a selection marker for <i>S. cerevisiae</i> (Marker). Vector types include CEN/ARS, 2Œº, or integrative <sup>36‚Äì39</sup>. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Flow cytometry of strain with GFP controlled by <b> <i>OptoEXP</i> </b>. </div> <p>Representative flow cytometry graphs from 3 biological replicates under the same conditions. Experiments were done in 24-well plates, in either glucose or glycerol and ethanol. Every graph is generated from 20,000 cells. ( <b>a</b>) Control strain (YEZ186) with GFP under P <sub>TEF1</sub> exposed for 3 hours to constant blue light (magenta) or kept for the same amount of time in the dark (cyan); these samples are almost completely superimposed on the right-hand side of graph. Control strain (YEZ140) with no GFP exposed for 3 hours of constant blue light (red) or kept for the same amount of time in the dark (grey); these samples are almost completely superimposed on the left-hand side of graph. Under these conditions, there is no detectable photo bleaching. ( <b>b</b>) Light-induced GFP expression in YEZ139, a strain with GFP controlled by <i>OptoEXP</i>. GFP expression in YEZ139 in SC-His + 2% glucose after 3 hours of exposure to blue light (green) is homogeneous across the cell population, and 37-fold higher than in YEZ139 cells kept in the dark for the same amount of time (orange). The maximum level of GFP expression obtained by <i>OptoEXP</i> in YEZ139 grown in full light for 3 hours (green) is 22.1% of what is achieved in YEZ186, which contains P <sub>TEF1</sub>-GFP, grown under the same conditions (magenta). Fluorescence from a control wild-type strain without GFP, YEZ140, grown for 3 hours in the light, is shown for comparison (red). <b>(c)</b> Light-induced GFP expression by <i>OptoEXP</i> in YEZ243 in SC-His + 3% glycerol + 2% ethanol media. Starting from cultures growing in the dark, samples were taken (in the exponential growth phase, OD <sub>600</sub> ~ 3) and incubated in 24-well plates in the following manner: 2 hours in the dark (orange); 1.5 hours in the dark followed by 30 minutes in light (green); 1 hour in the dark, followed by 1 hour in light (brown); or 2 hours in light (dark blue). YEZ140 (red) and YEZ186 (magenta) were used as controls for no GFP expression and GFP expression under a strong, constitutive promoter (P <sub>TEF1</sub>), respectively. <b>(d)</b> Example of the gating used to make the flow cytometry plots in (a), (b), and (c). All experiments were repeated at least three times. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 3</span> </a> </div>   <div class="title" tagx="title" title="title"> <i>OptoEXP</i> performance under LED panels of different intensities </div> <p>Light-induced expression of GFP in YEZ243 ( <i>OptoEXP</i> driving GFP) compared to constitutive GFP expression in YEZ186 (P <i> <sub>TEF1</sub> </i>-GFP), using light panels of different intensities. Pulsed light was applied at 10s ON/70s OFF. Data shown as mean values; dots represent individual data points; error bars represent one standard deviation from biological replicates (n = 4 biologically independent 1mL culture samples). All experiments were repeated at least three times. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Comparison of <i>OptoINVRT</i> light-repressible transcription circuits </div> <p>( <b>a</b>) <i>OptoINVRT</i> circuit design, based on the expression of Gal80p from <i>OptoEXP</i> and of Gal4p from constitutive promoters of different strength, with or without a photosensitive degron domain (PSD). ( <b>b</b>) Genes controlled by <i>OptoINVRT</i> circuits are repressed in the light and activated in the dark by the repression activity of Gal80p on Gal4p transcription factor. The PSD fused to Gal4p in <i>OptoINVRT3</i> stimulates protein degradation in the light. ( <b>c</b>) Screens for lactic acid production in 2% glucose of several colonies of strains YEZ144 ( <i>OptoINVRT1</i>), YEZ145 ( <i>OptoINVRT2</i>), and YEZ146 ( <i>OptoINVRT3</i>), using growth parameters: œÅ = 5 and Œ∏ = 6 hours. (n = 7 biologically independent colonies) ( <b>d</b>) Screens for isobutanol production in 2% glucose of several colonies of YEZ159 ( <i>OptoINVRT1</i>), YEZ156 ( <i>OptoINVRT2</i>), and HPY6 ( <i>OptoINVRT3</i>), using growth parameters: œÅ = 8 and Œ∏ = 3 hours. (n = 12 biologically independent colonies) ( <b>e</b>) Screens for isobutanol production in 2% glucose of several colonies of YEZ159 ( <i>OptoINVRT1</i>), YEZ156 ( <i>OptoINVRT2</i>), and HPY6 ( <i>OptoINVRT3</i>), using growth parameters: œÅ = 5 and Œ∏ = 3 hours. (n = 12 biologically independent colonies). These screens ( <b>c-e</b>) were performed once in our laboratory. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 5</span> </a> </div>   <div class="title" tagx="title" title="title">
Light-controlled lactic acid production</div> <p> <b>(a)</b> Lactic acid is produced by reduction of pyruvate by lactic acid dehydrogenase (LDH). <i>PDC1</i> is controlled by <i>OptoEXP</i> and LDH by <i>OptoINVRT</i> circuits. <b>(b)</b> With optogenetic controls, light can be used to separate fermentation into two phases: a growth phase, when exposed to light, in which <i>PDC1</i> is expressed and LDH is repressed, and a lactic acid production phase, when cells are in the dark, in which <i>PDC1</i> is not induced, and LDH is expressed. <b>(c)</b> Experimental design for screening of strains and optimization of conditions. Cell density at which cells are moved from light to dark (œÅ) and the time cells are incubated in the dark before starting the fermentation (Œ∏) were varied in these experiments. <b>(d)</b> Three <i>OptoINVRT</i> circuits were tested for lactic acid production: <i>OptoINVRT1</i> (YEZ144); <i>OptoINVRT2</i> (YEZ145), and <i>OptoINVRT3</i> (YEZ146). Top graphs: Dependence of lactic acid titers on œÅ. Bottom graphs: Dependence of the ratio of lactic acid to ethanol on œÅ. Fermentations were done in 26.5 g/L glucose and run for two days. All samples had Œ∏ = 6 hours. Data shown as mean values; dots represent individual data points; error bars represent standard deviations of 1mL culture replicates exposed to the same light conditions (n = 3 biologically independent 1mL culture samples). *, p &amp;lt; 0.05; **, p &amp;lt; 0.01; ***, p &amp;lt; 0.001. Statistics are derived using a one-sided t test. All experiments were repeated at least three times. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 6</span> </a> </div>   <div class="title" tagx="title" title="title">
Quantitative PCR experiments</div> <p> <b>(a)</b> Number of copies of P <sub>C120</sub> driving <i>PDC1</i> in key strains, determined with quantitative PCR done on genomic DNA samples (see Methods). All strains have one copy of <i>PDC1</i> integrated in the <i>HIS3</i> locus, and the rest are integrated in random Œ¥-integration sites (except YEZ50lost, which only has one copy in the <i>HIS3</i> locus). Data shown as mean values; dots represent individual data points; error bars represent one standard deviation from biological replicates (n = 3 biologically independent 1mL culture samples). All experiments were repeated at least three times. <b>(b)</b> qPCR of <i>PDC1</i> and <i>ILV2</i> mRNA levels during fed-batch fermentation with periodic light stimulation for isobutanol production in 0.5-liter fermentors. Quantitative PCR was performed on samples from fed-batch fermentations for isobutanol production ( <a href="#F13">Fig. 3e</a>) to measure concentrations of <i>PDC1</i> and <i>ILV</i>2 transcripts. Gene expression was normalized with <i>ACT1</i> transcripts. Lines represent average values from samples taken from two separate fermentations ran in the same conditions (n = 2 biologically independent samples). All experiments were repeated at least two times. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 7</span> </a> </div>   <div class="title" tagx="title" title="title">
Optimization of light-controlled isobutanol production</div> <p> <b>(a)</b> Dependence of isobutanol titers on œÅ. Cells were grown with Œ∏ = 3 hours; fermentations were done in 21.5 g/L glucose; isobutanol titers were measured after 2 days of fermentation in the dark. YZy335 is a control strain with a constitutive isobutanol pathway plasmid and wild-type <i>PDC1, PDC5, PDC6</i> and was used in 2-day high cell density fermentations as control. <b>(b)</b> Dependence of isobutanol titers on Œ∏. Cells were grown to œÅ = 8.5. Fermentations were again done in 20.8 g/L glucose for 2 days in the dark. All data shown as mean values; dots represent individual data points; error bars represent standard deviations of 1mL culture replicates exposed to the same light conditions (n = 3 biologically independent 1mL culture samples). All experiments were repeated at least three times. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 8</span> </a> </div>   <div class="title" tagx="title" title="title">
Optimization of high glucose fermentations</div> <p> <b>(a)</b> Glucose remainders as percentage of initial glucose concentration after 48-hours (20 g/L initial glucose) or 80-hours (150 g/L initial glucose) of fermentation of YEZ167-4 in the dark. Cell growth parameters: œÅ = 8.5 and Œ∏ = 4 hours (for fermentations in 20 g/L glucose) and œÅ = 9.5 and Œ∏ = 3 hours (for fermentations in 150 g/L glucose). Data shown as mean values; dots represent individual data points; error bars represent one standard deviation from biological replicates (n = 3 biologically independent 1mL culture samples). All experiments were repeated at least three times. <b>(b)</b> NAD <sup>+</sup>/NADH ratio recovery through light pulsing. NAD <sup>+</sup>/NADH ratios were measured in samples under similar conditions as batch fermentation shown in <a href="#F13">Fig. 3c</a> (See Methods). YEZ167-4 cultures were diluted into six 24-well plates and grown to an OD <sub>600</sub> of 9.5 and left in the dark for 3 hours before resuspending cells in 15% glucose media. Four of the plates were pulsed every 10 hours for 30 minutes at a duty cycle of 15s ON/65s OFF. Cells were harvested after 48h fermentations, and at different times since the last light pulse (0, 1.5, 3.5, or 7.5 hours). Control plates were kept under full light or in the dark throughout the 48-hour fermentations. NAD <sup>+</sup>/NADH ratios in pelleted cells were measured following a previously described method <sup>43</sup>. Data shown as mean values; dots represent individual data points; error bars represent one standard deviation from biological replicates (n = 4 biologically independent 1mL culture samples). All experiments were repeated at least three times. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 9</span> </a> </div>   <div class="title" tagx="title" title="title">
Diagrams of light-stimulated lab-scale fermentors used to test YEZ167-4. (a and b)</div> <p>The 2L bioreactor was set up so that 3 light panels could be placed around the fermentor. A magnetic stir plate and stir bar were used to mix the culture, and fermentations were done in a 30¬∞C warm-room. <b>(c and d)</b> The 500 mL fed-batch bioreactor was set up so that 2 light panels could be placed around the fermentor. Culture was mixed with motorized propeller and a heat plate with temperature control probe was used to maintain temperature of 30¬∞C. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 10</span> </a> </div>   <div class="title" tagx="title" title="title">
Light-dependent GFP expression in lab-scale fermentor at relatively high cell densities</div> <p> <b>(a)</b> Schematic of 5L fermentor setup with dimensions of the area exposed to light. Red is the heating blanket around the reactor. Brown depicts the cell culture (2.5L). Blue depicts the area being illuminated by blue LEDs. <b>(b)</b> Picture of the functioning 5L, light-stimulated fermentor. <b>(c)</b> Representative flow cytometry results from 2 fermentation replicas, using YEZ243, which has light-inducible GFP expression. Cells were grown in fed-batch mode using a glycerol feed to achieve the highest cell densities possible in this setup. Yeast cells were exposed to light starting at an OD <sub>600</sub> of 15 and left under continuous illumination for the rest of the experiment. Samples of the fermentor were fixed at the time of harvesting to prevent time-dependent variations. Gray was a sample of YEZ140 without GFP, used as a control. Light blue is pre-induction at an OD <sub>600</sub> of 15. Red is after 1 hour of induction at an OD <sub>600</sub> of 16. Orange is after 6 hours of induction at an OD <sub>600</sub> of 19. Green is after 24 hours of induction at an OD <sub>600</sub> of 41. Purple is after 32 hours of induction at an OD <sub>600</sub> of 46. Dark blue is after 40 hours of induction at an OD <sub>600</sub> of 50. Every curve is generated from 20,000 cell counts. Data from the other fermentor run, which is very similar, is available upon request. All experiments were repeated at least three times. </p>   </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Information</span> <div class="media" title="media"> <a href="NIHMS942948-supplement-Supplementary_Information.pdf">LINK</a> </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Dr. Kevin Gardner (CUNY Advanced Science Research Center) and Dr. Laura Motta-Mena (Optologix, Inc.) for providing the plasmids and maps for the EL222 system (pVP16-EL222 and pC120-Fluc) <sup> <a href="#R12">12</a> </sup>. We also thank Dr. David Pincus (Whitehead Institute for Biomedical Research) for plasmid pNH603, Dr. Jinsuk J. Lee (Samsung Advanced Institute of Technology) for plasmid pET28a LDH, and Dr. Christof Taxis (Philipps University of Marburg) for plasmid pDS143. We thank Dr. Celeste Nelson (Princeton University) for sharing her qPCR equipment and Siyang Han for assistance in qPCR experiments and figure presentation. We thank Dr. Joshua Rabinowitz and Dr. John Storey (Princeton University) for sharing their 500mL Sixfors fermentation system for fed-batch fermentation experiments, as well as Dr. Sanford Silverman (Princeton University) for technical assistance on this equipment. We also thank Dr. Christina DeCoste and the Princeton Molecular Biology Flow Cytometry Resource Center for assistance in flow cytometry experiments. This work was supported by the Alfred P. Sloan Foundation (to J.L.A), The Pew Charitable Trusts (to J.L.A.), NIH grant DP2EB024247 (to J.E.T.), and an Eric and Wendy Schmidt Transformative Technology Fund grant (to J.L.A. and J.E.T.). </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Data Availability</b> </p> <p>The authors declare that all data supporting the findings of this study are available within the paper (and its supplementary information files), but original data that supports the findings are available from the corresponding authors upon reasonable request. </p> </div> <div class="fn-type-con" title="con"> <p> <b>Author contributions</b> </p> <p>E.M.Z., J.E.T. and J.L.A. conceived this project and designed the experiments. E.M.Z., Y.Z., and J.L.A. constructed the strains and plasmids. E.M.Z and H.P. did experiments in <a href="#F11">Figure 1</a>; E.M.Z. and J.M. did experiments in <a href="#F12">Figure 2</a>.; E.M.Z did experiments in <a href="#F13">Figure 3</a>; E.M.Z, Y.Z, and M.A.L. did experiments in Extended Data. E.M.Z., J.E.T. and J.L.A. analyzed the data and wrote the paper. </p> </div> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing financial interests</b> </p> <p>The authors declare no competing financial interests.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="R1" /> <span class="label" tagx="label" title="label">1</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woolston</span> <span class="given-names" tagx="given-names" title="given-names">BM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stephanopoulos</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Metabolic Engineering: Past and Future</span> <span class="source" tagx="source" title="source">Annu Rev Chem Biomol Eng</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">259</span> <span class="lpage" tagx="lpage" title="lpage">288</span> <span class="year" tagx="year" title="year">2013</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23540289">23540289</a> </span> </span> </li> <li tag="ref"> <a name="R2" /> <span class="label" tagx="label" title="label">2</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Using unnatural protein fusions to engineer resveratrol biosynthesis in yeast and mammalian cells </span> <span class="source" tagx="source" title="source">J Am Chem Soc</span> <span class="volume" tagx="volume" title="volume">128</span> <span class="fpage" tagx="fpage" title="fpage">13030</span> <span class="lpage" tagx="lpage" title="lpage">13031</span> <span class="year" tagx="year" title="year">2006</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17017764">17017764</a> </span> </span> </li> <li tag="ref"> <a name="R3" /> <span class="label" tagx="label" title="label">3</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keasling</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Manufacturing Molecules Through Metabolic Engineering</span> <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">1355</span> <span class="year" tagx="year" title="year">2011</span> </span> </li> <li tag="ref"> <a name="R4" /> <span class="label" tagx="label" title="label">4</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ajikumar</span> <span class="given-names" tagx="given-names" title="given-names">PK</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Isoprenoid pathway optimization for Taxol precursor overproduction in Escherichia coli </span> <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">330</span> <span class="fpage" tagx="fpage" title="fpage">70</span> <span class="lpage" tagx="lpage" title="lpage">74</span> <span class="year" tagx="year" title="year">2010</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20929806">20929806</a> </span> </span> </li> <li tag="ref"> <a name="R5" /> <span class="label" tagx="label" title="label">5</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tan</span> <span class="given-names" tagx="given-names" title="given-names">SZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manchester</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prather</span> <span class="given-names" tagx="given-names" title="given-names">KLJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Controlling central carbon metabolism for improved pathway yields in <i>Saccharomyces cerevisiae</i> </span> <span class="source" tagx="source" title="source">ACS Synth Biol</span> <span class="comment">acssynbio.5b00164</span> <span class="year" tagx="year" title="year">2015</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acssynbio.5b00164">10.1021/acssynbio.5b00164</a> </span> </span> </li> <li tag="ref"> <a name="R6" /> <span class="label" tagx="label" title="label">6</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ro</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Production of the antimalarial drug precursor artemisinic acid in engineered yeast</span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">440</span> <span class="fpage" tagx="fpage" title="fpage">3</span> <span class="lpage" tagx="lpage" title="lpage">6</span> <span class="year" tagx="year" title="year">2006</span> </span> </li> <li tag="ref"> <a name="R7" /> <span class="label" tagx="label" title="label">7</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gu</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Su</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Tunable switch mediated shikimate biosynthesis in an engineered non-auxotrophic Escherichia coli </span> <span class="source" tagx="source" title="source">Sci Rep</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">29745</span> <span class="year" tagx="year" title="year">2016</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27406890">27406890</a> </span> </span> </li> <li tag="ref"> <a name="R8" /> <span class="label" tagx="label" title="label">8</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brockman</span> <span class="given-names" tagx="given-names" title="given-names">IM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prather</span> <span class="given-names" tagx="given-names" title="given-names">KLJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Dynamic knockdown of E. coli central metabolism for redirecting fluxes of primary metabolites </span> <span class="source" tagx="source" title="source">Metab Eng</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">104</span> <span class="lpage" tagx="lpage" title="lpage">113</span> <span class="year" tagx="year" title="year">2015</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25542851">25542851</a> </span> </span> </li> <li tag="ref"> <a name="R9" /> <span class="label" tagx="label" title="label">9</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toettcher</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voigt</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiner</span> <span class="given-names" tagx="given-names" title="given-names">OD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">WA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The promise of optogenetics in cell biology: interrogating molecular circuits in space and time </span> <span class="source" tagx="source" title="source">Nat Methods</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">35</span> <span class="lpage" tagx="lpage" title="lpage">38</span> <span class="year" tagx="year" title="year">2011</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21191370">21191370</a> </span> </span> </li> <li tag="ref"> <a name="R10" /> <span class="label" tagx="label" title="label">10</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milias-Argeitis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">In silico feedback for in vivo regulation of a gene expression circuit</span> <span class="source" tagx="source" title="source">Nat Biotechnol</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">1114</span> <span class="lpage" tagx="lpage" title="lpage">6</span> <span class="year" tagx="year" title="year">2011</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22057053">22057053</a> </span> </span> </li> <li tag="ref"> <a name="R11" /> <span class="label" tagx="label" title="label">11</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kennedy</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rapid blue-light-mediated induction of protein interactions in living cells</span> <span class="source" tagx="source" title="source">Nat Methods</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">973</span> <span class="lpage" tagx="lpage" title="lpage">975</span> <span class="year" tagx="year" title="year">2010</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21037589">21037589</a> </span> </span> </li> <li tag="ref"> <a name="R12" /> <span class="label" tagx="label" title="label">12</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Motta-Mena</span> <span class="given-names" tagx="given-names" title="given-names">LB</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An optogenetic gene expression system with rapid activation and deactivation kinetics</span> <span class="source" tagx="source" title="source">Nat Chem Biol</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">196</span> <span class="lpage" tagx="lpage" title="lpage">202</span> <span class="year" tagx="year" title="year">2014</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24413462">24413462</a> </span> </span> </li> <li tag="ref"> <a name="R13" /> <span class="label" tagx="label" title="label">13</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimizu-Sato</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huq</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tepperman</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quail</span> <span class="given-names" tagx="given-names" title="given-names">PH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A light-switchable gene promoter system</span> <span class="source" tagx="source" title="source">Nat Biotechnol</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">1041</span> <span class="lpage" tagx="lpage" title="lpage">1044</span> <span class="year" tagx="year" title="year">2002</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12219076">12219076</a> </span> </span> </li> <li tag="ref"> <a name="R14" /> <span class="label" tagx="label" title="label">14</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taslimi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optimized second-generation CRY2‚ÄìCIB dimerizers and photoactivatable Cre recombinase</span> <span class="source" tagx="source" title="source">Nat Chem Biol</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">8</span> <span class="year" tagx="year" title="year">2016</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nchembio.2063">10.1038/nchembio.2063</a> </span> </span> </li> <li tag="ref"> <a name="R15" /> <span class="label" tagx="label" title="label">15</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salinas</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rojas</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delgado</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agosin</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larrondo</span> <span class="given-names" tagx="given-names" title="given-names">LF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optogenetic switches for light-controlled gene expression in yeast</span> <span class="source" tagx="source" title="source">Appl Microbiol Biotechnol</span> <span class="year" tagx="year" title="year">2017</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00253-017-8178-8">10.1007/s00253-017-8178-8</a> </span> </span> </li> <li tag="ref"> <a name="R16" /> <span class="label" tagx="label" title="label">16</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nash</span> <span class="given-names" tagx="given-names" title="given-names">AI</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Structural basis of photosensitivity in a bacterial light-oxygen-voltage/helix-turn-helix (LOV-HTH) DNA-binding protein </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span> <span class="volume" tagx="volume" title="volume">109</span> <span class="year" tagx="year" title="year">2012</span> </span> </li> <li tag="ref"> <a name="R17" /> <span class="label" tagx="label" title="label">17</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rivera-Cancel</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Motta-Mena</span> <span class="given-names" tagx="given-names" title="given-names">LB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gardner</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of natural and artificial DNA substrates for light-activated LOV-HTH transcription factor EL222 </span> <span class="source" tagx="source" title="source">Biochemistry</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">10024</span> <span class="lpage" tagx="lpage" title="lpage">10034</span> <span class="year" tagx="year" title="year">2012</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23205774">23205774</a> </span> </span> </li> <li tag="ref"> <a name="R18" /> <span class="label" tagx="label" title="label">18</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zoltowski</span> <span class="given-names" tagx="given-names" title="given-names">BD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Motta-Mena</span> <span class="given-names" tagx="given-names" title="given-names">LB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gardner</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Blue-ligh induced dimerization of a bacterial LOV-HTH DNA-binding protein</span> <span class="source" tagx="source" title="source">Biochemistry</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">2273</span> <span class="lpage" tagx="lpage" title="lpage">2275</span> <span class="year" tagx="year" title="year">2013</span> </span> </li> <li tag="ref"> <a name="R19" /> <span class="label" tagx="label" title="label">19</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reade</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">TAEL: a zebrafish-optimized optogenetic gene expression system with fine spatial and temporal control </span> <span class="source" tagx="source" title="source">Development</span> <span class="volume" tagx="volume" title="volume">144</span> <span class="fpage" tagx="fpage" title="fpage">345</span> <span class="lpage" tagx="lpage" title="lpage">355</span> <span class="year" tagx="year" title="year">2017</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27993986">27993986</a> </span> </span> </li> <li tag="ref"> <a name="R20" /> <span class="label" tagx="label" title="label">20</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jayaraman</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Blue light-mediated transcriptional activation and repression of gene expression in bacteria </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="fpage" tagx="fpage" title="fpage">gkw548</span> <span class="year" tagx="year" title="year">2016</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkw548">10.1093/nar/gkw548</a> </span> </span> </li> <li tag="ref"> <a name="R21" /> <span class="label" tagx="label" title="label">21</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Da Silva</span> <span class="given-names" tagx="given-names" title="given-names">NA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srikrishnan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Introduction and expression of genes for metabolic engineering applications in Saccharomyces cerevisiae </span> <span class="source" tagx="source" title="source">FEMS Yeast Res</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">197</span> <span class="lpage" tagx="lpage" title="lpage">214</span> <span class="year" tagx="year" title="year">2012</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22129153">22129153</a> </span> </span> </li> <li tag="ref"> <a name="R22" /> <span class="label" tagx="label" title="label">22</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Usherenko</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Photo-sensitive degron variants for tuning protein stability by light</span> <span class="source" tagx="source" title="source">BMC Syst Biol</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">128</span> <span class="year" tagx="year" title="year">2014</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25403319">25403319</a> </span> </span> </li> <li tag="ref"> <a name="R23" /> <span class="label" tagx="label" title="label">23</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Maris</span> <span class="given-names" tagx="given-names" title="given-names">AJA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Directed evolution of pyruvate decarboxylase-negative <i>Saccharomyces cerevisiae</i>, yielding a C <sub>2</sub>-independent, glucose-tolerant, and pyruvate-hyperproducing yeast </span> <span class="source" tagx="source" title="source">Appl Environ Microbiol</span> <span class="volume" tagx="volume" title="volume">70</span> <span class="fpage" tagx="fpage" title="fpage">159</span> <span class="lpage" tagx="lpage" title="lpage">66</span> <span class="year" tagx="year" title="year">2004</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14711638">14711638</a> </span> </span> </li> <li tag="ref"> <a name="R24" /> <span class="label" tagx="label" title="label">24</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klein</span> <span class="given-names" tagx="given-names" title="given-names">JLC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olsson</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Glucose control in Saccharomyces cerevisiae‚ÄØ: the role of M/G7 in metabolic functions</span> <span class="source" tagx="source" title="source">Microbiology</span> <span class="volume" tagx="volume" title="volume">144</span> <span class="fpage" tagx="fpage" title="fpage">13</span> <span class="lpage" tagx="lpage" title="lpage">24</span> <span class="year" tagx="year" title="year">1998</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9467897">9467897</a> </span> </span> </li> <li tag="ref"> <a name="R25" /> <span class="label" tagx="label" title="label">25</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Avalos</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fink</span> <span class="given-names" tagx="given-names" title="given-names">GR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stephanopoulos</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Compartmentalization of metabolic pathways in yeast mitochondria improves the production of branched-chain alcohols </span> <span class="source" tagx="source" title="source">Nat Biotechnol</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">335</span> <span class="lpage" tagx="lpage" title="lpage">41</span> <span class="year" tagx="year" title="year">2013</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23417095">23417095</a> </span> </span> </li> <li tag="ref"> <a name="R26" /> <span class="label" tagx="label" title="label">26</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hammer</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Avalos</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Uncovering the role of branched-chain amino acid transaminases in Saccharomyces cerevisiae isobutanol biosynthesis </span> <span class="source" tagx="source" title="source">Metab Eng</span> <span class="year" tagx="year" title="year">2017</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ymben.2017.10.001">10.1016/j.ymben.2017.10.001</a> </span> </span> </li> <li tag="ref"> <a name="R27" /> <span class="label" tagx="label" title="label">27</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hahn</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Improvement of isobutanol production in Saccharomyces cerevisiae by increasing mitochondrial import of pyruvate through mitochondrial pyruvate carrier </span> <span class="source" tagx="source" title="source">Appl Microbiol Biotechnol</span> <span class="volume" tagx="volume" title="volume">100</span> <span class="fpage" tagx="fpage" title="fpage">7591</span> <span class="lpage" tagx="lpage" title="lpage">7598</span> <span class="year" tagx="year" title="year">2016</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27225475">27225475</a> </span> </span> </li> <li tag="ref"> <a name="R28" /> <span class="label" tagx="label" title="label">28</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deisseroth</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optogenetics</span> <span class="source" tagx="source" title="source">Nat Methods</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">26</span> <span class="lpage" tagx="lpage" title="lpage">29</span> <span class="year" tagx="year" title="year">2011</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21191368">21191368</a> </span> </span> </li> <li tag="ref"> <a name="R29" /> <span class="label" tagx="label" title="label">29</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerhardt</span> <span class="given-names" tagx="given-names" title="given-names">KP</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An open-hardware platform for optogenetics and photobiology</span> <span class="source" tagx="source" title="source">Sci Rep</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">35363</span> <span class="year" tagx="year" title="year">2016</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27805047">27805047</a> </span> </span> </li> <li tag="ref"> <a name="R30" /> <span class="label" tagx="label" title="label">30</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milias-Argeitis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rullan</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aoki</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buchmann</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khammash</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Automated optogenetic feedback control for precise and robust regulation of gene expression and cell growth </span> <span class="source" tagx="source" title="source">Nat Commun</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">12546</span> <span class="year" tagx="year" title="year">2016</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27562138">27562138</a> </span> </span> </li> <li tag="ref"> <a name="R31" /> <span class="label" tagx="label" title="label">31</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milne</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wahl</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Maris</span> <span class="given-names" tagx="given-names" title="given-names">AJA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pronk</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daran</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Excessive by-product formation: A key contributor to low isobutanol yields of engineered Saccharomyces cerevisiae strains </span> <span class="source" tagx="source" title="source">Metab Eng Commun</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">39</span> <span class="lpage" tagx="lpage" title="lpage">51</span> <span class="year" tagx="year" title="year">2016</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29142820">29142820</a> </span> </span> </li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Optogenetic circuits characterization</div> <p> <b>(a)</b> Reversible <i>OptoEXP</i> system based on VP16-EL222, sensitive to 450 nm light. <b>(b)</b> GFP expression with <i>OptoEXP</i> (YEZ139) tuned with different duty cycles. <b>(c)</b> Maximum GFP expression from <i>OptoEXP</i> compared to common constitutive promoters using strains YEZ27(P <sub>CYC1</sub>), YEZ28(P <sub>ADH1</sub>), YEZ29(P <sub>PGK1</sub>), YEZ30(P <sub>TDH3</sub>), YEZ31(P <sub>TEF1</sub>) and YEZ32(P <sub>C120</sub>). <b>(d)</b> <i>OptoINVRT</i> circuit diagram (details in <a href="#F4">Extended Data Fig. 4a, b</a>) <b>(e)</b> <i>OptoINVRT</i> circuits in CENPK.2-1C-derived ( <i>gal80Œî, gal4Œî</i>) strains YEZ100 ( <i>OptoINVRT1</i>), YEZ101 ( <i>OptoINVRT2</i>), and YEZ102 ( <i>OptoINVRT3</i>). <b>(f)</b> <i>OptoINVRT</i> circuits in BY4741-derived ( <i>gal80Œî</i>, <i>pdc1Œî, pdc5Œî, pdc6Œî</i>) strains YEZ115 ( <i>OptoINVRT1</i>), YEZ116 ( <i>OptoINVRT2</i>), and YEZ117 ( <i>OptoINVRT3</i>). All data shown as mean values; dots represent individual data points; error bars represent one standard deviation of replicates exposed to the same conditions (n = 4 biologically independent 1mL samples). All experiments were repeated at least three times. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Light-dependent metabolic valve for ethanol production</div> <p> <b>(a)</b> Using <i>OptoEXP</i> to build light-inducible metabolic valve for <i>PDC1</i>. <b>(b)</b> Light-dependent growth on glucose of a triple-PDCŒî strain containing <i>OptoEXP-PDC1</i> metabolic valve. Top left: BY4741 (Wild-type). Top right: YEZ61-23 ( <i>pdc1Œî, pdc5Œî, pdc6Œî</i>, P <sub>TEF1</sub>_VP16-EL222, P <sub>C120</sub>_ <i>PDC1</i>). Bottom left: YEZ50C ( <i>pdc1Œî, pdc5Œî, pdc6Œî</i>, P <sub>TEF1</sub>_VP16-EL222). Bottom right: YEZ61C ( <i>pdc1Œî, pdc5Œî, pdc6Œî</i>, P <sub>C120</sub>_ <i>PDC1</i>). YPD: Yeast peptone + 2% glucose; YPGE: Yeast peptone + 3% glycerol + 2% ethanol. <b>(c)</b> YEZ61-23 and BY4741 growth in liquid SC + 2% glucose media at different light doses, with best fits to the exponential growth curves. <b>(d)</b> Maximum growth rate of YEZ61-23 as a function of light. Solid line represents average of wild-type (BY4741) growth rate (dotted lines mark standard deviation). All data shown as mean values (individual points available in supplementary spreadsheet); error bars represent standard deviations of biological replicates exposed to the same conditions (n = 3). All experiments were repeated at least three times. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Light-controlled isobutanol production</div> <p> <b>(a)</b> Isobutanol and ethanol biosynthesis controlled by <i>OptoINVRT-ILV2</i> and <i>OptoEXP-PDC1</i>, respectively. <b>(b)</b> Blue light allows YEZ167-4 growth in glucose, alongside ethanol formation, while dark conditions initiate production phase by stopping <i>PDC1</i> induction and de-repressing <i>ILV2</i>. <b>(c)</b> Isobutanol and 2-MBOH production of YEZ167-4 in 80-hour long fermentations in 150g/L glucose and blue light pulses of 15s ON/65s OFF for 30 minutes every 5, 10, or 20 hours. YZy335 is a light-insensitive isobutanol-producing control strain. <b>(d)</b> Growth of YEZ167-4 in 2L bioreactor and constant blue light, compared to YZy335 wild-type control. <b>(e)</b> Isobutanol and 2-MBOH production of YEZ167-4 in a 0.5L fed-batch pH-controlled fermentor using periodic light stimulation during the production phase. Blue bar indicates the time of blue light exposure. Dotted lines indicate start and stop of glucose feed. All data shown as mean values; dots in (c) represent individual data points; individual points for (d) and (e) are available in supplementary spreadsheet and <a href="#SD1">Supplementary Table 5</a>, respectively); error bars represent standard deviations of three biological replicates (c), n=3; or three separate bioreactor runs (d and e), n=3. All experiments were repeated at least twice. </p>   </div> </div>  </body></html>