<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">ACS Synth Biol</span> <span class="iso-abbrev" title="iso-abbrev">ACS Synth Biol</span> <span class="publisher-id" title="publisher-id">sb</span> <span class="coden" title="coden">asbcd6</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">ACS Synthetic Biology</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2161-5063</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">American Chemical Society </span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5820651">5820651</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1021/acssynbio.7b00413">10.1021/acssynbio.7b00413</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Biosynthesis of Orthogonal Molecules Using Ferredoxin and Ferredoxin-NADP <sup>+</sup> Reductase Systems Enables Genetically Encoded PhyB Optogenetics </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kyriakakis</span> <span class="given-names" tagx="given-names" title="given-names">Phillip</span> </span> <a href="#cor1">*</a> <a href="#aff1">†</a> <a href="#notes2">¶</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Catanho</span> <span class="given-names" tagx="given-names" title="given-names">Marianne</span> </span> <a href="#aff1">†</a> <a href="#notes2">¶</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoffner</span> <span class="given-names" tagx="given-names" title="given-names">Nicole</span> </span> <a href="#aff3">§</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thavarajah</span> <span class="given-names" tagx="given-names" title="given-names">Walter</span> </span> <a href="#aff1">†</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">Vincent J.</span> </span> <a href="#aff1">†</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chao</span> <span class="given-names" tagx="given-names" title="given-names">Syh-Shiuan</span> </span> <a href="#aff4">∥</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hsu</span> <span class="given-names" tagx="given-names" title="given-names">Athena</span> </span> <a href="#aff5">⊥</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pham</span> <span class="given-names" tagx="given-names" title="given-names">Vivian</span> </span> <a href="#aff6">#</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naghavian</span> <span class="given-names" tagx="given-names" title="given-names">Ladan</span> </span> <a href="#aff1">†</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dozier</span> <span class="given-names" tagx="given-names" title="given-names">Lara E.</span> </span> <a href="#aff2">‡</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patrick</span> <span class="given-names" tagx="given-names" title="given-names">Gentry N.</span> </span> <a href="#aff2">‡</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coleman</span> <span class="given-names" tagx="given-names" title="given-names">Todd P.</span> </span> <a href="#cor2">*</a> <a href="#aff1">†</a> </span> <span class="citation_author_institution" id="aff1">[†], <span class="institution" title="institution">University of California, San Diego </span> <span class="country" tagx="country" title="country">United States</span> </span> <span class="citation_author_institution" id="aff2">[‡], <span class="institution" title="institution">University of California, San Diego</span> <span class="country" tagx="country" title="country">United States</span> </span> <span class="citation_author_institution" id="aff3">[§], <span class="institution" title="institution">University of California, San Diego </span> <span class="country" tagx="country" title="country">United States</span> </span> <span class="citation_author_institution" id="aff4">[∥], <span class="institution" title="institution">Quinnipiac University </span> <span class="country" tagx="country" title="country">United States</span> </span> <span class="citation_author_institution" id="aff5">[⊥], <span class="institution" title="institution">University of California, San Diego </span> <span class="country" tagx="country" title="country">United States</span> </span> <span class="citation_author_institution" id="aff6">[#], <span class="institution" title="institution">University of Iowa </span> <span class="country" tagx="country" title="country">United States</span> </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span>E-mail: <span class="email" tagx="email" title="email">pkyriaka@ucsd.edu</span>. </div> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span>E-mail: <span class="email" tagx="email" title="email">tpcoleman@ucsd.edu</span>. </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-01-01</span> </span> <span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2018-02-02</span> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">706</span> <span class="lpage" tagx="lpage" title="lpage">717</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-11-17</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">American Chemical Society </span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article published under an ACS AuthorChoice <a href="http://pubs.acs.org/page/policy/authorchoice_termsofuse.html">License</a>, which permits copying and redistribution of the article or any adaptations for non-commercial purposes. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p> <div class="graphic_UNKNOWN" /> </p> <p>Transplanting metabolic reactions from one species into another has many uses as a research tool with applications ranging from optogenetics to crop production. Ferredoxin (Fd), the enzyme that most often supplies electrons to these reactions, is often overlooked when transplanting enzymes from one species to another because most cells already contain endogenous Fd. However, we have shown that the production of chromophores used in Phytochrome B (PhyB) optogenetics is greatly enhanced in mammalian cells by expressing bacterial and plant Fds with ferredoxin-NADP+ reductases (FNR). We delineated the rate limiting factors and found that the main metabolic precursor, heme, was not the primary limiting factor for producing either the cyanobacterial or plant chromophores, phycocyanobilin or phytochromobilin, respectively. In fact, Fd is limiting, followed by Fd+FNR and finally heme. Using these findings, we optimized the PCB production system and combined it with a tissue penetrating red/far-red sensing PhyB optogenetic gene switch in animal cells. We further characterized this system in several mammalian cell lines using red and far-red light. Importantly, we found that the light-switchable gene system remains active for several hours upon illumination, even with a short light pulse, and requires very small amounts of light for maximal activation. Boosting chromophore production by matching metabolic pathways with specific ferredoxin systems will enable the unparalleled use of the many PhyB optogenetic tools and has broader implications for optimizing synthetic metabolic pathways. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
optogenetics</div> <div class="kwd" title="kwd">
phytochrome</div> <div class="kwd" title="kwd">
phycocyanobilin</div> <div class="kwd" title="kwd">
phytochromobilin</div> <div class="kwd" title="kwd">
ferredoxin</div> <div class="kwd" title="kwd">
FNR</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="document-id-old-9" value="sb7b00413" /> <meta name="document-id-new-14" value="sb-2017-00413t" /> <meta name="ccc-price" value="" /> </div> </div> </div> <div class="body" title="body"> <p>An established research practice used in synthetic biology is the transplantation of metabolic reactions from one species to another, with wide-ranging potential applications including metabolic gene therapy, <sup> <a href="#ref1">1</a>, <a href="#ref2">2</a> </sup> production of crops without fertilizer, <sup> <a href="#ref3">3</a>, <a href="#ref4">4</a> </sup> and more fundamental applications in research, such as optogenetics. The exquisite temporal and spatial precision achieved through optogenetics have been used to develop an assortment of powerful analytical tools to control biological functions such as gene expression, <sup> <a href="#ref5">5</a>− <a href="#ref10">10</a> </sup> neural activity, <sup> <a href="#ref11">11</a>, <a href="#ref12">12</a> </sup> cell signaling, <sup> <a href="#ref13">13</a> </sup> secretion, <sup> <a href="#ref14">14</a> </sup> peroxisomal trafficking, <sup> <a href="#ref15">15</a> </sup> and protein activity. <sup> <a href="#ref16">16</a> </sup> Metabolically engineering cells to endogenously produce specific chromophores enables many optogenetic applications, including genetically encoded systems for optical control of genes. <sup> <a href="#ref17">17</a> </sup> Many of the systems used and characterized for these applications utilize proteins that require red and far-red responsive phytobilin chromophores like phycocyanobilin (PCB) and phytochromobilin (PΦB). These molecules originate from phytochrome systems in cyanobacteria, algae, and plants, but are not naturally made in many fungal species, bacteria, or animal cells. <sup> <a href="#ref18">18</a>− <a href="#ref21">21</a> </sup> Production of these chromophores requires biliverdin IX-alpha (BV), a degradation product of heme, and the enzymes phycocyanobilin:ferredoxin oxidoreductase (PcyA) or phytochromobilin:ferredoxin oxidoreductase (HY2), respectively ( <a href="#fig1">Figure </a><a href="#fig1">1</a> A). <sup> <a href="#ref22">22</a>, <a href="#ref23">23</a> </sup> Several groups produced PCB and PΦB in <i>E. coli</i> by expressing PcyA or HY2 along with heme oxygenase (HO1), without adding a ferredoxin (Fd) and ferredoxin-NADP <sup>+</sup>-reductase (FNR) reduction system from the same species as the PcyA or HY2 enzymes. <sup> <a href="#ref24">24</a>− <a href="#ref29">29</a> </sup> Likewise, Müller <i>et al.</i> tested PCB production in mammalian cells by expressing cyanobacterial PcyA and HO1 in the mitochondria but did not cointroduce a cyanobacterial Fd-FNR system. <sup> <a href="#ref11">11</a> </sup> Müller <i>et al.</i> reasoned that localizing PcyA and HO1 in the same cellular compartment where the chromophore precursor (heme) is produced would enhance PCB production. <sup> <a href="#ref11">11</a> </sup> However, in addition to heme, HO1, PcyA, and HY2 also depend on Fd activity, leaving open the possibility that Fd and not heme was limiting. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>PCB and PφB production is limited by Fd+FNR in mammalian cells. (A) The metabolic pathway forPCB synthesis including the NADPH/FNR/Fd redox cascade (Heme: ChemSpider ID 4802, Bv: ChemSpider ID10628548, PCB: ChemSpider ID 16736730). (B) HEK293 cells were analyzed for phytobilin production using the plasmids shown. Phytobilin production was measured by covalent linkage to PhyB followed by immunoprecipitation with anti-HA, Zn-PAGE and Western blots. sPCYA and tPCYA produce PCB and aHY2 produces PΦB. Cells were either transfected with two ferredoxin-dependent enzymes (ho1 and pcyA or ho1 and HY2) alone (condition M2) or along with matching Fd+FNR (tpetF+tpetH) plasmids (condition M4). ho1 = heme oxygenase, pcyA = phycocyanobilin:ferredoxin oxidoreductase, HY2 = phytochromobilin:ferredoxin oxidoreductase, petF = ferredoxin, petH = ferredoxin:oxidoreductase/FNR, NE = No Enzymes, SYNP2 = Synechococcus PCC7002 and THEEB = <i>Thermosynechococcus elongatus</i>, ARATH = <i>Arabidopsis thaliana</i>, MTS = Mitochondrial Targeting Sequence, P2A = 2A self-cleaving peptide, IRES = Internal Ribosome Entry Site, NLS = Nuclear Localization Sequence, DBD = DNA Binding Domain. </p>   </div> <p>Most cells already contain endogenous Fd; therefore, researchers have not typically considered it when transplanting enzymes from one species to another. However, Beale <i>et al.</i> and Frankenberg <i>et al.</i> demonstrated that Fd activity on PcyA from <i>Anabaena</i> sp. PCC 7120 varies greatly depending on the species Fd comes from. <sup> <a href="#ref24">24</a>, <a href="#ref30">30</a> </sup> Similarly, mammalian Fds have also been shown to be highly specific to their target enzymes, suggesting that Fd and/or FNR may be limiting for chromophore production in mammalian cells. <sup> <a href="#ref31">31</a>, <a href="#ref32">32</a> </sup> Consequently, to increase production of molecules like PCB for optogenetic uses in animal cells, we investigated the limiting factors for the PCB and PΦB production in mammalian cells. </p> <p>To evaluate the rate-limiting reactants for endogenous chromophore production, we systematically tested each component of the biosynthetic pathway, including Fd and FNR. We showed that Fd+FNR is the primary rate-limiting component, followed by heme. The increased PCB production found with the addition of Fd+FNR was further improved by testing different stoichiometric expression levels of each enzyme. Endogenous PCB production was greatly increased compared to previous approaches <sup> <a href="#ref17">17</a> </sup> that did not consider metabolic engineering with Fd+FNR systems. </p> <p>To demonstrate the utility of increased chromophore production for optogenetic applications, we chose a PhyB-based optogenetic system, which utilizes PCB and has been used to control a wide array of biological processes. Since the light sensitivity of PhyB is proportional to the amount of chromophore in the cell, to apply PhyB optogenetic tools in transgenic animal models, it will be essential to genetically encode a high level of chromophore production. Able to produce significantly more chromophore than before, <sup> <a href="#ref17">17</a> </sup> we fully genetically encoded the red/far-red PhyB-PIF3 two-hybrid gene switch for the first time. </p> <p>A genetically encoded PhyB-PIF3 system with PCB production is particularly significant because when bound to PhyB, chromophores such as PCB: (i) are extremely sensitive to light (high absorbance/extinction coefficient), (ii) have a long-lived activation state, ranging from tens of minutes to hours, <sup> <a href="#ref33">33</a> </sup> (iii) are reversible upon illumination with a specific wavelength of far-red light, <sup> <a href="#ref14">14</a> </sup> and (iv) respond to wavelengths optimal for tissue penetration. The reversibility of this system with far-red light allows for additional spatial control by enabling suppression of gene activity with far-red light in specific locations. <sup> <a href="#ref34">34</a> </sup> After adapting the PhyB-PIF3 system from Shimizu Sato <i>et al.</i>( <a href="#ref5">5</a>) for mammalian cells, we found that it can induce gene expression by several hundred fold, it is reversible with a stable "on state" in the order of hours, and it requires very low amounts of red light for maximum activation (calculated to be below the equivalent of 40 nW/cm <sup>2</sup> of continuous light for full activation over 24 h). Since genetically encoding the system maintains a constant supply of chromophore, we were also able to find that the light intensity required for maximal gene activation depends on the duration of illumination. </p> <p>More generally than optogenetics, there are numerous biomolecules produced in bacteria and plants that are Fd-dependent. Matching the Fd species to a biosynthetic production pathway makes possible the metabolism of many other classes of molecules such as lipids, sterols, luciferins, quinones, carotenoids, nitrates/nitrogen, and sulfites not normally produced in those cells. <sup> <a href="#ref3">3</a>, <a href="#ref35">35</a>− <a href="#ref40">40</a> </sup> Increasing production of these classes of molecules can improve agriculture, increase the production of pharmaceuticals, and enable other tools for synthetic biology. </p> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="regulationofpcbproductioninmammaliancellsbyfd,fnr,andheme" title="sec"> <div class="title" tagx="title" title="title">
Regulation of PCB Production in Mammalian Cells by Fd, FNR, and Heme </div> <p>Given that previous studies have shown that PCB production can be limited by heme, Fd or FNR, <sup> <a href="#ref25">25</a>, <a href="#ref30">30</a> </sup> we tested limiting factors of PCB production in mammalian cells using combinations of these components in excess. Zinc-PAGE PhyB immunoprecipitation assays in Human Embryonic Kidney (HEK293) cells were used to test PCB production with metabolic enzymes from two species: <i>Synechococcus sp. PCC 7002 </i> (SYNP2/sPcyA) or <i>Thermosynechococcus elongatus </i> (THEEB/tPcyA). We tested PCB production under two conditions, either mitochondrial-HO1+PcyA ( <b>M2</b>) or mitochondrial-HO1+PcyA+Fd+FNR ( <b>M4</b>), ( <a href="#fig1">Figure </a><a href="#fig1">1</a> B). When either species of HO1+PcyA enzymes were expressed, we detected low levels of PCB ( <a href="#fig1">Figure </a><a href="#fig1">1</a> B, <b>M2</b>). However, when all four enzymes HO1+PcyA+Fd+FNR ( <b>M4</b>) were expressed, we observed a striking increase in PCB levels ( <a href="#fig1">Figure </a><a href="#fig1">1</a> B), which agrees with recent findings from Uda <i>et al. </i>( <a href="#ref41">41</a>) To exclude the possibility that this was specific to cyanobacterial enzymes, we also produced the plant chromophore PΦB, by replacing the cyanobacterial PcyA with a plant homologue <i>Arabidopsis</i> HY2. PcyA and HY2 showed the same Fd+FNR dependence ( <a href="#fig1">Figure </a><a href="#fig1">1</a> B, <b>M2-asHY2</b> <i>versus</i> <b>M4-asHY2</b>). It is noteworthy that the Fd+FNR-dependent increase in PΦB production was still observed when plant HY2 was used along with cyanobacterial HO1/Fd/FNR. We chose SYNP2 Fd+FNR for recycling HY2 because SYNP2 Fd was more similar than THEEB Fd in amino acid sequence identity to <i>Arabidopsis</i> Fds and specifically the major ferredoxin that recycles HY2 in <i>Arabidopsis</i> ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Table S1</a>). <sup> <a href="#ref42">42</a> </sup> However, PΦB production may be further increased by employing <i>Arabidopsis</i> Fd+FNR enzymes. It may be possible to predict compatibility of a transplanted ferredoxin-dependent pathway to the host cells Fd based on sequence similarity as shown in <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Table S1</a>. These findings show that excess Fd+FNR activity can increase PCB or PΦB production in mammalian cells ( <a href="#fig1">Figure </a><a href="#fig1">1</a> B). </p> <p>Next, we delineated the limiting factors for the endogenous production of chromophores in mammalian cells. We decided to test PCB production in both the cytoplasm and mitochondria because the endogenous ferredoxin system of mammalian cells is localized in the mitochondria; therefore, we considered the cytoplasmic enzyme localization as a condition with negligible endogenous Fd+FNR activity. We show in <a href="#fig2">Figure </a><a href="#fig2">2</a> A that expression of cytoplasmic-PcyA+HO1 ( <b>C2</b>) is not sufficient to produce significant levels of PCB (lane 3 <i>vs</i> lane 2). When cytoplasmic-PcyA+HO1 was cotransfected along with cytoplasmic Fd+FNR ( <b>C4</b>) higher, but statistically nonsignificant levels of PCB were detected (lane 3 <i>vs</i> 4, <i>p</i> &amp;gt; 0.05). Similarly, when PcyA+HO1 were localized to the mitochondria ( <b>M2</b>), very low levels of PCB were detected (lane 5). However, when PcyA+HO1 and Fd+FNR were all localized to the mitochondria ( <b>M4</b>), PCB production was significantly increased when compared to PcyA+HO1 only ( <b>M2</b>) (lane 5 <i>vs</i> 6, <i>p</i> &amp;lt; 0.001). These findings were corroborated by imaging PhyB-bound PCB using the Cy-5 channel (blue) ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Figure S1</a>). These results demonstrate that the Fd+FNR system is the primary limiting factor of the PCB production pathway in mammalian mitochondria, but it is not sufficient for high levels of PCB production when expressed in the cytoplasm. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Order of rate limiting factors of PCB production in mammalian cells. (A,B) HEK293 cells were analyzed for PCB production using the plasmids shown. PCB production was measured by covalent linkage to PhyB followed by immunoprecipitation with anti-HA, Zn-PAGE and Western blots. (A) PCB production was compared with excess (+heme) and without (−heme), using the cytoplasmic expression of pcyA+ho1 alone (condition C2) or with cytoplasmic pcyA+ho1+fd+fnr (condition C4); mitochondrial expression of pcyA+ho1 alone (condition M2) or with mitochondrial pcyA+ho1+fd+fnr (condition M4) ( <i>n</i> = 4). (B) Cells were either transfected with two ferredoxin-dependent enzymes alone, ho1 and pcyA (condition M2), or along with a matching fd:tpetF (condition M3) or along with matching fd+fnr:tpetF + tpetH (condition M4) ( <i>n</i> = 4). ho1 = heme oxygenase, pcyA = phycocyanobilin:ferredoxin oxidoreductase, HY2 = phytochromobilin:ferredoxin oxidoreductase, petF = ferredoxin/fd, petH = ferredoxin:oxidoreductase/fnr, NE = No Enzymes, SYNP2 = Synechococcus PCC7002 and THEEB = <i>Thermosynechococcus elongatus </i>, ARATH= <i>Arabidopsis thaliana</i>, IRES = Internal Ribosome Entry Site, NLS = Nuclear Localization Sequence, MTS = Mitochondrial Targeting Sequence, P2A = 2A self-cleaving peptide, DBD = DNA Binding Domain. One-way ANOVA with Bonferroni post-test was used to calculate <i>p</i> values using GraphPad Prism 5.01. (*) = <i>p</i> &amp;lt; 0.05, (**) = <i>p</i> &amp;lt; 0.01, (***) = <i>p</i> &amp;lt; 0.001. Error bars = Standard Deviation. <i>n</i> = independent experiments. </p>   </div> <p>Since heme is a metabolic precursor in the PCB production pathway, we systematically tested if it was limiting for PCB production in either the cytoplasm or in the mitochondria. We hypothesized that if heme was a limiting factor for PCB production in the cytoplasm, then the addition of excess heme would increase production. While a faint band was visible in <b>C2</b>+heme ( <a href="#fig2">Figure </a><a href="#fig2">2</a> A lane 9), it was indistinguishable from cells transfected with PhyB and no enzymes and given excess heme ( <a href="#fig2">Figure </a><a href="#fig2">2</a> A lane 8). However, excess heme significantly increased levels of PCB production in the <b>C4</b> condition (lanes 4 and 10, <i>p</i> &amp;lt; 0.01). In addition, we found that Fd+FNR was limiting when comparing <b>C2</b>+heme to <b>C4</b>+heme (lanes 9 and 10, <i>p</i> &amp;lt; 0.01). This demonstrates that heme is the limiting factor for PCB production when an excess of Fd+FNR is present in the cytoplasm. Importantly, PCB production was not influenced by excess heme when enzymes were localized to the mitochondria ( <b>M4</b>–heme and <b>M4</b>+heme, lanes 6 and 12). This confirms that Fd+FNR is primarily limiting in both the cytoplasm and the mitochondria and that heme is secondarily limiting only in the cytoplasm. </p> <p>To further investigate the PCB production dependence on Fd, we transfected cells with two, three or all four enzymes in the pathway: PcyA-HO1 ( <b>M2</b>), PcyA+HO1+Fd ( <b>M3</b>), or PcyA+HO1+Fd+FNR ( <b>M4</b>), along with PhyB for all conditions ( <a href="#fig2">Figure </a><a href="#fig2">2</a> B). We show in <a href="#fig2">Figure </a><a href="#fig2">2</a> B that the addition of Fd to PcyA+HO1 ( <b>M3</b>) significantly increased PCB production compared to PcyA+HO1 alone ( <b>M2</b>) ( <i>p</i> &amp;lt; 0.05). Importantly, Fd+FNR ( <b>M4</b>) produces significantly more PCB than adding Fd alone ( <i>p</i> &amp;lt; 0.01), demonstrating that for maximum PCB production both Fd and FNR are required. </p> <p>While we considered testing the overexpression of the host cell’s Fd+FNR, there are noteworthy advantages to using orthogonal Fd+FNR matching the species of the transplanted metabolic pathway. The mammalian Fd+FNR may be able to reduce BV bound to PcyA but only at a fraction of the rate of the cyanobacterial Fd+FNR. The required overexpression needed for the host cell’s system to perform at the same production rate would therefore more likely disturb the cell’s metabolism. Using an orthogonal system would be more efficient and would also less likely interact with the host cell’s metabolic proteins. Matching the orthogonal enzyme species thus allows for minimal perturbation of the normal host cell physiology and at the same time maximize production rates. </p> </div> <div class="effectsofpcya,ho1andfd+fnrstoichiometryonpcbproductionlevels" title="sec"> <div class="title" tagx="title" title="title">
Effects of PcyA, HO1 and Fd+FNR Stoichiometry on PCB Production Levels </div> <p>Okada <i>et al.</i>( <a href="#ref43">43</a>) demonstrated that Fd forms stable complexes with both HO1 and PcyA. Therefore, we hypothesized that PCB production may be further optimized through enzyme stoichiometry. We transfected separate PcyA+HO1 and Fd+FNR plasmids at different ratios and observed that PCB production was highly dependent on the ratio between PcyA+HO1 and Fd+FNR ( <a href="#fig3">Figure </a><a href="#fig3">3</a> A). Considering this, to serve as a quantitative guide for optimizing PCB production, we developed computational models of this pathway using coupled ordinary differential equations (model details in <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Supporting Information</a>). We tested the enzyme stoichiometry using a functional PhyB-PIF3 luciferase gene expression system adapted from Shimizu Sato <i>et al.</i>( <a href="#ref5">5</a>) ( <a href="#fig3">Figure </a><a href="#fig3">3</a> B). First, we used optimized versions of the PhyB-PIF3 switch, including optimizing DNA binding domains ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Figure S2</a>), activation domains ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Figure S3</a>), and reporter constructs ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Figure S4</a>). Next, the stoichiometry was tested by transfecting different ratios of the PcyA+HO1 and Fd+FNR plasmids and illuminating the cells with red light for 24 h (timeline of illumination as shown in <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Figure S3A</a>), followed by a luciferase assay to compare gene induction levels. We found that gene activation levels were also highly dependent on enzyme stoichiometry, with only the 17:1 PcyA+HO1:Fd+FNR showing any measurable response to light ( <a href="#fig3">Figure </a><a href="#fig3">3</a> C and <a href="#fig3">3</a>D, <i>p</i> &amp;lt; 0.01). This demonstrates how chromophore levels influence the performance of PhyB optogenetic systems. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Stoichiometry of PCB production constructs. (A) PCB production assay comparing plasmid ratios of pcyA+ho1 to fd+fnr using the plasmids shown. Transfection ratios are indicated in boxes below the Western blot. PCB production was measured by covalent linkage to PhyB followed by immunoprecipitation with anti-HA, Zn-PAGE and Western blots. (B) Schematic of the PhyB-PIF3 light switch. PhyB is fused to a DNA Binding Domain (DBD) and bound to a light-sensitive chromophore (PCB). The PhyB-DBD fusion remains bound to the UAS promoter. PIF3 is fused to an Activation Domain (AD). Upon absorption of a red photon (660 nm), PhyB changes conformation and recruits PIF3 to the promoter region. The AD fused to PIF3 then activates the gene downstream of the promoter. Upon absorption of a far-red photon (735 nm), PhyB changes conformation that leads to PIF3 unbinding, removing the AD from the promoter, shutting the downstream gene off. (C) Plasmid maps for endogenous PCB production and PhyB-PIF3 light switchable promoter. (D) Luciferase gene activation levels using endogenously produced PCB with several ratios of pcyA+ho1:petF+petH ( <i>n</i> = 3). (E) Three construct designs consisting of all four biosynthetic enzymes on a single plasmid and a single plasmid for PIF3 and PhyB. (F) Testing gene activation comparing single plasmid biosynthetic plasmids ( <i>n</i> = 7). ho1 = heme oxygenase, pcyA = Phycocyanobilin:ferredoxin oxidoreductase, petF = ferredoxin, petH = ferredoxin:oxidoreductase/FNR, MTS = Mitochondrial Targeting Sequence, P2A = 2A self-cleaving peptide, NLS = Nuclear Localization Sequence, IRES = Internal Ribosome Entry Site, AD = Activation Domain, DBD = DNA Binding Domain, R/FR = Red light/Far-red light. Error bars = Standard Deviation, (*) = <i>p</i> &amp;lt; 0.05, (**) = <i>p</i> &amp;lt; 0.01. Statistics were calculated using one-way ANOVA with Bonferroni post-test using GraphPad Prism 5.01. <i>n</i> = individual experiments. </p>   </div> </div> <div class="mammalianphyb-pifgeneswitchusingendogenouslyproducedpcb" title="sec"> <div class="title" tagx="title" title="title">
Mammalian PhyB-PIF Gene Switch Using Endogenously Produced PCB</div> <p>After identifying the requirements for high levels of endogenous PCB production, we sought to encode all four biosynthetic enzymes on a single plasmid. Our original four enzyme plasmid (pPKm-245) contained all PCB biosynthetic enzymes separated by P2A sequences to achieve a 1:1:1:1 expression level of each enzyme. <sup> <a href="#ref44">44</a> </sup> However, the results in <a href="#fig3">Figures </a><a href="#fig3">3</a> A–D suggested that PCB production could be further optimized by modifying the plasmid’s expression stoichiometry. To this end, we replaced one of the P2A sequences with an Internal Ribosomal Entry Site (IRES), which typically gives 1 order of magnitude lower expression to the gene following the IRES sequence. <sup> <a href="#ref45">45</a>− <a href="#ref47">47</a> </sup> The plasmid pPKm-244 was generated by placing an IRES between <i>pcyA</i> and <i>Fd</i>, leading to higher PcyA-HO1 levels and lower Fd+FNR levels ( <a href="#fig3">Figure </a><a href="#fig3">3</a> E). We also constructed a plasmid, pPKm-248, containing <i>HO1</i>, <i>Fd</i>, and <i>FNR</i> all placed after the IRES sequence. This plasmid results in minimized heme oxygenase and Fd+FNR activity while keeping higher levels of PcyA ( <a href="#fig3">Figure </a><a href="#fig3">3</a> E). Using the experiment timeline in <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Figure S3A</a>, we found that lowering HO1 and Fd+FNR levels with the pPKm-248 plasmid produced 1.8-fold ( <i>p</i> &amp;lt; 0.05) and 2.2-fold ( <i>p</i> &amp;lt; 0.01) higher gene activation levels than pPKm-244 and pPKm-245 respectively ( <a href="#fig3">Figure </a><a href="#fig3">3</a> F). In addition to producing more PCB, lower expression of HO1, Fd and FNR should provide maximal PCB levels with minimal interference in the host cells metabolism. </p> </div> <div class="lightsensitivityofthemammalianphyb-pif3geneswitchusingendogenouslyproducedpcb" title="sec"> <div class="title" tagx="title" title="title">
Light Sensitivity of the Mammalian PhyB-PIF3 Gene Switch Using Endogenously Produced PCB </div> <p>PhyB-PIF optogenetic systems in animal cells have mostly been characterized in conditions where PCB is added externally. However, PCB degrades rapidly in cell culture media, <sup> <a href="#ref7">7</a> </sup> which affects PhyB’s light sensitivity over long time spans. <sup> <a href="#ref48">48</a> </sup> Since our constructs enable constant endogenous production of PCB, we sought to test the light sensitivity of the PhyB-PIF3 switch (pPKm-230) with the endogenously produced chromophore. We illuminated transfected cells with the activating red light, at different intensities for 24 h, and found that light intensities of 1.00 μmol/m <sup>2</sup>/s, 0.1 μmol/m <sup>2</sup>/s, and 0.01 μmol/m <sup>2</sup>/s achieved similar high levels of gene activation ( <a href="#fig4">Figure </a><a href="#fig4">4</a> B and <a href="#fig4">4</a>D). In contrast, transfected cells illuminated with a light intensity of 0.001 μmol/m <sup>2</sup>/s had a significantly lower gene response ( <i>p</i> &amp;lt; 0.05). Since the system is bistable, <sup> <a href="#ref33">33</a> </sup> we reasoned that activating with intensities between 1.0 and 0.01 μmol/m <sup>2</sup>/s, which activate the system over a long time span (24 h), may not represent saturating amounts of light for shorter illumination times. <sup> <a href="#ref49">49</a> </sup> To test this hypothesis, we characterized the gene switch using these same light intensities, but with a single 1 min pulse of red light ( <a href="#fig4">Figure </a><a href="#fig4">4</a> C and <a href="#fig4">4</a>E). Unlike the 24-h illumination experiment, we found that when we illuminated the cells with red light for 1 min, light intensities of 0.1 μmol/m <sup>2</sup>/s and 0.01 μmol/m <sup>2</sup>/s had a significantly lower gene response than an intensity of 1.0 μmol/m <sup>2</sup>/s ( <i>p</i> &amp;lt; 0.001). This finding highlight that for characterizing these light responsive bistable proteins, we should consider both the light intensity and duration of illumination. For example, our results using 0.1 μmol/m <sup>2</sup>/s and 0.01 μmol/m <sup>2</sup>/s show that those intensities are not saturating with a 1 min pulse, but those same intensities induce saturating activation levels over 24 h ( <a href="#fig4">Figure </a><a href="#fig4">4</a> D and <a href="#fig4">4</a>E). This is expected from a system that is bistable with a long-lived activation state, <sup> <a href="#ref49">49</a> </sup> inactive molecules not activated in the first minute will be activated later if light is continuously applied, eventually activating all of the light-sensitive molecules. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Light sensitivity of the genetically encoded PhyB-PIF3 switch. (A) Plasmids optimized for an endogenous PhyB-PIF3 light switchable promoter. (B) Pulsing program for 24-h illumination experiments. (C) Pulsing program for 1 min illumination experiments. (D) Gene response to a 24-h pulse with several light intensities ( <i>n</i> = 4). (E) Gene response to a 1 min pulse with several light intensities ( <i>n</i> = 4). (F) Gene activation responses using 1 μmol/m <sup>2</sup>/sec or 0.1 μmol/m <sup>2</sup>/sec of continuous light compared with using 0.1 μmol light at different pulse intervals for 24 h ( <i>n</i> = 3). The blue stars indicate the minimal light dose for saturating activation using 24-h illuminations. (G) Pulsing program for testing the duration of activation. Pulsing was done as in B. (H) Gene response to pulsing at increasing intervals. Cells were pulsed for 1 min using 1 μmol/m <sup>2</sup>/sec 660 nm light, followed by darkness for the indicated times for a total of 24 h ( <i>n</i> = 5). The blue arrows indicate the minimal light dose for saturating activation using 24 h illuminations. (I) Total light flux during 24 h period of illumination for experiments in <a href="#fig4">Figure </a><a href="#fig4">4</a> D and <a href="#fig4">Figure </a><a href="#fig4">4</a> H. Cont. = continuous illumination, 1 min/4 min = 1 min red light, 4 min darkness, 1 min/9 min = 1 min red light, 9 min darkness, 1 min/29 min = 1 min red light, 29 min darkness. ho1 = heme oxygenase, pcyA = Phycocyanobilin:ferredoxin oxidoreductase, petF = ferredoxin, petH = ferredoxin:oxidoreductase/FNR IRES = Internal Ribosome Entry Site, MTS = Mitochondrial Targeting Sequence, NLS = Nuclear Localization Sequence, P2A = 2A self-cleaving peptide, AD = Activation Domain, DBD = DNA Binding Domain, R/FR = Red light/Far-red light. Error bars = Standard Deviation, (*) = <i>p</i> &amp;lt; 0.05, (***) = <i>p</i> &amp;lt; 0.001. Statistics were calculated using one-way ANOVA with Bonferroni post-test using GraphPad Prism 5.01. <i>n</i> = individual experiments. </p>   </div> </div> <div class="endogenousmammalianphyb-pif3geneswitchbistabilityandreversibilitywithfar-redlight" title="sec"> <div class="title" tagx="title" title="title">
Endogenous Mammalian PhyB-PIF3 Gene Switch Bistability and Reversibility with Far-Red Light </div> <p>We further tested the light sensitivity and bistability by shining activating red light at different pulse intervals ( <a href="#fig4">Figure </a><a href="#fig4">4</a> F). As controls, we illuminated HEK293 cells with continuous 1.0 μmol/m <sup>2</sup>/s or 0.1 μmol/m <sup>2</sup>/s red light for 24 h and found they reach similar levels of gene activation. In addition to continuous illumination, we utilized alternating light/dark cycles composed of 1 min of red light and 4, 9, or 29 min of darkness (1 min/4 min, 1 min/9 min, 1 min/29 min respectively) for 24 h. Continuous red light at 0.1 μmol/m <sup>2</sup>/s, as well as the 1 min/4 min and 1 min/9 min conditions, did not produce statistically different activation levels ( <a href="#fig4">Figure </a><a href="#fig4">4</a> F). In contrast, the condition with 0.1 μmol/m <sup>2</sup>/s of red light pulsed at 1 min/29 min had significantly lower activation levels than continuous light and pulsed light in the 1 min/4 min and 1 min/9 min conditions ( <a href="#fig4">Figure </a><a href="#fig4">4</a> F, <i>p</i> &amp;lt; 0.05). Because the 1 min/9 min (blue star) condition has one-tenth the number of photons as 0.1 μmol/m <sup>2</sup>/s in total photon flux, it is equivalent in the number of photons to 0.01 μmol/m <sup>2</sup>/s of continuous illumination or 183 nW/cm <sup>2</sup> for 660 nm red light. This agrees with the result where the same total amount of light is applied continuously, suggesting that the activation state of PhyB is much longer than the 9 min dark interval ( <a href="#fig4">Figure </a><a href="#fig4">4</a> D and <a href="#fig4">4</a>F). </p> <p>Interestingly, we also found that cells containing the PhyB-PIF3 system had a slightly higher level of gene activation in the darkness than cells in the presence of far-red light, potentially due to the bistability of the protein ( <a href="#fig4">Figure </a><a href="#fig4">4</a> F). Thermodynamically, in darkness, a mixed population of species (Pf and Pfr forms) is the expected nature of a bistable molecule, since some PhyB molecules can spontaneously switch to the "activated state". Therefore, the proportion of activated PhyB molecules should be higher in darkness than when PhyB is illuminated with a deactivating far-red light. </p> <p>Since pulsing the light on a minute time scale achieved similar levels of activation as continuous light ( <a href="#fig4">Figure </a><a href="#fig4">4</a> F), we decided to test the duration of the activated state of PCB bound PhyB (PhyB·PCB) by increasing the spacing between red light pulses as shown in <a href="#fig4">Figure </a><a href="#fig4">4</a> G. Our results show similar levels of gene activation for red light pulses delivered for 1 min every 8, 6, 4, 2, 1 h, and a half hour at 1 μmol/m <sup>2</sup>/s ( <a href="#fig4">Figure </a><a href="#fig4">4</a> H). However, a pulse delivered every 12 h (a total of two pulses in the 24 h period) produced significantly lower gene activation than the pulses delivered in the shorter intervals ( <a href="#fig4">Figure </a><a href="#fig4">4</a> H). It is possible that those two pulses in the 24-h period delivered too little total amount of light to fully activate the system ( <a href="#fig4">Figure </a><a href="#fig4">4</a> I). However, this data still supports that the switch effectively stays "on" for at least 8 h following a 1 min pulse of 1 μmol/m <sup>2</sup>/s of red light ( <a href="#fig4">Figure </a><a href="#fig4">4</a> H, blue arrow). In terms of total light delivery (μmol/m <sup>2</sup>), the 1 min pulses every 8 h using 1.0 μmol/m <sup>2</sup>/s is effectively equivalent to the number of photons with continuous light at 0.0021 μmol/m <sup>2</sup>/s or 38nW/cm <sup>2</sup> for 660 nm light, which is a strikingly small amount of light and speaks to the high sensitivity of this system. </p> <p>One hallmark of PhyB based optogenetic switches is their conformational reversibility upon absorption of another photon of a different wavelength. <sup> <a href="#ref33">33</a> </sup> While the ability for PCB bound PhyB (PhyB·PCB) to isomerize upon red light absorption and reverse upon far-red light absorption has been previously shown, <sup> <a href="#ref5">5</a> </sup> whether the PhyB(1–621)-DBD and PIF3(1–524)-AD interaction was reversible by far-red light when expressed in mammalian cells has not been tested. <sup> <a href="#ref13">13</a>, <a href="#ref50">50</a> </sup> To test the reversibility of the switch, we exposed HEK293 cells, transfected with the PhyB-PIF3 switch and endogenously producing PCB constructs ( <a href="#fig5">Figure </a><a href="#fig5">5</a> A), to either 24 h of red light, 12 h of red light followed by 12 h of darkness, or 12 h of red light followed by 12 h of far-red light ( <a href="#fig5">Figure </a><a href="#fig5">5</a> B). Luciferase expression was significantly lower in cells shifted into darkness after 12 h of continuous red light than cells exposed to 24 h of light ( <i>p</i> &amp;lt; 0.05), indicating PhyB reversed to its inactive state once red-light illumination ended. Compared to switching from red light to darkness, switching from red to far-red light showed significantly lower luciferase expression, indicating that the far-red light inactivated the gene switch (red box, <i>p</i> &amp;lt; 0.05). This result indicates that after red light activation, the switch remains on for some time in the darkness and that it can be switched off with far-red light. This finding has important implications for the switch’s ability to control genes since it shows that the gene expression levels can be titrated temporally by timing the duration of red light or by red light followed by far-red light. Thus, this system can be used for spatial control by patterning red and far-red light for targeted localization of gene activation. <sup> <a href="#ref34">34</a> </sup> </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>PhyB-PIF3 light switch bistability and reversibility with far-red light and performance in several cell types. (A) Plasmids optimized for an endogenous PhyB-PIF3 light switchable promoter. (B) Testing the reversibility of the PhyB-PIF3 light-switchable promoter in mammalian cells. Cells were in darkness, illuminated with 735 nm far-red light, 660 nm red light for 24 h, or with 12 h or red light followed by darkness or followed by far-red light ( <i>n</i> = 3). (C) Testing the PhyB-PIF3 light switch in four different cell types. Cells were transfected, then illuminated with red light for 24 h as shown in <a href="#fig3">Figure </a><a href="#fig3">3</a> C ( <i>n</i> = 4). ho1 = heme oxygenase, pcyA = phycocyanobilin:ferredoxin oxidoreductase, petF = ferredoxin, petH = ferredoxin:oxidoreductase/fnr, IRES = Internal Ribosome Entry Site, MTS = Mitochondrial Targeting Sequence, NLS = Nuclear Localization Sequence, P2A = 2A self-cleaving peptide, AD = Activation Domain, DBD = DNA Binding Domain, R/FR = Red light/Far-red light. Error bars = s.d., (*) = <i>p</i> &amp;lt; 0.05, Statistics were calculated using one-way ANOVA with Bonferroni post-test using GraphPad Prism 5.01. <i>n</i> = individual experiments. </p>   </div> </div> <div class="geneticallyencodedphyb-pif3geneswitchinseveralmammaliancelllines" title="sec"> <div class="title" tagx="title" title="title">
Genetically Encoded PhyB-PIF3 Gene Switch in Several Mammalian Cell Lines </div> <p>We also tested the PhyB-PIF3 gene switch performance in different cell types containing endogenously produced PCB. We transfected HEK293, hepatocellular carcinoma (HUH-7), HeLa, and mouse fibroblasts (3T3) cells with the PhyB-PIF3 gene switch and HO1+PcyA+Fd+FNR plasmids (pPKm-230 and pPKm-248, respectively). We used 1 μmol/m <sup>2</sup>/s of red light illumination in a cycle composed of 1 min pulses of red light followed by 4 min of darkness, for a duration of 24 h ( <a href="#fig5">Figure </a><a href="#fig5">5</a> C). The PhyB-PIF3 switch with endogenously produced PCB activated luciferase about 280-fold in HEK293 cells, 70-fold in HUH-7 cells, 300-fold in HeLa cells and 440-fold in 3T3 cells. These findings show that the system is effective in producing PCB and activating different mammalian cell types. While we have highly optimized the PhyB-PIF3 light switch with endogenously produced PCB, there are several ways to customize the levels of activation or leakiness to tailor it to specific cell types and applications. For example, different activation or repression domains could be used ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Figure S3</a>). In addition, there are still other permutations of gene fusions that can be tested in future studies that may further enhance this system, such as using a DBD on the N-terminus of PhyB or optimizing linker sequences. Using a stronger or tissue-specific promoter to drive expression of PCB or PΦB biosynthetic enzymes may also lead to higher activation levels or can restrict light sensitivity to specific cell types. <sup> <a href="#ref51">51</a> </sup> As presented in this research, using wavelengths that are optimal for tissue penetration, <sup> <a href="#ref10">10</a>, <a href="#ref12">12</a> </sup> the PhyB(1–621)-PIF3 gene switch with endogenously produced PCB is among the most light-sensitive optogenetic switches. </p> </div> </div> <div class="summary" title="sec"> <div class="title" tagx="title" title="title">
Summary</div> <p>We have shown that the Fd+FNR system is the rate-limiting factor for the production of the chromophores PCB and PΦB in the mitochondria of mammalian cells, and is limited by the Fd+FNR system followed by heme in the cytoplasm. The ability to produce PCB and PΦB with PcyA and HY2, respectively, suggests that matching reduction systems that efficiently supply electrons to a metabolic pathway can also enhance the production of other bilins and other classes of molecules. This finding creates new opportunities for engineering synthetic systems to produce these chromophores, along with many other molecules. This has potential industrial applications in decreasing costs of crop production, producing plant molecules in microbes, or delivering therapeutic molecules <i>via</i> genetically encoded pathways. </p> <p>Genetically encoding endogenous production of chromophores like PCB also enables the use of several existing and compatible optogenetic tools to regulate cell signaling, <sup> <a href="#ref13">13</a>, <a href="#ref52">52</a> </sup> cell migration, <sup> <a href="#ref13">13</a> </sup> or protein localization <sup> <a href="#ref13">13</a> </sup> without the addition of exogenous chemicals. This makes possible the use of PhyB when constant levels of PCB are required, facilitating potential <i>in vivo</i> applications, or when the addition of PCB to samples is not practical (such as when samples are in a sealed container or for long illumination times). This study achieves the long-sought goals in optogenetics of enabling high-level production of the chromophores PCB and PΦB in mammalian cells and demonstrates a more general method for efficiently producing molecules from one species in another. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="zinc-page-immunoprecipitationassays" title="sec"> <div class="title" tagx="title" title="title">
Zinc-PAGE-Immunoprecipitation Assays </div> <p>Protein G PLUS-Agarose (ThermoFisher, 22851) beads were prepared by adding 200 μg anti-HA (clone HA-7, Sigma H9658) into 2 mL 25% agarose. After overnight binding at 4 °C, unbound anti-HA was washed off four times with 1× Phosphate-buffered Saline (PBS, pH 7.4, ThermoFisher, 10010023). For each 6-well plate, 500 000 HEK293 cells (ATCC, CRL-1573) were transfected using 2.5 μg DNA and 6 μL of Lipofectamine 2000 per well (ThermoFisher Scientific, 11668019). For heme experiments, media or media containing 10 μM heme (Frontier Scientific, H651–9), was exchanged 18 h after transfection and again 43 h after transfection. Heme was dissolved at 10 mM in 100 mM NaOH and sterile filtered with a 0.22 μM filter (Millipore, SLGP033RS). Cells were then harvested with RIPA buffer (1% Triton X-100, 0.5% Sodium Deoxycholate, 25 mM Tris pH 8.0, 150 mM NaCl, 0.10% SDS and 2.5 mM EDTA, and 2× protease inhibitors (Sigma, P8340–1 ML)), immediately placed on ice, sonicated briefly and then centrifuged for 30 min at 21 000 <i>g</i>. BCA assays (ThermoFisher Scientific, 23225) were used to determine the protein concentration of resulting supernatant/lysates. Equal masses for each protein sample were diluted with two parts of cold PBS, then loaded onto Protein G PLUS-Agarose beads containing anti-HA (preparation above), for overnight binding while mixing at 4 °C. Next beads were washed and boiled in sample buffer (30% glycerol, 10% SDS, 300 mM Tris pH 6.8, 0.03% Bromophenol Blue, 179 mM 2-Mercaptoethanol). After loading and running the samples in a SDS-PAGE gel, the gels were incubated in SDS-PAGE Running Buffer (25 mM Tris, 192 mM glycine, 0.1% SDS) containing 10 mM Zinc Acetate for 10 min prior to imaging in a Fluorochem E (Protein Simple). Gels were then transferred onto nitrocellulose and probed with the primary antibody anti-HA 1:5000 (Sigma, clone HA-7, H9658), and by Goat anti-Mouse secondary antibody 1:5000 (ThermoFisher, 32230). Western blots were imaged in a Fluorochem E (Protein Simple). Gel bands were quantified using the FIJI (ImageJ) gel analysis tool. <sup> <a href="#ref53">53</a> </sup> </p> </div> <div class="imagingpcbproduction" title="sec"> <div class="title" tagx="title" title="title">
Imaging PCB Production</div> <p>HEK293 cells (ATCC, CRL-1573), plated at 100 000 cells per well in a 24-well plate, were transfected 24 h after plating on polylysine (Sigma P6407–5 mg) coated coverslips in each well. 43 h later, the media was exchanged with fresh media or media+5 μM PCB (Frontier Scientific, P14137) for the NE+PCB control. One hour later, cells were rinsed in 1× PBS and then fixed in 4% Paraformaldehyde in 1× PBS for 10 min. Cells were then washed with 1× PBS before incubating in permeabilization buffer (5% BSA + 0.3% TritonX-100 in PBS) for 30 min, followed by incubating with primary antibodies, anti-FLAG mouse monoclonal 1:1000 (Sigma, F3165) and polyclonal anti-HA rabbit 1:500 (Santa Cruz, Y-11) in antibody buffer (2% BSA + 0.2% TritonX-100 in PBS) at 4 °C overnight. Next coverslips were rinsed twice and washed three times in 1× PBS and then incubated in antibody buffer containing goat anti-mouse AlexaFluor 488 1:1000 (ThermoFisher, A11001), and goat anti-rabbit AlexaFluor 568 1:1000 (ThermoFisher, A11011). Coverslips were rinsed and washed again, then mounted with Fluoromount-G (SouthernBiotech, 0100–20). Images were taken using a DeltaVision RT Deconvolution Microscope. </p> </div> <div class="cellculture,transfection,lightinductionandreportergeneassays" title="sec"> <div class="title" tagx="title" title="title">
Cell Culture, Transfection, Light Induction and Reporter Gene Assays </div> <p>Human Embryonic Kidney 293 cells (HEK293, ATCC CRL-1573) were cultivated in Dulbecco’s Modified Eagle Medium (DMEM, Gibco, 11965–092) supplemented with 10% fetal bovine serum (FBS, Omega Scientific, FB-02) and 100 U/mL of penicillin and 0.1 mg/mL of streptomycin (Gibco, 11548876). All cells were cultured under 5% CO <sub>2</sub> at 37 °C. Cells were seeded at 100 000 HEK293 cells per well in 24-well plates, 24 h before transfection. Transfection of plasmids was achieved through lipofection following the manufacturer’s instructions and protocol (Lipofectamine 2000, ThermoFisher, 11668019). For each transfection reaction, a total of 0.5 μg of plasmid DNA was combined with specific plasmid ratios for each experiment as detailed in <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Table S2</a>. A construct with Renilla luciferase reporter plasmid DNA was included as an internal transfection control in all transfections. The culture medium was replaced with fresh medium 24 h after transfection and the plates were placed inside black boxes (Hammond Manufacturing Company, 1591ESBK) for the remainder of the experimental procedure. For conditions where external PCB is added, 15 μM of PCB (Frontier Scientific, P14137) from a 20 mM stock dissolved in DMSO (Santa Cruz Biotechnology, sc-202581) was supplemented in fresh medium 24 h after transfection ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Figure S2A</a>). </p> <p>Light induction was programmed to start 12 h after medium replacement. Each black box was equipped with a circuit consisting of six red LEDs (660 nm, Thorlabs, M660L3), except for the dark boxes and far-red boxes which had no LEDs or a single far-red LED (735 nm, Thorlabs, M735L2), respectively. In addition, each black box circuit was designed to allow for fine adjustment of light intensity (circuitry is shown <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Figure S5</a>), from 0.0008 to 200 μmol/m <sup>2</sup>/s. Light intensity was measured in μW at the cell level, converted to μmol/m <sup>2</sup>/s (light sensor area = 63.6 mm <sup>2</sup>), and adjusted for each experiment design using Sper Scientific Direct’s Laser Power Meter (SSD, 8400). Detailed information on wavelengths, illumination intensity, and duration used for each experimental procedure and data shown are detailed in <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Table S6</a>. Pulse duration and total illumination times were electronically controlled <i>via</i> a LabVIEW computer driving an Arduino microprocessor and custom-made circuits (see <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Supporting Information</a>). </p> </div> <div class="luciferaseactivityassay" title="sec"> <div class="title" tagx="title" title="title">
Luciferase Activity Assay </div> <p>Luciferase assays were carried out using the Dual-Luciferase Assay system (Promega, PRE1960), and following the manufacturer’s protocol. Cells were lysed immediately after removing from the incubator using the manufacturer’s instructions. Firefly and Renilla Luciferase activities were measured from cell lysates using the luminometer module of the Infinite 200 PRO multimode reader (Tecan). Results of luciferase activity assays are expressed as a ratio of firefly luciferase (Fluc) activity to Renilla luciferase (Rluc) activity. </p> </div> <div class="illuminationcircuitsandsoftware" title="sec"> <div class="title" tagx="title" title="title">
Illumination Circuits and Software </div> <p>The light control system employs an Arduino Uno and a light intensity control circuit ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Figure S6</a>) driven by a user interface developed in LabVIEW (National Instruments) to control each box’s LED intensity ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Figure S5</a>). This system is ideal for precise timing and light-intensity control of each experimental box while allowing for user-determined experimental start delay, illumination frequencies, and control of the total duration of the experiment. <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Supporting Information</a> contains a full description of the illumination apparatus, user interface, and circuitry. </p> </div> <div class="kineticmodel" title="sec"> <div class="title" tagx="title" title="title">
Kinetic Model </div> <p>Using PySB, <sup> <a href="#ref54">54</a> </sup> we generated an <i>in silico</i> model to describe the biochemical interactions among the enzymes that compose the hypothesized PCB-production pathway, as seen in <a href="#fig1">Figure </a><a href="#fig1">1</a> A. The quantitative mathematical model was parametrized ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Table S4</a>) by experimental data and uses ordinary differential equations to describe the changes in the concentration of the molecular components of the reaction. We probed the proposed model directly as proposed in the literature and similar pathways published. <sup> <a href="#ref17">17</a>, <a href="#ref25">25</a>, <a href="#ref43">43</a> </sup> We complement this work showing the model’s agreement with the tested pathway, demonstrating how heme, Fd, and FNR are rate limiting factors for the production of PCB, as confirmed experimentally in <a href="#fig1">Figures </a><a href="#fig1">1</a> , <a href="#fig2">2</a> and <a href="#fig3">3</a>. A full description of the kinetic model can be found in <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">Supporting Information</a>. </p> </div> <div class="marvin" title="sec"> <div class="title" tagx="title" title="title">
Marvin</div> <p>Marvin was used for drawing and displaying chemical structures in <a href="#fig1">Figure </a><a href="#fig1">1</a> A and Graphical Abstract, Marvin 17.28.0, 2017, ChemAxon ( <span class="uri" title="uri"> <a href="http://www.chemaxon.com">http://www.chemaxon.com</a> </span>). </p> </div> </div> </div> <div class="back" title="back"> <div class="title" tagx="title" title="title">
Supporting Information Available</div> <p>The Supporting Information is available free of charge on the <a href="http://pubs.acs.org">ACS Publications website</a> at DOI: <a href="http://pubs.acs.org/doi/abs/10.1021/acssynbio.7b00413">10.1021/acssynbio.7b00413</a>. </p><ul> <li> <p>Illumination circuits and software; Kinetic model development and parametrization; Table S1: Similarity tables for ferredoxin and ferredoxin-dependent bilin reductases; Table S2: Plasmids used in this study; Table S3: Transfection and illumination details for of the study’s figures; Table S4: Parameters for the model; Figure S1: Imaging PCB production in mammalian cells; Figure S2: Optimizing PhyB and PIF light switches for mammalian cells; Figure S3: Optimizing PhyB and PIF gene switch; Figure S4: Comparing reporter constructs; Figure S5: Illumination setup; Figure S6: Circuit design for LED illumination; Figure S7: Model results (Initial Heme and Fd activation levels); Figure S8: Model results (Fd dependence and Heme dependence) ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acssynbio.7b00413/suppl_file/sb7b00413_si_001.pdf">PDF</a>) </p> </li> </ul> <p /> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="sb7b00413_si_001.pdf">LINK</a>  <p>sb7b00413_si_001.pdf</p>  </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p> <sup>¶</sup> P.K. and M.C. contributed equally to this work. P.K. conceived the original concept of mitochondrial targeting for connecting PCYA-HO1 for Fd-FNR and using it to genetically encode the PhyB-PIF3 gene switch. P.K., M.C., N.H., and T.P.C. initiated this project. M.C., W.T., L.N., and V.J.H. performed light induction experiments. P.K., M.C., N.H., V.P., C.S., L.E.D., A.H., and L.N., cloned the plasmids. M.C. modeled the metabolic pathways. M.C. and V.J.H. designed and built illumination apparatus and software. P.K. ran Zn-PAGE experiments; P.K. and L.E.D. imaged PCB in cells. G.P. provided expertise on experimental design and critical feedback. P.K., M.C., W.T., T.P.C., N.H., and G.P. wrote the manuscript with input from other authors. </p> <p>The authors declare no competing financial interest. </p> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>We acknowledge Wilfried Weber for his kind gift of plasmids (pKM-087, pKM-022, and pMZ-802), William McGinnis for his kind gift of the Fluc minimal promoter, Steven P. Briggs for the MTAD construct, Peter Quail for PhyB and PIF3 genes, and Alexey Veraksa for helping initiate this project with P.K. Supported by the Kavli Institute for Brain and Mind at UC San Diego and the Salk Institute, National Science Foundation through the NSF Center for Science of Information under Grant CCF-0939370, NIH Grant NS060847, UCSD School of Medicine Microscopy Core Grant P30 NS047101. P.K. was supported by Training in Multiscale Analysis of Biological Structure and Function, Training Grant: NIH grant T32 EB009380 and by the UCSD Cellular and Molecular Genetics Training Program through an institutional grant from the National Institute of General Medicine (T32 GM007240). M.C. was a fellow of the AAUW (American Association of University Women) 2015 International Fellowship. </p> </div> <div class="glossary" title="glossary"> <div class="def-list" title="def-list"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-item" title="def-item"> <div class="term" title="term">
PCB</div> <div class="def" title="def"> <p>phycocyanobilin</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PΦB</div> <div class="def" title="def"> <p>phytochromobilin</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Fd</div> <div class="def" title="def"> <p>ferredoxin</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
FNR</div> <div class="def" title="def"> <p>ferredoxin-NADP+-reductase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Fd+FNR</div> <div class="def" title="def"> <p>endogenous oxidation–reduction system containing Fd and FNR </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PcyA</div> <div class="def" title="def"> <p>PcyAphycocyanobilin:phycocyanobilin:ferredoxin oxidoreductase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HY2</div> <div class="def" title="def"> <p>phytochromobilin:ferredoxin oxidoreductase </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BV</div> <div class="def" title="def"> <p>biliverdin IX-alpha</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HO1</div> <div class="def" title="def"> <p>heme oxygenase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PhyB</div> <div class="def" title="def"> <p>phytochrome B</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PIF</div> <div class="def" title="def"> <p>phytochrome-interacting factor</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PhyB·PCB</div> <div class="def" title="def"> <p>PCB-bound PhyB</p> </div> </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="ref1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaspar</span> <span class="given-names" tagx="given-names" title="given-names">H. B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cooray</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gilmour</span> <span class="given-names" tagx="given-names" title="given-names">K. C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parsley</span> <span class="given-names" tagx="given-names" title="given-names">K. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adams</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bjorkegren</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bayford</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davies</span> <span class="given-names" tagx="given-names" title="given-names">E. G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Veys</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairbanks</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bordon</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petropoulou</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kinnon</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thrasher</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Hematopoietic stem cell gene therapy for adenosine deaminase-deficient severe combined immunodeficiency leads to long-term immunological recovery and metabolic correction </span>. <span class="source" tagx="source" title="source">Sci. Transl. Med.</span> <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">97ra80</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/scitranslmed.3002716">10.1126/scitranslmed.3002716</a> </span>. </span> </li> <li tag="ref"> <a name="ref2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">X. Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morreau</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rottier</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bonten</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gillemans</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wenger</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grosveld</span> <span class="given-names" tagx="given-names" title="given-names">F. G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doherty</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzuki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grosveld</span> <span class="given-names" tagx="given-names" title="given-names">G. C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">d Azzo</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> ( <span class="year" tagx="year" title="year">1995</span>) <span class="mixed-article-title" title="mixed-article-title">Mouse model for the lysosomal disorder galactosialidosis and correction of the phenotype with overexpressing erythroid precursor cells </span>. <span class="source" tagx="source" title="source">Genes Dev.</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">2623</span>– <span class="lpage" tagx="lpage" title="lpage">2634</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gad.9.21.2623">10.1101/gad.9.21.2623</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7590240">7590240</a> </span> </span> </li> <li tag="ref"> <a name="ref3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burén</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">E. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sweeny</span> <span class="given-names" tagx="given-names" title="given-names">E. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lopez-Torrejón</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Veldhuizen</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voigt</span> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rubio</span> <span class="given-names" tagx="given-names" title="given-names">L. M.</span> </span> ( <span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Formation of Nitrogenase NifDK Tetramers in the Mitochondria of Saccharomyces cerevisiae </span>. <span class="source" tagx="source" title="source">ACS Synth. Biol.</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">1043</span>– <span class="lpage" tagx="lpage" title="lpage">1055</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acssynbio.6b00371">10.1021/acssynbio.6b00371</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28221768">28221768</a> </span> </span> </li> <li tag="ref"> <a name="ref4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shintani</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DellaPenna</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> ( <span class="year" tagx="year" title="year">1998</span>) <span class="mixed-article-title" title="mixed-article-title">Elevating the vitamin E content of plants through metabolic engineering </span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">282</span>, <span class="fpage" tagx="fpage" title="fpage">2098</span>– <span class="lpage" tagx="lpage" title="lpage">2100</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.282.5396.2098">10.1126/science.282.5396.2098</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9851934">9851934</a> </span> </span> </li> <li tag="ref"> <a name="ref5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimizu-Sato</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huq</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tepperman</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quail</span> <span class="given-names" tagx="given-names" title="given-names">P. H.</span> </span> ( <span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">A light-switchable gene promoter system </span>. <span class="source" tagx="source" title="source">Nat. Biotechnol.</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">1041</span>– <span class="lpage" tagx="lpage" title="lpage">1044</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nbt734">10.1038/nbt734</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12219076">12219076</a> </span> </span> </li> <li tag="ref"> <a name="ref6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Spatiotemporal control of gene expression by a light-switchable transgene system </span>. <span class="source" tagx="source" title="source">Nat. Methods </span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">266</span>– <span class="lpage" tagx="lpage" title="lpage">269</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.1892">10.1038/nmeth.1892</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22327833">22327833</a> </span> </span> </li> <li tag="ref"> <a name="ref7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Müller</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Engesser</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Metzger</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schulz</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kämpf</span> <span class="given-names" tagx="given-names" title="given-names">M. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Busacker</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steinberg</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tomakidi</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ehrbar</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagy</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Timmer</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zubriggen</span> <span class="given-names" tagx="given-names" title="given-names">M. D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">A red/far-red light-responsive bi-stable toggle switch to control gene expression in mammalian cells </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res. </span> <span class="volume" tagx="volume" title="volume">41</span>, <span class="fpage" tagx="fpage" title="fpage">e77</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt002">10.1093/nar/gkt002</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23355611">23355611</a> </span> </span> </li> <li tag="ref"> <a name="ref8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pathak</span> <span class="given-names" tagx="given-names" title="given-names">G. P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strickland</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vrana</span> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tucker</span> <span class="given-names" tagx="given-names" title="given-names">C. L.</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Benchmarking of optical dimerizer systems </span>. <span class="source" tagx="source" title="source">ACS Synth. Biol.</span> <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">832</span>– <span class="lpage" tagx="lpage" title="lpage">838</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/sb500291r">10.1021/sb500291r</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25350266">25350266</a> </span> </span> </li> <li tag="ref"> <a name="ref9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Folcher</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oesterle</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zwicky</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thekkottil</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heymoz</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hohmann</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christen</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daoud El-Baba</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buchmann</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fussenegger</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Mind-controlled transgene expression by a wireless-powered optogenetic designer cell implant </span>. <span class="source" tagx="source" title="source">Nat. Commun.</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">5392</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms6392">10.1038/ncomms6392</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25386727">25386727</a> </span> </span> </li> <li tag="ref"> <a name="ref10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaberniuk</span> <span class="given-names" tagx="given-names" title="given-names">A. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shemetov</span> <span class="given-names" tagx="given-names" title="given-names">A. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Verkhusha</span> <span class="given-names" tagx="given-names" title="given-names">V. V.</span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">A bacterial phytochrome-based optogenetic system controllable with near-infrared light </span>. <span class="source" tagx="source" title="source">Nat. Methods</span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">591</span>– <span class="lpage" tagx="lpage" title="lpage">597</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.3864">10.1038/nmeth.3864</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27159085">27159085</a> </span> </span> </li> <li tag="ref"> <a name="ref11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boyden</span> <span class="given-names" tagx="given-names" title="given-names">E. S. </span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bamberg</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagel</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deisseroth</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Millisecond-timescale, genetically targeted optical control of neural activity </span>. <span class="source" tagx="source" title="source">Nat. Neurosci.</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">1263</span>– <span class="lpage" tagx="lpage" title="lpage">1268</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nn1525">10.1038/nn1525</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16116447">16116447</a> </span> </span> </li> <li tag="ref"> <a name="ref12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">J. Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knutsen</span> <span class="given-names" tagx="given-names" title="given-names">P. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muller</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kleinfeld</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsien</span> <span class="given-names" tagx="given-names" title="given-names">R. Y.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">ReaChR: a red-shifted variant of channelrhodopsin enables deep transcranial optogenetic excitation </span>. <span class="source" tagx="source" title="source">Nat. Neurosci. </span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">1499</span>– <span class="lpage" tagx="lpage" title="lpage">1508</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nn.3502">10.1038/nn.3502</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23995068">23995068</a> </span> </span> </li> <li tag="ref"> <a name="ref13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levskaya</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiner</span> <span class="given-names" tagx="given-names" title="given-names">O. D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">W. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voigt</span> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Spatiotemporal control of cell signalling using a light-switchable protein interaction </span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">461</span>, <span class="fpage" tagx="fpage" title="fpage">997</span>– <span class="lpage" tagx="lpage" title="lpage">1001</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature08446">10.1038/nature08446</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19749742">19749742</a> </span> </span> </li> <li tag="ref"> <a name="ref14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibson</span> <span class="given-names" tagx="given-names" title="given-names">E. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kennedy</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">A light-triggered protein secretion system </span>. <span class="source" tagx="source" title="source">J. Cell Biol.</span> <span class="volume" tagx="volume" title="volume">201</span>, <span class="fpage" tagx="fpage" title="fpage">631</span>– <span class="lpage" tagx="lpage" title="lpage">640</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1083/jcb.201210119">10.1083/jcb.201210119</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23671313">23671313</a> </span> </span> </li> <li tag="ref"> <a name="ref15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spiltoir</span> <span class="given-names" tagx="given-names" title="given-names">J. I.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strickland</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Glotzer</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tucker</span> <span class="given-names" tagx="given-names" title="given-names">C. L.</span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Optical control of peroxisomal trafficking </span>. <span class="source" tagx="source" title="source">ACS Synth. Biol.</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">554</span>– <span class="lpage" tagx="lpage" title="lpage">560</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acssynbio.5b00144">10.1021/acssynbio.5b00144</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26513473">26513473</a> </span> </span> </li> <li tag="ref"> <a name="ref16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">X. X.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chung</span> <span class="given-names" tagx="given-names" title="given-names">H. K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lam</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">M. Z.</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Optical control of protein activity by fluorescent protein domains </span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">338</span>, <span class="fpage" tagx="fpage" title="fpage">810</span>– <span class="lpage" tagx="lpage" title="lpage">814</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1226854">10.1126/science.1226854</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23139335">23139335</a> </span> </span> </li> <li tag="ref"> <a name="ref17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Müller</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Engesser</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Timmer</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagy</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zurbriggen</span> <span class="given-names" tagx="given-names" title="given-names">M. D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Synthesis of phycocyanobilin in mammalian cells</span>. <span class="source" tagx="source" title="source">Chem. Commun. (Cambridge, U. K.)</span> <span class="volume" tagx="volume" title="volume">49</span>, <span class="fpage" tagx="fpage" title="fpage">8970</span>– <span class="lpage" tagx="lpage" title="lpage">8972</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/c3cc45065a">10.1039/c3cc45065a</a> </span>. </span> </li> <li tag="ref"> <a name="ref18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodriguez-Romero</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hedtke</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kastner</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Müller</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fischer</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Fungi, hidden in soil or up in the air: light makes a difference </span>. <span class="source" tagx="source" title="source">Annu. Rev. Microbiol.</span> <span class="volume" tagx="volume" title="volume">64</span>, <span class="fpage" tagx="fpage" title="fpage">585</span>– <span class="lpage" tagx="lpage" title="lpage">610</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.micro.112408.134000">10.1146/annurev.micro.112408.134000</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20533875">20533875</a> </span> </span> </li> <li tag="ref"> <a name="ref19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karniol</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagner</span> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walker</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vierstra</span> <span class="given-names" tagx="given-names" title="given-names">R. D.</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis of the phytochrome superfamily reveals distinct microbial subfamilies of photoreceptors </span>. <span class="source" tagx="source" title="source">Biochem. J.</span> <span class="volume" tagx="volume" title="volume">392</span>, <span class="fpage" tagx="fpage" title="fpage">103</span>– <span class="lpage" tagx="lpage" title="lpage">116</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1042/BJ20050826">10.1042/BJ20050826</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16004604">16004604</a> </span> </span> </li> <li tag="ref"> <a name="ref20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Auldridge</span> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forest</span> <span class="given-names" tagx="given-names" title="given-names">K. T.</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Bacterial phytochromes: more than meets the light </span>. <span class="source" tagx="source" title="source">Crit. Rev. Biochem. Mol. Biol. </span> <span class="volume" tagx="volume" title="volume">46</span>, <span class="fpage" tagx="fpage" title="fpage">67</span>– <span class="lpage" tagx="lpage" title="lpage">88</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3109/10409238.2010.546389">10.3109/10409238.2010.546389</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21250783">21250783</a> </span> </span> </li> <li tag="ref"> <a name="ref21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rockwell</span> <span class="given-names" tagx="given-names" title="given-names">N. C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Su</span> <span class="given-names" tagx="given-names" title="given-names">Y. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lagarias</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Phytochrome structure and signaling mechanisms </span>. <span class="source" tagx="source" title="source">Annu. Rev. Plant Biol.</span> <span class="volume" tagx="volume" title="volume">57</span>, <span class="fpage" tagx="fpage" title="fpage">837</span>– <span class="lpage" tagx="lpage" title="lpage">858</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.arplant.56.032604.144208">10.1146/annurev.arplant.56.032604.144208</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16669784">16669784</a> </span> </span> </li> <li tag="ref"> <a name="ref22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frankenberg</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mukougawa</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kohchi</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lagarias</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Functional genomic analysis of the HY2 family of ferredoxin-dependent bilin reductases from oxygenic photosynthetic organisms </span>. <span class="source" tagx="source" title="source">Plant Cell </span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">965</span>– <span class="lpage" tagx="lpage" title="lpage">978</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2307/3871353">10.2307/3871353</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11283349">11283349</a> </span> </span> </li> <li tag="ref"> <a name="ref23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kohchi</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mukougawa</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frankenberg</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Masuda</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yokota</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lagarias</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">The Arabidopsis HY2 gene encodes phytochromobilin synthase, a ferredoxin-dependent biliverdin reductase </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">425</span>– <span class="lpage" tagx="lpage" title="lpage">436</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2307/3871286">10.2307/3871286</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11226195">11226195</a> </span> </span> </li> <li tag="ref"> <a name="ref24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beale</span> <span class="given-names" tagx="given-names" title="given-names">S. I.</span> </span> ( <span class="year" tagx="year" title="year">1993</span>) <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of phycobilins </span>. <span class="source" tagx="source" title="source">Chem. Rev.</span> <span class="volume" tagx="volume" title="volume">93</span>, <span class="fpage" tagx="fpage" title="fpage">785</span>– <span class="lpage" tagx="lpage" title="lpage">802</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/cr00018a008">10.1021/cr00018a008</a> </span>. </span> </li> <li tag="ref"> <a name="ref25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gambetta</span> <span class="given-names" tagx="given-names" title="given-names">G. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lagarias</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Genetic engineering of phytochrome biosynthesis in bacteria </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span> <span class="volume" tagx="volume" title="volume">98</span>, <span class="fpage" tagx="fpage" title="fpage">10566</span>– <span class="lpage" tagx="lpage" title="lpage">10571</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.191375198">10.1073/pnas.191375198</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11553807">11553807</a> </span> </span> </li> <li tag="ref"> <a name="ref26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tooley</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">Y. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Glazer</span> <span class="given-names" tagx="given-names" title="given-names">A. N.</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of a fluorescent cyanobacterial C-phycocyanin holo-alpha subunit in a heterologous host </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span> <span class="volume" tagx="volume" title="volume">98</span>, <span class="fpage" tagx="fpage" title="fpage">10560</span>– <span class="lpage" tagx="lpage" title="lpage">10565</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.181340998">10.1073/pnas.181340998</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11553806">11553806</a> </span> </span> </li> <li tag="ref"> <a name="ref27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mukougawa</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanamoto</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kobayashi</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yokota</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kohchi</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Metabolic engineering to produce phytochromes with phytochromobilin, phycocyanobilin, or phycoerythrobilin chromophore in Escherichia coli </span>. <span class="source" tagx="source" title="source">FEBS Lett.</span> <span class="volume" tagx="volume" title="volume">580</span>, <span class="fpage" tagx="fpage" title="fpage">1333</span>– <span class="lpage" tagx="lpage" title="lpage">1338</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.febslet.2006.01.051">10.1016/j.febslet.2006.01.051</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16458890">16458890</a> </span> </span> </li> <li tag="ref"> <a name="ref28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Landgraf</span> <span class="given-names" tagx="given-names" title="given-names">F. T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forreiter</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hurtado Picó</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lamparter</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hughes</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Recombinant holophytochrome inEscherichia coli </span>. <span class="source" tagx="source" title="source">FEBS Lett.</span> <span class="volume" tagx="volume" title="volume">508</span>, <span class="fpage" tagx="fpage" title="fpage">459</span>– <span class="lpage" tagx="lpage" title="lpage">462</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0014-5793(01)02988-X">10.1016/S0014-5793(01)02988-X</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11728472">11728472</a> </span> </span> </li> <li tag="ref"> <a name="ref29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shin</span> <span class="given-names" tagx="given-names" title="given-names">A. Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">Y. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">P. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J. I.</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Expression of recombinant full-length plant phytochromes assembled with phytochromobilin in Pichia pastoris </span>. <span class="source" tagx="source" title="source">FEBS Lett.</span> <span class="volume" tagx="volume" title="volume">588</span>, <span class="fpage" tagx="fpage" title="fpage">2964</span>– <span class="lpage" tagx="lpage" title="lpage">2970</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.febslet.2014.05.050">10.1016/j.febslet.2014.05.050</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24911206">24911206</a> </span> </span> </li> <li tag="ref"> <a name="ref30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frankenberg</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lagarias</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Phycocyanobilin:ferredoxin oxidoreductase of Anabaena sp. PCC 7120. Biochemical and spectroscopic </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="volume" tagx="volume" title="volume">278</span>, <span class="fpage" tagx="fpage" title="fpage">9219</span>– <span class="lpage" tagx="lpage" title="lpage">9226</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M211643200">10.1074/jbc.M211643200</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12514179">12514179</a> </span> </span> </li> <li tag="ref"> <a name="ref31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheftel</span> <span class="given-names" tagx="given-names" title="given-names">A. D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stehling</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pierik</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elsässer</span> <span class="given-names" tagx="given-names" title="given-names">H. P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mühlenhoff</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Webert</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hobler</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hannemann</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bernhardt</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lill</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Humans possess two mitochondrial ferredoxins, Fdx1 and Fdx2, with distinct roles in steroidogenesis, heme, and Fe/S cluster biosynthesis </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A. </span> <span class="volume" tagx="volume" title="volume">107</span>, <span class="fpage" tagx="fpage" title="fpage">11775</span>– <span class="lpage" tagx="lpage" title="lpage">11780</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1004250107">10.1073/pnas.1004250107</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20547883">20547883</a> </span> </span> </li> <li tag="ref"> <a name="ref32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aliverti</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandini</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pennati</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Rosa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zanetti</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Structural and functional diversity of ferredoxin-NADP(+) reductases </span>. <span class="source" tagx="source" title="source">Arch. Biochem. Biophys.</span> <span class="volume" tagx="volume" title="volume">474</span>, <span class="fpage" tagx="fpage" title="fpage">283</span>– <span class="lpage" tagx="lpage" title="lpage">291</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.abb.2008.02.014">10.1016/j.abb.2008.02.014</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18307973">18307973</a> </span> </span> </li> <li tag="ref"> <a name="ref33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">R. W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Helwig</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Westphal</span> <span class="given-names" tagx="given-names" title="given-names">A. H.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pel</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hörner</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beyer</span> <span class="given-names" tagx="given-names" title="given-names">H. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samodelov</span> <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zurbriggen</span> <span class="given-names" tagx="given-names" title="given-names">M. D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borst</span> <span class="given-names" tagx="given-names" title="given-names">J. W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fleck</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Unearthing the transition rates between photoreceptor conformers </span>. <span class="source" tagx="source" title="source">BMC Syst. Biol.</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">110</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12918-016-0368-y">10.1186/s12918-016-0368-y</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27884151">27884151</a> </span> </span> </li> <li tag="ref"> <a name="ref34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adrian</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nijenhuis</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoogstraaten</span> <span class="given-names" tagx="given-names" title="given-names">R. I.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willems</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kapitein</span> <span class="given-names" tagx="given-names" title="given-names">L. C.</span> </span> ( <span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">A Phytochrome-Derived Photoswitch for Intracellular Transport </span>. <span class="source" tagx="source" title="source">ACS Synth. Biol. </span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">1248</span>– <span class="lpage" tagx="lpage" title="lpage">1256</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acssynbio.6b00333">10.1021/acssynbio.6b00333</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28340532">28340532</a> </span> </span> </li> <li tag="ref"> <a name="ref35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanke</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mulo</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Plant type ferredoxins and ferredoxin-dependent metabolism</span>. <span class="source" tagx="source" title="source">Plant, Cell Environ.</span> <span class="volume" tagx="volume" title="volume">36</span>, <span class="fpage" tagx="fpage" title="fpage">1071</span>– <span class="lpage" tagx="lpage" title="lpage">1084</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/pce.12046">10.1111/pce.12046</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23190083">23190083</a> </span> </span> </li> <li tag="ref"> <a name="ref36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cahoon</span> <span class="given-names" tagx="given-names" title="given-names">E. B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shanklin</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> ( <span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Substrate-dependent mutant complementation to select fatty acid desaturase variants for metabolic engineering of plant seed oils </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span> <span class="volume" tagx="volume" title="volume">97</span>, <span class="fpage" tagx="fpage" title="fpage">12350</span>– <span class="lpage" tagx="lpage" title="lpage">12355</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.210276297">10.1073/pnas.210276297</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11027301">11027301</a> </span> </span> </li> <li tag="ref"> <a name="ref37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Curatti</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rubio</span> <span class="given-names" tagx="given-names" title="given-names">L. M.</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Challenges to develop nitrogen-fixing cereals by direct nif-gene transfer </span>. <span class="source" tagx="source" title="source">Plant Sci.</span> <span class="volume" tagx="volume" title="volume">225</span>, <span class="fpage" tagx="fpage" title="fpage">130</span>– <span class="lpage" tagx="lpage" title="lpage">137</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plantsci.2014.06.003">10.1016/j.plantsci.2014.06.003</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25017168">25017168</a> </span> </span> </li> <li tag="ref"> <a name="ref38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rekittke</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olkhova</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wiesner</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Demmer</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Warkentin</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jomaa</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ermler</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Structure of the (E)-4-hydroxy-3-methyl-but-2-enyl-diphosphate reductase from Plasmodium falciparum </span>. <span class="source" tagx="source" title="source">FEBS Lett.</span> <span class="volume" tagx="volume" title="volume">587</span>, <span class="fpage" tagx="fpage" title="fpage">3968</span>– <span class="lpage" tagx="lpage" title="lpage">3972</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.febslet.2013.10.029">10.1016/j.febslet.2013.10.029</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24188825">24188825</a> </span> </span> </li> <li tag="ref"> <a name="ref39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pinto</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harrison</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hsu</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobs</span> <span class="given-names" tagx="given-names" title="given-names">W. R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leyh</span> <span class="given-names" tagx="given-names" title="given-names">T. S.</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Sulfite reduction in mycobacteria </span>. <span class="source" tagx="source" title="source">J. Bacteriol.</span> <span class="volume" tagx="volume" title="volume">189</span>, <span class="fpage" tagx="fpage" title="fpage">6714</span>– <span class="lpage" tagx="lpage" title="lpage">6722</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JB.00487-07">10.1128/JB.00487-07</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17644602">17644602</a> </span> </span> </li> <li tag="ref"> <a name="ref40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yonekura-Sakakibara</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Onda</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ashikari</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanaka</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kusumi</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hase</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> ( <span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Analysis of reductant supply systems for ferredoxin-dependent sulfite reductase in photosynthetic and nonphotosynthetic organs of maize </span>. <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="volume" tagx="volume" title="volume">122</span>, <span class="fpage" tagx="fpage" title="fpage">887</span>– <span class="lpage" tagx="lpage" title="lpage">894</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.122.3.887">10.1104/pp.122.3.887</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10712553">10712553</a> </span> </span> </li> <li tag="ref"> <a name="ref41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uda</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goto</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oda</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kohchi</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsuda</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aoki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> ( <span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Efficient synthesis of phycocyanobilin in mammalian cells for optogenetic control of cell signaling </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A. </span> <span class="volume" tagx="volume" title="volume">114</span>, <span class="fpage" tagx="fpage" title="fpage">11962</span>– <span class="lpage" tagx="lpage" title="lpage">11967</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1707190114">10.1073/pnas.1707190114</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29078307">29078307</a> </span> </span> </li> <li tag="ref"> <a name="ref42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chiu</span> <span class="given-names" tagx="given-names" title="given-names">F. Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Y. R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tu</span> <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Electrostatic interaction of phytochromobilin synthase and ferredoxin for biosynthesis of phytochrome chromophore </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="volume" tagx="volume" title="volume">285</span>, <span class="fpage" tagx="fpage" title="fpage">5056</span>– <span class="lpage" tagx="lpage" title="lpage">5065</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M109.075747">10.1074/jbc.M109.075747</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19996315">19996315</a> </span> </span> </li> <li tag="ref"> <a name="ref43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okada</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">HO1 and PcyA proteins involved in phycobilin biosynthesis form a 1:2 complex with ferredoxin-1 required for photosynthesis </span>. <span class="source" tagx="source" title="source">FEBS Lett. </span> <span class="volume" tagx="volume" title="volume">583</span>, <span class="fpage" tagx="fpage" title="fpage">1251</span>– <span class="lpage" tagx="lpage" title="lpage">1256</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.febslet.2009.03.052">10.1016/j.febslet.2009.03.052</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19328793">19328793</a> </span> </span> </li> <li tag="ref"> <a name="ref44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szymczak</span> <span class="given-names" tagx="given-names" title="given-names">A. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Workman</span> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vignali</span> <span class="given-names" tagx="given-names" title="given-names">K. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dilioglou</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vanin</span> <span class="given-names" tagx="given-names" title="given-names">E. F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vignali</span> <span class="given-names" tagx="given-names" title="given-names">D. A.</span> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Correction of multi-gene deficiency in vivo using a single "self-cleaving" 2A peptide-based retroviral vector </span>. <span class="source" tagx="source" title="source">Nat. Biotechnol.</span> <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">589</span>– <span class="lpage" tagx="lpage" title="lpage">594</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nbt957">10.1038/nbt957</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15064769">15064769</a> </span> </span> </li> <li tag="ref"> <a name="ref45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Licursi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christian</span> <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pongnopparat</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirasawa</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">In vitro and in vivo comparison of viral and cellular internal ribosome entry sites for bicistronic vector expression </span>. <span class="source" tagx="source" title="source">Gene Ther.</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">631</span>– <span class="lpage" tagx="lpage" title="lpage">636</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/gt.2011.11">10.1038/gt.2011.11</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21368899">21368899</a> </span> </span> </li> <li tag="ref"> <a name="ref46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bochkov</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmenberg</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Translational efficiency of EMCV IRES in bicistronic vectors is dependent upon IRES sequence and gene location </span>. <span class="source" tagx="source" title="source">BioTechniques</span> <span class="volume" tagx="volume" title="volume">41</span>, <span class="fpage" tagx="fpage" title="fpage">283</span>– <span class="lpage" tagx="lpage" title="lpage">292</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2144/000112243">10.2144/000112243</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16989088">16989088</a> </span> </span> </li> <li tag="ref"> <a name="ref47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mizuguchi</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishii-Watabe</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uchida</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayakawa</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> ( <span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">IRES-dependent second gene expression is significantly lower than cap-dependent first gene expression in a bicistronic vector </span>. <span class="source" tagx="source" title="source">Mol. Ther.</span> <span class="volume" tagx="volume" title="volume">1</span>, <span class="fpage" tagx="fpage" title="fpage">376</span>– <span class="lpage" tagx="lpage" title="lpage">382</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1006/mthe.2000.0050">10.1006/mthe.2000.0050</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10933956">10933956</a> </span> </span> </li> <li tag="ref"> <a name="ref48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang Deng</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Phytochrome signaling mechanisms</span>. <span class="source" tagx="source" title="source">Arabidopsis Book</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">e0148</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1199/tab.0148">10.1199/tab.0148</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22303272">22303272</a> </span> </span> </li> <li tag="ref"> <a name="ref49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mattis</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tye</span> <span class="given-names" tagx="given-names" title="given-names">K. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferenczi</span> <span class="given-names" tagx="given-names" title="given-names">E. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramakrishnan</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Shea</span> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prakash</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gunaydin</span> <span class="given-names" tagx="given-names" title="given-names">L. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hyun</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fenno</span> <span class="given-names" tagx="given-names" title="given-names">L. E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gradinaru</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yizhar</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deisseroth</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Principles for applying optogenetic tools derived from direct comparative analysis of microbial opsins </span>. <span class="source" tagx="source" title="source">Nat. Methods</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">159</span>– <span class="lpage" tagx="lpage" title="lpage">172</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.1808">10.1038/nmeth.1808</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22179551">22179551</a> </span> </span> </li> <li tag="ref"> <a name="ref50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beyer</span> <span class="given-names" tagx="given-names" title="given-names">H. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Juillot</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herbst</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samodelov</span> <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Müller</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schamel</span> <span class="given-names" tagx="given-names" title="given-names">W. W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Römer</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schäfer</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagy</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strähle</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zurbriggen</span> <span class="given-names" tagx="given-names" title="given-names">M. D.</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Red Light-Regulated Reversible Nuclear Localization of Proteins in Mammalian Cells and Zebrafish </span>. <span class="source" tagx="source" title="source">ACS Synth. Biol.</span> <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">951</span>– <span class="lpage" tagx="lpage" title="lpage">958</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acssynbio.5b00004">10.1021/acssynbio.5b00004</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25803699">25803699</a> </span> </span> </li> <li tag="ref"> <a name="ref51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qin</span> <span class="given-names" tagx="given-names" title="given-names">J. Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clift</span> <span class="given-names" tagx="given-names" title="given-names">K. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hulur</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiang</span> <span class="given-names" tagx="given-names" title="given-names">A. P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">B. Z.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lahn</span> <span class="given-names" tagx="given-names" title="given-names">B. T.</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Systematic comparison of constitutive promoters and the doxycycline-inducible promoter </span>. <span class="source" tagx="source" title="source">PLoS One</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">e10611</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0010611">10.1371/journal.pone.0010611</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20485554">20485554</a> </span> </span> </li> <li tag="ref"> <a name="ref52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toettcher</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gong</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">W. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiner</span> <span class="given-names" tagx="given-names" title="given-names">O. D.</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Light-based feedback for controlling intracellular signaling dynamics </span>. <span class="source" tagx="source" title="source">Nat. Methods</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">837</span>– <span class="lpage" tagx="lpage" title="lpage">839</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.1700">10.1038/nmeth.1700</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21909100">21909100</a> </span> </span> </li> <li tag="ref"> <a name="ref53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schindelin</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arganda-Carreras</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frise</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaynig</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Longair</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pietzsch</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Preibisch</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rueden</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saalfeld</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmid</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tinevez</span> <span class="given-names" tagx="given-names" title="given-names">J. Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">White</span> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hartenstein</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eliceiri</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tomancak</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cardona</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Fiji: an open-source platform for biological-image analysis </span>. <span class="source" tagx="source" title="source">Nat. Methods </span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">676</span>– <span class="lpage" tagx="lpage" title="lpage">682</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.2019">10.1038/nmeth.2019</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22743772">22743772</a> </span> </span> </li> <li tag="ref"> <a name="ref54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lopez</span> <span class="given-names" tagx="given-names" title="given-names">C. F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muhlich</span> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bachman</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sorger</span> <span class="given-names" tagx="given-names" title="given-names">P. K.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Programming biological models in Python using PySB </span>. <span class="source" tagx="source" title="source">Mol. Syst. Biol.</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">646</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/msb.2013.1">10.1038/msb.2013.1</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23423320">23423320</a> </span> </span> </li> </ul> </div> </div>  </body></html>